<!doctype html><html lang=en-us><head><meta charset=utf-8><title>Symbiota Docs</title><meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=1"><meta name=generator content="Hugo 0.79.1"><link rel=stylesheet href=/symbiota-docs/plugins/bootstrap/bootstrap.min.css><link rel=stylesheet href=/symbiota-docs/plugins/themify-icons/themify-icons.css><link rel=icon href=/symbiota-docs/images/favicon.png type=image/x-icon><link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel=stylesheet><style>:root{--primary-color:#428500;--body-color:#f9f9f9;--text-color:#636363;--text-color-dark:#242738;--white-color:#ffffff;--light-color:#f8f9fa;--font-family:Lato}</style><link href=https://biokic.github.io/symbiota-docs/css/style.min.css rel=stylesheet media=screen><link href=https://biokic.github.io/symbiota-docs/css/custom.min.css rel=stylesheet media=screen><script src=/symbiota-docs/plugins/jquery/jquery-1.12.4.js></script><script src=/symbiota-docs/plugins/jquery/jquery-ui.js></script><script src=/symbiota-docs/plugins/bootstrap/bootstrap.min.js></script><script src=/symbiota-docs/plugins/match-height/jquery.matchHeight-min.js></script><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create','G-MPX39Z5FZQ','auto');ga('send','pageview');}</script><script async src="https://www.googletagmanager.com/gtag/js?id=G-MPX39Z5FZQ"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
gtag('js',new Date());gtag('config','G-MPX39Z5FZQ');</script></head><body><header class="banner overlay bg-cover" data-background=https://biokic.github.io/symbiota-docs/images/annie-spratt-5cFwQ-WMcJU-unsplash.jpg><nav class="navbar navbar-expand-md navbar-dark"><div class=container><a class="navbar-brand px-2" href=/symbiota-docs>Symbiota Docs</a>
<button class="navbar-toggler border-0" type=button data-toggle=collapse data-target=#navigation aria-controls=navigation aria-expanded=false aria-label="Toggle navigation">
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse text-center" id=navigation><ul class="navbar-nav ml-auto"><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/>Home</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/about>About</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/faq>Faq</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/citing>Citing The Docs</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/contact>Contact</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/contribute>Contributing</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/search>Search</a></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle text-dark" href=# role=button data-toggle=dropdown aria-haspopup=true aria-expanded=false>Docs</a><div class=dropdown-menu><a class=dropdown-item href=https://biokic.github.io/symbiota-docs/user>User Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/editor>Editor Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/coll_manager>Collection Manager Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/dev>Developer Guide</a></div></li></ul><select class=lang-list id=select-language onchange="location=this.value;">
<option id=en value=https://biokic.github.io/symbiota-docs/ selected>En</option>
<option id=es value=https://biokic.github.io/symbiota-docs/es/>Es</option>
<option id=fr value=https://biokic.github.io/symbiota-docs/fr/>Fr</option></select></div></div></nav><div class="container section"><div class=row><div class="col-lg-8 text-center mx-auto"><h1 class="text-white mb-3">Symbiota Documentation</h1><p class="text-white mb-4">Growing archive of help files for one of the most popular Natural History content management systems</p><div class=position-relative><input id=search class=form-control placeholder="Quick search for terms here. Press enter to go to results page.">
<i class="ti-search search-icon"></i><script>$(function(){var projects=[{value:"[contribute] How to run a local version of Symbiota Docs",label:"<p>You can run the Symbiota Docs website in your own computer if you\u0026rsquo;d like to contribute with content and you\u0026rsquo;re interested in previewing changes before pushing them to us, or for development purposes as well, in case you\u0026rsquo;d like to suggest theme modifications.\nSymbiota Docs is a static documentation website compiled with Hugo. The files that create each page and topic are written in Markdown (with the possibility to add some HTML), and \u0026ldquo;translated\u0026rdquo; by Hugo into static HTML files.\nIn a nutshell, what you need to run a local version of the Symbiota Docs website is:\n Install Hugo in your computer Download or clone the symbiota docs GitHub repository Run a Hugo server Enjoy!  Install Hugo Hugo is a framework for building static websites. To install Hugo, please visit the official Hugo installation documentation.\nGet the Symbiota Docs files  Visit the Symbiota Docs GitHub repository. Click the \u0026ldquo;Code\u0026rdquo; button on the right side of the page. Select the option your most comfortable with (clone, open with GitHub desktop or download zip). Go to the folder where you have placed your downloaded files. If you have a zipped archive, unzip it. Open a terminal\/shell\/command-line tool in your operating system and navigate to your folder (in the example below my files are in the Documents directory, inside another directory called symbiota-docs):  cd Documents\/symbiota-docs\/ Run a development Hugo server with the following command in the same command-line tool:  hugo server -D If your website doesn\u0026rsquo;t pop up automatically, in a browser, navigate to the address indicated by your terminal (normally it\u0026rsquo;s http:\/\/localhost:1313\/ or some similar with different port numbers). You are now live locally! To make changes, open the desired file (ending in .md for Markdown) in a text editor, make changes and save to see the modifications instantly reflected in your local website.  </p><p>contribute</p><p>develop</p><p>hugo</p><p>local</p><p>documentation</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contribute\/local_version\/"},{value:"[editor] Annotation Label Printing",label:"<p> This page describes how to print annotation labels in a Symbiota portal.\n To use this tool, you will need to have first added the annotated specimens to the annotation print queue. This can be done either individually, via the Occurrence Editor, or in batch using the Add Batch Determinations\/Nomenclatural Adjustments tool (see Annotations\/Determinations for instructions).\nTo print annotation labels, navigate to the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection) and click “Print Annotations Labels.” A table will be displayed that lists all the specimens in the Annotation Print Queue. Check the boxes in the leftmost column next to the specimen records for which you wish to print annotations.\nTo view a specimen record, either click the collector name and number (to view public record display) or click the pencil icon next to the collector name and number (to view occurrence editor for that record).\nCustomize your annotation label using the provided options and select either Print in Browser or Export to DOCX (to export as a Microsoft Word document).\nCheck the box next to “Remove selected annotations from queue” to remove these specimens from the print queue after you click one of the print buttons.\n</p><p>labels</p><p>printing</p><p>annotation</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/label\/annotation\/"},{value:"[coll_manager] Compare Duplicate IDs Across Collections",label:"<p> This page describes how you can identify specimens in your collection that have different identifications (i.e., scientific names) than duplicates of that specimen housed at another collection.\n Navigate to this tool through the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Duplicate Cleaning Tools. In the bottom box under Taxonomy, click \u0026ldquo;Duplicate specimens with potential identification conflicts\u0026hellip;\u0026rdquo;.\nThis tool lists the duplicate clusters of specimens (specimens that have the same collector, number, and date, even across different collections) for which one or more of the specimens does not have the same taxonomic name. The tool can be used to identify whether there are specimens in your collection that should be annotated to the most recently identified taxonomic name. For example, in the cluster below, OBI and UCR may consider annotating their specimens according to the most recent identification of the SBBG duplicate. The specimen records associated with each specimen can be viewed by clicking on the catalog number of the desired record.\n</p><p>data cleaning</p><p>taxonomy</p><p>duplicates</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/id_conflict\/"},{value:"[user] User Guide",label:"<p>Welcome to the User Guide! This guide describes how to use tools and functions that any member of the public can access. For many of these functions, you will need a user account, but you do not need to be granted editor or administrator permissions to do these things (e.g., make checklists).\nFor information about becoming a data editor, see our User Permissions page.\nTo create an account, click the New Account option in the toolbar of any Symbiota portal, enter the requested information, and click Create Login.\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/"},{value:"[coll_manager] Batch Uploading\/Linking Images",label:"<p> This page describes general workflows for adding multiple images to a portal at one time. Technically, images are not stored within a Symbiota database. Instead, the URLs of the images are mapped to specimen records. Contact the portal manager to inquire about the workflows supported by a given portal.\n Images associated with records in a Symbiota portal may be Externally Hosted or Internally Hosted.\n  Externally Hosted images are those stored on servers that are unrelated to the Symbiota portal (e.g., are housed at another institution). These external servers provide image links, which can then be loaded into the Symbiota database.\n  Internally Hosted images are those stored on servers that are write-accessible to the Symbiota portal server. For example, if you coordinate with Arizona State University to batch upload images to one of their hosted portals (e.g., SEINet, Bryophyte Portal), your images are Internally Hosted.\n  Images uploaded or linked to specimens in a Symbiota portal should be JPGs or in another web-friendly format. It is recommended that individual files are no larger than 8 MB to allow users with slower connectivity to view them within a reasonable timeframe.\n Batch Adding Externally Hosted Images Externally Hosted images can be added to Symbiota records using one of three primary methods: (1) using the URL mapping tool and a spreadsheet of catalog numbers with image URLs (documentation coming soon!); (2) mapping a column of image links to the associatedMedia field when conducting a Full Text File Upload or Skeletal File Upload. For more information about uploading text files, see this page. If you already have data in the portal, and you just want to add image links, do NOT use Full Text File Upload; or (3) importing a Darwin Core Archive of your data with a fully-populated multimedia extension.\nBatch Adding Internally Hosted Images There are several workflows used to batch upload\/link internally hosted specimen images. Batch processing typically consists of two separate steps: (1) Loading images onto a web server, and (2) Mapping image URLs to Symbiota occurrence records. Step (2) is possible when images are named according to the catalog number or other catalog number associated with the record in the portal.\nStorage of a large number of images on servers associated with a Symbiota portal may require an image-hosting agreement and\/or incur image-hosting costs. Check with the portal administrator for more information about your portal\u0026rsquo;s image hosting allowances and workflows. For ASU-hosted servers, contact the Symbiota Support Hub (help@symbiota.org).\n</p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/images\/batch\/"},{value:"[contribute] Formatting Symbiota Docs",label:"<p>Style Guide If you intend to contribute Markdown (.md) files to Symbiota Docs, please review the Style Guide to ensure your contributions are formatted consistently with other pages in this repository.\n</p><p>contribute</p><p>documentation</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contribute\/style_guide\/"},{value:"[editor] Batch Uploading Images",label:"<p> This page describes some workflows for batch uploading images into a Symbiota portal.\n Symbiota portals do not necessarily store images, rather, they store links to where images can be accessed via the web. For this reason, there are many ways that images can be linked to occurrence records in the portal. Any storage solution that can provide a web-accessible URL to an image can be used to link images to occurrences in a Symbiota portal.\nImage Storage at ASU For relatively small collections with little to no IT support, the Arizona State University may be able to web-host the jpeg versions (\u0026lt;10 MB) of images for display in a Symbiota portal. In this case, images are uploaded to ASU via Globus and transferred to ASU servers. The URLs generated by these servers are then linked to your records in a Sybmiota portal. To determine whether this is possible for your collection, contact the Symbiota Support Hub.\nImage Storage through an Institutional Server If your institution has the ability to serve images in a web-accessible format, these URLs can be easily linked to their occurrences in your portal through a batch processing script. Contact your portal administrator or the Symbiota Support Hub for more information or to set up this workflow.\n</p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/batch\/"},{value:"[editor] Deleting Images or Transferring Between Records",label:"<p> This page describes how to transfer images to other records in your collection or how to delete images associated with a record.\n Transferring an Image to Another Record  Navigate to the occurrence record that has the image you would like to transfer. (e.g., My Profile \u0026gt; Occurrence Management \u0026gt; name of collection \u0026gt; Edit Existing Occurrence Records \u0026gt; use record search form to find specific occurrence). Open the occurrence record in the Occurrence Editor (see screenshot below). Click the Images tab.  Click the pencil icon to the right of the image you would like to transfer. If you would like to transfer the image to a brand new record, rather than a record that already exists, scroll down to the box labeled Link to a New Blank Occurrence Record within Collection and click the \u0026ldquo;Link to New Occurrence\u0026rdquo; button.  The SymbiotaID number associated with the new record you have created with the image will be displayed at the top of the page after you click this button. To navigate to that record, click the blue SymbiotaID number.   If you would like to transfer the image to a record that already exists, scroll down to the box labeled Remap to Another Specimen. You can either enter the SymbiotaID number for the record (if you have it on hand) or use the Occurrence Linking Aid (where you can search by catalog number, collector, and\/or collector number). To do the latter, click \u0026ldquo;Open Occurrence Linking Aid\u0026rdquo; to the right of the field. In the pop-up window, enter the search terms you would like to use to find the occurrence to which you would like to link the image. You can include several terms in this search concurrently. Then click the Search Occurrences button to see the search results. If there are results of your search, they will be listed below the Voucher Search Pane as shown below. To select one of the listed occurrences for linking, click Select Occurrence Record. This will close the window and import that record\u0026rsquo;s SymbiotaID number into the Remap to Another Specimen panel.  If you don\u0026rsquo;t find the record you are looking for, try using different search terms. If the record does not exist, you can click the Create New Occurrence button to create a brand new occurrence for the image.    Back on the images tab of the Occurrence Editor page, click the Remap Image button.  Any data associated with that image (e.g., its catalog number) will NOT be transferred along with the image. You will need to copy over this information in another way, if applicable.\n Deleting an Image  Complete steps 1-4 in the instructions above. Scroll down to the box labeled Delete Image.  If your images are stored on ASU servers (i.e., you upload them to Dropbox or otherwise transfer them to ASU), check the \u0026ldquo;Remove image from server\u0026rdquo; box. Otherwise, leave this box unchecked If your images are stored on another server, such as CyVerse, leave the \u0026ldquo;Remove image from server\u0026rdquo; box unchecked.    </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/delete\/"},{value:"[coll_manager] Duplicate Catalog Numbers",label:"<p> This page describes how to identify and merge (when necessary) specimens with duplicate catalog numbers or other catalog numbers.\n This tool can be found in the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection).\nTo use the tool, first select whether you would like to list duplicates based on Catalog Numbers or Other Catalog Numbers from the “List Duplicates based on\u0026hellip;” box. If duplicates exist, a table of duplicate records will appear on the resulting page.\nFrom here, you can evaluate whether the duplicate records should be edited, merged, or ignored. To view and\/or edit a record, click the number in the ID column next to the record you wish to view. This will open that specimen’s Record Editor page.\nIf you decide that the two records should be merged, check the boxes next to both duplicates, then select the radio button that corresponds to the record you wish to make the primary record. If both records have a value in a given field, Symbiota will keep the value that belongs to this primary record and discard the other value. When you are satisfied that the correct information will be retained, click the Merge Duplicate Records button.\nIf there are many duplicate records to evaluate, you can click the checkbox at the top of the checkbox column. This will check all of the boxes in the entire table. You can also select the radio button column header, which will allow Symbiota to auto-select a record to retain for each pair. You will want to vet this list to ensure that the correct record is selected for retention.\n </p><p>data cleaning</p><p>duplicates</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/dupes\/"},{value:"[editor] Linking New Images to Old Records",label:"<p> This page addresses a common digitization workflow challenge: how do you link images of specimens (with new barcode numbers) to specimen records that already exist in the portal if the barcode number isn\u0026rsquo;t associated with the old data yet? Here we describe two options to solve this problem.\n The following instructions assume that the barcode number is stored in the Catalog Number field, and the accession number (usually a stamped number) is stored in the Other Catalog Numbers field.\n Option 1: Add catalog (barcode) numbers to occurrence records while imaging This protocol describes how to add a barcode number to each of the specimens that you image during the imaging process.\n (Example instructions for an herbarium collection)\n Acquire a specimen. Log in to your account and navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Edit Existing Occurrence Records. Place a barcode on your specimen. Look for a stamped number (“accession number”) on the specimen.  If a stamped number does not exist, proceed to step 8.   In the Record Search Form, enter the stamped number into the Other Catalog Numbers field. Do not include leading zeros (e.g., if the stamp is 01499, enter 1499). Click Display Editor.  If no record is returned, double check that you entered the accession number correctly into the Other Catalog Numbers field and that there are no extra spaces before or after the numbers you typed. If the number is correct, contact a supervisor. If a record is returned, check that the data on the specimen record match the data on the label of the specimen in front of you, particularly collector, collector number, and date. i. If these data match, place your cursor in the Catalog Number field (top left) and scan the barcode of the specimen in front of you. The barcode number will then appear in this field. Press Tab on your keyboard, followed by Enter. Proceed with imaging the specimen and return to step 1. ii. If these data do NOT match, contact a supervisor.    To re-open the Record Search Form after you have performed a search, click the magnifying glass icon at the top right side of the specimen record.\n In the Record Search Form, enter the last name of the collector preceded by a percent sign (%), the collector number, and the date (in YYYY-MM-DD format) into the top three fields (see example below) and click Display Editor.  If no record is returned, double check that you entered data correctly into the appropriate fields. If everything looks correct, contact a supervisor. If a record is returned, check that the data on the specimen record match the data on the label of the specimen in front of you, particularly collector, collector number, and date. i.\tIf these data match, place your cursor in the Catalog Number field (top right) and scan the barcode of the specimen in front of you. The barcode number will then appear in this field. Press Tab on your keyboard, followed by Enter. Proceed with imaging the specimen and return to step 1. ii.\tIf these data do NOT match, contact a supervisor.    Option 2: View new images in the portal and link by manually entering the accession number This protocol describes how to update your old accession number with the specimen’s new barcode number. You can do this only after you have uploaded specimen images into your portal through a batch process that has created new \u0026ldquo;unprocessed\u0026rdquo; records consisting of only a barcode number and an image.\n (Example instructions for an herbarium collection)\n Log in to your account and navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Edit Existing Occurrence Records. In the Record Search Form, click the Processing Status field and select “Unprocessed” from the dropdown menu. In the Custom Field 1 field, select Other Catalog Number from the first dropdown menu and IS NULL from the second dropdown menu. Click the “Display Editor” button. Locate the accession number (the stamped number, rather than the barcoded number or the collector number) on the specimen in the image to the right of the Occurrence Editor form.  You can zoom in by pressing Command (Mac) or Control (Windows) and clicking on the image where you want to zoom in. Alternatively, you can hold Shift, click on the area where you want to zoom in, and move the mouse up (to zoom in) or down (to zoom out).   Enter the accession number, without leading zeros (e.g., if the number reads “0145”, you will enter “145”), into the Other Cat. #s field (circled in screenshot below).  Click outside of the Other Cat. #s field (or press the Tab button). A message should show up under the “Dupes?” button. Make sure that your browser’s pop-up blocker is disabled so you can see these messages.  If the message shows “No Dupes Found”:  Check that the specimen sheet is not stamped “Databased” or has any other indication that it SHOULD have a record in the database. If it is, try adding a leading zero to the accession number and repeat step 9. Press the Save Edits button (or press Tab and Enter on your keyboard) Move on to step 9.   If a pop-up window shows up that says “Record(s) using the same identifier already exists. Do you want to view this record?”:  Click OK. Check that the identified duplicate does not already have an image associated with it (if it does, you will see a bold barcode number on that record). If the duplicate does have an image already, close the window, scroll down to the Processing status field in the Curation box, select “Expert Required” from the dropdown menu, and then click the Save Edits button. Otherwise, move to step c. Check that the data of the identified duplicate matches the data on the label of the specimen image you were viewing. You need not check every field, just two or three important fields such as Scientific Name, Collector, and Locality. If the data match up, click “Merge Records” and exit the duplicate window. If the data do not match up, close the window, scroll down to the Processing status field in the Curation box, select “Expert Required” from the dropdown menu, and then click the Save Edits button.     Click the double arrow icon at the top right corner of the occurrence editor. Repeat steps 6-9.  This protocol is demonstrated in the following video:\n  </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/link\/"},{value:"[editor] Editor Guide",label:"<p>Welcome to the Editor Guide! This guide contains instructions for users with Editor permissions for a collection in a Symbiota site.\nUsers with Administrator permissions automatically inherit Editor permissions as well, but not vice versa, so much of this content pertains to Administrators as well.\nNot sure what level of permissions you are working with? See the User Permissions page.\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/"},{value:"[col_obs] Managing Data in a Personal Observation Profile",label:"<p>Users must be sponsored by a collection administrator in order to add and edit records to this collection profile.\nSponsored users have the following options for managing their data in the Personal Observations Profile collection. Related tutorial documentation is linked below.\n Search Records: search through the records that you have entered into this profile. You can conduct custom, detailed searches from this interface. Tutorial information can be found on this page. Add a New Record: tutorials for adding records here and for editing records here Print Labels: print labels for specimens of all types, including specimen packets. Tutorials here. Print Annotation Labels: print annotation\/determination\/identification labels for specimens that already have full labels. Tutorials here. Submit image-vouchered observation: use an image to create a new record, if you did not collect a specimen for the record. The available data entry fields are similar to those used to add a new specimen record (see above). Review\/Verify Occurrence Edits: view, revert, or apply edits that you have made to your records. Tutorial here. Download backup file (CSV extract): download a copy of your data to review or send to a collection. CSV files may be opened in Excel or a similar spreadsheet software. View User Comments: because your data are publicly visible, users may leave comments on your records. Use this tool to view and manage those comments. See this page for more information.  </p><p>transfer</p><p>export</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/col_obs\/add\/"},{value:"[col_obs] Sending Data from a Personal Observation Profile",label:"<p> This page describes how a collector or other personal observation profile editor can transfer or send their data to a collection\n Copying data from one (or few) specimens Use this option if you would like to send a copy of some data to another collection, rather than transferring the data to the collection (and removing it from your own).\nOnce your specimen data are entered into the Symbiota portal, if the collection to which you would like to send your data is \u0026ldquo;live-managed\u0026rdquo; in a Symbiota portal (i.e., they use the portal as their main database), that collection can easily import your data as they process the incoming specimens. To do so, they will use the Add New Record tool and enter in only the collector name, number, and date for the specimen. They can then use the Duplicates Tool to \u0026ldquo;Transfer All Fields\u0026rdquo; or \u0026ldquo;Transfer to Empty Fields\u0026rdquo;, which will bring in all data associated with the matching specimen.\nCopying data for many specimens Use this option if you would like to send a copy of a lot of data to another collection, rather than transferring the data to the collection (and removing it from your own).\nIf you are sending data for a great number of specimens, or if the collection to which you are sending the data is not \u0026ldquo;live-managed\u0026rdquo; in the portal, you may want to send your data as a CSV file. To do this:\n Navigate to your collection profile by logging into the Symbiota portal and clicking My Profile \u0026gt; Occurrence Management tab. Click \u0026ldquo;Download backup file (CSV extract)\u0026rdquo; in the top box of this tab. Remove all the specimens that you do not wish to send from this CSV (e.g., by sorting by date or location in Excel). Send this CSV along with your specimens or via email to the collections manager of the collection where you are depositing your specimens.  Transferring individual specimens from a personal profile to a collection Use this option if you would transfer ownership of the data in your personal observation profile to the collection where you are depositing your specimen(s). This will remove the specimens from your personal observation profile.\nTo transfer a specimen from your personal observation profile, you must have administrator permissions for the collection into which you will deposit the specimens\/data. Contact the collections manager to determine whether this is a preferred option. They may prefer that you send them the data for them to upload, and you can delete the specimen data from your own collection yourself (or contact a portal administrator for batch deletion).\n Open the specimen that you would like to transfer and click the Admin tab (far right).  From the dropdown menu, select the name of the collection to which you would like to transfer your record. This action will remove the specimen from your personal observation profile!  Batch transferring specimens from a personal profile to a collection Contact your portal administrator to batch transfer many specimens from your personal observation profile to another collection in the portal.\n</p><p>transfer</p><p>export</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/col_obs\/send\/"},{value:"[coll_manager] Management Types",label:"<p>There are two main ways that collections present their data in Symbiota portals: live or as a snapshot. Each collection\u0026rsquo;s data management method in Symbiota can be found on its respective profile. For example:\n\nA live collection is one that manages their data directly in the Symbiota portal. Data entry, data editing, and digitization are conducted via a web browser within Symbiota portal. Live collections can start with as much or as little data as a collection wishes; if the collection already has a spreadsheet, database, or Darwin Core Archive of their collection data, they can import the data into the Symbiota portal and manage the data in the portal henceforth. A collection can also start the digitization process in the portal without any starting data. Some advantages to live management in a Symbiota portal include:\n Management of data through a web browser, enabling multiple users (once granted appropriate permissions) to manage data remotely Access to efficient digitization tools, such as batch georeferencing and duplicate data harvesting Access to data cleaning tools for taxonomy, geography, and more  A snapshot collection is one that manages their data in a different content management system (e.g., Access, Specify, CSpace) and imports a copy of their data into a Symbiota portal so it can be discovered by more users and\/or published to global aggregators. Snapshot collections can import their data into a Symbiota portal via a spreadsheet, Darwin Core Archive, or through an IPT (Integrated Publishing Toolkit) or other web-based system. A collection can set up and save an import profile that will allow future updates of the data as long as the unique identifier from that source (called \u0026ldquo;dbpk\u0026rdquo; in a Symbiota portal) remains the same between imports.\nIn both cases, collections in Symbiota portals benefit from:\n Participation in a network with special regional and\/or taxonomic expertise Assistance with backend queries and daily troubleshooting through the Symbiota Support Hub Quick and easy publication of your data to global aggregators (e.g., GBIF), if desired  </p><p>management</p><p>live</p><p>snapshot</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/join\/mng\/"},{value:"[coll_manager] Collection Manager Guide",label:"<p>Welcome to the Collection Manager Guide! In this guide, you will find instructions for more advanced functions that are available to users with Administrator permissions.\nAdministrators automatically gain Editor permissions as well, so much of the content in the Editor Guide will also be useful for Collection Managers \/ Administrators.\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/"},{value:"[coll_manager] Geography Cleaning Tools",label:"<p> This page describes how to use the two geographic cleaning tools in Symbiota portals to batch clean geographic data for occurrences.\n The two geographic cleaning tools are the Geographic Distribution viewer and the Geography Cleaning Tool.\nGeography Distribution Viewer Navigate to this tool through the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Data Cleaning Tools, then Geographic Distributions.\nThe Geography Distribution viewer can be used to examine the countries, states, and counties that exist in your database. Any misspellings, non-standardized entries (e.g., “USA” rather than “United States”), or suspected mistakes (e.g., “United Arab Emirates” rather than “United States”) can be detected using this tool. To view the state\/province values for each country, click the name of the country. Then, to view the county values for each state\/province, click the name of the state\/province.\nA user with administrator permissions can correct errors in country, state\/province, and\/or county individually by clicking the number next to the place name (circled on next screenshot), or you can search for those records using the record search form and batch edit them.\nGeography Cleaning Tool Navigate to this tool through the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Data Cleaning Tools, then Geography Cleaning Tools.\nThe Geography Cleaning Tool will search your database for non-standardized geographical terms (countries, states\/provinces, and counties) used in your collection. These will be listed as “questionable.” To view and potentially edit these records, you can click the “List [terms]\u0026hellip;” link (one example circled below).\nThis tool will also check whether there are records that lack data in the country, state\/province, or county fields, yet have geographic data in other fields. For example, the line “Null country with non-Null state\/province” lists all records that do not have a country value in the specimen record even though there is a state or province listed in the state\/province field of that record. You can click “List records\u0026hellip;” and assign a higher geographic value to those records (see example below).\nSimilar lists are provided for records with empty state\/province fields but filled county fields and for records with empty county fields but filled locality fields.\n</p><p>geography</p><p>data cleaning</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/geography\/"},{value:"[coll_manager] Adding Specimens to Crowdsourcing Queue",label:"<p> This page describes how a collection manager or administrator can add specimen records to the queue to be crowdsourced and how to review crowdsourced transcriptions.\n A webinar demonstrating this process can be found here: https:\/\/youtu.be\/1IZhUMqCGvs\n Specimens can be crowdsourced for transcription if they (1) have an associated image, and (2) have a processing status of \u0026ldquo;unprocessed.\u0026rdquo; To manage the processing status of specimens, you can use the Batch Editor Tool if necessary. Specimen images that are added to a portal without associated data are automatically assigned a processing status of \u0026ldquo;unprocessed\u0026rdquo;; however, if skeletal data were uploaded with the images, the processing status was defined by the person who uploaded the data.\n Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). In the Administration Control Panel, click Processing Toolbox. Click the Crowdsourcing tab.  Don\u0026rsquo;t see a Crowdsourcing tab? Crowdsourcing may not be enabled in your portal. Contact your portal administrator for more information.\n On the resulting page, you will see tallies of how many records are in the queue, how many have been processed, and how many could be added to the queue. Click the Add to Queue button to see additional options for refining which specimens you will include in the queue (shown in screenshot below).  You can\u0026rsquo;t conduct a search for values that don\u0026rsquo;t exist! Many \u0026ldquo;unprocessed\u0026rdquo; records don\u0026rsquo;t have data for any fields; therefore, they might not be included if you narrow your search by one of the additional fields shown above.\n Define how many specimens you will add to the queue at one time by modifying the number in the Record Limit field.  It is recommended to not add many thousands of records to the queue at one time. Too many records may leave your crowdsourcers feeling hopeless to finish the task!   Click the Add to Queue button.  </p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/crowdsource\/edit\/"},{value:"[coll_manager] Removing Specimens from Crowdsourcing Queue",label:"<p>Specimens will remain in the crowdsourcing queue until their processing status has been changed from \u0026ldquo;unprocessed\u0026rdquo;. Use the occurrence editor or the Batch Editor tool to change the processing status of the specimen or specimens that you would like removed from the queue.\n</p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/crowdsource\/remove\/"},{value:"[coll_manager] Reviewing and Approving Crowdsourcing",label:"<p> This page describes how to review and approve edits that have been made to records in the crowdsourcing module.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). In the Administration Control Panel, click Processing Toolbox. Click the Crowdsourcing tab.  Don\u0026rsquo;t see a Crowdsourcing tab? Crowdsourcing may not be enabled in your portal. Contact your portal administrator for more information.\n If there are records that need approval, they will show up after Pending Approval. Click Review to review them.  Alternatively, you can review records edited by individual users by clicking Review in the Pending Review column in the tables at the bottom of the page.   Review records in the provided table, or click the pencil icon to view individual records along with their images. To approve records, click the check box next to the records you wish to approve, scroll down to the bottom of the page, and click Submit Reviews.  Click \u0026ldquo;Additional Options\u0026rdquo; at the bottom of the page for more options. From here you can, instead of reviewing a record, \u0026ldquo;Remove Points and change to Not Reviewed\u0026rdquo; (if the record had previously been reviewed\u0026quot; or \u0026ldquo;Move back into crowdsourcing queue as Open Records\u0026rdquo;\n </p><p>crowdsourcing</p><p>volunteer</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/crowdsource\/review\/"},{value:"[editor] Crowdsourcing (Volunteer Instructions)",label:"<p> This page provides basic instructions that you can give to your crowdsourcers to get them started in the crowdsourcing module. Note that you will want to add specific transcription instructions for your collection.\n Creating an Account First, you’ll need to create an account in the portal for which you will be crowdsourcing.\n Navigate to the portal in which you will be georeferencing. On the right side of the grey menu, click New Account. Enter the requested information, check the I’m not a robot box, and click Create Login.  After you’ve created an account once, you will use this login information for all future sessions. Just click Login and enter your username and password.\n Navigating to Records to Transcribe  Once you are logged in, click the Crowdsourcing tab from the main menu. Your stats will be shown in the top right side of the page under Your User Status. Here you can see how many specimens you have transcribed, as well as how many points are awaiting approval and how many have been approved.  Pending points are those that have not yet been reviewed by a curator  Curators are very busy, please be patient if your pending points don’t turn into approved points very quickly!   Approved points are those that have been reviewed and approved by a curator   In the bottom table that reads Your Stats by Collections, decide which herbarium you would like to work on (or select the herbarium instructed by your event leader). Click the number in the Open Records column that belongs to that collection’s row. On the resulting page, you’ll see a table of specimen records that require transcription. Randomly select one of the specimens by clicking on its Symbiota ID number (leftmost column).  If you see a message that says Record Locked!, go to the next specimen by clicking the double arrows at the top of the form, just under the magnifying glass   Transcribe the record as instructed.  This material is based upon work supported by the National Science Foundation under Grant Numbers 2001500, 2001509, 2001394, 2001422, 2001427, 2001431, 2001299, 2001354, 2001288, and 2001345. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation.\nAdditional resources about crowdsourcing in a Symbiota portal can be found below:\n Transcription Guide for the Mid-Atlantic Megalopolis project Intro to Crowdsourcing for the GLOBAL project Navigating Crowdsourcing in the Pteridophyte Portal Transcription Rules for Crowdsourcing in Pteridoportal  </p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/crowdsource\/individual\/"},{value:"[contribute] Contributing",label:"<p>Contributing to the Symbiota Docs Symbiota Docs is a collaborative effort of the Symbiota community, coordinated by the Symbiota Support Hub. We welcome and encourage all contributions.\nWhat can I contribute?  Minor error fixing (misspellings, formatting issues, broken link reporting) Reviewing existing content Creating new content or expanding existing content Translating content (currently we support Spanish, French and Brazilian Portuguese) Bug reporting Content ideas  How to contribute We\u0026rsquo;ll be offering hands-on workshops on how to manage content in GitHub! Stay tuned.\n Technical details (for users familiar with GitHub)    Content on this site is open source and managed through the GitHub development platform. The front-end is compiled with Hugo. Each topic is structured in directories that include a markdown file that receives the name of _index.\u0026lt;language tag\u0026gt;.md. Anyone with a GitHub account can create a fork of this repository and contribute directly in the folder structure, or look for the \u0026ldquo;Edit this content on GitHub\u0026rdquo; button in the bottom of a particular page to edit its content in a new fork. To submit your contribution, please open a pull request. To submit bug reports, please open an issue in our GitHub repository. To see discussions or start a new conversation, please visit our Discussions tab in our GitHub repository.   If you are willing to learn to manage content in GitHub     Set up a user account in GitHub. Familiarize yourself with Markdown. Here\u0026rsquo;s a link to a nice cheatsheet provided by the Markdown Guide. Learn the basics of the Git flow on GitHub: forking a repository, pushing and opening a pull request on GitHub. Browse the structured topics inside the content folder in the Symbiota Docs GitHub repository. Explore the available formatting options inside the resources\/templates directory. Optionally: contact the Symbiota Support Hub indicating your intent to contribute via GitHub. We will meet with you to discuss your contributions and answer any remaining questions.     If you are not familiar with GitHub or unwilling to learn (don\u0026rsquo;t be afraid to learn new things!)    Send your contributions as Word or text documents to the Symbiota Support Hub. Make sure you include your full name so we can credit you as the author! Please include any images as separate files.   </p><p>contribute</p><p>translations</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contribute\/"},{value:"[coll_manager] **Joining an Existing Portal**",label:"<p>Click the button below to find a list of active Symbiota portals. You will want to select a Symbiota portal that matches the scope of your data. You can add your data to a Symbiota portal as a \u0026ldquo;live\u0026rdquo; collection or a \u0026ldquo;snapshot\u0026rdquo; (see this page for more information about management types in Symbiota portals).\nList of Symbiota Portals Request to Join a Symbiota Portal Once you have identified an existing Symbiota portal to which you want to publish specimen data, contact the project or portal managers to become a data provider. Portal contacts are typically listed on the homepage or within the left menu of the data portal. Contact help@symbiota.org for help if you are unable to find the portal contact.\nYou may publish data to more than one portal; however, it is recommended to only manage a particular dataset or collection in one Symbiota portal, and all other portals would host only a snapshot of those data. Different collections (e.g., vascular plants vs. lichens vs. bryophytes) can be live-managed on their respective portals.\nTypically, the portal manager will need the information in the following list to initiate your collection within the portal. Once the collection has been added, data can be loaded. See the page on Uploading Specimen Data for details on methods and acceptable data formats for upload specimen.\nInformation needed to create a new collection within a portal:\n Collection title that will be listed on the search page Institution and collection codes (e.g. acronyms) Short description of collection holdings and preparation types Contact(s): name, title, email, and ORCID, if available Link(s) to homepage and\/or other external resources Collection or institution icon – a small image that visually represents the collection, icons that are roughly the same height and width work best Preferred usage rights. The portal manager should be able to advice you on this. It may also be useful to read the following Creative Commons links: https:\/\/creativecommons.org\/licenses\/, http:\/\/wiki.creativecommons.org\/Data Whether you are publishing a snapshot of your data that will be periodically updated, or you plan to manage the data as a live dataset directly within the portal Source of the Global Unique identifier (GUID) for the specimen records. If the specimen records are managed directly within the portal, we recommend letting the portal automatically generate a GUID (UUID) when records are created. If a data snapshot is published from a local in-house data management system, GUIDs need to be generated locally and then included in the data published to the portal.  What if I have multiple collections or data types? If you have multiple collections (e.g., vascular plants, fungi, and bryophytes), it is recommnded to manage these separately in their respective data portals. This is because the taxonomic thesauri of the portals are well-curated to their focal taxonomic group, and you are more likely to be able to take advantage of digitization efficiencies (e.g., duplicate matching) in that portal. See more information on our Managing Multiple Portals page.\n</p><p>join</p><p>contribute</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/join\/"},{value:"[col_obs] Collector\/Observer Guide",label:"<p>This guide will describe how collectors of data and\/or specimens can add records to a Symbiota portal\u0026rsquo;s Personal Observations Profile collection. Editors of Personal Observation Profiles need not be affiliated with any particular institution or collection represented in the portal.\nBy managing collection data in a Symbiota portal, collectors can take advantage of the Symbiota portal\u0026rsquo;s data entry form (including dropdown lists and conversion tools), georeferencing tools, and label printing features. Furthermore, this workflow saves collections time and energy because they do not need to re-transcribe data.\n Most Symbiota portals have a central collection profile generally referred to as the \u0026ldquo;personal observation profile\u0026rdquo;. It is sometimes called the \u0026ldquo;general research observations\u0026rdquo; profile or \u0026ldquo;specimens being processed\u0026rdquo; profile. For example, see SEINet\u0026rsquo;s \u0026ldquo;General Research Observations\u0026rdquo; collection here. This collection profile is shared among all users who have been sponsored as editors of this collection profile; however, users may only edit records that they have entered into this profile. Records in this profile are publicly viewable once they are entered, but they may only be edited by the user who added them.\nThe personal observation profile is intended to be a place where collectors can manage their records (e.g., add and edit data, print specimen labels) and distribute the data to the collections where they will send their specimens, if applicable. A diagram of a generalized workflow is shown below.\nThis workflow saves everyone time and energy because collections do not need to re-transcribe the data. Morever, even though there may be many duplicates for a specimen, the data only need to be entered once. This single data entry also reduces the likelihood of data entry error.\nTo manage observation-only data (i.e., no specimens) see iNaturalist.\nUsers must be sponsored by a collection administrator in order to add and edit records to this collection profile.\nSponsored users have the following options for managing their data in the Personal Observations Profile collection. Related tutorial documentation is linked below.\n Search Records: search through the records that you have entered into this profile. You can conduct custom, detailed searches from this interace. Tutorial information can be found on this page. Add a New Record: tutorials for adding records here and for editing records here Print Labels: print labels for specimens of all types, including specimen packets. Tutorials here. Print Annotation Labels: print annotation\/determination\/identification labels for specimens that already have full labels. Tutorials here. Submit image-vouchered observation: use an image to create a new record, if you did not collect a specimen for the record. The available data entry fields are similar to those used to add a new specimen record (see above). Review\/Verify Occurrence Edits: view, revert, or apply edits that you have made to your records. Tutorial here. Download backup file (CSV extract): download a copy of your data to review or send to a collection. CSV files may be opened in Excel or a similar spreadsheet software. View User Comments: because your data are publicly visible, users may leave comments on your records. Use this tool to view and manage those comments. See this page for more information.  </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/col_obs\/"},{value:"[coll_manager] Managing Multiple Portals",label:"<p>What do I do if I have multiple collections? Should I keep them all in one portal? If you have multiple collections (e.g., vascular plants, fungi, and bryophytes) that you would like to manage live in Symbiota-based platforms, it is recommnded to manage these separately in their respective data portals. This is because the taxonomic thesauri of the portals are well-curated to their focal taxonomic group, and you are more likely to be able to take advantage of digitization efficiencies (e.g., duplicate matching) in that portal. For example, the Chico State University Herbarium manages their vascular plant data in the Consortium of California Herbaria (CCH2) portal (the focus of which is vascular plants from herbaria in California), their bryophyte data in the Bryophyte Portal, and their Lichen data in the Lichen Portal. However, a copy of their bryophyte and lichen data can be found in the CCH2 portal as well, since it is a resouce for the California community.\nWhat if I want to have the same collection in more than one portal? No problem! In some cases, multiple portals may benefit from including some or all of your collection\u0026rsquo;s data. If your collection(s) fit(s) within the scope of multiple portals, you can load a snapshot of your data (or a subset of your data) into various portals, even if you do not manage your data in that portal. This can be done easily by creating a \u0026ldquo;Darwin Core Archive Provider\u0026rdquo; upload profile that maps a Darwin Core Archive from another Symbiota portal. To pass only a subset of your data to the other portal, you can use Custom Occurrence Record Import Filters, available for Darwin Core Archive and IPT imports, or you can work with your portal administrator to set up a Stored Procedure that will filter your dataset to include only specific records.\nWhat if my collection is currently in one portal, but I want to split it into multiple portals? As described above, you can pass only a subset of your data into another portal using Custom Occurrence Record Import Filters, available for Darwin Core Archive and IPT imports, or you can work with your portal administrator to set up a Stored Procedure that will filter your dataset to include only specific records. Once this is complete, your portal administrator(s) can remove the specimen records from the originating portal, if desired.\n</p><p>multiple collections</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/join\/mult\/"},{value:"[coll_manager] Collection Statistics",label:"<p> This page describes how you can find information about how many occurrences and images you have in your collection.\n Viewing Collection Statistics Collection statistics are generated on command and are not created \u0026ldquo;on the fly\u0026rdquo;. A collection administrator should periodically refresh collection statistics. Collection statistics are also automatically refreshed when a new Darwin Core Archive is published\/created.\n Statistics relating to the number of specimens, images, georeferences, and taxa within a collection can be found on that collection\u0026rsquo;s Collection Profile page. An example collection profile is shown in the screenshot below and can be found here. The collection statistics are found at the bottom of the page and include:\n Number of total records Number of records that are georeferenced Number of records that have associated images Total number of images in the collection Number of specimens that are identified at least to the level of species Number of families, genera, species, and total taxa that represented in the collection (NOTE: these numbers are calculated using only taxonomic names that have been indexed to the taxonomic thesaurus)  Collection profiles can be accessed by visiting the Collection Search Page (click Search Collections) at the following URL: [BASE URL]\/collections\/index.php. For example, for the CCH2 portal, the full URL is https:\/\/cch2.org\/portal\/collections\/index.php.\nIf you are an editor or administrator for a collection, you can also access your statistics by clicking My Profile \u0026gt; Occurrence Management \u0026gt; name of collection.\nRefreshing Collection Statistics Collection statistics are generated on command and are not created \u0026ldquo;on the fly\u0026rdquo;. To refresh your collection statistics, navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection) and click Update Statistics (bottom link in the Administration Control Panel).\nProcessing Status Statistics in the Reports Tab To view statistics regarding the processing statuses in your collection, navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection), then click Processing Toolbox. Click the Reports tab. A table will be displayed that shows you the number of specimens in each processing status. To view occurrences one by one, click the edit (pencil) icon in the Count column. To view occurrences as a list, click the table icon in the Count column.\nThis page will also show you how many of your occurrences do not have linked images and how many lack skeletal data (i.e., a value in the Scientific Name field).\n</p><p>statistics</p><p>number of specimens</p><p>reports</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/stats\/"},{value:"[portal_manager] Portal Manager Guide",label:"<p>Welcome to the Portal Manager Guide! This guide will describe how to setup and maintain a Symbiota portal. It will cover (coming soon!) how to change configuration files, access the Symbiota backend, and perform other advanced functions.\nFor current notes related to the ongoing Symbiota developments and how to implement new features in your portal, please refer to the Current Notes section.\n </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/"},{value:"[coll_manager] Taxonomic Cleaning Tools",label:"<p> This page describes how to use the two taxonomic cleaning tools in Symbiota portals. For more information about the taxonomic thesaurus, visit the Taxonomy page.\n The taxonomic cleaning tools are great resources for cleaning up misspellings and other taxonomic name entries that do not link to names in the central taxonomic thesaurus. They are meant to aid in locating and fixing taxonomic errors and inconsistencies.\nAnalyze Taxonomic Names   This function analyzes the values that exist in the Scientific Name field in your database and flags the values that are not linked to the taxonomic thesaurus (i.e., are not linked to a recognized name). Note that this tool does NOT evaluate whether that name is accepted according to current taxonomy. The number of unrecognized scientific names is listed at the top of the Action Menu box.\nYou can evaluate the unrecognized scientific names and link them to the names of the correct taxa by clicking the name of the Taxonomic Resource to which you want to compare the names, selecting the target kingdom, and clicking the Analyze Taxonomic Names button. If you wish to begin at a certain name (e.g., you have already checked names up to Mentzelia), you can enter this name in the Start Index field. You can also alter the number of names you would like the portal to analyze per run in the Names Processed per Run field. There are multiple types of results from running this tool. In general, the tool will list the name that it is attempting to index (i.e., match to an existing taxonomic name), search your defined taxonomic resource for that name, and, if that name is not found in the taxonomic resource, check the taxonomic thesaurus for similar names to the unrecognized name. You then have several options for resolving the unrecognized name, depending on the type of result. To view the specimen(s) associated with any unrecognized name, click the pencil icon to the right of the name and number of specimens, in brackets. The types of results are explained in more detail below.\nExample 1: Misspelling, incomplete name, or orthographic variant This is the most common type of result. In this case, you discover that the unrecognized name was misspelled or spelled differently than the recognized name. You can click “remap to this taxon” to change the name of the specimen(s) to this recognized name.\nExample 2: Unpublished or unrecognized taxon name In this case, you find that the name listed on the specimen was not published or is not currently listed in the taxonomic thesaurus. You must then use taxonomic expertise to decide whether this specimen should be remapped to a different taxonomic name (if you are confident that they are synonymous) and\/or annotated, or if the taxonomic name should be left as is and it should be included in the taxonomic thesaurus. If you believe that a name should be found in the taxonomic thesaurus and you do not find it with a manual search, contact the portal manager.\nExample 3: Name exists but is not found in taxonomic thesaurus When the portal does not find a name in the taxonomic thesaurus, but it does find the name in the taxonomic resource, it will import the taxonomic name into the taxonomic thesaurus. This will automatically map the specimen’s taxonomic name to this newly added entry in the taxonomic thesaurus as well.\nIf you have not analyzed all the taxonomic names in one pass, you can click the Continue Analyzing Names button to have Symbiota check the next 20 (or whatever number is defined by the user) names.\nTaxonomic Distributions Much like the Geographic Distribution viewer, the Taxonomic Distribution viewer can be used to examine the families, genera, species, and infraspecific taxa that exist in your database. Misspellings, non-standardized entries, or suspected mistakes can be detected using this tool. To view the genera for each family, click the name of the family. Then, to view the species for each genus, click the name of the genus, and so on. A user with administrator permissions can correct errors in taxonomic names individually by clicking the number next to the taxonomic name (circled below), or the user can search for those records using the record search form and batch edit them.\n</p><p>taxonomy</p><p>data cleaning</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/taxonomy\/"},{value:"[editor] Rotating Images",label:"<p> This page describes a common issue of image rotation and suggests how to make sure your images show up in the correct orientation in a Symbiota portal.\n Specimen images are often taken with the camera oriented at a 90° angle, which produces a specimen image that is rotated sideways. To ensure images are in the correct orientation in a Symbiota portal, you must ensure that the image has been rotated using a program that actually rotates the image data rather than just adding a EXIF metadata orientation tag (which is how most cameras and file explorers rotate images).\nAdjusting the camera settings to define the correct orientation, or batch rotating the images within file explorer on a PC are not ideal because they only modify the EXIF metadata orientation tag within the file and do NOT actually rotate the image file (pixel data). While many software programs (e.g. Chrome, Firefox, MS Windows Explorer, etc) will auto-rotate correctly based on the orientation tag and display the image correctly, not all software programs will do this (e.g. Internet Explorer, some browser plugins, etc). Furthermore, when images are upload on the web, the orientation tags are often not carried over to web derivatives generated by the file upload program (e.g. thumbnail, web views, etc). For example, the image links below will all have the wrong orientation when viewed in Internet Explorer (PC version), and only the full size (original) image will self-orient within Chrome or Firefox.\nThumbnail\nWeb View\nFull Size\nThe safest solution is to rotate the actual image data (pixel definition) rather than just the setting the orientation tag. This can be done as a batch processing using image editing programs such as Adobe Lightroom or Photoshop. The link below provides a list of a number of free options for lossless rotation of images. For example, you can download “JPEG Lossless Rotator”, which is easy to use, shows the true rotation of the images within a folder, and provides lossless rotation tools.\nList of Lossless Rotation Apps\nLossless Rotator App by Anny Studio\nTroubleshooting Certain metadata can affect how images display in your web browser. If an image appears incorrectly rotated in your portal, try switching browsers to see if this corrects its orientation.\n</p><p>images</p><p>image orientation</p><p>image rotating</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/rotate\/"},{value:"[portal_manager] Adding Taxa to the Taxonomic Thesaurus",label:"<p> A user must have Super Administrator or Taxonomic Editor permissions to add to or edit the thesaurus.\n There are three main ways that a user can add taxonomic names to the taxonomic thesaurus:\n Individually via the Taxonomy Explorer interface Automatically via the Taxonomic Cleaning Tools Batch via the Taxonomic File Upload Tool  Additional information about options 1 and 2 are provided below (see the link above for instructions for option 3).\nAdding taxa individually Both the parent taxon (e.g., genus of the species you would like to add) and the accepted name of the taxon (if not the name you are adding) must already be in the taxonomic thesaurus for you to add it. If either one of these is missing, you will get an error when you try to complete the form as below.\n  Navigate to the taxonomic thesaurus (Sitemap \u0026gt; scroll down to Taxonomy \u0026gt; Taxonomic Tree Viewer), or just add \u0026ldquo;taxa\/taxonomy\/taxonomydisplay.php\u0026rdquo; to your base path. Click the green plus sign at the top right corner of the screen. Enter information about the taxon in the provided form.    Enter the taxonomic name without the authorship in the Taxon Name field Enter the authorship into the Author field For hybrid taxa, select the multiplication symbol from the dropdown list in front of the UnitName2 field For infraspecific taxa, use the small UnitName3 field for the infraspecific epithet type (e.g., var., subsp., f.), if applicable, and the long UnitName3 field for the infraspecific epithet. Select the appropriate taxon rank from the dropdown Taxon Rank field The form will automatically parse out the UnitName1, UnitName2, and UnitName3. Adjust these if necessary. IMPORTANT: Add a link or citation for the source of the scientific name you are adding, for example, a Plants of the World Online entry, a literature citation with DOI, etc. If the taxonomic name you are adding is the currently accepted name, leave the Accepted radio button checked. If the taxonomic name is not accepted, select the Not Accepted radio button and enter the accepted name in the provided field. If the name of the accepted species is not found in the taxonomic thesaurus (i.e., does not appear in the dropdown menu), you must add that name to the thesaurus first.  If you edit anything in the Taxon Name field after you have added to the Author, Notes, or Source fields, it will reset the form and clear anything you have entered into these fields.\n Batch adding taxa using the Taxonomic Cleaning Tool The Taxonomic Cleaning Tool is useful not only for cleaning misspellings and errors, but also for adding taxaonomic names to the taxonomic thesaurus. If a taxonomic authority has been enabled in the symbini file, you will be able to cross-reference the un-indexed name in a collection with that taxonomic authority. Running the taxonomic cleaning tool will then auto-import any names that are not indexed in the collection that can be found in that taxonomic thesaurus. See the instructions for using the Taxonomic Cleaning Tools.\n</p><p>taxonomy</p><p>taxonomic thesaurus</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/taxonomy\/add\/"},{value:"[user] Creating Datasets",label:"<p> To create a checklist, you now need to obtain permissions from a portal administrator or the Symbiota Support Hub. For assistance, email the Hub.\n You can access the tools to create a new dataset either (A) through the general search page or (B) through your Occurrence Management Page.\nThrough the general search page  Conduct a search from the main search page of the Symbiota portal. This is usually accessed by clicking \u0026ldquo;Search\u0026rdquo; or \u0026ldquo;Search Collections\u0026rdquo; on the home page of the portal. For more information about searching in a Symbiota portal, visit this page. When you conduct the search, click the \u0026ldquo;List Display\u0026rdquo; button\u0026quot;. From the List Display, click the button with the dataset icon.  This will open the Dataset Management box. Select the dataset to which you would like to add the specimens from the \u0026ldquo;Dataset target\u0026rdquo; dropdown menu, or select \u0026ldquo;Create New Dataset\u0026rdquo; to start building a new dataset. From here, you can select the specimens you would like to include in your dataset from your query by checking the boxes outlined by dotted lines for each specimen.  To only add a few specimens from the page, check the boxes next to the desired specimens, then click the \u0026ldquo;Add Selected Records to Dataset\u0026rdquo; button. To add all the specimens on the page, check the box next to \u0026ldquo;Select all records on page\u0026rdquo;, then click the \u0026ldquo;Add Selected Records to Dataset\u0026rdquo; button. To add all specimens resulted from your query, click the \u0026ldquo;Add Complete Query to Dataset\u0026rdquo; button.    Once you have added occurrences to the dataset, you will be taken to the Dataset Manager page. Click the General Management Tab to can edit or add a name, internal notes, or a description to the dataset, or to make the dataset public (check the \u0026ldquo;Publicly Visible\u0026rdquo; box).  Through the dataset management page  Click My Profile. Click Occurrence Management. In the \u0026ldquo;Miscellaneous Tools\u0026rdquo; box, click \u0026ldquo;Dataset Management\u0026rdquo;. Click \u0026ldquo;Create a New Dataset\u0026rdquo; (if you have no existing datasets) or the green plus sign at the top right of the page. Enter a name, notes, and description (if desired) for your dataset. You can also make the dataset public by checking the \u0026ldquo;Publicly Visible\u0026rdquo; box. Click Create Dataset. This will take you back to the \u0026ldquo;Occurrence Dataset Management\u0026rdquo; page. To add occurrences to a dataset, click the name of the dataset you created, then click the \u0026ldquo;occurrence search page\u0026rdquo; link. Or you can start at step 1 in the instructions for creating datasets Through the general search page above.  </p><p>dataset</p><p>inventory</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/dataset\/create\/"},{value:"[editor] Trait Scoring from Images",label:"<p> This page describes how to score traits for an occurrence from an image of that occurrence available in a Symbiota portal\n Trait scoring tools are not activated in all portals. Contact your portal manager for more information.\n Scoring traits for a single specimen Navigate to the Occurence Editor for the occurrence you wish to score (see this page for help finding specimens) and click on the Traits tab. Any traits available for scoring will be displayed in boxes in this tab. To view additional scoring optiosn for a given trait, click the black triangle at the top right of the box (highlighted below). From here, you can score the traits from any available images. You can also remove trait information by clicking the Delete Coding button.\nScoring traits using the Trait Scoring from Images tools This tool can be used to efficiently score specimens of the same taxon or from the same region.\nFrom the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection)), click Occurrence Trait Coding Tools, then Trait Coding from Images. In the Filter box, select the trait that you wish to score, followed by the name of the taxon for which you wish to score (if applicable). Phenological Traits from the Trait dropdown menu. You can also refine your search by using the All Counties \u0026amp; All States dropdown list. This field provides a dropdown menu of all countries and states that are indexed in CCH2. If you select one of these geographic criteria, the tool will only show you specimens from that country\/state.\nClick the Load Images button. On the next page, you will see an image of a specimen of the selected taxon on the left, and an Action Panel on the right (see screenshot below). Note that you can edit the contents of the Filter box here as well. Depending on the scoring schema available for the trait you are scoring, you may be able to view more scoring options by clicking the black triangle inside the Action Panel. Select the trait values that you wish to apply to the specimen and click Save and Next.\nTo view the record associated with the specimen, click the catalog number that is displayed just above the image. To skip the specimen, click SKIP just to the right of the catalog number.\nNote that you can also switch between the Medium and High resolution versions of the image using the Med Res. and High Res. radio buttons on the top left of the image.\n</p><p>trait scoring</p><p>attribute scoring</p><p>phenology</p><p>traits</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/trait\/img\/"},{value:"[coll_manager] Public Comments",label:"<p> This page describes how to view comments on your occurrences. Any Symbiota user can create a comment on an occurrence record. This can be a useful feature for community members to ask questions or flag data quality issues.\n To view and vet comments posted on records from your collection, navigate to the Administration Control Panel: My Profile \u0026gt; Occurrence Management \u0026gt; name of collection and click “View Posted Comments.” If there is no orange text to the right of this option, you have no comments that have not yet been reviewed.\nOn the Comments Listing page, you can:\n hide comments from the public (in cases of sensitive or personal data) mark comments as reviewed (i.e., you have resolved the issue described by the comment) delete comments (in cases of spam or inappropriate content) view specimen records associated comments by selecting the URLs above comment details (catalog number, collector, collector number, and date)  In the event that you have many comments to work through, you use the Filter Options box (top right) to view only comments that were posted by a certain user, have a certain status (public\/non-public\/reviewed), or were posted within a certain date range.\n⚠️ Spam comments are so far rare, but one should regularly review posted comments to ensure no inappropriate content has been posted to your collection. If you find any inappropriate comments, please contact the portal manager immediately so they can remove this user from the portal.\n </p><p>comments</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/public_feedback\/comment\/"},{value:"[editor] Adding Records",label:"<p>There are several ways to add occurrence records, depending on your level of permissions. Editors can add specimens as skeletal records, as full records, and by uploading an image.\n</p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/"},{value:"[editor] Tagging Images",label:"<p> This page describes how to tag images of occurrences.\n Tagging individual images Occurrence images can be tagged with important metadata about what is contained within that image. For example, a specimen can be marked as representing an adult organism or an immature organism. Some possible tags are shown in the image below:\nTo tag an image, navigate to the occurrence to which that image belongs (see this page). Then click the Images tab. Click the edit button and scroll down to the checklist of tags that you can add to the image. Click the Submit Image Edits button when complete.\nBatch tagging images  Navigate to the Image Search interface in your portal. This is usually listed on main menu of your portal, potentially nested in an \u0026ldquo;Images\u0026rdquo; dropdown section. If you can\u0026rsquo;t find it here, navigate to your Sitemap and click \u0026ldquo;Interactive Search Tool\u0026rdquo; under the Image Library header.  Even though you access this tool through the Image Search interface, you will need editor permissions to tag images in the search results.\n Enter the search criteria for the images you would like to tag in the Search Criteria box.   To select images from a specific collection, select the \u0026ldquo;Specimen\/Vouchered Images\u0026rdquo; from the Image Type dropdown box and then use the checkboxes to select the collection(s) from which you would like to see images. Click the plus icon to expand the lists of collections, if applicable.  Click the Load Images button. Click the pencil icon in the top right corner of the Search Criteria box to toggle the batch tagging interface ON. A new box labeled Action Panel will show up below the Search Criteria box. Select the Image Tag from the dropdown list that you would like to apply, then check the boxes next to the images to which you would like to apply that tag. Click the Batch Assign Tag button to apply that tag to the selected images.  </p><p>image tags</p><p>image annotation</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/tag\/"},{value:"[coll_manager] Image URL Upload",label:"<p> This page describes how to associate externally hosted images with records in your portal using a CSV of image links\/URLs.\n Adding Images via URLs  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Import\/Update Specimen Records, then select \u0026ldquo;Extended Data Import\u0026rdquo;. Click the \u0026ldquo;Choose File\u0026rdquo; button to upload a properly formatted associations file into the uploader (see sections below for formatting requirements). Select \u0026ldquo;Image Field Map\u0026rdquo; from the Import Type dropdown menu. Click the \u0026ldquo;Initialize Import\u0026rdquo; button. Map the fields in your input file (shown on the left) to appropriate target fields (see table below). If you would like to create a new record for each identifier that does not match an existing identifier in the system, check the box labeled \u0026ldquo;Link image to new blank record if catalog number does not exist.\u0026rdquo; Click the Import Data button.  ⚠️ When providing images via URLs, the URLs should begin with \u0026ldquo;https:\/\/\u0026rdquo; (not \u0026ldquo;http:\/\/\u0026quot;) in order for them to display correctly in various browsers. This is especially important for thumbnail images. If you provide images using URLs containing \u0026ldquo;https\u0026rdquo; and they do not display correctly in your Symbiota portal, check that the server that hosts your images maintains an SSL certificate that supports the use of \u0026ldquo;https\u0026rdquo;.\n Setting Up an Image Import File A template for this upload type can be found here.\nThe required fields for this upload type are (1) a subject identifier for the occurrence you are linking to (occurrenceID, catalog number, and\/or other catalog number), and (2) originalUrl. The originalUrl should include a direct link to the desired image (i.e., ideally ending in .jpg or .tif or some other image file extension). Note that uploading a link to a webpage where that image may be found rather than directly to the image may result in images not displaying as expected.\nOptional fields are listed in the table below.\nImage Url Mapping Fields    Field Name Data Type (Length in characters) Description     subject identifier: catalogNumber Text (32) The primary human-readable identifier for the record you are linking the image to.   subject identifier: otherCatalogNumbers Text (45) An alternative catalog number stored as an \u0026ldquo;Additional Identifier\u0026rdquo; in the portal for the record you are linking the image to. See this page for more context.   subject identifier: occurrenceID Text (255) The global unique identifier (GUID) of the record you are linking the image to.   accessRights Text (255) A url to the page describing who can access the resource or an indication of its security status.   anatomy Text (100) Textual description of the anatomical features visible in the image or media item.   archiveurl Text (255) A stable link to a publicly available archival image (generally a large TIF or RAW file). Not commonly used.   caption Text (100) A textual description of the image or media item. The caption will be publicly viewable and should ideally be kept brief.   copyright Text (255) The organization to which a copyright or license belongs.   format Text (100) The media type of the media item, such a JPG, TIFF, video, GIF, etc.   hashFunction Text (45) Name of the function that was used to generate a hash (fixed-size value representation of the item) for the media item. See this article for more information about hashing.   hashValue Text (45) A fixed-size value representation of the media item generated using a given hashing algorithm. See this article for more information about hashing.   mediaMD5 Text (45) Hash for the image or media item created using the MD5 algorithm. See this article for more information about hashing.   notes Text (350) Comments or other notes regarding the image or media item.   originalUrl Text (255) The large or highest-quality version of the image or media item. This url will be used to create thumbnail and web-ready versions of images using the thumbnail maintenance tool, if used.   owner Text (250) The legal owner of the image or media item.   photographer Text (100) The creator of the image or media item.   photographerUid Integer (10) The unique number assigned to the Symbiota portal user who created the image or media item. Note that this can only be populated in cases when the photographer has a user account in the portal.   referenceUrl Text (255) A link to a URL or page that features the image in the original software.   rights Text (255) The rights applied to the image or media item dictating appropriate reuse of the item. For example, a Creative Commons license may be applied here.   sortOccurrence Integer (11) The sort order of the image as it will appear on the page related to the occurrence record. Lower numbers will be displayed before higher numbers.   sourceIdentifier Text (150) A unique identifier belonging to the image or media item provided by the source of the media item.   thumbnailUrl Text (255) A thumbnail-sized version (e.g., ~300 pixels in the widest dimension) of the image or media item to be shown quickly in search results.   url Text (255) A web-ready version of the image or media item, such as a JPG no larger than 2 MB. This is the version that is displayed as \u0026ldquo;medium resolution\u0026rdquo; in the occurrence editor.    </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/images\/url_upload\/"},{value:"[user] Contacting a Collection",label:"<p>To contact the collections manager or curator of a collection, visit the Collection Details page of that collection by doing the following:\n Select Search Collections from the portal menu. Click \u0026ldquo;more info\u0026hellip;\u0026quot; after the name of the collection you are interested in. Contact information for the person or people responsible for the collection will be listed at the top of this page, as illustrated below.          Contact information can be located at the top of each collection\u0026rsquo;s profile page. The example above is from the CCH2 Data Portal.    </p><p>contact</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/public_feedback\/contact\/"},{value:"[portal_manager] Citing Symbiota portals and occurrences",label:"<p>August 2022-current: Citation formats and introduction to CITEME.txt Citation formats and the creation of a CITEME.txt file have been introduced recently in Symbiota (Pull Request #261).\nDefault formats Symbiota portals have now a standardized way for users to cite them and their data. These default, dynamic, citation formats have been introduced (and can be customized as necessary) and can be found in the [root]\/includes directory:\n General portal: found in the citationportal_template.php; Collection (for collections not published in GBIF): citationcollection_template.php; Collection published in GBIF: citationgbif_template.php; Dataset: citationdataset_template.php;  Additionally, the template for the Usage Policy page ([root]\/includes\/usagepolicy_template.php) has been updated to include the dynamic formats as well. This means that portal managers that would like to customize citation formats should only do it by modifying the formats themselves.\nWhere in Symbiota are formats used? The citation formats are inserted in certain context-specific pages and in downloads.\nThe general portal format will be seen by default in the Usage Policy page.\nThe collection formats will be inserted in each Collection Profile page, depending on whether each collection is published to GBIF or not.\nExample of collection profile page that has not been published to GBIF in the NEON Biorepository Data Portal.\nExample of collection profile page that has been published to GBIF in the NEON Biorepository Data Portal.\nThe dataset format will be inserted in each Dataset page.\nAll of the formats might be included, one at a time, in the CITEME.txt file included with the downloaded package from a Symbiota portal, depending on the context in which a package was requested. For instance, downloading the results of a search will include a CITEME.txt file that will contain a citationportal.php format. On the other hand, downloading a DwCA from a collection page (which is published to GBIF), will include the citationgbif.php format. This behavior was designed to better inform users where their occurrence data is coming from and when.\nExample of a CITEME.txt file packaged within a DwCA download from the NEON Biorepository Data Portal.\nActivating format usage To make sure your portal uses the citation formats, all you have to do is make a copy of each citation format template, and remove the _template string from their file name.\nCustomizing formats Please note that the citation formats should only include plain text and\/or PHP. No HTML should be included in the formats, because they are used in different contexts.\n To customize each format, open the copies of the templates and make the desired changes. Save them. As the copies of the templates (without the _template string in the file name) won\u0026rsquo;t be tracked by Git, they won\u0026rsquo;t be overwritten by future releases.\nGBIF widget The Collection Profile page ([root]\/collections\/misc\/collprofiles.php) has been updated to include each appropriate citation format (if not published to GBIF, it will include by default the citationcollection.php format; if published to GBIF, it will include the citationgbif.php format).\nIf a collection has been published to GBIF, the page will fetch the GBIF citations widget using the collection-specific GBIF Dataset Key.\nExample of a GBIF widget added to a NEON Biorepository Data Portal collection profile.\nDependencies For collections that publish to GBIF, data referring to the GBIF-specific citation format will be fetched from the uses GBIF\u0026rsquo;s API v1 to fetch collections information (like gbiftitle and DOI).]\n</p><p>citing</p><p>citations</p><p>downloads</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/current_notes\/citations\/"},{value:"[portal_manager] Batch Loading Taxonomic Data",label:"<p> This page describes how taxonomy can be added to a Symbiota portal in bulk.\n A user must have Super Administrator or Taxonomic Administrator permissions to add to or edit the thesaurus.\n Format the Input File Taxonomic names can be batch loaded into the thesaurus from a flat, structured comma- or tab-delimited text file. To upload an Excel file, first save as a CSV (comma delimited) text file. Formatting this file correctly is important to ensuring your taxonomy is successfully imported. At minimum, the file must include a column for ScientificName and the parent taxon (i.e. the taxon that nests immediately above the scientificName in your taxonomic hierarchy).\nFor example, to import the name Buccinum polaris Gray, 1839, your input file could include the following fields:\n   Taxon Rank Parent ScientificName Family Author RankID     species Buccinum Buccinum polaris Buccinidae Gray, 1839 220    However, if the higher taxonomy for this name does not already exist in your portal, the following rows should also be included in your file:\n   Taxon Rank Parent ScientificName Family Author RankID     division Animalia Mollusca   30   class Mollusca Gastropoda   60   order Gastropoda Neogastropoda   100   superfamily Neogastropoda Buccinoidea   130   family Buccinoidea Buccinidae Buccinidae  140   genus Buccinidae Buccinum Buccinidae  180   species Buccinum Buccinum polaris Buccinidae Gray, 1839 220    Download example CSV View example as a CSV\nTips for preparing your input file  Parent taxa must already exist in the portal\u0026rsquo;s thesaurus or be included in the input file. As an alternative to a parent taxon column, as shown above, the input file can contain columns for the core hierarchy that is defined within the taxonunits table (e.g. kingdom, division, class, subclass, order, family, etc). Values for rankid are assigned on the backend of your database in the table taxonunits. These value may vary between Symbiota portals. If you do not have access to these values, the Symbiota Support Hub can provide them to you for SSH-hosted portals. If a taxon\u0026rsquo;s acceptance status is not defined (0 = not accepted, 1 = accepted ), all taxa will be assumed to be accepted. Include a separate column in your input file to denote acceptance status, if desired. In the absence of a hierarchical definition (e.g. the parent taxa), infraspecific, specific, and genus linkages will be determined from ScientificName and linked to Family. If the Family value does not yet exist in the thesaurus and the hierarchy above Family is not defined in upload field, the taxon will be linked directly to Kingdom. You can only batch input taxonomy for one Kingdom at a time (Animalia, Fungi, etc.) Make sure not to map more than one source column to the same target, and that source column names are unique from one another. The batch loader will import new taxonomy in the order it appears in the file, starting with the first row. It is therefore best to organize the input file such that highest ranks appear first, followed by lower ranking taxa. Non-ranked nodes \/ unranked clades are supported by the bulk loader and do not require rankid values. However, it can be easier to ingest these names using the frontend interface, especially if only a few unranked names are to be added. Many fields that appear on Taxon Profile pages can be batch imported using this tool (e.g. vernacular names, language values, etc.). To preview what fields are available for bulk importing, create a small test CSV and open it using the steps 1-4 outlined in the following section. Contact the Hub or someone with backend access to add new ranks. Ranks are added on a per-kingdom basis and may require community discussion in order to be inserted into the hierarchy.  Ingest the File Using the Batch Loader Once your file is prepared, if your user account has the appropriate permissions, you can batch upload taxonomy from a spreadsheet (CSV) or text file by navigating to Sitemap \u0026gt; Taxonomy \u0026gt; Batch Upload a Taxonomic Data File \u0026gt; Taxa Upload.   Select your formatted input file.\n  Select the Target Thesaurus. (Your portal may only have one option.)\n  Select the Target Kingdom (all names in the input file will be associated with this Kingdom).\n  Once you select Map Input File, you will be prompted to map the fields in your input file to target fields in the database. Note that the parent taxon = parentstr and scientificName = scinameinput.   Once the fields are mapped, select Verify Mapping.\n  Once the file is processed, you will be given the option to review the new names as they were imported into the database\u0026rsquo;s internal staging tables. The is your last chance to preview changes before they are made permanent in the thesaurus; do so by selecting the link, Download CSV Taxa File. Look for an errorstatus column in the CSV explaining any records that failed to import.\n  If you are satisfied with the import, select Activate Taxa to transfer the new taxonomy to the portal\u0026rsquo;s thesaurus. If not, close the window and start over.\n  Tips for batch importing taxonomy  If you are new to this process, complete a few test imports to be sure your new taxon records are ingesting correctly. You can review new records in the Taxonomy Explorer (Sitemap \u0026gt; Additional Resources \u0026gt; Taxonomy Explorer). For especially complex taxonomic trees, import higher taxonomy first, then import new genera and species-level names. When batch importing synonyms, import all accepted names (and, if needed, their higher taxonomy) first. Then import the synonyms and their associated higher taxonomy. A column containing the accepted name for each synonym can be included in your input file.  </p><p>taxonomy</p><p>taxonomic thesaurus</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/taxonomy\/batch_load\/"},{value:"[portal_manager] Checklists",label:"<p>June 2022-current: Checklists permissions The \u0026ldquo;Create a Checklist\u0026rdquo; permission has been granted for the majority of existing users in active Symbiota portals, for those accounts created before June 2022. If a user does not have \u0026ldquo;Create a Checklist enabled\u0026rdquo;, and needs it, you might be asked to add these user permissions to their account. Additionally, all new users will need to request permission to create checklists\/datasets moving forward.\n\u0026ldquo;Create a Checklist\u0026rdquo; permissions may now be granted by the administrator of any collection in a portal. In your Administration Control Panel, click \u0026ldquo;Manage Permissions\u0026rdquo;, and scroll all the way down to the \u0026ldquo;Checklist\/Dataset Management Sponsorship\u0026rdquo; box. From here, you can grant checklist\/dataset creation privileges to any user. If you are a superadministrator, go to the \u0026ldquo;User Permissions\u0026rdquo; page in your \u0026ldquo;Sitemap\u0026rdquo;, and click the user that needs permissions.\nIn the user profile page, find the \u0026ldquo;Assign New Permissions\u0026rdquo; options, and mark the checkbox that says \u0026ldquo;Create a Checklist\u0026rdquo;, and then click the \u0026ldquo;Add Permission\u0026rdquo; button to complete the process.\n</p><p>checklist</p><p>inventory</p><p>permission</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/current_notes\/checklists\/"},{value:"[portal_manager] Current notes for Portal Managers",label:"<p>This section should be used by portal managers or admins interested in learning more about new Symbiota developments, how to implement new released features and code-related advisory.\n</p><p>managers</p><p>admins</p><p>release notes</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/current_notes\/"},{value:"[user] Editing Datasets",label:"<p>To edit an existing dataset  Click My Profile. Click Occurrence Management. In the \u0026ldquo;Miscellaneous Tools\u0026rdquo; box, click \u0026ldquo;Dataset Management\u0026rdquo;. Click the name of the dataset that you wish to edit.\nIn the General Management tab, you can:  edit the name, notes, and description of the dataset make the dataset private or publicly available delete the dataset (bottom of the page)\nIn the User Access tab, you can: provide other users access to view (read), add\/delete occurrences (read\/write), or fully manage (full access) the dataset\nIn the Occurrence List tab, you can: add occurrences to a dataset. Click the name of the dataset you created, navigate to the Occurrences tab and click the \u0026ldquo;occurrence search page\u0026rdquo; link. Or you can start from the general search page for the Symbiota portal.    To add occurrences to an existing dataset  Conduct a search from the main search page of the Symbiota portal. This is usually accessed by clicking \u0026ldquo;Search\u0026rdquo; or \u0026ldquo;Search Collections\u0026rdquo; on the home page of the portal. For more information about searching in a Symbiota portal, visit this page. When you conduct the search, click the \u0026ldquo;List Display\u0026rdquo; button\u0026quot;. From the List Display, click the button with the dataset icon.  This will open the Dataset Management box. Select the dataset to which you would like to add the specimens from the \u0026ldquo;Dataset target\u0026rdquo; dropdown menu. From here, you can select the specimens you would like to include in your dataset from your query by checking the boxes outlined by dotted lines for each specimen.  To only add a few specimens from the page, check the boxes next to the desired specimens, then click the \u0026ldquo;Add Selected Records to Dataset\u0026rdquo; button. To add all the specimens on the page, check the box next to \u0026ldquo;Select all records on page\u0026rdquo;, then click the \u0026ldquo;Add Selected Records to Dataset\u0026rdquo; button. To add all specimens resulted from your query, click the \u0026ldquo;Add Complete Query to Dataset\u0026rdquo; button.     </p><p>dataset</p><p>inventory</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/dataset\/edit\/"},{value:"[editor] Trait Scoring from Text",label:"<p> This page describes how to batch score traits for occurrences in your collection from text (e.g., description, notes) in the occurrence record. The example provided displays how to score phenological traits for vascular plant specimens, though other traits may be score-able depending on your Symbiota portal.\n Trait scoring tools are not activated in all portals. Contact your portal manager for more information.\n Navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection), click Occurrence Trait Coding Tools, and select “Trait Mining from Verbatim Text”. In the Harvesting Filter box, select the Occurrence trait that you would like to score.\nFor the “Verbatim text source” field, select the database field that you want the tool to display. The options include: Habitat, Substrate, Occurrence Remarks (Notes), Dynamic Properties, Verbatim Attributes (Description), Behavior, Reproductive Condition (Phenology), Life Stage, and Sex. In the “Filter by text (optional)” field, you can enter any text string by which you wish to filter the results. For example, if you enter “fw”, the tool will provide all values for that field that contain “fw”. This field is optional, but including a search string will significantly narrow your search results. When choosing strings to enter in this field, consider searching for misspelled and abbreviated versions of the values you expect. For example, if you are searching for specimens that you can score as flowering, you could search for “fowering” or “fwrr” as well as “flowering” and “fwr”. In the “Filter by taxon (optional)” field, you can enter the name of a taxon by which you wish to filter the results. This field is also optional.\nOnce you have filled out the desired fields, click the Get Field Values button. The result will look similar to the screenshot on the next page. The Harvesting Filter box is still visible, and the box below will list all the unique strings (values) found in the specified database field that align with your search criteria. The numbers in brackets to the right of each string value indicates how many specimen records have that exact value in the specified database field.\nSelect the value or values from the list that you wish to score. To select several values from the list that you wish to give the same phenological score, hold Ctrl and click on each value in the list. Select the appropriate score from the trait scoring schema at the bottom of the lower box. Depending on the scoring schema available for the trait you are scoring, you may be able to view more scoring options by clicking the black triangle next to the schema. From this list, select the traits that you wish to apply to this specimen. Traits that you do not wish to score can be left blank. When you are satisfied with your scoring(s), click the Batch Assign State(s) button. If you make a mistake, click Reset Form (Caution: this will un-select all of your selected text fields as well, so be careful when selecting traits!).\n</p><p>trait scoring</p><p>attribute scoring</p><p>phenology</p><p>traits</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/trait\/text\/"},{value:"[editor] Annotations \/ Determinations",label:"<p> This page describes how to add annotations individually or in batch.\n Annotations (also called Determinations) in Symbiota portals are taxonomic identifications provided for an occurrence.\nAdding Individual Determinations\/Annotations To annotate an individual record or add previous determinations, navigate to the specimen in the Occurrence Editor (My Profile \u0026gt; Occurrence Records \u0026gt; name of collection \u0026gt; Edit Existing Records) and click the Determination History tab. If no previous annotations exist, you will see a message that says “There are no historic annotations for this specimen” and a box labeled Add a New Determination. If an annotation already exists on the specimen, you will see that annotation. In the latter situation, click the green plus sign to add a new annotation.\nIn the Add a New Determination Box, enter, at minimum, the scientific name, determiner (person who identified the specimen, if known. Enter “unknown” if not known), and the date (or “n.d.” if not known), at minimum. You can also add an identification qualifier (e.g., “cf.” or “aff.”), a reference, and notes, and you can select a confidence of determination, if desired. By default, if the date of determination is newer than that previous date of determination, the Make this the current determination box will be checked once you move your cursor out of the Date box. If you want this determination to be the most up-to-date identification for the specimen, keep this box checked (or check it, if it was not auto-checked). If you wish to print this annotation label now or in the future, also check the Add to Annotation Print Cue box. Click Submit Determination.\nIf the specimen previously had an identification and you have checked the box next to “Make this the current determination,” a historic annotation will be added to the record automatically as shown in the next screenshot. For this reason, you do not need to manually add a historic determination before you add a new determination.\n Batch Adding Annotations\/Determinations You can add many annotations at once using the Add Batch Determinations\/Nomenclatural Adjustments tool. Navigate to this tool by accessing the Data Editor Control Panel (My Profile \u0026gt; Occurrence Records \u0026gt; name of collection) and clicking “Add Batch Determinations\/Nomenclatural Adjustments.” To select the specimens to which determination data will be added (“Define Specimen Recordset”), either enter a list of catalog numbers (separated by commas) in the “Catalog Number:” field or select a taxon to evaluate by entering its name in the “Taxon:” field. Click the Add Record(s) to List button.\nA table of specimens of the indicated taxon will appear below the “Define Specimen Recordset” box (see screenshot below). You can then select all the specimens that you wish to annotate by checking or unchecking the boxes in the leftmost column of the table. Use the “Select\/Deselect all Specimens” box to uncheck or check all the boxes if you would prefer to select specimens one by one.\nAdd information about the name change in the “New Determination Details” box. Here you can indicate whether the change is due to a new identification (“Identification Adjustment\/Verification”) or a nomenclatural change (“Nomenclatural Adjustment”). An identification qualifier (e.g., “aff.” or “cf.”) can be added in the “Identification Qualifier:” field. Enter the new scientific name in the “Scientific Name” field, and the “Author:” field will automatically populate if the taxon is already in the taxon table. If the taxon is not in the taxon table, you will have to manually enter the author of the scientific name. You should also enter the name of the determiner in the “Determiner:” field, as well as the full date of determination in the “Date:” field. You can indicate the confidence in the determination (Low, Medium, or High) in the “Confidence of Determination:” field, list your determination references in the “Reference:” field, or include any other notes in the “Notes:” field. Finally, if you check the “Make this the current determination” box, the scientific name will be updated for the selected records. Otherwise, the determination will be added to the specimens’ records, but the most current ID name will not be changed.\nIf you wish to print the annotation labels in the future, check the “Add to Annotation Print Queue” box.\n</p><p>edit</p><p>annotation</p><p>identification</p><p>determination</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/annotations\/"},{value:"[coll_manager] Data Import Fields",label:"<p> This page lists the possible specimen fields that can be imported into a Symbiota data portal. The type of field (text, number, etc.) is listed in the Type field. The number of characters allowed in that field is displayed in parentheses, when applicable.\n You can select any number of fields in the table below to include in your file upload, but some fields are more commonly used than others. At the button below, you will find a template based on plant collections in Excel format. The second row provides field explanataions and the third row provides an example. Delete these rows and save the file as a CSV (UTF-8) before attempting to upload.\nDownload Example Template If the data portal is being used to displaying a “snapshot” of your data stored in your local central database, the upload file MUST have a field that serves as the unique identifier for each incoming specimen record (dbpk). This field serves a link between the source record and the snapshot record within the portal. If the import is a CSV file, the first row must contain field names. Note that field names do not have to match the name used below, but there cannot be any characters ($#@\u0026amp;%) in the column names. If you have issues saving your import profile, your field names may be too long! Try shortening the field names in the CSV file.\nFor more information about data fields, also see our Symbiota Data Fields page.\nItalic = Darwin Core fields Bold = strongly encouraged fields, though none of the fields are technically required * = for import only, merged into another field after import\nFor Darwin Core fields, click on the link in the \u0026ldquo;Name\u0026rdquo; column to see the DwC field definition.\n    Name Type Notes     associatedCollectors Text (255) All collectors except the primary collector, separated by commas or semicolons.   associatedMedia Text (255) URL to jpg images, delimited by commas or semicolons. More information about uploading images can be found on this page.   associatedSequences Text (65,535) Note: This field is not yet accessible in the occurrence editor.   associatedTaxa Text (65,535) Associated taxa delimited by commas or semicolons   authorInfraspecific Text The authorship of the infraspecific epithet, if different than the authorship of the specific epithet. If you do not provide authorships, authorships will be automatically assigned according to the taxonomic thesaurus (if the name is found in the thesaurus). See also specify:subspecies_author, specify:variety_author, and specify:forma_author.   authorSpecies Text The authorship of the specific epithet of the species. If you do not provide authorships, authorships will be automatically assigned according to the taxonomic thesaurus (if the name is found in the thesaurus).   basisOfRecord Text (32)    behavior Text (500)    catalogNumber Text (32) Barcode or Accession number   collectionCode Text (32) Populate only if different than source collection (i.e., the manager\/metadata\/) associated with your collection)   collectorFamilyName Text (255) The family name or last name of the collector (if in its own field). If your collector names are not parsed into initials and family name, use the recordedBy field. See the specify:collector fields if your collector\u0026rsquo;s name is split between first, middle, and last names.   collectorInitials Text (255) The initials (excluding family name) of the collector. If your collector names are not parsed into family name and initials, use the recordedBy field. See the specify:collector fields if your collector\u0026rsquo;s name is split between first, middle, and last names.   continent Text (45) Note: This field is not yet accessible in the occurrence editor.   coordinateUncertaintyInMeters Integer    coordinateUncertaintyRadius Integer The value of the uncertainty radius of the georeference (used in concert with coordinateUncertaintyUnits), which is converted into coordinateUncertaintyInMeters upon upload. If all your coordinate uncertainty values are in meters, use the coordinateUncertaintyInMeters field instead.   coordinateUncertaintyUnits Text The units of the uncertainty radius of the georeference (used in concert with coordinateUncertaintyRadius).   country Text (64)    countryCode Text (5)    county Text (255)    cultivationStatus Integer 0 = wild, 1 = cultivated   dataGeneralizations Text (250) Notes or remarks about the record. This field is not publicly accessible, so it is a good place to put curatorial notes or temporary comments.   dateEntered Date\/Time The date when the record was first added to the source database.   dateIdentified Text (45)    day Integer (4) If eventDate is null, year-month-day will be used to build the eventDate.   dbpk Text (45) Specimen record unique identifier (primary key) of source database (record id). Barcode, occurrenceID (GUID), or a database Primary Key is ideal. Can also be catalogNumber, given that it is populated for each record and enforced as unique. Required if collection is “snapshot” of a central database. Not needed if collection is managed directly within portal.   decimalLatitude Decimal number (8)    decimalLongitude Decimal number (8) For USA records, value is negative   disposition Text (32) Can be used to put storage location information, if needed.   duplicateQuantity   Used for printing labels. This field is not publicly accessible.   dynamicProperties Text (65,535) This field should ideally be formatted in JSON. For non-formatted descriptions, use the verbatimAttributes field.   *elevationNumber Integer Use this field for the elevation values (i.e., numbers) when your elevation values and units fields are separated. These will be concatenated into verbatimElevation.   *elevationUnits Text (45) Use this field for the elevation units when your elevation values and units fields are separated. These will be concatenated into verbatimElevation.   endDayOfYear Integer The numeric value (1-365) of the ending date of an eventDate range. Used in concert with startDayOfYear.   establishmentMeans Text (45) Use of a controlled vocabulary is preferred (see Darwin Core link).   eventDate Date\/Time Date collected, or earliest date of collection, if a range is provided, formatted as YYYY-MM-DD. If other formatting is used, or if date ranges are included, map to the \u0026ldquo;verbatimEventDate\u0026rdquo; field instead.   eventDate2 Date\/Time Latest date collected, when a range is provided. Note: This field is not yet accessible in the occurrence editor.   eventID Text (45) The unique identifier for a collection event, if provided by the source database. Note: This field is not yet accessible in the occurrence editor.   eventTime Text (45) Note: This field is not yet accessible in the occurrence editor.   exsiccatiIdentifier Integer (11) Identifier from exsiccati indexing table   exsiccatiNotes Text (250) (Documentation coming soon!)   exsiccatiNumber Text (45) (Documentation coming soon!)   family Text (255) This need only be provided when the scientific name is not in the taxonomic thesaurus, or you would like to override the automatically-assigned family that would come from the taxonomic thesaurus.   fieldNumber Text (45)    footprintWKT Text (65,535)    genus Text (255)     geodeticDatum Text (255) WGS84, NAD83, NAD27, etc   georeferencedBy Text (255)    georeferencedDate Date\/Time Note: This field is not yet accessible in the occurrence editor.   georeferenceProtocol Text (255)    georeferenceRemarks Text (255)    georeferenceSources Text (255)    georeferenceVerificationStatus Text (32)    habitat Text (65,535) Information about the environmental conditions in which the specimen was collected.   host  Currently concatenated into associatedTaxa   identificationQualifier Text (255) cf, aff. etc   identificationReferences Text (2000)    identificationRemarks Text (2000)    identifiedBy Text (255)    individualCount Text (45)    informationWithheld Text (250) E.g., \u0026ldquo;coordinates not provided due to rare species\u0026rdquo;. If a record\u0026rsquo;s locality data are redacted using localitySecurity, the public version of this field will autopopulate a list field names that contain redacted data. More information here.   infraspecificEpithet Text (255)    institutionCode Text (32) Populate only if different than source collection (i.e., the manager\/metadata\/) associated with your collection)   island Text (75) Note: This field is not yet accessible in the occurrence editor.   islandGroup Text (75) Note: This field is not yet accessible in the occurrence editor.   labelProject Text (45) Used for printing labels. This field is not publicly accessible.   language Text (20) The language of the original record.   *latDeg Integer (11) Latitude degrees   *latMin Decimal number Latitude minutes   *latNS Text (3) North\/south hemisphere. Should be either N or S.   *latSec Decimal number Latitude seconds   lifeStage Text (45)    *lngDeg Integer (11) Longitude degrees   *lngEW Text (3) East\/west hemisphere. Should be either E or W.   *lngMin Decimal number  Longitude minutes   *lngSec Decimal number Longitude seconds   locality Text (65,535)    localitySecurity Integer 0=don’t hide locality details from general public, 1=hide locality, coordinates, and images . For more information about redacting locality information, see this page.   localitySecurityReason Text (100) Description of why a locality is hidden from public view.   locationID Text (150)    locationRemarks Text (65,535)    materialSampleJSON Text (65,535) Documentation coming soon!   maximumDepthInMeters Integer (11)    maximumElevationInMeters Integer (6)    minimumDepthInMeters Integer (11)    minimumElevationInMeters Integer (6) If the elevation is a single value in meters, only use this field   modified Date\/Time Date last modified in the source database. Further edits within the Symbiota portal will be stored in a separate table.   month Integer (4) If eventDate is null, year-month-day will be used to build the eventDate.   municipality Text (255)    observerUID  A unique identifier applied to the person referred to in \u0026ldquo;recordedBy\u0026rdquo;.   occurrenceId Text (255) Occurrence Global Unique Identifier (GUID)   occurrenceRemarks Text (65,535) General notes or remarks about the occurrence\/specimen.   organismID     otherCatalogNumbers Text (255) To take advantage of the Tag Name \u002b Identifier system (in which you can tag an identifier\/other catalog number with a specific title), enter the tag name followed by a colon and then the identifier value, e.g., \u0026ldquo;Old Accession Number: 12345\u0026rdquo;. For multiple identifiers, separate the tag name \u002b identifiers by semicolons, e.g., \u0026ldquo;NP #: 4321; Accession #: 9876\u0026rdquo;   ownerInstitutionCode Text (32)    paleoJSON Text (65,535) A JSON-formatted field containing the data to be included in the Symbiota paleo module. Documentation coming soon!   parentLocationID  Note: This field is not yet accessible in the occurrence editor.   preparations Text (100)    processingStatus  Processing status for digitization tasks. This field is not publicly accessible.   recordedBy Text (255) Primary collector\/observer name. All other collectors should be placed in the \u0026ldquo;associatedCollectors\u0026rdquo; field. If the primary collector\/observer name is parsed into multiple fields, see the collectorFamilyName, collectorInitials, and specify:collector fields.   recordEnteredBy Text (250) Data entry personnel   recordNumber Text (45) Collector number   *recordNumberPrefix Text (45)  Merged into recordNumber   *recordNumberSuffix Text (45)  Merged into recordNumber   reproductiveCondition Text (255) e.g. sterile, flw, frt, asci, etc. Use of controlled vocabulary is preferred.   samplingEffort Text (200)    samplingProtocol Text (100)    scientificname Text (255) Scientific name w\/ authorship. The authorship will be parsed from the name.   scientificNameAuthorship Text (255) Author of scientific name   sciname Text (255) Scientific name without author   sex Text (45)    specificEpithet Text (255)    startDayOfYear Integer The numeric value (1-365) of the starting date of an eventDate range. Used in concert with endDayOfYear.   stateProvince Text (255)    substrate Text (500) The soil or other substrate (e.g., bark, rock) on which a sessile organism was found. In Darwin Core Archive exports, this field is concatenated into \u0026ldquo;habitat\u0026rdquo;.   taxonRank Text (32) Rank name of infraspecific abbreviation (e.g., var., subsp.) allowed   taxonRemarks Text (2000)    tempfield  The tempfields are provided as temporary holding locations for data that needs to be concatenated or otherwise transformed by a Stored Procedure before it can be moved into its final database field.   *trsRange Text (45) The range value (with direction) for U.S. township-range-section (TRS, or public land survey system) coordinates. (e.g., \u0026ldquo;23E\u0026rdquo; for the TRS coordinates T6S R23E section 34)   *trsSection Text (45) The section for U.S. township-range-section (TRS, or public land survey system) coordinates. (e.g., \u0026ldquo;34\u0026rdquo; for the TRS coordinates T6S R23E section 34)   *trsSectionDetails Text (45) Any additional details, such as quarter or sixteenth sections, for U.S. township-range-section (TRS, or public land survey system) coordinates.   *trsTownship Text (45) The township value (with direction) for U.S. township-range-section (TRS, or public land survey system) coordinates. (e.g., \u0026ldquo;6S\u0026rdquo; for the TRS coordinates T6S R23E section 34)   typeStatus Text (255)    *UtmEasting Text (45) The easting value for UTM coordinates (e.g., \u0026ldquo;334543\u0026rdquo; for the UTM coordinates 12N 334543 5463754)   *UtmNorthing Text (45) The northing value for UTM coordinates (e.g., \u0026ldquo;5463754\u0026rdquo; for the UTM coordinates 12N 334543 5463754)   *UtmZoning Text (45) The zone value for UTM coordinates (e.g., \u0026ldquo;12N\u0026rdquo; for the UTM coordinates 12N 334543 5463754)   verbatimAttributes Text (65,535) Verbatim description of organism (e.g. 1.5 m tall, flowers white with purple tips, etc). In the occurrence editor, this field is usually labeled \u0026ldquo;Description\u0026rdquo;.   *verbatimLatitude Text (255) Used to generate decimal latitude. This field will be merged into verbatimCoordinates.   verbatimCoordinates Text (255) e.g. UTM: 12N 334543 5463754; 34° 25’N 113° 43’W   verbatimDepth Text (50)    verbatimElevation Text (255) Use this field when your elevation values and units are in the same field (e.g., \u0026ldquo;1200 ft\u0026rdquo;) and when your elevations are not consistently in meters.   verbatimEventDate Text (255) Map collection\/observation date to this field when it is not YYYY-MM-DD standardized, when dates are incomplete, or when date ranges are present.   *verbatimLongitude Text (255) Used to generate decimal longitude. This field will be merged into verbatimCoordinates.   waterBody Text (150) Note: This field is not yet accessible in the occurrence editor.   year Integer (4) If eventDate is null, year-month-day will be used to build the eventDate.   specify:subspecies Text The infraspecific epithet of a subspecies only (without subsp. prefix).   specify:subspecies_author Text The authorship of the subspecific epithet.   specify:variety Text The infraspecific epithet of a variety only (without var. prefix).   specify:variety_author Text The authorship of the varietal epithet.   specify:forma Text The infraspecific epithet of a forma only (without f. prefix).   specify:forma_author Text The authorship of the forma.   specify:collector_first_name Text The first (given) name of the primary collector. Will be concatenated into recordedBy along with middle_initial and last_name fields.   specify:collector_middle_initial Text The middle initial name of the primary collector. Will be concatenated into recordedBy along with first_name and last_name fields.   specify:collector_last_name Text The last (family) name of the primary collector. Will be concatenated into recordedBy along with first_name and middle_initial fields.   specify:determiner_first_name Text The first (given) name of the primary person who applied the taxonomic identification to the record. Will be concatenated into identifiedBy along with middle_initial and last_name fields.   specify:determiner_middle_initial Text The middle initial of the primary person who applied the taxonomic identification to the record. Will be concatenated into identifiedBy along with first_name and last_name fields.   specify:determiner_last_name Text The last (family) name of the primary person who applied the taxonomic identification to the record. Will be concatenated into identifiedBy along with first_name and middle_initial fields.   specify:qualifier_position Text When an identification qualifier is provided, this field will be concatenated along with the qualifier to indicate the part of the scientific name that the qualifier applies to. For example, if the record you are adding reads \u0026ldquo;Amaranthus cf. albus\u0026rdquo;, enter \u0026ldquo;Amaranthus albus\u0026rdquo; in the sciname field, \u0026ldquo;cf.\u0026rdquo; in the identificationQualifier field, and \u0026ldquo;albus\u0026rdquo; in the qualifier_position field.   specify:latitude1 Decimal number The westernmost latitude in a range of latitudes provided as a georeference for the record. Will be concatenated along with latitude2 into verbatimCoordinates.   specify:latitude2 Decimal number The easternmost latitude in a range of latitudes provided as a georeference for the record. Will be concatenated along with latitude1 into verbatimCoordinates.   specify:longitude1 Decimal number The northernmost longitude in a range of longitudes provided as a georeference for the record. Will be concatenated along with longitude2 into verbatimCoordinates.   specify:longitude2 Decimal number The southernmost longitude in a range of longitudes provided as a georeference for the record. Will be concatenated along with longitude1 into verbatimCoordinates.   specify:land_ownership Text Will be concatenated into the \u0026ldquo;locality\u0026rdquo; field separated by a semicolon.   specify:topo_quad Text Will be concatenated into the \u0026ldquo;locality\u0026rdquo; field separated by a semicolon.   specify:georeferenced_by_first_name Text The first (given) name of the primary person who applied the georeference to the record. Will be concatenated into georeferencedBy along with middle_initial and last_name fields.   specify:georeferenced_by_middle initial Text The middle initial of the primary person who applied the georeference to the record. Will be concatenated into georeferencedBy along with first_name and last_name fields.   specify:georeferenced_by_last_name Text The last (family) name of the primary person who applied the georeference to the record. Will be concatenated into georeferencedBy along with first_name and middle_initial fields.   specify:locality_continued Text Will be concatenated into the \u0026ldquo;locality\u0026rdquo; field with no delimiter.   specify:georeferenced_date Date\/Time Will be concatenated into the \u0026ldquo;georeferencedBy\u0026rdquo; field with the following formatting: \u0026ldquo;georef date: georeferenced_date\u0026rdquo;   specify:elevation_(ft.) Integer The value of elevation in units of feet.   specify:preparer_first_name Text The first (given) name of the person who prepared the specimen. Will be concatenated into \u0026ldquo;preparations\u0026rdquo; along with middle_initial and last_name fields as follows: \u0026ldquo;preparer: first_name middle_initial last_name\u0026rdquo;.   specify:preparer_middle_initial Text The middle initial of the person who prepared the specimen. Will be concatenated into \u0026ldquo;preparations\u0026rdquo; along with first_name and last_name fields as follows: \u0026ldquo;preparer: first_name middle_initial last_name\u0026rdquo;.   specify:preparer_last_name Text The last (family) name of the person who prepared the specimen. Will be concatenated into \u0026ldquo;preparations\u0026rdquo; along with first_name and middle_initial fields as follows: \u0026ldquo;preparer: first_name middle_initial last_name\u0026rdquo;.   specify:prepared_by_date Date\/Time The date the specimen was prepared. Will be concatenated into \u0026ldquo;preparations\u0026rdquo;.   specify:cataloger_first_name Text The first (given) name of the person who first cataloged the record. Will be concatenated into \u0026ldquo;recordEnteredBy\u0026rdquo; along with middle_initial and last_name fields.   specify:cataloger_middle_initial Text The middle initial of the person who first cataloged the record. Will be concatenated into \u0026ldquo;recordEnteredBy\u0026rdquo; along with first_name and last_name fields.   specify:cataloger_last_name Text The last name of the person who first cataloged the record. Will be concatenated into \u0026ldquo;recordEnteredBy\u0026rdquo; along with first_name and middle_initial fields.   specify:cataloged_date Text Will be concatenated into the \u0026ldquo;recordEnteredBy\u0026rdquo; field with the following formatting: \u0026ldquo;cataloged date: cataloged_date\u0026rdquo;    </p><p>data upload</p><p>data import</p><p>file upload</p><p>IPT</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/upload\/fields\/"},{value:"[user] Adding Taxa to Checklist",label:"<p> This page describes how to add taxa (species) to a checklist in a Symbiota portal.\n There are three ways to add taxa to a checklist, by querying the portal for taxa that meet your search criteria, by adding taxa individually, or by batch uploading a list of taxa and notes.\nBatch Adding Taxa by Searching the Portal This method should be used if you do not already have a list of taxa prepared, and you want to build a checklist based on occurrences in the portal. You can define certain search criteria (e.g., a location or polygon in the map interface), view the results of this search, and add the taxa (and\/or vouchers) that meet your search criteria.\n On the checklist page, click the center pencil icon with a v (\u0026ldquo;Manage Linked Vouchers\u0026rdquo;) at the top right of the page. This will take you to the voucher administration tools. If this is your first time defining search criteria, you will be prompted with a \u0026ldquo;Edit Search Statement\u0026rdquo; box. Use the fields available here to define the search criteria for the occurrences you would like to see. For example, you can list a state or county, taxon, a bounding box, or any combination of these (or additional) fields.  If you have previously created a search statement, you can edit your search statement by clicking the magnifying glass icon at the top of the page.    Take care to make your search statement general enough that you will get the maximum number of specimens; fewer criteria is better. For example, rather than including both \u0026ldquo;United States\u0026rdquo; in country and \u0026ldquo;Arizona\u0026rdquo; in state, try just using \u0026ldquo;Arizona\u0026rdquo; in state so that even records with \u0026ldquo;USA\u0026rdquo; or \u0026ldquo;United States of America\u0026rdquo; are included in your search results. You will likely need to conduct several different searches to ensure a complete result (e.g., do a search for \u0026ldquo;AZ\u0026rdquo; as well as one for \u0026ldquo;Arizona\u0026rdquo;).\n If you restrict your search based on coordinates, your results will be limited to specimens that have coordinate data. Depending on the portal, this may represent only 30-50% of available records.\n After you have defined your search criteria, click the Missing Taxa tab. In the Display Mode dropdown list, select Batch Linking.  From here you will see a table consiting of the first 1000 records that match your search criteria and that represent taxa that are not already in your checklist. To add taxa to the checklist from this page, check the boxes next to the vouchers with taxonomic names that you wish to add to your checklist (check the box in the top left corner of the table to check all the records in the table).\nTo add only the taxonomic names represented by these specimens to your checklist, check the \u0026ldquo;Add names without linking vouchers\u0026rdquo; box, then click the Submit Vouchers button. To add the taxonomic names and link vouchers to your checklist, leave this box unchecked and click the Submit Vouchers button.  After linking the desired taxa, refresh the list and link the next batch as necessary until all desired taxonomic names have been added.  Adding Individual Taxa    On the checklist page, click the rightmost pencil icon (labeled spp.) at the top right of the page. This will bring up a panel on the lower right side of the screen. Add the name of a taxon you wish to add. Do NOT include the authorship of the taxon. You can start typing, and a dropdown list will allow you to select the taxon from available options.  You cannot add a name that is not present in the taxonomic thesaurus (it will tell you if this is the case). If you find that the taxon you wish to add is not present:  Check that your spelling is correct. Use a vetted taxonomic resources, such as Tropicos or IPNI to check. If your spelling is correct, contact your portal manager with the name, authorship, and a reference or link to that name in another taxonomic database, and it will be added for you.     Fill in the other fields if desired. All of these fields can be left blank:  Family Override: If you enter the family name here, you will override the family provided by the taxonomic thesaurus. It is advised not to enter anything in this field. Habitat: If the taxon is usually found in a particular habitat in the area circumscribed by the checklist, you can include this information here. Abundance: Here you can note how frequently\/abundantly this taxon is found in the area circumscribed by the checklist. For example, you can use terms like “common,” “frequent,” or “rare.” Notes: Any other comments about this taxon pertaining to your checklist area can be put here, for example: “Cultivated ornamental” or “favored by honeybees” Internal notes: This field, unlike the others, will be visible only to people authorized to edit the checklist. You might make a note “need to check the species” or “no picture available” for example. Source: This field is more likely to be used for research and teaching checklists. For such checklists, it is useful to know whether the information came from a flora, some other publication, or a report by some   Click the “Add species to List” button.  Batch Uploading Taxa from a File It is recommended to add a few taxa individually before attempting a batch upload so that you can get familiar with the data types and formats.\n   If you have several taxa to add at one time, you can upload them in a CSV (comma separated value) file. The easiest way to create the desired listing is to use Microsoft Excel or OpenOffice Calc and then save the file as a CSV file. In your CSV file, one column in the file should be labeled sciname. This should contain the scientific name of the taxon WITHOUT the authorship included. In this CSV file, you must be very careful to spell the scientific names correctly. If you have an error in your spelling, you will not be able to load that name into your checklist. You can also include “family”, “habitat”, “abundance”, and “notes” columns (see above for descriptions of these fields) in this file. An example is shown below.\n   Sciname Habitat Abundance Notes     Asystasia gangetica Middle of the field Rare Demonstration; correct spelling   Asystasia gangetca field unknown Demonstration; wrong spelling    Below is a screenshot of uploading the previous example “file”. Note that only one name, the one that was correctly spelled, was uploaded.\nTo return to the Checklist, click “Return to Checklist” at the top left of the page (underlined below).\nYou can then view the taxa that were added to the checklist.\n</p><p>checklist</p><p>inventory</p><p>flora</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/add\/"},{value:"[coll_manager] Creating a Darwin Core Archive",label:"<p> This page describes how you can package your data as a Darwin Core Archive (standard format for data publishing and sharing) in a Symbiota portal.\n A Darwin Core Archive (DwC-A) is a data standard that is commonly used to package species occurrence data into a single, self-contained dataset. A DwC-A includes metadata, a file of occurrence data, and, often, files for determinations (identifications), images, and any other data extensions.\nYou can only publish a Darwin Core Archive of occurrences that have unique occurrence IDs and GUIDs. In your collection metadata page, make sure that you have selected a source for your GUID and that this source contains unique values for each occurrence.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Darwin Core Archive Publishing. Click the Create\/Refresh Darwin Core Archive button.  </p><p>data publishing</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/dwc\/"},{value:"[editor] Uploading Images",label:"<p> This page describes how to upload individual field and specimen images directly into a Symbiota portal. It is also possible to link images that are stored in external servers. For information about the latter option, visit the Batch Adding\/Linking Images page.\n There are three catefories of images that can be linked to a Symbiota portal. Instructions for uploading each of these types are provided below.\nIndividual specimen images  Log in to your account in the portal. Navigate to the Occurrence Editor page of the specimen to which you would like to add an image (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection \u0026gt; Edit Existing Occurrence Records \u0026gt; conduct search for specimen) Click the Images tab.  If an image has already been added to the specimen, and you wish to add another click the green plus sign at the top right of the window. Otherwise, the image uploading interface will be highlighted in yellow on this page. Click the Choose File button and navigate to the image file you wish to upload. If you instead have a URL to the image, you can click Enter URL and paste the URL into the resulting field instead. Enter any additional information, if desired, in the other image uploading fields.  You can enter a number into the Sort field if you wish to define the order in which mutliple images will appear on the occurrence page. If you enter a number greater than 500 in the Sort field, that image will be displayed on the Occurrence Details page but on the Taxon Profile page for that taxon. This is ideal for poor-quality or sensitive images (e.g. road kills).\n Click the Submit New Image button.  Field image without location information Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed on the Taxon Profile Page, which has general information such as descriptions, distribution maps, synonyms, and common names. To upload an image:\n Log in to your account in the portal. You must have Taxon Profile Editor permissions to do the following\n  Navigate to the page of the taxon you wish to edit. To do so, you may be able to:  Click Sitemap, then Taxonomic Tree Viewer or Taxonomy Explorer. Search for the taxon of interest and click its name. Perform a quick search on the home page for the taxon of interest.   Click the pencil icon at the top right corner of the taxon page. Click the Add Image tab. Select the image file you would like to upload from Choose File, then enter any additional information in the provided fields.  The Sort Sequence field allows you to determine the order of the images that will show up on the taxon profile. The higher the number, the further down the priority list the image will be.   Click the Upload Image button. Field images are uploaded and managed through the Taxon Profile Editing interface. Users with Taxon Profile editing permissions can submit an image by clicking on the editing symbol located in the upper right of any Taxon Profile page, or through the image submission links available on the sitemap page. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see below).  Image Vouchers (field images with location information) Field images with specific locality information can serve as vouchers for field observations of many birds, mammals, and easy-to-identify plants. There are, however, some taxonomic groups that typically require microscopic or chemical analysis for accurate identification (e.g. lichens, bryophytes, fungi). For such organisms, field images will have limited value. For this reason, some data portals might not allow the submission of observations without a physical specimen.\nIt is particularly important that images intended as vouchers for an observation be of high quality and clearly display diagnostic morphology of the organism. Data fields required for an image to qualify as a voucher include: observer name, observation date, country, state, locality description, latitude, longitude, and at least one image.\nAs is the case with images of physical specimen, Image Vouchers are displayed on both the Occurrence Details and the Taxon Profile pages. Image-supported observations can be uploaded in a General Research Observation profile. The following video describes how to gain access to and submit observations to such a collection:\n  </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/upload\/"},{value:"[editor] Specimen Packet Printing",label:"<p> This guide describes the use of the label printing tool in Symbiota to print lichen and bryophyte packets: full sheets of paper with a 3”x5” label printed on the bottom third of the sheet. For instructions on folding packets, see this guide.\n   Using the Default Packet-Printing Profile The default packet label looks like this:\n To print a packet, navigate to the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Print Specimen Labels from the Data Editor Control Panel. Enter search terms into the Define Specimen Recordset form to search for the specimens for which you would like to print packets. Then click the Filter Specimen Records button. Check the boxes next to the specimens for which you would like to print packets, or click the checkbox at the top of the checkboxes column to select all the records.  Scroll down to the Label Printing form at the bottom of the page. From the Label Profiles dropdown menu, select Generic Lichen Packet. From here, you can customize your packet in a limited number of ways: To change the heading prefix: edit the text in the Heading Prefix field To change the auto-generated middle section of the heading by selecting a button from the Heading Mid-Section options. To change the end of the heading: edit the text in the Heading Suffix field To change the name of your collection that will show up on top of your barcode: edit the text in the Label Footer field  NOTE: There is no actual “Footer” on lichen and bryophyte packets. Instead, the label footer is shown atop the printed barcode. Leave this blank if you would not like any text on top of your barcode OR if you are not including a barcode on your packet   To show the authorship of the species before the infraspecific epithet, if you are printing taxa with infraspecific epithets: check the box next to “Print species authors for infraspecific taxa” You do NOT need to check the box next to Print Catalog Numbers. The catalog number will be displayed underneath the barcode if “Include barcode of Catalog Number” is checked. Clicking “Print only Barcode” will print a barcode only with no packet or other data Note that Other Catalog Numbers are not listed on packets by default. To add this field, create a custom label packet profile as described in the next section and include Other Catalog Numbers in your list of printed fields. When printing your packet, make sure to print at 100% scale. Use Print Preview and adjust the margins as necessary so that the packet is centered and the correct size. If printing from Google Chrome, click the More Settings option and uncheck the Headers and footers box.  Customizing Your Own Packet-Printing Profile  Complete steps 1-7 as above. Click the pencil icon next to Label Profiles.  Find the profile labeled Generic Lichen Packet. To customize the packet-printing profile for your own personal use, click the Select Target option under Generic Lichen Packet and select “User Profile”. To customize the 5. packet-printing profile for your entire collection (i.e., other people will also be able to use this profile), select “Collection Profile” from the dropdown menu. Click the Clone Profile button under Generic Lichen Packet. When the page reloads, you will see a new profile under your collection profiles or user profiles (depending on what you selected in step 5 above). Click the pencil icon next to the title of the new, cloned label profile to edit the profile. Do NOT edit the Default CSS, Custom CSS, and Custom JS fields, but you can edit the title, prefix, etc. as above.  To remove the barcode from your packet, uncheck the box next to Display barcode. If you do this, make sure to delete “Custom Collection Name” from the Footer text field.   To edit which fields are listed on your packet and\/or the orientation of those fields, scroll down to the JSON field and click “visual interface”.   In the bottom center of the screen, you will see a preview of the label that you are customizing. The left-hand Fields Available column contains a bank of fields that you can draw from and include on your label The grey Label Content Area in the center is where you can place and move the fields that you have selected from the Fields Available column for your label The Field Options on the top right can be used to edit the prefix, suffix, font size, position, and font family of a field after selecting a field in the Label Content Area. The Line Options on the bottom right can be used to select a delimiter between fields on the same line and edit the text alignment and spacing after selecting a line in the Label Content Area. To add a new line, scroll to the bottom of the Label Content Area and click the ADD LINE button. Beware of adding additional lines to the Label Content Area! If you have too much content, it may not fit on the packet label.  Customize your label by adding fields (drag and drop fields from the Field Available column into the Label Content Area), removing fields (click the x icon next to the field in the Label Content Area), editing fields (using Field Options), and editing lines (using Line Options) as desired. Click the SAVE FORMAT \u0026amp; CLOSE button. Follow steps 1-7 of Using the Default Packet-Printing Profile. From the Label Profiles field, select your new label printing format from the dropdown list. Click the Print in Browser button. If your new label profile does not meet your expectations, return to the label profile editor (steps 7-9 above) and edit the profile as desired. For small edits that are specific to the specimen data you are working with (e.g., special character issues or spacing issues), you can also edit the label in the print preview. After clicking Print in Browser as above, click the Edit Labels Content button in the top left corner of your screen. From here, you can make small adjustments to the label(s) that you currently intend to print. Note that these changes do not change the label profile that you have created.  </p><p>labels</p><p>printing</p><p>packet</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/label\/packet\/"},{value:"[coll_manager] Checklists",label:"<p> The \u0026ldquo;Create a Checklist\u0026rdquo; permission has been granted for the majority of existing users (accounts created before June 2022). If a user does not have \u0026ldquo;Create a Checklist enabled\u0026rdquo;, and needs it, ask your portal admin (super admin) or the Support Hub to add these user permissions to their account. All new users will need to request permission to create checklists\/datasets moving forward.\n For more information on checklists, please visit the Checklist chapter in the User Guide\n </p><p>checklists</p><p>inventories</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/checklist\/"},{value:"[user] Leaving Comments",label:"<p>If you notice an issue or would like to ask a question about a particular occurrence in a Symbiota portal, you can leave a comment on that occurrence that will then be automatically emailed to the collection manager and, in some cases, the portal manager.\nFirst, make sure you are logged in to your account. Any user can create an account by clicking New Account in the site menu.\nNext, open the public display page of the record. To do so from the results of a search in \u0026ldquo;list display\u0026rdquo;, click Full Record Details below the occurrence. To do so from the results of a search in \u0026ldquo;table display\u0026rdquo;, click on the SymbiotaID number (far left column) of the occurrence. To do so from a map search, hover over the point, then click See Details.\nFrom the public display page of the record, click the Comments tab, enter your comment, and click Submit Comment.\n⚠️ Symbiota does not automatically associate your contact information with your comments on specimen records. If you would like to be contacted in response to your comment, please include your name and contact information to assist the collections administrator with reaching out.\n </p><p>comment</p><p>correction</p><p>contact</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/public_feedback\/comment\/"},{value:"[coll_manager] Comments \u0026 Feedback",label:"<p> Sourcing public feedback on your specimen occurrences is one way to improve the \u0026ldquo;cleanliness\u0026rdquo;, or quality, of your data. Symbiota offers several tools to help you gather this information from anyone with a portal user account.\n  Commenting Public Edits  Related Resources   </p><p>crowdsourcing</p><p>annotations</p><p>comments</p><p>public feedback</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/public_feedback\/"},{value:"[coll_manager] Public Edits",label:"<p> This page describes how any Symbiota user can suggest modifications to an occurrence record. This can be a useful feature for community members to remediate data quality issues.\n Enable Public Edits The option to enabled public edits is managed on a per-collection basis and is activated through the Collection Metadata Editor. Turn this on by navigating to the Administration Control Panel \u0026gt; Edit Metadata and check the box for Allow Public Edits.\nOnce activated, any portal user who is logged in can suggest changes to your occurrence records using a paired-down version of the Occurrence Editor form. Public users cannot suggest edits to Catalog Number or Scientific Name values.\n        Once Public Edits are enabled via the Collection Metadata Editor, any user who is logged into the portal can suggest changes to individual occurrence records.    Review Public Edits To review suggested changes made by public users:\n Navigate to the Administration Control Panel \u0026gt; Review\/Verify Occurrence Edits. In the Filter panel, set Applied Status = \u0026ldquo;Not Applied\u0026rdquo; and select Submit Filter to filter out pending public edits to your occurence records. Proceed to use the Specimen Edit Reviewer to accept and reject suggested changes to your records.  What do public portal users see? Refer to the User Guide for more information.\n</p><p>crowdsourcing</p><p>annotations</p><p>comments</p><p>public feedback</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/public_feedback\/public_edits\/"},{value:"[user] Suggesting Edits",label:"<p> This page describes how any logged-in Symbiota user can suggest modifications to an occurrence record. The editing tools can be used to remediate data quality issues or add missing information.\n Portal users can only leave suggested edits on occurrence records if (1) they are logged into the portal and (2) public editing is enabled for a given collection.\n To suggest changes to an occurrence record:\n  Open the occurrence record and select the link at the bottom of the record that reads, \u0026ldquo;Occurrence Editor\u0026rdquo;, as in the example below.   Open the record and make changes using the Occurrence Data form. Note that public users cannot make suggested changes to Catalog Number or Scientific Name values; use the commenting tools to leave remarks related to these fields.\n  More information about editing records can be found on the Editing Records page.\n Select the \u0026ldquo;Save Edits\u0026rdquo; button at the bottom of the page once your modifications are complete. Your suggested changes must be approved by an administrator of the collection before they will become public.   </p><p>crowdsourcing</p><p>annotations</p><p>comments</p><p>public feedback</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/public_feedback\/public_edits\/"},{value:"[portal_manager] Editing Taxonomic Names \/ Synonymy",label:"<p> A user must have Super Administrator or Taxonomic Editor permissions to add to or edit the thesaurus.\n The taxonomic thesaurus of a Symbiota portal requires frequent curation due to taxonomic changes. Editing taxonomic names and synonymy requires some care because of the many linkages between taxa and their children taxa and synonyms.\n Navigate to the taxonomic thesaurus (Sitemap \u0026gt; scroll down to Taxonomy \u0026gt; Taxonomic Tree Viewer), or just add \u0026ldquo;taxa\/taxonomy\/taxonomydisplay.php\u0026rdquo; to your base path. Search for the name of the taxon you would like to edit in the Taxon box. Click the pencil icon to the right of the taxon you would like to edit.  Editing basic information (name, authorship, source, locality security)  In the Editor tab, you can edit the scientific name, rank, notes, source, and whether the taxon\u0026rsquo;s locality information is globally redacted for all occurrences in the portal. Click the pencil icon in the top right corner of the page to edit these fields.  Changing the parent or accepted name of a taxon  In the Taxonomic Status tab, you can change the parent taxon name or switch the taxon from being accepted to not accepted (or vice versa). Click the pencil icon to the right of the name type that you would like to change. Whenever possible, provide a link or citation for the change in acceptance. For example, you might add a link to the a Plants of the World Online entry, a literature citation with DOI, etc.  If you are changing a taxon to Not Accepted, you must also resolve the accepted status of all children taxa of that taxon individually before the target taxon. The children taxa are listed in the Children Taxa tab and can be accessed and modified through links in that tab.\n Deleting a taxon  In the Delete tab, you can delete a taxon from the taxonomic thesaurus. Before doing so, you must resolve all linkages to that taxon as listed in this tab. To move all the linkages and resources from one taxon to another (e.g., in the case of duplicate taxonomic names entered in the thesaurus), enter the name of the taxon in the Remap Resources to Another Taxon box and click the Remap Taxon button.  </p><p>taxonomy</p><p>taxonomic thesaurus</p><p>synonymy</p><p>accepted names</p><p>locality security</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/taxonomy\/edit\/"},{value:"[coll_manager] GEOLocate CoGe",label:"<p>The GEOLocate Collaborative Georeference Tool (CoGe) can be activated and integrated in a Symbiota portal to engage multiple users in a central georeferencing platform outside of the portal. More information about this platform can be found here: GeoLocate CoGe page.\nA webinar introduction to CoGe can be found here.\nThe general steps for a CoGe workflow are as follows:\n Create a profile in the CoGe website Create a community on the CoGe website Add members to your community in CoGe Add data to the CoGe community from your Symbiota portal Have members georeference in the CoGe web client.  You can either have data pushed to CoGe directly from your Symbiota portal, or you can download records as a CSV file, upload them to CoGe, download them once they are done, and then re-upload them to Symbiota. However, if you push data to CoGe directly, all georeferences made in CoGe will be automatically applied to your records in the Symbiota portal. Updates will be \u0026ldquo;live\u0026rdquo; in the portal.\n A manual for collections managers\/administrators using the Collaborative Georeferencing (CoGe) tool can be found here: https:\/\/epicc.berkeley.edu\/wp-content\/uploads\/2015\/11\/UsingGeoLocateforCollaborativeGeoreferencing_2016.pdf. This manual covers steps 1-3 in the above workflow, but does not cover how to add occurrences to CoGe from a Symbiota portal. Video tutorials can be found here: https:\/\/www.geo-locate.org\/community\/default.html.\nInstructions for step 4 are provided below.\nExample instructions for step 5 can be found here (GEOLocate CoGe Training Course), and a written protocol can be found here (GEOLocate CoGe Protocol).\nPushing Specimens from a Symbiota Portal to GEOLocate CoGe (step 4 from above workflow)  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Editor \u0026gt; name of collection). Click Processing Toolbox. If CoGe has been enabled for your portal, a tab labeled GeoLocate CoGe will be visible. Click this tab. If CoGe is not enabled, contact your portal manager to request this feature.  Click Login to CoGe in the CoGe Status box on the resulting page and log in to CoGe using your CoGe user account information in the pop-up window\/tab. Return to your Symbiota portal page.  If you have previously logged in to CoGe, click the Check Status button instead of logging in. If \u0026ldquo;Connected\u0026rdquo; does not show up in green after clicking this button, try logging out and then logging back in to CoGe. If this still does not work, check with your portal manager to ensure that your portal is set up to interact with CoGe.\n The CoGe communities that you manage will now be listed in the Available Communities box. If no communities are visible, make sure that you have created a community on the CoGe website. At the top of the page, use the Processing Status and\/or additional filters to search for the records you would like to push to CoGe (i.e., that you would like to be georeferenced in the CoGe client). Once you have entered the search parameters\/filters, click out of this field, and you will see how many records were found in the CoGe Status box under Match Count. In the Available Communities box, click the radio button next to the CoGe community to which you would like to assign the records. Enter a brief name for the data set in the \u0026ldquo;Data source identifier (primary name):\u0026rdquo; box and a description, if desired. If you would like the georeferences from CoGe to be automatically applied to their records in the Symbiota portal, click the \u0026ldquo;Push Data to GeoLocate CoGe\u0026rdquo; button. If you plan to use the download (Symbiota -\u0026gt; CSV), upload (CSV -\u0026gt; CoGe), download (CoGe -\u0026gt; CSV), upload (CSV -\u0026gt; Symbiota) workflow, click the \u0026ldquo;Download Records Locally\u0026rdquo; button.  You can also use the \u0026ldquo;Download Records Locally\u0026rdquo; button to quickly check whether the specimens that you are pushing to CoGe are as expected.\n Return to the CoGe site, navigate to your Dashboard, click the name of the community to which you just pushed records, and click the GO TO COMMUNITY button. Click the name of the dataset that you just pushed to CoGe.  Click the Update Cache button. Once the cache has been fully updated, they will be available via the CoGe web client.  Reviewing Georeferences Created in GEOLocate CoGe  To view georeferences that have been created in CoGe, navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Editor \u0026gt; name of collection). Click Review\/Verify Occurrence Edits. In the Filter box at the top right, select External from the dropdown menu of the Editing Source field. Then click Submit Filter. The edits that were applied in CoGe will now be visible in the reviewer table.  </p><p>georeferencing</p><p>collaborative georeferencing</p><p>GEOLocate</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/georeference\/coge\/"},{value:"[user] Adding Vouchers to Checklist",label:"<p> This page describes how to add voucher specimens (physical evidence of the existence of a taxon in a checklist) to a checklist in a Symbiota portal.\n There are two main ways to add voucher specimens to a research checklist. You can add specific, individual specimens from your own collection to the checklist through the occurrence editor (the page of the specimen record) or add specimens from any collection through the Manage Linked Voucher tool on the checklist page.\nAdding Specimens from Any Collection (Manage Linked Voucher Tool)    Navigate to the homepage of your checklist. Click the pencil icon that has a V next to it (the center icon in the top right corner of the page, circled below).  If this is the first time you have used this tool, you will be asked to first define the search statement (i.e., search parameters) that will be used to find the specimens that might pertain to your checklist. You can use a variety of fields to define your search, as described below.  You will be able to change the search statement in the future, so you are not necessarily defining the bounds forever.\n  You can enter search terms for a number of fields on the left side of the form. You can use the polygon footprint that you defined when creating the checklist (see 29:1 Creating a checklist, step 3c) to search for specimen records by checking the “Search based on polygon defining checklist research boundaries” box. If you did not define a polygon previously, you can also do so from here by clicking the pencil icon to the right of this option and following the steps outlined in 29:1 Creating a checklist, step 3c. You can define a specific bounding box by entering the latitude and longitude values that delimit the box, or by defining the lat\/long values on a map. To do the latter:  Click the globe icon to the right of the Lat North field In the pop-up window, pan and zoom to the location where you want to place the bounding box. Click on the approximate center where you want the bounding box to be located. Adjust the location and size of the bounding box as desired by clicking and dragging inside the box (to move the box) or clicking and dragging one of the circles on the sides or corners of the box (to resize the box). To save the bounding box, click Save and Close.     There are a number of important checkboxes below the Lat\/Long fields that you can use to refine your search. For example, you can allow the search to return specimens with the desired search terms OR within certain latitude and longitude coordinates. You can use any combination of the above options to search for voucher specimens.  Once you have entered the desired search terms, click the Save Search Terms button. There are a number of ways to add voucher specimens from this point.   If you have already defined a taxon list for your checklist, you can add voucher specimens from the New Vouchers tab. From this tab, you can:  View taxa that do not currently have voucher specimens in your checklist (select Non-vouchered taxa list from the Display Mode dropdown list). View specimen records that match your search criteria and are one of the taxa defined in your taxon list that is NOT currently vouchered (select Occurrences for non-vouchered taxa from the Display Mode dropdown list).  To add voucher specimens from this list, click the box to the left of the specimen record(s) that you wish to add and click the Add Vouchers button.   View specimen records that match your search criteria and are one of the taxa defined in your taxon list, regardless of whether that taxon has been vouchered previously (select New occurrences for all taxa from the Display Mode dropdown list).   To add voucher specimens from this list, click the box to the left of the specimen record(s) that you wish to add and click the Add Vouchers button. Regardless of whether you have previously defined a taxon list for your checklist, you can add voucher specimens through the Missing Taxa tab. From this tab, you can:  View the taxa that are not found in the checklist currently but are represented by one or more specimens that align with your search criteria (select Species List from the Display Mode dropdown list).  You can then view a list of the specimens associated with each taxon by clicking the page and link icon . From there, you can link the desired specimens individually by clicking the add voucher icon to the right of the specimen record in the list.   View the specimens that are not yet added as voucher specimens for your checklist but do align with your search criteria (select Batch Linking from the Display Mode dropdown menu).  To add voucher specimens from this list, click the box to the left of the specimen record(s) that you wish to add and click the Add Vouchers button.   View specimen records that align with your search criteria, but have scientific names that are not linked to a particular name in the taxonomic thesaurus (select Problem Taxa from the Display Mode dropdown menu).  To add voucher specimens from this list, enter the name of the taxon to which this specimen belongs into the text box to the right of the specimen’s scientific name. Note that the name you enter in this box must already be found in the list of taxa in your checklist. Click the Link Voucher button.      Adding Individual Specimens from Your Own Collection You must be an editor of a collection to assign specimens in this way.\n  Look up a specific specimen in your own collection (e.g., by clicking Edit Existing Occurrence Records in your Data Editor Control Panel and searching by a specific catalog number or by using other search terms). On the resulting page, click the Linked Resources tab. In the Checklist Voucher Linkages box (see below), select the desired Checklist from the dropdown menu and click Link to Checklist as Voucher. A new window will likely pop up as a result, but you can ignore this window and close the editor. The specimen has been successfully linked as a voucher to the desired checklist.  </p><p>checklist</p><p>inventory</p><p>flora</p><p>voucher</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/voucher\/"},{value:"[coll_manager] Crowdsourcing",label:"<p></p><p>crowdsourcing</p><p>volunteer</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/crowdsource\/"},{value:"[user] Accessibility Options",label:"<p>Users who create accounts in the portal can adjust Accessibility Preferences, which are designed to improve accessibility to the site for people with disabilities. More information about accessibility as defined by U.S. Section 508 can be found here.\nCurrently, users can select to the Vertical Form Layout option. This option makes content and forms easier to navigate using a screen reader and expands layouts, which can improve accessibility for users with differing experience, knowledge, language skills, or concentration levels.\nTurning on Accessibility Options temporarily Your portal may have an Accessibility Options link in the footer. Click this, if so. If not, proceed to the next section.\nTurn on Accessibility Options for your user profile You can make certain Accessibility Options the default for whenever you log in to the portal by following these steps:\n Log in to your portal and navigate to your profile by clicking My Profile. Click the User Profile tab. Click Edit Profile. Check the box next to \u0026ldquo;Check to indicate a preference for vertical form layout\u0026rdquo;.  </p><p>508</p><p>disability</p><p>screen reader</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/accessibility\/"},{value:"[portal_manager] Generating Taxon Map Thumbnails",label:"<p> Each taxon in the taxonomic thesaurus has a Taxon Profile page that includes images, descriptions, links, and other data (when they exist) regarding that taxon. Using the Taxon Profile Map Thumbnail tool, a portal manager (superadministrator) can also generate thumbnail-sized distribution maps for taxa using record data from the portal. These maps can be dot maps or heat maps, and both rely entirely on the distributions of georeferenced records available in the portal. These maps must therefore be interpreted in the context of the geographic and\/or taxonomic focus on the specific portal.\n Navigate to the tool through the Sitemap by scrolling down to the Administrative Functions (Super Admins only) heading and clicking Manage Taxon Profile Map Thumbnails.\nThe map on the resulting page will be automatically centered to the default map coordinates as defined in the Symbini file. To adjust the bounds of the map, click and drag on the map, use the plus and minus buttons to zoom in or out, respectively, and\/or use the rectangle drawing tool, which will zoom you closer to a specific area on the map. Double click the map to zoom in.\nYou can also manually set the boundaries of the map viewing box by editing the latitude and longitude values in the Bounds box. Click the \u0026ldquo;Global Bounds\u0026rdquo; button to represent a global map.\nYou can change the base layer of the map by clicking the layers icon in the top right corner of the map and selecting the appropriate radio button from the options.\nStart typing the name of taxon for which you would like to generate a map in the \u0026ldquo;Taxon Name\u0026rdquo; field and select a matching taxon from the taxonomic thesaurus. If you can\u0026rsquo;t find your taxon of interest in the dropdown list, add it to the taxonomic thesaurus.\nIf you select a taxon with any children taxa (e.g., you select a genus or family name), clicking the \u0026ldquo;Build Map(s)\u0026rdquo; button will run through all the children taxa of that taxon and build thumbnail maps for all with available specimens using the parameters you have selected. A thumbnail map will not be created for the parent taxon (genus or family)\n Adjust the parameters and click the \u0026ldquo;Preview Map\u0026rdquo; button to adjust the appearance of your thumbnail map. Depending on number of available records and the distribution of your taxon, you may need to play with the available parameters to produce a useful thumbnail.\n If you want to create a map of points where records are located, select the \u0026ldquo;Dot Map\u0026rdquo; option from the Map Type box. Select \u0026ldquo;Heat Map\u0026rdquo; for a heat map. Check the box next to \u0026ldquo;Automatically adjust bounds to fit coordinate data\u0026rdquo; if you would like the map boundaries to adjust automatically to include all the georeferenced records of that taxon. ⚠️ If you adjust the boundaries of the map manually and would like to keep those boundaries, make sure you uncheck the \u0026ldquo;Automatically adjust bounds to fit coordinate data\u0026rdquo; box before clicking \u0026ldquo;Build Map(s)\u0026rdquo;.  Once you are satisfied with the appearance of your map, click \u0026ldquo;Build Map(s)\u0026rdquo;.\nHeat Map Parameters The appearance of the heat map can be adjusted by changing the heat radius (using the slider) and the heat density (using the minimum and maximum density fields).\nA larger heat radius would mean that each point is represented by a larger colored area on the map.\nWhereas a smaller heat radius makes each point represent a smaller colored area on the map.\nSetting a smaller Minimum Density value will require a certain number of records to be present in an area before it will show up as \u0026ldquo;heat\u0026rdquo; on the map, and it will intensify the \u0026ldquo;heat\u0026rdquo; of locations with multiple records. Setting a larger Minimum Density value will lower the relative \u0026ldquo;heat\u0026rdquo; of hot spots.\nDot Map Parameters To adjust the color of the map, click the grey square and adjust the hue and saturation in the pop-up box. Click outside of the pop-up box to close.\n</p><p>taxonomy</p><p>taxon map</p><p>taxon profile</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/taxonomy\/thumbnails\/"},{value:"[editor] Duplicate Matching",label:"<p> This page describes how to use the Duplicate Matching tool in the occurrence editor to accelerate data entry.\n The Duplicate Matching tool allows the user to search the entire database (i.e., all collections in the Symbiota portal) for records that share the same collector name, collector number, and date and link them to the record they are entering\/editing.\nSearching for duplicates  Search for and open a record that you wish to edit or add a new record. Enter the name of the collector\/observer into the Collector\/Observer field, the collector\u0026rsquo;s unique collector number (if applicable) into the Number field, and the date of collection into the Date field (in YYYY-MM-DD format). Click the \u0026ldquo;Duplicates\u0026rdquo; button.  You can turn an automatic duplicate search on or off using the \u0026ldquo;Auto search\u0026rdquo; checkbox. This will conduct a search for duplicates automatically after you enter the collector name, number, and date.\n There are three possible results of a duplicate search using this tool.\nPossible EXACT Duplicates This result occurs when a record with a matching collector last name, collector number, and date is found in a collection other than the one you are currently working in. This may represent a true duplicate specimen.\nPossible EXACT Duplicates with NOTICE This result occurs when a record with a matching collector last name, collector number, and date is found within the collection you are currently working in. This may represent a true duplicate specimen, if you have them in your collection, or it could represent unintentional duplicate data entry (i.e., an error).\nPossible Matching Duplicate EVENTS This result occurs when a record with a matching collector last name and date was found, but the exact collector number was not found. In this case, the results presented will include collector numbers slightly above and slightly below the number that you entered. These records may still be useful because they may share, e.g., locality data with the record that you are currently entering\/editing.\nHandling duplicate match results You will have several options for using the results of the duplicate matching search. These will be listed at the bottom of the duplicate result that has been identified (see example below). You may:\nTransfer All Fields Transfer the data from all the fields in the identified duplicate specimen or event into the data entry page that you are currently working on. This will replace any data that you have already entered (e.g., if you entered \u0026ldquo;Luis Gonzalez\u0026rdquo; in the collector\/observer name, and the duplicate had was \u0026ldquo;Gonzalez, Luis\u0026rdquo; in this field, \u0026ldquo;Gonzalez, Luis\u0026rdquo; would overwrite your previous entry). Clicking this option will transfer the data and close the duplicate matching window.\nTransfer to Empty Fields Only Transfer the data from all fields in the identified duplicate specimen or event into the data entry page that you are currently working on, unless that field already has data in it. Only empty fields will be populated. Your previous entries will remain the same. Clicking this option will transfer the data and close the duplicate matching window.\nLink as Duplicate Checking this box and then clicking one of the additional options will create a duplicate linkage between the record you are editing and the duplicate record. For more information about duplicate linkages, visit the Duplicate Clustering documentation.\nGo to Record This option is only available if the identified duplicate belongs to the same collection that you are currently editing. Clicking this option will take you to the occurrence editor page for the duplicate record so you can view it (e.g., if you want to verify the data or view the image). Once you have looked at the image, if you want to transfer any of the data into the record you are working on, you will need to click the \u0026ldquo;Duplicates\u0026rdquo; button on the record that you were editing again.\nMerge Records This option is only available if the identified duplicate belongs to the same collection that you are currently editing. If you click this option, the two records will be merged together. The record in the pop-up indow will be prioritized over data in your occurrence editor. For example, if the record listed in the pop-up window has \u0026ldquo;USA\u0026rdquo; in the country field, and the record in the occurrence editor has \u0026ldquo;United States\u0026rdquo; in the country field, the merged record will have \u0026ldquo;USA\u0026rdquo; in the country field. Fields that are empty in one of the two records will be filled in with the data from the record that already has data in that field. All images and linked resources will be included in the merged record. All determinations will be included in the merged with the exception of any \u0026ldquo;current determination\u0026rdquo; belonging to the record in the occurrence editor, which will be discarded. As a result, you will have one current determination (belonging to the record in the pop-up window) and all other previous determinations associated with the record. For this reason, it is a good idea to check the Determinations tab of the merged record before moving on.\n</p><p>duplicate</p><p>copy</p><p>transfer</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/duplicates\/"},{value:"[user] Creating a Checklist",label:"<p> This page describes how to create a checklist or biotic inventory in a Symbiota portal.\n The \u0026ldquo;Create a Checklist\u0026rdquo; permission has been granted for the majority of existing users (accounts created before June 2022). If you do not have \u0026ldquo;Create a Checklist enabled\u0026rdquo;, and need it, ask your portal admin (super admin) or the Support Hub to add these user permissions to your account. All new users will need to request permission to create checklists\/datasets moving forward.\n  Log in to the Symbiota profile (if you don\u0026rsquo;t already have an account, click New Account to create one). Click “My Profile” Under the “Species Checklists” tab, click the green plus sign.  Fill out the “Create New Checklist” box with the desired information.  Most checklists will be a General Checklist. Alternatively, you can another list type from the Checklist Type dropdown menu. A Species Exclusion List is used to curate a subset of taxa that are purposely excluded from another list, and a Rare, threatened, protected species list is used to compile a state-level list of rare, threatened, and protected taxa for which locality data will be redacted for specimens in that state. To indicate that this checklist is a subset within a larger checklist, select the “parent” (larger) checklist from the More Inclusive Reference Checklist dropdown menu. You can define the area included in the checklist by describing it in the Locality and Abstract fields, including Latitude and Longitude coordinates, and\/or you can define the boundaries of that area geographically by setting a “footprint polygon”. To define a “footprint polygon”:  Click the globe icon inside the Polygon Footprint box. A Google map will appear a new window. Drag the map until it shows the region covered by your checklist, then zoom in until you can see the boundaries of the area. Click the polygon icon in the top center of the window then click around the boundaries of your area. When you get back to the point where you started, double click the starting point. This will lock the coordinates into the text box at the bottom of the window. Your defined area will shaded in. If you are dissatisfied with your polygon, you can delete or redraw it by clicking the appropriate button on the bottom right. When you are satisifed with your polygon, click the Save Polygon button.   You can decide whether you want to keep this checklist private, or if you want it to be available to the public by selecting Private or Public from the Access dropdown menu.    To add taxa (species) to your list, see the section Add Taxa to Checklist.\n</p><p>checklist</p><p>inventory</p><p>flora</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/create\/"},{value:"[coll_manager] Data Citations",label:"<p> Some citation features will require initial configuration by your portal manager or the Symbiota Support Hub prior to full implementation. Refer to the relevant portal manager documentation for more information.\n As collections become available online, properly citing them has become increasingly necessary to enable full digital connectivity between your specimens, their extended data, and the published literature. Symbiota offers a suite of options to enable proper attribution of data shared through our portals. Collections Managers are strongly encouraged to instruct users of their data to include all elements of these citations to fully benefit from their functionality.\n  Collection Citations Collections published to GBIF Collections that publish their data to GBIF automatically have access to robust data usage tracking through the generation of a DOI that uniquely identifies your collection online. When this DOI is included in citations of your data, automated citation tracking becomes possible. GBIF\u0026rsquo;s documentation on this topic can be found here.\nWhere is my collection\u0026rsquo;s properly formatted citation? In order for citations to link to your collections in GBIF and Symbiota, data users must cite your data properly. If enabled in your portal, suggested citations will be automatically generated on your collections profile page (see image directly below). While these citations may be reformatted to conform to citation styles required by publishers, each element of the citation must be included, the most important element being the DOI expressed as a URL.\n Example citation: NEON Biorepository Data Portal (2022). NEON Biorepository Carabid Collection (Pinned Vouchers). Occurrence dataset https:\/\/doi.org\/10.15468\/zyx3fn accessed via the NEON Biorepository Data Portal, https:\/\/biorepo.neonscience.org\/ on 2022-10-25.\n It is wise to preemptively encourage researchers to properly cite your data according to these guidelines, which can be found on your collections profile:\nWhat happens once my data are cited? If your data are cited properly in digitally available published literature, GBIF will track these citations once they are indexed by Google Scholar. In turn, these citations will tally in the \u0026ldquo;citation widget\u0026rdquo; that appears at the top of your collections profile. Clicking on the widget will direct you to a bibliography of works that have used your digitized collections data:\nRead more about GBIF\u0026rsquo;s citation guidelines here.\n Collections not published to GBIF If your collection is not publishing to GBIF, you can still encourage researchers to cite your collections data using the citation that automatically generates in your profile above \u0026ldquo;Collection Statistics\u0026rdquo;. This citation can be adapted to meet various citation style requirements; however, each element of the citation should be included, and especially the dataset ID and URLs, which are unique to your collection.\n Example citation: Soil Collection (Distributed Periodic). Occurrence dataset (ID: cfb05bfe-b267-471a-b538-e5b644e3afa7) https:\/\/biorepo.neonscience.org\/portal\/content\/dwca\/NEON-SOIC-DP_DwC-A.zip accessed via the NEON Biorepository Data Portal, https:\/\/biorepo.neonscience.org\/, 2022-10-25).\n Learn how to publish your data to GBIF here.\n Data Downloads When your data are downloaded from a Symbiota portal, a \u0026ldquo;CITEME.txt\u0026rdquo; file will be included in the data package. This file will include a suggested citation as well as the URL to your portal\u0026rsquo;s data usage policy, if one is maintained.\nExample CITEME.txt content:\n This data package was downloaded from the Ecdysis Portal on 2022-10-25 17:03:40. Please use the following format to cite this dataset:\nBiodiversity occurrence data published by: Ecdysis Portal (accessed through the Ecdysis Portal Portal, https:\/\/serv.biokic.asu.edu\/ecdysis, 2022-10-25). For more information on citation formats, please see the following page: https:\/\/serv.biokic.asu.edu\/ecdysis\/includes\/usagepolicy.php\n Data Usage Policy \u0026amp; Portal Citations Some portal communities maintain their own portal-wide data usage policies for media and specimen records, which includes a recommended citation format for the portal. This information can typically be found on the Sitemap \u0026gt; Image Library \u0026gt; Usage Policy and Copyright Information. To request modifications to your portal\u0026rsquo;s data usage policy, or to have one added to your portal, contact your portal manager.\n        Citation guidelines provided in the CCH2 portal\u0026rsquo;s data usage policy    </p><p>citations</p><p>gbif</p><p>data publishing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/citation\/"},{value:"[user] Changing your Password or Login Info",label:"<p>To change your password, username, or any other personal information in a Symbiota portal:\n Log in to the portal. Navigate to My Profile. Click the User Profile tab. From here, use the bold black links to:  Edit Profile: change first name, last name, email address, ORCID, title, institution, or address Change Password: enter your current password, followed by your new password twice Change Login: enter a new username, followed by your current password to confirm Manage Access: here you can clear your access tokens, which are like \u0026ldquo;pathways\u0026rdquo; to editing data on your behalf in the portal. If the number seems high, you can clear the access tokens. This will require you to enter your username and password anew upon logging in from any computer or device.    Resetting your Password If you no longer have access to your user account, reset your password by clicking Log In, then the Reset Password link. You will be sent an email with a new password that you can use to log in. This email is often filtered by institutional email addresses. Please check your spam folder, then email help@symbiota.org if you still don\u0026rsquo;t see it.\n</p><p>password</p><p>profile</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/pwd\/"},{value:"[editor] Material Samples",label:"<p> This page describes how to add and edit material sample records associated with specimen occurrences. This tab will only be visible if it has been activated for your collection. Contact your Portal Manager to determine whether this tab has been configured for your Symbiota portal.\n         Material Sample tab within the Occurrence Editor    Material Sample Data Fields Definitions for data fields displayed on the Material Sample tab are defined in the Symbiota Data Fields documentation.\nAdding\/Editing Material Sample Records One specimen occurrence in a Symbiota portal can be associated with one or more material samples.\n Adding Individual Material Samples         Specimen occurrence with multiple associated material sample records as viewed from within the Occurrence Editor. This example is from the NEON Biorepository\u0026rsquo;s Symbiota portal.    To add a new material sample to an existing catalog record:\n Navigate to the Occurrence Editor: My Profile \u0026gt; Occurrence Records \u0026gt; name of collection \u0026gt; Edit Existing Records Search for the record that the sample will be associated with. Open the record so that the Occurence Editor form appears. At the top of the form, select the Material Sample tab. Select the plus sign icon to begin adding a new material sample. As you fill out the form, consult the Symbiota definitions for Material Sample data fields if you are unfamiliar with these terms. Select the \u0026ldquo;Add Record\u0026rdquo; button to save the new material sample record. Repeat, starting at step 5, to add additional material sample records.  To edit existing material sample records, navigate to the Material Sample tab and select the pencil icon (✏️) to reopen the editable form.\n Batch Adding Material Samples At present, material sample records cannot be batch uploaded from within a Symbiota portal\u0026rsquo;s user interface. Collection Administrators are advised to contact the Symbiota Support Hub for assistance if this is desired.\nDownloading Material Samples Data entered into the Material Sample tab can be downloaded from Symbiota as Darwin Core Archive or Symbiota data backup file. The output file will be named materialSample.csv.\n</p><p>Material Sample</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/materialsample\/"},{value:"[portal_manager] Adding\/Editing Glossary Terms",label:"<p> Only users with Superadministrator or Glossary Editor permissions may add or edit terms in the glossary.\n The glossary is publicly available on most portals via the sitemap only after at least one entry has been added to the glossary. Until then, the glossary is linked in the Sitemap under Administrative Functions (Super Admins only).\nOn the glossary page, click the name of an entry (to edit) or click \u0026ldquo;Add New Term\u0026rdquo; at the top right corner of the page to add a new glossary entry. Note that you can use the Search box to search for an existing entry.\nAdding a New Glossary Term The following fields are available when you add a new term.\nTo link a glossary term to a specific taxonomic group, begin to the type the name of the taxonomic group in the Taxonomic Group field and select the appropriate name from the dropdown menu.\nTo link a glossary term to another glossary term as a synonym or translation, select the appropriate relationship from the dropdown menu following Relationship and the desired term from the dropdown menu following the Related Term field. To link the term to another term that is not a synonym or translation, click the Create Term button and then edit that term using the instructions below.\nEditing an Existing Glossary Term Once a glossary term is created, you can edit the previous data you entered and\/or add additional data such as images, sources, and translations.\nOn the glossary page, click the name of an entry. In the pop-up window, click the pencil icon in the top right corner of the window.\nIn the Details tab, you can edit any of the basic information for the term, such as definition, source, or language.\nFrom here, you can link the term to another related term (e.g., subclass, part) using the Related Terms tab.\nYou can add one or many translations of the term using the Translations tab.\nYou can add one or many images using the Images tab (you can either upload a file or provide a URL to an appropriate image).\nYou can delete a term using the Admin tab.\n</p><p>vocabulary</p><p>identification keys</p><p>character matrix</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/glossary\/edit\/"},{value:"[portal_manager] Glossary",label:"<p> In Symbiota portals, the glossary is a list of terms and their definitions relating to the content of the portal.\n Key points regarding Symbiota glossaries:\n Glossary terms can be linked to both characters and character states used in identification keys. Glossary entries can include definitions, images, links to outside resources, and translations. Glossary entries can be linked to a specific taxonomic group above the level of family (i.e., to the entire clade to which the term applies) and\/or to other, related glossary terms. The glossary can be edited by superadministrators and anyone who has been granted Glossary Editor permissions. Glossary Editor permissions can only be granted by a Superadministrator (i.e., portal manager). See User Permissions. The glossary is publicly available on most portals via the sitemap only after at least one entry has been added to the glossary. Until then, the glossary is linked in the Sitemap under Administrative Functions (Super Admins only). To make the glossary more accessible, you can add a menu item or link to the \u0026ldquo;\/glossary\/index.php\u0026rdquo; page on the homepage or another location on the site. The glossary is not pre-populated by default and must be manually curated. There is currently no user interface for batch uploading glossary entries. Batch additions must be facilitated by a portal administrator on the backend.  </p><p>vocabulary</p><p>identification keys</p><p>character matrix</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/glossary\/"},{value:"[portal_manager] Linking Glossary Terms to Identification Keys",label:"<p> Only users with Superadministrator or Identification Keys Administrator permissions may add or edit terms in the glossary.\n Glossary terms can be linked to both characters and character states used in identification keys to help define these characters and character states.\nGlossary terms can be linked in the same interface used to edit and add characters and character states. Instructions for doing so can be found here.\n</p><p>vocabulary</p><p>identification keys</p><p>character matrix</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/glossary\/link\/"},{value:"[user] Checklists",label:"<p> To create a checklist, you now need to obtain \u0026ldquo;Create a Checklist\u0026rdquo; permissions from a portal administrator or the Symbiota Support Hub. For assistance, email the Hub.\n A checklist is a list of taxa for a certain region or use (e.g., a teaching list so that students can learn a certain set of taxa, or a list of rare or threatened taxa). The checklist tools in Symbiota offer:\n Viewing a checklist by scientific names, common names, images in the image library, or a combination of these options Linking voucher specimens from the Symbiota portal Management of notes (either public or private) associated with taxa in the checklist Identification and name recognition\/spelling games (study aids) Identification keys consisting of only taxa from the checklist  To create a checklist, you can:\n Upload a pre-existing list of taxonomic names using the batch uploader. Find taxa that should be included in your checklist based on voucher specimens that meet user-specified geographic criteria (e.g., a polygon, bounding box, or text field). Instructions here.    Additional resources about creating checklists in association with a floristics project can be found below:\n Mid-Atlantic Megalopolis Creating a Flora  Checklist Use Cases  Generate lists for field use (species inventories) Link taxa and inventories in publications to voucher specimens Visualize species distributions within a given area Export lists as Word documents and spreadsheets for printing and field use Create study aids for students via games  Sample checklist  West Fork of Oak Creek Flora  Related Resources  Bell, M., E., \u0026amp; Landrum, L. R. (2021). Making Checklists with the SEINet Database\/Symbiota Portals. Canotia, 17, 1–24. PDF  </p><p>checklist</p><p>inventory</p><p>flora</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/"},{value:"[user] Dynamic Checklists",label:"<p>This page will describe how to create and use dynamic checklists.\nWant this page faster? Contribute!\n</p><p>checklists</p><p>dynamic</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/dynamic\/"},{value:"[editor] Checklists",label:"<p> The \u0026ldquo;Create a Checklist\u0026rdquo; permission has been granted for the majority of existing users (accounts created before June 2022). If a user does not have \u0026ldquo;Create a Checklist enabled\u0026rdquo;, and needs it, ask your portal admin (super admin) or the Support Hub to add these user permissions to their account. All new users will need to request permission to create checklists\/datasets moving forward.\n For more information on checklists, please visit the Checklist chapter in the User Guide\n </p><p>checklists</p><p>inventories</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/checklist\/"},{value:"[coll_manager] Importing from Specify",label:"<p>Collections that manage their data in Specify 6 can push a copy of their data to a Symbiota portal using built-in Specify tools. Instructions for this process can be found in the Specify forum.\n</p><p>specify</p><p>import</p><p>upload</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/upload\/specify\/"},{value:"[user] Exporting Checklist Reports",label:"<p>You can generate a number of useful reports using the Reports tab in the Manage Linked Vouchers tool, as shown in the menu below.\nYou can print a list of species, list of vouchers, list of potential species additions, and other options from this page.\n</p><p>checklist</p><p>report</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/report\/"},{value:"[user] Download",label:"<p></p><p>download</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/download\/"},{value:"[user] Datasets",label:"<p> This page describes how to create lists of occurrences, called \u0026ldquo;datasets\u0026rdquo;, in a Symbiota portal.\n A dataset is a list of occurrences created and\/or curated by a user. Datasets may be used to keep track of specimens associated with certain inventory, research, data curation, or interest projects. For example, a researcher might save a datset consisting of all the occurrence records they used in a particular analysis, a collector might save a dataset consisting of all specimens from a particular collecting trip, or a collection manager might save a dataset of all the specimens that they need to physically inspect for curation purposes. Some examples in practice can be found here and here.\nDatasets can be private or publicly available.\n  </p><p>dataset</p><p>inventory</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/dataset\/"},{value:"[portal_manager] Importing Determinations",label:"<p> This page provides instructions for bulk uploading specimen determinations\/annotations and associating them with their respective specimen records. For instructions for adding individual determinations or the batch annotation tool, see this page. This tool can only be accessed by a superadministrator.\n If you are new to using this tool, try importing only a few new determinations to familiarize yourself with how the tool behaves before ingesting a large batch. For instance, this tool cannot be used to overwrite existing determinations; it can only be used to add new determinations to one or more occurrence records.\n         Data associated with specimen identifications (aka taxonomic determinations or annotations) can be added to specimen records in bulk using the Extended Data Importer tool.    Initiating the Upload  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Within the Administration Control Panel, navigate to Import\/Update Specimen Records \u0026gt; Extended Data Import. Select the file containing your determinations to be imported. See below for tips regarding the contents and formatting of this file. Choose Import Type = \u0026ldquo;Determinations\u0026rdquo;. Select the desired Upload Type from the dropdown menu. If your upload was successful, the message \u0026ldquo;Done process input file\u0026rdquo; will appear in the Action Panel.  Example Data Import File Files containing determinations to be bulk imported must be in CSV format and include the following: 1) an identifier for the associated specimen occurrence (catalogNumber, otherCatalogNumber, or occurrenceID), 2) dateIdentified, 3) identifiedBy, and 4) sciname.\nFor example:\n   catalogNumber identifiedBy dateIdentified sciname     OBI000001 Don Cole 1950 Rosa woodsii var. gratissima   OBI000001 unknown s.d. Rosa   OBI000002 Edward Gilbert 2023-10-01 Rosa woodsii    Data Import Fields Refer to the Symbiota Data Fields and Data Import Fields guides for more information about some of the fields listed below.\n Not all fields available through the Extended Data Importer for bulk determination ingestion are visible through the Occurrence Editor interface. However, once imported, these values can be exported, for example, through a data backup.\n    Target Field (🔸 = required) Data Type (Length) Notes     subject identifier 🔸  catalogNumber, otherCatalogNumber, or occurrenceID   dateIdentified 🔸 Text (45) The date the identification was made. If the date is unknown, use \u0026ldquo;s.d.\u0026rdquo; in this field.   family (Override) Text (255) The family to which the taxon belongs. If the scientific name provided is matched to one in the taxonomic thesaurus, this will be auto-filled according to the thesaurus entry.   higherClassification (Override) Text (150) A list (concatenated and separated) of taxa names terminating at the rank immediately superior to the referenced dwc:Taxon. If the scientific name provided is matched to one in the taxonomic thesaurus, this will be auto-filled according to the thesaurus entry for the current identification.   identificationQualifier Text (255) The determiner\u0026rsquo;s expression of uncertainty in their identification. This will be listed on the label along with the scientific name. (e.g., cf., aff.)   identificationReferences Text (2000) The reference source used to make the identification.   identificationRemarks Text (2000) Any additional notes regarding the identification of the specimen.   identificationUncertain Integer (2) Use the value \u0026ldquo;1\u0026rdquo; to indicate an uncertain identification or \u0026ldquo;0\u0026rdquo; for certain identifications.   identificationVerificationStatus Text (45) A categorical indicator of the extent to which the taxonomic identification has been verified to be correct.   identifiedBy🔸 Text (255) The name of the person who identified the specimen. Also called a determiner. If the determiner is unknown, use \u0026ldquo;unknown\u0026rdquo; in this field.   isCurrent Integer (2) Use the value \u0026ldquo;Y\u0026rdquo; to flag a determination as the most current. However, doing so will not override existing determinations that have already been flagged as the most current.   printQueue Integer (2) Use the value \u0026ldquo;1\u0026rdquo; to add a determination to the annotation print queue.   scientificNameAuthorship Text (255) The name of the person who first named the taxon. If the scientific name provided is matched to one in the taxonomic thesaurus, this will be auto-filled according to the thesaurus entry.   sciname 🔸  Taxonomic identification without authorship string. See Data Import Fields.   securityStatus Integer (11) Use the value \u0026ldquo;1\u0026rdquo; to redact images and locality data for the associated occurrence record; use \u0026ldquo;0\u0026rdquo; to keep all details visible. See Symbiota Data Fields.   securityStatusReason Text (100) Explanation for data redaction. See Symbiota Data Fields.   sortSequence Integer (10) Specifies the sequential sort order of multiple determinations on the determination history tab and in the public display. Low numbers will sort higher than high numbers.   sourceIdentifier Text (45) An identifier for the Identification (the body of information associated with the assignment of a scientific name). May be a global unique identifier or an identifier specific to the data set.   taxonConceptID Text (45) An identifier for the taxonomic concept to which the determination refers.   taxonRemarks Text (2000) Any additional notes regarding the taxonomic name to which the specimen was identified.   verbatimIdentification Text (250) A string representing the taxonomic identification as it appeared in the original record.    </p><p>data upload</p><p>data import</p><p>determinations</p><p>annotations</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/determinations\/"},{value:"[coll_manager] Linked Resources \/ Associations",label:"<p> This page describes how to batch upload associated occurrences or other linked resources to your records. For general information about linked resources or to upload individual linked resources, see this page.\n This tool uses terms subject and object to refer to two different types of occurrences. The subject record is the occurrence that you are adding associations to. The object record is the association or other occurrence that is being added or linked to the subject record.\nUploading Linked Resources \/ Associations  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Import\/Update Specimen Records, then select \u0026ldquo;Extended Data Import\u0026rdquo;. Click the \u0026ldquo;Choose File\u0026rdquo; button to upload a properly formatted associations file into the uploader (see sections below for formatting requirements). Select \u0026ldquo;Associations\u0026rdquo; from the Import Type dropdown menu. Select the desired Association Type from the next dropdown menu: Non-occurrence Resource Link, Occurrence - Internal (this portal), Occurrence - External Link, or Taxon Observation. Only one Association Type may be uploaded per import file. Click the \u0026ldquo;Initialize Import\u0026rdquo; button. Select the desired Relationship Type from the dropdown menu.  The available Relationship Type values are defined per portal and ideally correspond to a community-created controlled vocabulary, such as terms from the Relation Ontology. The term you select refers to the occurrence as the \u0026ldquo;subject\u0026rdquo; and the linked resource as the \u0026ldquo;object\u0026rdquo;. For example, if you select the relationship \u0026ldquo;siblingOf\u0026rdquo;, the inferred relationship is \u0026ldquo;My occurrence is the sibling of this other resource\/link\/occurrence.\u0026rdquo;\n (Optional) Select the desired Relationship Subtype from the dropdown menu. The options in this list are also defined per portal as with Relationship Type. Map the fields in your input file (on the left of the resulting page) to appropriate target fields (see Table 1. Linked Resources Upload Fields). The fields available in the dropdown list will depend on the Association Type you are uploading. If you would like to overwrite previously-uploaded associations with identical values of the \u0026ldquo;identifier\u0026rdquo; field, check the box labeled \u0026ldquo;Update records with matching identifiers.\u0026rdquo; Click the Import Data button.  Association Types 1. \u0026ldquo;Non-occurrence Resource Link\u0026rdquo; Uploads Non-occurrence Resource: a URL to an external resource that provides information or extended data relating to the occurrence, but is itself not an occurrence. Examples include field notes, a compiled dataset, etc.\nDownload Example Template The required fields for this upload type are (1) a subject identifier for the occurrence you are linking to (catalog number, other catalog number, or occurrenceID), and (2) resourceUrl. The resourceUrl should be a link to the external resource that you would like to be associated with your records.\nOptional fields (defined below) include accordingTo, basisOfRecord, establishedDate, notes, object identifier (catalog number, occurrenceID, or occid), and relationshipID.\n2. \u0026ldquo;Occurrence - Internal (this portal)\u0026rdquo; Uploads Occurrence - Internal (this portal): a link to an occurrence (specimen\/observation) that exists in the same portal as the occurrence you are linking to; when creating associations within a portal, the portal will automatically update the corresponding occurrence with the reciprocal relationship\nDownload Example Template The required fields for this upload type are (1) a subject identifier for the occurrence you are linking to (catalog number, other catalog number, and\/or occurrenceID), and (2) an object identifier for the occurrence object you are linking to the subject occurrence (catalog number, occurrenceID, or occid\/Symbiota number). The object identifier will be used to link to an existing record within the portal.\nOptional fields (defined below) include accordingTo, basisOfRecord, establishedDate, notes, and relationshipID.\n3. \u0026ldquo;Occurrence - External Link\u0026rdquo; Uploads         Linkages to occurrences and similar resources that are managed outside of your Symbiota portal can be bulk imported using the \u0026ldquo;Occurrence - Externally Managed\u0026rdquo; upload option.    Occurrence - External Link: a link to an occurrence (specimen\/observation) that is available in another Symbiota-based portal.\nDownload Example Template The required fields for this upload type are (1) a subject identifier for the occurrence you are linking to (catalog number, other catalog number, or occurrenceID), and (2) resourceUrl. It is also strongly recommended to include a value for verbatimSciname (scientific name) so that the relationship can be searchable.\nOptional fields (defined below) include accordingTo, basisOfRecord, establishedDate, notes, objectID, relationshipID, resourceUrl, and verbatimSciname.\n4. \u0026ldquo;Taxon Observation\u0026rdquo; Uploads         Observations that are not associated with digitized occurrences can be bulk imported using the \u0026ldquo;Taxon Observation\u0026rdquo; upload option.    Taxon Observation: the assertion of a taxon being associated with the occurrence you are linking to. This may be, for example, the host taxon of the occurrence, a parasite, a taxon sharing the same habitat, etc.\nAssociated taxa added in the Linked Resource tab (either individually or in batch) will be placed in the associatedTaxa field when you download the data (e.g., as a Darwin Core Archive). In the download, any information stored in the associatedTaxa field will NOT be included in deference to the provided Linked Resources. Therefore, if you translate any associatedTaxa into Linked Resources, make sure to translate all the associated taxa!\n Download Example Template The required fields for this upload type are (1) an identifier for the occurrence (subject) you are linking to (catalog number, other catalog number, or occurrenceID) and (2) scientific name (of the object association being added).\nOptional fields (defined below) include accordingTo, basisOfRecord, establishedDate, notes, objectID, and relationshipID.\nTable 1. Linked Resources Upload Fields The Data Type column defines what type of data (e.g., text or numeric) can be included in this field, as well as the maximum number of characters that are allowed in the field. If the number of characters in your input file exceeds this maximum, the upload process will fail.\n    Field Name Data Type (Length in characters) Description     subject identifier: catalogNumber Text (32) The primary human-readable identifier for the subject record you are linking to.   subject identifier: otherCatalogNumbers Text (45) An alternative catalog number stored as an \u0026ldquo;Additional Identifier\u0026rdquo; in the portal for the subject record you are linking to. See this page for more context.   subject identifier: occurrenceID Text (255) The global unique identifier (GUID) of the subject record you are linking to.   accordingTo Text (45) The source asserting the relationship between the subject and object.   basisOfRecord Text (45) The nature of the record, from the Darwin Core controlled vocabulary. The most commonly used are PreservedSpecimen, FossilSpecimen, and HumanObservation.   establishedDate Datetime The date when the relationship between the subject and the object was asserted.   notes Text (250) Comments about the relationship between the subject and the object.   objectID Text (250) An identifier for a related resource (the object, rather than the subject of the relationship).   object identifier: catalogNumber Text (32) The primary human-readable identifier for the object record you are linking to the subject.   object identifier:occid Integer (10) The occid (internal Symbiota reference number) for the object record you are linking to the subject. The occid is not the same as the occurrenceID (see below).   object identifier: occurrenceID Text (255) The global unique identifier (GUID) of the object record you are linking to the subject.   relationshipID Text (45) An identifier for the relationship type that connects the subject to its object.   resourceUrl Text (250) A URL\/link to the object in the association\/relationship.   verbatimSciname Text (250) The taxonomic name of the object of the association\/relationship.    </p><p>associated occurrences</p><p>extended specimen</p><p>linked resources</p><p>batch upload</p><p>relationships</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/upload\/links\/"},{value:"[user] Downloading Images",label:"<p> This page describes how to download images from a Symbiota portal.\n It is incumbent upon data users to abide by usage rights when downloading images from Symbiota portals. Usage rights appear on collection profiles, occurrence records, and in each portal community\u0026rsquo;s respective Data Usage Guidelines (Sitemap \u0026gt; Image Library \u0026gt; Usage Policy and Copyright Information).\n Download Images Individually Images can be downloaded directly by opening images from occurrence records and saving them locally: Batch Image Downloading An R script has been developed to batch download images listed as URLs in a CSV or Darwin Core Archive-derived multimedia file:\n Pearson, Katelin. (2023, May 10). R Script for Batch Downloading Images from a CSV File (from a Darwin Core Archive). Zenodo. https:\/\/doi.org\/10.5281\/zenodo.7922418\n </p><p>Download</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/download\/download_images\/"},{value:"[user] (Admin) Adding\/Editing ID Key Characters \u0026 States",label:"<p> This page describes how to add or edit the characters and states that are available in Identification Keys. To edit which character states are applied to a given taxon, visit the Editing ID Key States for Taxa page.\n A user must have Identification Keys Administrator access to edit character and state values in a portal, since these edits affect the entire portal, not just a single checklist.\n Structure of Identification Keys Identification keys are built from a list of characters that each have multiple states.\nCharacters are categories of traits that are shared across all members of a taxon (e.g., \u0026ldquo;average wing length\u0026rdquo; or \u0026ldquo;leaf phyllotaxy\u0026rdquo;).\nA state is the specific trait that is shared within the taxon (e.g., \u0026ldquo;3-15 mm\u0026rdquo; or \u0026ldquo;opposite\u0026rdquo;, for each of the characters above). At this time, states can only be categorical (i.e., you cannot enter a number value).\nSetting up identification keys in a portal requires the administrator to import or add character and state values (see ID Keys Administrator instructions here. Then, an identification keys editor can assign certain state values to certain taxa (see ID Keys Editor Instructions here.\nAdding\/Editing Characters Changes made to characters and character states affect all identification keys and checklists across the portal. Please coordinate changes with other ID Key Administrators, when possible.\n Navigate to the Character Management page by visiting the page\u0026rsquo;s sitemap (\u0026ldquo;portal domain\u0026rdquo;\/sitemap.php) or going directly to the page (\u0026ldquo;portal domain\u0026rdquo;\/ident\/admin\/index.php).\nCharacters may be grouped together, for example, by anatomy or other similarities, as shown in the example from SEINet below.\nClicking on the name of a grouping will collapse or expand the characters underneath that grouping.\nTo add a new character, click the green plus sign at the top right corner of the page. You will be prompted to add some information that will later be found in the Details tab of the character editor (see below).\nTo edit a character, click on the name of the character.\nOn the first Details tab, you can change the name of the character, change the type of character (if available), change the grouping (that showed on the previous page), add a URL to any resources that can help a user understand the character, link the character to the glossary, add notes, or change the sort order of the character (i.e., where it shows up relative to other characters in the identification key).\nYou can also decide whether to show or hide characters by selecting \u0026ldquo;Hidden\u0026rdquo; from the Difficulty dropdown menu.\nIn the Character States tab, you can add or edit the states that belong to the character. Remember that character states are the options that the user can choose from when trying to identify a sample. The character states are what the ID Key Editor will choose between when assigning a state to a certain taxon. See below for instructions on editing character states.\nIn the Taxonomic Linkages tab, you can apply the character to the highest phylogenetic\/taxonomic node to which a character belongs. For example, if a character applies to all plants, you would apply the character to the entire kingdom Plantae. Taxonomic branches can also be excluded. For example, if a character was relevant to all families in an order except one, you could set the \u0026ldquo;Relevance to taxon\u0026rdquo; = \u0026ldquo;Relevant\u0026rdquo; for the order, then add the excluded family with \u0026ldquo;Relevance to taxon\u0026rdquo; = \u0026ldquo;Exclude\u0026rdquo;.\nThe final \u0026ldquo;Admin\u0026rdquo; tab allows you to delete the character. All character states must be removed from the character before it can be deleted.\nAdding\/Editing Character States From the Character Management page, click on the name of the character that belongs to the state you wish to edit. Navigate to the Character States tab. To edit an existing character state, click on the name of the character state. In the Character State Details box, you can edit the state name, the description, the related glossary term, the notes, and the sort sequence of the character. Scroll down to the Illustration box to add an illustration of the character state. Scroll down to the Delete Character State box to evaluate the state for deletion. All images, languages, and descriptions to the character state must be unlinked before a state can be deleted.\nTo add a new character state, click the green plus sign at the top right side of the page (highlighted below).\n</p><p>checklist</p><p>keys</p><p>identification</p><p>character</p><p>state</p><p>ID</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/idkeys\/admin\/"},{value:"[user] (Editor) Editing ID Key States for Taxa",label:"<p> This page describes how to edit the charater states that are applied to given taxa for identification keys.\n A user must have Identification Keys Editor access to edit character and state values in a portal, since these edits affect the entire portal, not just a single checklist.\n Structure of Identification Keys Identification keys are built from a list of characters that each have multiple states.\nCharacters are categories of traits that are shared across all members of a taxon (e.g., \u0026ldquo;average wing length\u0026rdquo; or \u0026ldquo;leaf phyllotaxy\u0026rdquo;).\nA state is the specific trait that is shared within the taxon (e.g., \u0026ldquo;3-15 mm\u0026rdquo; or \u0026ldquo;opposite\u0026rdquo;, for each of the characters above). At this time, states can only be categorical (i.e., you cannot enter a number value).\nSetting up identification keys in a portal requires the administrator to import or add character and state values (see ID Keys Administrator instructions here. Then, an identification keys editor can assign certain state values to certain taxa (see ID Keys Editor Instructions here).\nEditing Character States Assigned to Taxa The character states applied to taxa can be edited from the checklist editor. If you would like to edit the character states for taxa, but don\u0026rsquo;t have a specific checklist in which to do so, you can create a temporary checklist in which to do this and then delete the checklist once you are done.\nEnsure that you are logged in, then navigate to the checklist containing the taxa you would like to edit. Click the small pencil icon with \u0026ldquo;CM\u0026rdquo; next to it in the top right corner of the page.\nClick the name of the character for which you would like to edit the character states per taxon. On the resulting page, you will see the character state matrix. The name of the character is shown in the top left corner of the matrix, and the possible character states are listed as columns across the top row.\nTo assign a character state to a taxon, check the box in the taxon row and corresponding character state column. Character states assigned to a parent taxon (e.g., a family or genus) will be automatically applied to all of that taxon\u0026rsquo;s children taxa. You can, however, uncheck individual taxa if desired. You can also select multiple character states for a single taxon if both apply to that taxon.\nAlternatively, to edit all of the character states relating to a single taxon (rather than all of the taxa related to one character), click the pencil icon to the right of any taxonomic name.\nHere, you can check \u0026ldquo;Add\u0026rdquo; boxes to apply that character state to a taxon, or you can check \u0026ldquo;Remove\u0026rdquo; boxes to remove that character state from that taxon. Character states that are already applied to that taxon are shown in bold. Click any \u0026ldquo;Submit Changes\u0026rdquo; button to apply the character state changes to the taxon.\nTo view the character matrix associated with the parent taxon of the taxon you are currently viewing, click the \u0026ldquo;parent\u0026rdquo; link to the right of any character.\n</p><p>checklist</p><p>keys</p><p>identification</p><p>character</p><p>state</p><p>ID</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/idkeys\/edit\/"},{value:"[editor] OCR",label:"<p>Optical Character Recognition (OCR) is the use of computer algorithms to detect text information from images. If OCR tools are activated in your Symbiota portal, you can use OCR to attempt to parse textual information from pictures of, e.g., specimen labels and enter data into the appropriate fields from there. Instructions for using OCR in the Occurrence Editor are provided below. For batch OCR workflows, see the OCR page in the Collection Manager Guide.\nUsing OCR in the Occurrence Editor  Navigate to the occurrence editor of the specimen you would like to transcribe\/OCR. See this page for help with this step. On the right side of the page, under the image, you will find a yellow Options box. Select whether you would like to OCR the whole image and\/or whether you would like the portal to analyze the OCR results.   If you do not check \u0026ldquo;OCR Whole Image\u0026rdquo; the OCR will only run on whatever is visible in the viewing panel. Thus, if you would like to OCR only the label of the image, you can zoom in so that only the label is visible, then click OCR Image.  Click the OCR Image button. After OCR has run, a box will appear below the Options box with the parsed text. From here, you can save the OCR by clicking the Save OCR button and\/or you can copy and paste text from the box into the fields in the Occurrence Editor.  </p><p>optical character recognition</p><p>automated transcription</p><p>OCR</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/ocr\/"},{value:"[editor] Batch Georeferencing",label:"<p> This page describes the functionality of the Batch Georeferencing tool. Distinct protocols for georeferencing vary by institution but can utilize any of the tools described here.\n  Log in to your account in the portal, click “My Profile,” navigate to the “Occurrence Management” tab, and click the name of the institution for which you will be georeferencing. In the Data Editor Control Panel, click “Batch Georeference Specimens.” Enter your desired search terms in the provided Query Form box. You can search by Country, State, County, Municiplaity, Processing Status, and\/or Locality Terms. Then click Generate List.  Select the entry that you want to work on and click the Geolocate icon at the top right of the search results. This will open a new GeoLocate window (see next screenshot) displaying a map and, in some cases, possible location(s) for the specimen that GeoLocate inferred from the text.  The green dot is GeoLocate’s best guess for the locality of the specimen. You may also see red dots, which are alternative guesses. To view the reasons why GeoLocate selected these dots, click the “# possible locations found” tab to the right of the Workbench tab. The words in all caps are those that GeoLocate used to infer a location.    Click the “Options” button on the bottom left of the GeoLocate window and ensure the “Do Uncertainty” box is checked. Close the Georeferencing options window. Make sure that all of the boxes (latitude, longitude, uncertainty, and error polygon) near the bottom right of the window are checked. If there are dots on the map, investigate whether any of them could be a good starting point for your georeferencing by comparing the dots’ locations on the map to the text in the Locality String (see dashed box in screenshot above). It is possible that none of the dots indicate the correct locality. If this is the case, you can ignore the dots or remove them by clicking the “# possible locations found” tab and clicking the circled ‘x’ to the left of each rejected entry (see below).  When deciding whether to use one of the GeoLocate dots, make sure to check that the state and county where the dot has been placed matches the state and county indicated by the record.    Either starting with a GeoLocate dot, if appropriate, or from scratch, use the Locality String to determine an approximate location for the specimen. This will likely require searching other references (e.g., Google maps) for location and feature names.  You can change the base layer (i.e., the type of map shown in the GeoLocate window) by clicking the plus symbol in the upper right corner of the window (see next screenshot). To measure a distance on the map, click the button next to “Measure” and click on a starting point on the map. You can then move your cursor anywhere on the map and a line will be measured between those two points. The length of the line will be shown in green next to the line. To measure between more than one point, click again to anchor another point. To stop measuring, double click on the ending point of your measurement. If there is not enough information in the Locality String to assign an approximate point, view the specimen record by returning to the search results page (see screenshot below) and clicking the pencil icon on the top right of the search results box. This will bring up the specimen record, where you can see whether there is more locality information in another data field (e.g., habitat) or on the specimen image (if available).  If you are still not able to assign an approximate point from this information, scroll down to the bottom of the specimen record until you see the Processing Status field. Select “Expert Required” from the dropdown menu of this field. In the “Notes (Occurrence Remarks)” field above and to the left, enter a brief note in brackets, such as “[Not enough data to georeference]”. Click the “Save Edits” button, close the record, and select a new record to georeference.       Once you have found an approximate location for the specimen record, click the button next to “Place Marker” and click on the map where you want to place the GeoLocate dot. We recommend using the Georeferencing Quick Guide to determine where to place the point.\n  Set the error radius and\/or create an error polygon to indicate the uncertainty of the specimen’s locality. We recommend using the Georeferencing Quick Guide to determine how large of an error radius to use for a given point and whether you should use an error polygon.\n To edit the size of the error radius, click the green dot on the map and select “Edit Uncertainty” in the pop-up box. Click and drag the grey arrowhead that will appear on the outer rim of the error circle to resize the error radius. To create an error polygon, click the button next to “Create polygon” and click on the map where you want to begin drawing the polygon. A corner of the polygon will be created time you click on the map. To finish the polygon, double click. Once you have completed the polygon, click the green dot again and select “Resize uncertainty to polygon.”  To redo a polygon, click the “Clear Polygon” button to the right of the “Options” button.   Make sure to note any uncertainty you experienced when setting the error radius. For example, if you were georeferencing a named place that did not have clear boundaries, explain how you determined a suitable error radius by putting a note in the Remarks field on the search results page where you first selected your specimen record.    When you are confident in your GeoLocate point and error radius, click the “Save To Your Application” button at the bottom of the GeoLocate window. You will be taken back to the search results page where you first selected your specimen record (see first screenshot).  The coordinates and error will now show up below the search results in the appropriate Latitude, Longitude, and Error fields. If you created an error polygon, its coordinates will be listed in the Footprint WKT field.   Click the “Update Coordinates” button at the bottom of the page. Select a new record or set of records to georeference from the search results list and repeat.  Example Batch Georeferencing Protocols  California Phenology Network georeferencing guides Florida Museum of Natural History georeferencing workflow overview Microfungi TCN batch georeferencing instructions MSU Herbarium Georeferencing Guide  </p><p>georeferencing</p><p>batch edits</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/georeference\/batch\/"},{value:"[user] Downloading Data",label:"<p>Downloading data from Symbiota portals can easily be done from a public search.\nAfter conducting a search, click the button with the download icon This is usually displayed at the top right corner of the page, as shown below:\n</p><p>download</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/download\/download_data\/"},{value:"[coll_manager] Data Cleaning",label:"<p>Symbiota has a number of useful built-in tools for data cleaning processes. Each of the tools described in this section can be accessed through the Administration Control Panel under Data Cleaning Tools.\n Compare Duplicate IDs Across Collections Duplicate Catalog Numbers Geographic Cleaning Tools Taxonomic Cleaning Tools Data Quality Toolkit (designed for Collection Administrators and Editors)  A webinar about data cleaning can be found here:\n  </p><p>data cleaning</p><p>taxonomy</p><p>geography</p><p>duplicates</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/"},{value:"[editor] Data Quality Toolkit",label:"<p> This page includes several common data quality errors that you can find and fix in your dataset using some creative searching and\/or existing data quality tools. Data quality issues are grouped into data categories.\n For more help with data quality, see the following resources:\n Bob Mesibov\u0026rsquo;s Data Cleaner\u0026rsquo;s Cookbook GBIF\u0026rsquo;s data quality flags iDigBio\u0026rsquo;s data quality flags  Table of Contents  Catalog Numbers and Other Identifiers  Duplicate Catalog Numbers   Dates  Date Hasn\u0026rsquo;t Happened Yet Date is Suspiciously Old Identified Date Earlier than Collected Date Year, Month, and Day Values Do Not Match Date   Geography  Coordinates are Zero Coordinates Do Not Fall Within Named Geographic Unit Georeference Metadata with no Associated Georeference Elevation is Unlikely Improperly Negated Latitudes\/Longitudes Invalid Coordinates Lower Geography Values are Provided, but No Higher Geography Minimum and Maximum Elevation Values Mismatched Mismatched Country and CountryCode Values Mismatched Geographic Terms Missing Geodetic Datum Missing Latitudes\/Longitudes Misspelled Geographic Unit Names   Taxonomy  Misspelled or Invalid Taxonomic Names Unknown Higher Taxonomy   Other Issues  Incorrect Character Encodings Incorrect Line Endings Invalid Individual Count Non-standardized BasisOfRecord Values   Recordings  Catalog Numbers and Other Identifiers Duplicate Catalog Numbers Problem: The same catalog number is used multiple times within your dataset. (This problem may or may not be intentional, depending on your collection\u0026rsquo;s policies. It is generally best to not duplicate catalog numbers, when possible).\nSolution: Use the duplicate catalog number tool to view, edit, and\/or merge duplicates. Note that only users with administrator permissions can use this tool.\nDates Date Hasn\u0026rsquo;t Happened Yet Problem: The date the specimen was collected (often designated using the eventDate field) is in the future.\nSolution: There are two ways you can find records with this problem:\nMethod 1:\n Navigate to the Record Search Form for your collection. In the Sort By field, select Date. Then select \u0026ldquo;descending\u0026rdquo; in the second dropdown list.  Click Display Table. After the page loads, the highest dates will be listed at the top of the resulting table. Edit these records by clicking the link in the Symbiota ID column (far left).  Click the box and arrow icon to the right of the Symbiota ID number to open the record in a new window. That way, you will not need to re-conduct your search after editing every record.\n Method 2:\n Navigate to the public search page of your portal. This is most commonly done from the home page by selecting \u0026ldquo;Search\u0026rdquo; or \u0026ldquo;Search Collections\u0026rdquo; from the header menu. If you are taken to a list of collections, check the \u0026ldquo;Select\/Deselect all collections\u0026rdquo; box to deselect all the collections, then check the box next to your collection. Click the Search button. If you are not taken to a list of collections, proceed to step 3. In the Collector Criteria (or Collecting Event) section, enter today\u0026rsquo;s date in the first Collection Date field (or the Collection Start Date field) and 9999-12-31 in the second field (or the Collection End Date field). If you see a \u0026ldquo;Collections\u0026rdquo; section of the search page, check the \u0026ldquo;Select\/Deselect all collections\u0026rdquo; box to deselect all the collections, then check the box next to your collection. If there is no \u0026ldquo;Collections\u0026rdquo; section on the search page, you should have already selected your collection in step 2. Click the List Display button (or the Search button, if provided). The search results will show you all specimens supposedly collected after today\u0026rsquo;s date. You can view and edit these records by clicking the \u0026ldquo;Full Record Details\u0026rdquo; link underneath the specimen\u0026rsquo;s summary, then clicking the Occurrence Editor link on the resulting pop-up window.  Date is Suspiciously Old Problem: The date the specimen was collected (often designated using the eventDate field) is outside the expected historical date range. The expected date range depends on the institution, but it is unlikely that most collections have specimens with dates prior to 1600.\nSolution: See the methods described in the Date Hasn\u0026rsquo;t Happened Yet section, but do the following modifications:\nMethod 1:\nIn step 2, select \u0026ldquo;ascending\u0026rdquo; in the second dropdown list in the Sort By field. You will also likely want to filter out records without dates by selecting Date from Custom Field 1 and selecting IS NOT NULL from the second dropdown list. This will remove any blank dates from your search results, which would normally show up at the beginning of your ascending list.\nMethod 2:\nIn step 3, enter 0001-01-01 in the first Collection Date field (or the Collection Start Date field) and the earliest date you think would be possible in your collection (e.g., 1700-01-01) in the second field (or the Collection End Date field).\nIdentified Date Earlier than Collected Date Problem: The date the specimen was identified (dateIdentified field) is earlier than the date the specimen was collected (eventDate).\nSolution: This problem cannot be identified using Symbiota portal tools. To locate records with this issue, download your data from the public search page, as a backup file, or using the exporter tool. You can then use a spreadsheet program to compare the dateIdentified to the eventDate field (see Excel instructions here).\nYear, Month, and Day Values Do Not Match Date Problem: The event year, month, and day values do not match the provided event date. The event date is often the date of collection for preserved specimens.\nSolution: This problem cannot be identified using Symbiota portal tools. To locate records with this issue, download your data from the public search page, as a backup file, or using the exporter tool. You can then use a spreadsheet program to compare the dateIdentified to the eventDate field (see Excel instructions here).\nGeography Coordinates are Zero Problem: The provided latitude and\/or longitude values are 0.\nSolution:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select Decimal Latitude from the first dropdown menu, select EQUALS from the second dropdown menu, and enter 0 into the blank field. Either individually edit erroneous records by clicking the link in the Symbiota ID column (far left), or batch edit all entries using the Batch Editing Tool. Repeat steps 2 and 3 for the Decimal Longitude field. Alternatively, you can search for records with 0 for both latitude and longitude by adding another custom search term. To do so, click the pencil icon to the right of Custom Field 1 and adjust the fields of Custom Field 2 accordingly.  Coordinates Do Not Fall Within Named Geographic Unit Problem: The provided coordinates do not fall within the geographic boundaries of the named country, state, and\/or county.\nSolution: The problem cannot currently be identified using Symbiota portal tools. We recommend using the GBIF Reverse Geocoding API to verify coordinate-country matching, or by simply publishing your data to GBIF and viewing the data quality flags of your dataset.\nGeoreference Metadata with no Associated Georeference Problem: Metadata fields regarding coordinates, such as coordinateUncertaintyInMeters, georeferenceProtocol, georeferenceSources, georeferencedBy, georeferenceRemarks, and geodeticDatum are provided, but no coordinates are present. This is sometimes intentional, particularly when georeferencedBy and georeferencedRemarks are used to indicate whether a record was purposefully not georeferenced. However, it is rare that the other metadata fields can be used without associated coordinates (i.e., decimalLatitude, decimalLongitude, or verbatimCoordinates).\nProblem:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select Decimal Latitude from the first dropdown menu and select IS NULL from the second dropdown menu. Click the pencil icon to the right of Custom Field 1 to add another custom field. In Custom Field 2, select the georeference metadata field you would like to compare (see list of suggestions above) and select IS NOT NULL from the second dropdown menu. Click Display Table. Edit erroneous records by clicking the link in the Symbiota ID column (far left).  Click the box and arrow icon to the right of the Symbiota ID number to open the record in a new window. That way, you will not need to re-conduct your search after editing every record.\n Elevation is Unlikely Problem: Elevation values are either too high (\u0026gt;17000 m) or too low (-11000 m) to occur on Earth.\nSolution:\n Navigate to the Record Search Form for your collection. Do one of the following:  Minimum Elevation: In Custom Field 1, select Elevation Minimum from the first dropdown menu, select LESS THAN from the second dropdown menu, and enter -11000 into the blank field. Maximum Elevation: In Custom Field 1, select Elevation Maximum from the first dropdown menu, select GREATER THAN from the second dropdown menu, and enter 17000 into the blank field.   Click Display Table. Edit erroneous records by clicking the link in the Symbiota ID column (far left).  Improperly Negated Latitudes\/Longitudes Problem: The sign of the latitude (decimalLatitude) or longitude (decimalLongitude) does not match the sign\/hemisphere of the given country. For example, all longitudes in the U.S. should be negative.\nSolution: The problem cannot currently be identified using Symbiota portal tools. We recommend using the GBIF Reverse Geocoding API to verify coordinate-country matching, or by simply publishing your data to GBIF and viewing the data quality flags of your dataset.\nInvalid Coordinates Problem: Coordinates deviate from accepted ranges or formats, like decimalLatitude and decimalLongitude exceeding -90 to 90 and -180 to 180, respectively. verbatimCoordinates have to be valid values for coordinates in decimal degrees, degrees decimal minutes, degrees minutes second.\nSolution: Some types of invalid coordinates can be identified using the Record Search Form.\n Navigate to the Record Search Form for your collection. Do one of the following:  In Custom Field 1, select Decimal Latitude from the first dropdown menu, select LESS THAN from the second dropdown menu, and enter -90 in the blank field. In Custom Field 1, select Decimal Latitude from the first dropdown menu, select GREATER THAN from the second dropdown menu, and enter 90 into the blank field. In Custom Field 1, select Decimal Longitude from the first dropdown menu, select LESS THAN from the second dropdown menu, and enter -180 into the blank field. In Custom Field 1, select Decimal Longitude from the first dropdown menu, select GREATER THAN from the second dropdown menu, and enter 180 into the blank field.   Click Display Table. Edit erroneous records by clicking the link in the Symbiota ID column (far left).  Lower Geography Values are Provided, but No Higher Geography Problem: Lower geography (e.g., county, state\/province) values exist, but no higher geography values (e.g., country) are provided.\nSolution: This issue can be quickly identified and fixed using the geography cleaning tools. Note that you must have administrator permissions to use these tools.\nMinimum and Maximum Elevation Values Mismatched Problem: The minimum elevation (minimumElevationInMeters) has a greater value than the maximum elevation (maximumElevationInMeters).\nSolution: This problem cannot currently be identified using Symbiota portal tools. To locate records with this issue, download your data from the public search page, as a backup file, or using the exporter tool. You can then use a spreadsheet program to compare the minimumElevationInMeters to the maximumElevationInMeters field (see Excel instructions here).\nMismatched Country and CountryCode Values Problem: The provided value for country and countryCode do not match.\nSolution: This problem cannot currently be identified using Symbiota portal tools. To locate records with this issue, download your data from the public search page, as a backup file, or using the exporter tool. You can then use a spreadsheet program to compare the unique combinations of country and countryCode to look for deviations (see Excel instructions here).\nMissing Geodetic Datum Problem: Geodetic datum is a key piece of a properly georeferenced specimen, but is usually left blank. Although it is commonly assumed to be in \u0026lsquo;WGS84\u0026rsquo;, this should be added and noted as such.\nSolution:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select Geodetic Datum from the first dropdown list and IS NULL from the second dropdown. Click the pencil icon to the right of Custom Field 1 to add another Custom field. In Custom Field 2, select Decimal Latitude from the first dropdown list and IS NOT NULL from the second dropdown. Either individually edit erroneous records by clicking the link in the Symbiota ID column (far left), or batch edit all entries using the Batch Editing Tool.  Mismatched Geographic Terms Problem: A record has lower geographic terms (e.g., state\/province, county) that do not exist under the provided higher geographic term(s). For example, country = Canada and stateProvince = Sussex. There is no Sussex province in Canada.\nSolution: This issue can be quickly identified and fixed using the geography cleaning tools. Note that you must have administrator permissions to use these tools.\nMissing Latitudes\/Longitudes Problem: A record has a latitude value, but not a longitude value, or vice versa.\nSolution:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select Decimal Latitude (or Decimal Longitude) from the first dropdown list and IS NOT NULL from the second dropdown. Click the pencil icon to the right of Custom Field 1 to add another Custom field. In Custom Field 2, select Decimal Longitude (or Decimal Latitude, whichever is not the same as what you entered in Custom Field 1) from the first dropdown list and IS NULL from the second dropdown. Edit erroneous records by clicking the link in the Symbiota ID column (far left).  Misspelled Geographic Unit Names Problem: The geographic units (e.g., country, state\/province, county) are misspelled, resulting in poor matching of geographic unit names to existing geographic lists.\nSolution: This issue can be quickly identified and fixed using the geography cleaning tools. Note that you must have administrator permissions to use these tools.\nTaxonomy Misspelled or Invalid Taxonomic Names Problem: Scientific names are misspelled, resulting in poor matching of taxonomic names to taxonomic databases.\nSolution: This issue can be quickly identified and fixed using the taxonomic cleaning tools. Note that you must have administrator permissions to use these tools.\nUnknown Higher Taxonomy Problem: Species may be missing higher taxonomic information.\nSolution: This is only a problem in Symbiota portals if you have scientific names that are not included in the taxonomic thesaurus. You can use the [taxonomic cleaning tools]](https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/taxonomy\/) to automatically import names from Catalog of Life or other resources into the thesaurus (your ability to do this depends on the portal), or contact your portal administrator about adding missing names to the thesaurus.\nOther Issues Incorrect Character Encodings Problem: Data inconsistencies arise when incorrect character encodings are used during data manipulation or transfer. This issue occurs when datasets are opened, downloaded, or imported across different software platforms, leading to misinterpretation and garbled text. For instance, special characters like accents or symbols may be rendered incorrectly, affecting the readability and accuracy of the data. (e.g., Carl LinnÃ©).\nSolution: There is no cross-field searching tools that would enable you to find mis-rendered symbols across all fields, but you can search certain fields. For example:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select a field to search from the first dropdown list, select CONTAINS from the second dropdown, and enter a mis-converted character (e.g., Ã©) into the blank field. This page provides a helpful table with common problems with character encodings. You can conduct searches for the values in the \u0026ldquo;Actual\u0026rdquo; column and replace them with the values in the \u0026ldquo;Expected\u0026rdquo; column. For example, if you suspect there are \u0026ldquo;ë\u0026rdquo; values in a certain field, you\u0026rsquo;ll want to search on \u0026ldquo;Ã«\u0026rdquo;. Either individually edit erroneous records by clicking the link in the Symbiota ID column (far left), or batch edit all entries using the Batch Editing Tool.  Incorrect Line Endings Problem: When transferring text files between Unix\/Linux and DOS\/Windows systems, line endings can become inconsistent. Unix\/Linux systems typically use line feed (LF) characters, while DOS\/Windows systems use carriage return (CR) and line feed (LF) combinations. This mismatch can result in extra characters appearing in the data, causing visual artifacts and processing errors.\nSolution: This is unlikely to be a problem for data that has already been imported into a Symbiota portal. It is possible that erroneous (¶) symbols will be retained. In this case:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select a field to search from the first dropdown list (such as Locality), select CONTAINS from the second dropdown, and enter ¶ into the blank field. Either individually edit erroneous records by clicking the link in the Symbiota ID column (far left), or batch edit all entries with this symbol using the Batch Editing Tool.  Invalid Individual Count Problem: individualCount values may not make sense as a positive integer.\nSolution:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select Individual Count from the first dropdown list, select LESS THAN from the second dropdown list, and enter 1 in the blank field. Edit erroneous records by clicking the link in the Symbiota ID column (far left).  Alternatively, you can just scan through the table where Individual Count IS NOT NULL and look for discrepancies by eye.\nNon-standardized BasisOfRecord Values Problem: Values in the BasisOfRecord field do not match the recommended controlled vocabulary. While using standardized terms in this field is not strictly necessary, doing so does improve the discoverability and interoperability of your data.\nThe currently accepted values for BasisOfRecord include: MaterialEntity, PreservedSpecimen, FossilSpecimen, LivingSpecimen, MaterialSample, Event, HumanObservation, MachineObservation, Taxon, Occurrence, MaterialCitation.\nNote that even punctuation and capitalization differences in these values (e.g., Preserved Specimen) are discouraged.\nSolution:\n Navigate to the Record Search Form for your collection. In Custom Field 1, select Basis of Record from the first dropdown list, select NOT EQUALS from the second dropdown list, and enter \u0026ldquo;PreservedSpecimen\u0026rdquo; in the blank field. Click the pencil icon to the right of Custom Field 1 to add another Custom field. In Custom Field 2, select Basis of Record from the first dropdown list, select NOT EQUALS from the second dropdown list, and enter \u0026ldquo;FossilSpecimen\u0026rdquo; in the blank field. Repeat steps 3-4 for as many other valid BasisOfRecord values you think might exist in your collection. Either individually edit erroneous records by clicking the link in the Symbiota ID column (far left), or batch edit all entries using the Batch Editing Tool.  Recordings Workshop: Data Cleaning for Maximum Impact      </p><p>QA</p><p>QC</p><p>quality control</p><p>error checking</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/quality\/"},{value:"[user] Games in Checklists",label:"<p> This page describes how to use the games features of checklists for educational purposes. No user permissions or portal login is required to play games created from preexisting, publicly visible checklists. Learn how to create checklists here. Games created from checklists can be used as study aids.\n   Log into your Symbiota profile and navigate to your portal\u0026rsquo;s list of existing checklists. Look for a \u0026ldquo;Projects\u0026rdquo; or \u0026ldquo;Checklists\u0026rdquo; tab at the top of your portal interface.   Select the checklist containing content that you wish to create a game from. When the checklist\u0026rsquo;s page opens, hover over the \u0026ldquo;Games\u0026rdquo; icon to select the game you wish to use.   Name Game Summary \u0026ldquo;Name Game\u0026rdquo; is a memory recall game that prompts users to guess (and spell) a genus and species included in a checklist.\nHow to play  Begin by guessing a letter that you believe might occur in the genus or species epithet. Every time you incorrectly guess a letter, your remaining number of chances (3-12) will decrease. The :rose: :sunflower: :apple: icons simply visualize your progress and remaining number of changes (i.e., the rose withers, the apple is consumed, and the flower petals are removed with diminishing chances). Hints: Selecting a hint will cost you one chance, and they cannot be used on your last guess. On the keypad, select the ? (wildcard) button to fill in one letter for you; the Show Family button will display the species\u0027 family. If you correctly guess the genus and species before your chances run out (0), your number of games won will increase. If you do not correctly guess the species, select Reveal the Answer to display the answer. In either case, a link will then appear to the taxon record for further study. New Game will prompt the next question. Your number of games played, won, and success rate will be tracked as you continue playing and initiate new games.  Other tips  Your selected difficulty level affects how many chances you get: 3 (hardest), 6, or 12 (easiest). Spaces are already displayed for you. You do not have to guess them. You cannot change the difficulty level mid-game.  Flash Card Quiz Summary \u0026ldquo;Flash Card Quiz\u0026rdquo; prompts users to identify species from a checklist using images.\nHow to play  Choose the species displayed in the image from the Name of Organism dropdown menu. Up to 5 images of the mystery species will be recalled. You can limit the images to a certain family by additionally using the Filter Quiz by Taxonomic Group dropdown menu. If you identify the species correctly, a message will alert you to this success, and one point will tally below the image. Additionally, if you identify the species correctly on your first guess, that success will also be tracked below the image. If you cannot identify a specimen, select Tell Me What It Is! and the taxon record will display for further study. Selecting Display Common Names will add common names to the species list that displays in the Name of Above Organism dropdown menu. Reset Game will reset your score to zero.  Other tips  If the images are not displaying, try using a different browser. The quiz will bring up the first image associated with the taxon record, up to a maximum of five images. The sort order of these images can be edited on the taxon record to display in a different order (see \u0026ldquo;Game administration\u0026rdquo; below).  Game administration Checklists The species displayed in each game are dependent on the contents of the selected checklist. Editing a checklist requires a portal login and possibly additional permissions depending on who created (owns) the list. You can add\/remove taxa that display in games by editing checklists, if your account has permission to do so.\nImages If your account has Taxon Profile Editor permissions, you can edit which images display in the flash card quiz by changing the sort order of images in the Taxon Profile Editor, such that the images with a value of \u0026ldquo;1\u0026rdquo; will appear first, \u0026ldquo;2\u0026rdquo; second, and so on. The default sort value is \u0026ldquo;50\u0026rdquo;. However, be aware that image sorting changes made in the Taxon Profile Editor will affect how the images display across the entire portal\u0026ndash;not just within your game.\n</p><p>checklist</p><p>inventory</p><p>flora</p><p>games</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/games\/"},{value:"[coll_manager] Data Publishing",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/"},{value:"[coll_manager] Requesting Endorsement as a Publisher",label:"<p> For first-time data publishers, requesting endorsement as a publisher is a required first step in the data publishing process. Endorsement essentially establishes your organization as an entity (a \u0026ldquo;publisher\u0026rdquo;) within GBIF\u0026rsquo;s data portal, thereby enabling GBIF to index your collections dataset as a contribution from your home university, museum, etc. This process is typically straightforward, as described below.\n         Sample GBIF publisher page for the ASU Biocollections     Determine whether your organization is already endorsed:   Go to GBIF\u0026rsquo;s Become A Publisher page and scroll to the bottom of the page. Thoroughly search for your institution. For example, simply searching on \u0026ldquo;ASU\u0026rdquo; will not bring up \u0026ldquo;Arizona State University\u0026rdquo;, so try all possible combinations before concluding your organization is not registered. You can also search the GBIF portal for all endorsed publishers here. ⚠️ If your organization is found in the search form, do not request endorsement. Look for the contacts listed on your organization\u0026rsquo;s publisher page. If you are unsure whether your collection should be published as a dataset in connection with this existing publisher page, reach out to the listed contacts for assistance before proceeding with data publishing. You can also ask the Support Hub for advice in this situation if you\u0026rsquo;re not sure what to do. If your organization is not found in the search form, proceed with the remaining steps on this page.          It\u0026rsquo;s possible your organization is already registered as a publisher, as is often the case for museums and universities that maintain multiple natural history collections. If you determine that your organization is already registered as a publisher, reach out to the contact(s) listed on your institution\u0026rsquo;s publisher page to have them add your collection added as a separate dataset. For example, the ASU Biocollections are published to GBIF as a series of datasets.     If you have not already done so, familiarize yourself with the content on GBIF\u0026rsquo;s Become A Publisher page, which explains why the endorsement process is necessary.\n  Select \u0026ldquo;My organization is not already registered\u0026rdquo;, then review and agree to GBIF\u0026rsquo;s terms and conditions. Once you do so, a lengthier form will appear.\n  To complete the form:\nOrganizational Details:   Add your organizational details. This information may be somewhat similar to your collections profile in your Symbiota portal. However, if you are requesting endorsement for an organization that houses multiple collections that could potentially publish data to GBIF (now or in the future), this description should be more generalized. Refer to the description on the publisher page for the ASU Biocollections as an example. Use the map interface to create a placemark for your organization\u0026rsquo;s physical location.   Endorsing Node: If you are publishing your dataset through a Symbiota portal, select \u0026ldquo;Symbiota Support Hub\u0026rdquo; as your endorsing node if your collection is located in the United States. If your insitution is not US-based, please contact the GBIF-designated Node Manager for your country to see if they would like to complete your endorsement. If your country does not have a node, the Symbiota Support Hub may be able to assist with endorsement. Contact our Help Desk if you have questions about this step.\n  GBIF Projects: For most collections in Symbiota portals, the answer is \u0026ldquo;no\u0026rdquo;. Answer \u0026ldquo;yes\u0026rdquo; only if you are funded through a BID project.\n  Contacts: We strongly recommend adding multiple contacts for your organization and providing at least one generic institutional email address, if possible. This will minimize the chance that your organization\u0026rsquo;s publisher page will become orphaned due to staff turnover.\n  What and How:\n   For most collections in Symbiota portals, the answer is \u0026ldquo;Occurrence-only data\u0026rdquo;. This means specimen data. In Data Description, we suggest entering something like:   I intend to publish specimen occurrences through the Consortium of Southern Rocky Mountain Herbaria, which is a Symbiota portal, available from soroherbaria.org.\n  For the remaining three questions, most collections in Symbiota portals should select \u0026ldquo;no\u0026rdquo;.   Select \u0026ldquo;Register Organization\u0026rdquo;. At this point, if you selected the Symbiota Support Hub as your endorsing node, GBIF will notify the Hub that your organization has requested endorsement as a publisher.\n  Once approved, your organization will be listed as an \u0026ldquo;Endorsed Publisher\u0026rdquo; on the Support Hub\u0026rsquo;s Participant page on GBIF. When you open your new publisher page, note the publisher key that appears in the URL for the webpage; use this key when you go to publish your data to GBIF (Step 4).\n    </p><p>aggregator</p><p>gbif</p><p>data publishing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/endorsement\/"},{value:"[user] Splitting Checklists",label:"<p> This page describes how to split a single checklist into multiple checklists based on taxonomic groups. For example, you can make a master checklist of all taxa in a certain region, and then you can make child checklists that contain different taxonomic groups from that checklist (e.g., \u0026ldquo;Birds of Nevada County\u0026rdquo; can come from the parent checklist \u0026ldquo;Vertebrates of Nevada County\u0026rdquo;).\n To split a single checklist into multiple checklists, or just create child checklists that duplicate the information in the parent checklist, use the Batch Parse Species List Tool.\nFrom your parent checklist (i.e., the checklist that you would like to split\/parse), click the Checklist Administration button (top right corner with a pencil and A symbol), then click the Related Checklists tab. Scroll down, and you will see the Batch Parse Species List tool.\nIn the first box of the taxonomic node field, begin to type the name of the parent node for all the taxa you would like to include in your child checklist, then select the matching name from the dropdown list. You can add multiple nodes to the checklist by running this tool several times and adding to the same child checklist each time. You can alternatively add the taxon ID number from the portal into the second field. This will automatically be populated if you select a taxon name from the dropdown list.\nIn the target checklist field, select the name of the checklist that you would like to add or transfer the taxa to, or select Create New Checklist to add these taxa to a brand new checklist.\nSelect transfer taxa if you would like to remove the taxa from your parent list and add them to the new list that you selected in the target checklist field. Select copy taxa if you would like to copy the taxa from the parent list, but not delete them from the parent list.\nSelect a checklist from the Link to parent checklist field if you would like to link the newly defined checklist taxa to a parent checklist other than the parent checklist that you are parsing.\nIf you would also like to add the newly defined checklist to a specific project to which you have administrator access, select the project from the Add to project dropdown list.\nCheck the copy over permission and general attributes box if you would like to transfer user permissions and other attributes (such as whether it is public or private) to the newly defined checklist.\nClick the Parse Checklist button to complete the split\/copy action.\nIf you would like to add a polyphyletic group or multiple groups to a single checklist from a parent checklist, you can run this parsing tool multiple times on the parent checklist, each time selecting a different taxonomic node. For example, from a \u0026ldquo;Flora of Sequoia National Forest\u0026rdquo; checklist, you could create a \u0026ldquo;Graminoids of Sequoia National Forest\u0026rdquo; list by running the tool three times: once for Poaceae (grasses), once for rushes (Juncaceae), and once for sedges (Cyperaceae), all pointing to the same (new) child checklist.\n</p><p>parent checklist</p><p>child checklist</p><p>parent</p><p>child</p><p>taxonomic list</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/split\/"},{value:"[portal_manager] Taxonomy",label:"<p></p><p>taxonomy</p><p>taxonomic thesaurus</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/portal_manager\/taxonomy\/"},{value:"[editor] Downloading Data",label:"<p></p><p>download</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/download\/"},{value:"[editor] Crowdsourcing",label:"<p>Symbiota has a built-in Crowdsourcing module that can enable any logged-in user to transcribe specimen data from provided images. These specimens must have a Processing Status of \u0026ldquo;Unprocessed\u0026rdquo; and have at least one associated image.\nOnce a crowdsourcer has transcribed a specimen record and saved the edits, the processing status of the record will be automatically changed to \u0026ldquo;Pending Review\u0026rdquo; and the record will be removed from the queue. All edits that have been made to the record will be instantly publicly visible but can be reverted by a collection administrator. The transcribed record is now considered \u0026ldquo;Pending\u0026rdquo;, and the crowdsourcer will have \u0026ldquo;pending\u0026rdquo; points. Once the record is reviewed and approved by a collection administrator (i.e., processing status changed to \u0026ldquo;Reviewed\u0026rdquo;), these points will become \u0026ldquo;approved\u0026rdquo; points.\nUsers with administrator permissions can add occurrences to the crowdsourcing queue. Instructions can be found on this page.\n</p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/crowdsource\/"},{value:"[coll_manager] Publishing Data to GBIF",label:"<p> This document describes how to publish the data from your collection to global aggregator GBIF.\n Collections managed as \u0026ldquo;live datasets\u0026rdquo; within a Symbiota portal can immediately publish to GBIF without issues. Collections that make use of an in-house management system (e.g. Specify, Ke-Emu, etc.) and only publish a snapshot of their data to a Symbiota instance can also use the portal to publish their data to GBIF, but only if: 1) they are not publishing their data through another means (e.g. IPT installation, VertNet, etc.), and 2) an occurrenceID GUID is included in the data being pushed from their in-house database to the Symbiota dataset. If the collection is using the Symbiota publishing tool built into Specify, the occurrenceID GUID will be automatically included in the data upload from Specify.\nYour portal must be set up as a GBIF Publishing Installation to publishing your data to GBIF. This can be done by your portal manager.\n  Use these instructions to set up an institutional account with GBIF so that there is a direct publishing agreement established between GBIF and the institution. Since the institutional account may be used to list multiple collection datasets associated with that institution (e.g. https:\/\/www.gbif.org\/publisher\/4c0e9f60-c489-11d8-bf60-b8a03c50a862 ), you should coordinate with other collections within your institution, if applicable. Note that the institutional datasets can be published to GBIF using different publishing resources. For instance, the zoological collections could import their data from VertNet IPT (http:\/\/ipt.vertnet.org) or their institutional IPT, vascular plant data from the SEINet https:\/\/swbiodiversity.org, and lichens from CNALH (https:\/\/lichenportal.org).  If you are sure your institution is not yet registered, complete the registration form and follow the instructions provided by GBIF. If your institution is already registered, review the GBIF metadata for your organization and existing datasets and contact GBIF to make any necessary changes. Be sure that none of the existing datasets contain the same data you are trying to publish. If they do, make the appropriate arrangements with GBIF so that the old dataset can be archived BEFORE re-publishing the new dataset.   Login to your Symbiota portal, go to your Administrator Control Panel (via My Profile =\u0026gt; Specimen Management tab =\u0026gt; click your collection name), and click on \u0026ldquo;Edit Metadata and Contact Information\u0026rdquo; link in the Administration Control Panel. Verify your collection name and description (these will be used within the GBIF page), check the GBIF box to the right of \u0026ldquo;Publish to Aggregators\u0026rdquo;, and click the \u0026ldquo;Save Edits\u0026rdquo; button. If you do not see a GBIF publishing checkbox, contact your portal manager and ask them to configure the portal for GBIF publishing. Return to the Administrator Contol Panel and click on the \u0026ldquo;Darwin Core Archive Publishing\u0026rdquo; link in the Administration Control Panel. Click \u0026ldquo;Create\/Refresh Darwin Core Archive\u0026rdquo; button to package up your data within a Darwin Core Archive data package. Enter your institution\u0026rsquo;s GBIF publication key and click the Validate Key button. If you key validates, more instructions will be displayed along with a Submit Data button. The GBIF Publisher Key should have the following format: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx (e.g. 4c0e9f60-c489-11d8-bf60-b8a03c50a872). You can also enter the full URL to your GBIF publishing page, and the key will be automatically extracted. Before you can submit data, you will need to contact GBIF help desk (helpdesk@gbif.org) and request for the portal\u0026rsquo;s GBIF user account to be given permission to create and update datasets within your institution\u0026rsquo;s publishing instance. The GBIF username associated with the Symbiota portal installation is displayed in paragraph above the Submit Data button. Click on the GBIF email address to automatically generate a message within your email client. Once you hear back from GBIF affirming that the portal has permission to submit data to your publisher, click the Submit Data button. A link to your GBIF dataset will be immediately displayed, though it may take an hour or so for your data to be loaded, indexed, and available.    </p><p>aggregator</p><p>gbif</p><p>data publishing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/gbif\/"},{value:"[user] Identification Keys",label:"<p> This page describes how to navigate identification keys built in to checklists in Symbiota portals. View our documentation about checklists, editing characters and character states, and editing character states applied to taxa.\n Structure of Identification Keys Identification keys are built from a list of characters that each have multiple states.\nCharacters are categories of traits that are shared across all members of a taxon (e.g., \u0026ldquo;average wing length\u0026rdquo; or \u0026ldquo;leaf phyllotaxy\u0026rdquo;).\nA state is the specific trait that is shared within the taxon (e.g., \u0026ldquo;3-15 mm\u0026rdquo; or \u0026ldquo;opposite\u0026rdquo;, for each of the characters above). At this time, states can only be categorical (i.e., you cannot enter a number value).\nSetting up identification keys in a portal requires the administrator to import or add character and state values (see ID Keys Administrator instructions here. Then, an identification keys editor can assign certain state values to certain taxa (see ID Keys Editor Instructions here).\nUsing Identification Keys Identification keys are not turned on and\/or configured in all portals and require extensive configuring in order to be used across many taxa. To access this feature, contact your portal administrator.\n Identification keys can be used within checklists to identify a taxon within the list that possesses certain character states. A user can click the character states that match those of their sample\/individual of interest, and the taxon\/taxa with the same character state(s) will be displayed on the checklist.\nNote that the taxa you will see on the left will only include those that were in the original checklist.\nRelated Resources   </p><p>checklist</p><p>keys</p><p>identification</p><p>character</p><p>state</p><p>ID</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/idkeys\/"},{value:"[coll_manager] Downloading a Copy of your Database",label:"<p> This page describes how to download a copy of your data, including occurrence records, determinations, images (as links only), and any other extensions enabled in your portal (e.g., attribute data). It is strongly recommended that curators or collection managers regularly download and internally archive a backup data file in case of emergency. Doing so is quick and easy.\n Review the Symbiota Support Hub\u0026rsquo;s Statement on Cybersecurity here.\n To download a copy of your specimen data from a Symbiota portal:\n Navigate to the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection) \u0026gt; Download Data Backup File (under General Maintenance Tasks) A new window will appear asking you to select the character set (ISO-8859-1 or UTF-8) that will be used in your downloaded dataset. Click the Perform Backup button. The resulting file will be a zipped Darwin Core Archive.  To access your backup data, unzip\/open the Darwin Core Archive folder. This folder will contain several files, including the following CSV (Comma Separated) files that can be opened as a spreadsheet in MS Excel, Google Sheets, etc:\n occurrences.csv = your specimen records identifications.csv = the specimen identifications associated with each specimen record measurementOrFact.csv = trait data associated with each specimen record multimedia.csv = links to media\/images associated with each specimen record  The archive also contains metadata about your collection and about the fields contained in each of the CSV files. For more information about the format and use of Darwin Core Archives, see the following resources:\n https:\/\/github.com\/gbif\/ipt\/wiki\/DwCAHowToGuide https:\/\/en.wikipedia.org\/wiki\/Darwin_Core_Archive  </p><p>download</p><p>backup</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/download\/"},{value:"[coll_manager] Duplicate Clustering",label:"<p> This page describes how to view and batch link duplicate specimens (specimens of the same taxon collected on the same day by the same person in the same place) using the Duplicate Clustering tool.\n Occurrences can be linked as duplicates individually during or after data entry using tools in the occurrence editor. See this page for more information about linking duplicates on an individual basis and this page for information about using the duplicate matching tool during data entry.\nOccurrences can also be batch-linked automatically by the Duplicate Clustering tool. This tool creates a temporary index of your occurrences\u0027 collection dates, collector numbers, and collector last names, then links any occurrences that share all three of these characteristics.\nBecause creating duplicate specimens is not universal among collection types, tools that facilitate batch duplicate matching are not available in all portals. Contact your portal administrator to activate this function, if necessary.\n To view or link duplicates, navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection) and click Duplicate Clustering.\n To view existing duplicates, click Specimen duplicate clusters To view duplicates with taxonomic identifications that do not match, click Specimen duplicate clusters with conflicted identifications. An example output of this tool is shown in the screenshot below. To batch link duplicates, click Batch link specimen duplicates. This will automatically run the batch linking script to create duplicate clusters.  When viewing clustered duplicates, you can view the record for any occurrence by clicking the catalog number in blue font.\n A video walkthrough of how duplicate clustering tools can be used to resolve conflicting identifications can be found here:\n  </p><p>duplicates</p><p>duplicate specimens</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/dup\/"},{value:"[user] Voucher Conflicts Tab",label:"<p>The Voucher Conflicts tab allows you to view a list of specimen vouchers for which their current identifications do not match how they were originally added as part of the checklist. This is generally due to recent annotations.\nTo view this tool, navigate to the Manage Linked Vouchers tool in your checklist. If any specimens were re-identified to another taxon since you added that voucher to your checklist, it will be displayed in the Conflict Table. From here, you can view the voucher specimens by clicking on the name of the collection in the Voucher Specimen column. If you click the value in the Checklist ID column, you will be able to access the editing pane for that taxon in your checklist.\nYou can also update your checklist from these new identifications by clicking the checkbox next to the voucher you would like to update, and then clicking the Link Vouchers to Corrected Identification button. If the \u0026ldquo;Remove taxa from checklist if all vouchers are removed\u0026rdquo; box is unchecked, any non-vouchered taxa in your checklist (resulting from updating the vouchers\u0027 identifications in your checklist) will be removed from your checklist as well.\n</p><p>checklist</p><p>voucher</p><p>identification</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/conflict\/"},{value:"[coll_manager] Deleting Records",label:"<p> This page describes how to delete records from your collection.\n Only users with Administrator permissions can delete a specimen record, and only the portal manager(s) or someone with backend access can delete more than one specimen record at a time. This is designed to protect data integrity, such as GUIDs and links to other tables in the database.\nDeleting a specimen record is only appropriate when that specimen no longer exists or the record was added erroneously (e.g., it was an exact duplicate of an existing record). You should not delete a record for the purpose of updating it or adding a new version of the record.\nTo delete a record:\n Navigate to the specimen record that you would like to delete and open the Occurrence Editor form for that record. (See this page for help navigating to specific records.) Open the Admin tab. Select the “Evaluate record for deletion” button to determine whether the record can be safely deleted. If an image is associated with the record, you will need to disassociate the image from the specimen record before it can be deleted (see the deleting\/remapping images page). Likewise, a warning will appear if the specimen record is linked to a checklist, which must be resolved before the specimen record can be deleted. If there are no warnings at this point, click the \u0026ldquo;Delete Occurrence\u0026rdquo; button to remove the record from your dataset.  To batch delete records, contact your portal manager.\n</p><p>delete</p><p>remove</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/delete\/"},{value:"[coll_manager] Editing Occurrences",label:"<p>General instructions for editing records can be found in the Editor Guide.\nCollection managers (i.e., users with administrator permissions) can additionally conduct batch edits and delete records.\n</p><p>editing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/edit\/"},{value:"[coll_manager] Editing Collection Metadata and Contact Info",label:"<p> This page describes how to change or add contact information or information about your collection (e.g., homepage, collection title, acronym, description). New collections will be asked to complete this form to facilitate the intial setup of your of collection profile; this information can be modified once the profile is established using the instructions below. If you opt to publish data to GBIF, the same metadata provided about your collection will also be shared to GBIF.\n Collections Metadata  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). In the Administration Control Panel, click Edit Metadata. In the first tab you can change:  Institution Code - The name (or acronym) in use by the institution having custody of the occurrence records. This field is required. For more details, see Darwin Core definition. Herbaria should also register this code with IndexHerbariorum. Collection Code - The name, acronym, or code identifying the collection or data set from which the record was derived. This field is optional. For more details, see Darwin Core definition. Collection Name - Title of your collection, e.g. \u0026ldquo;Arizona State University Vascular Plant Herbarium\u0026rdquo;. Description - A brief description of your collection and what it contains (2000 character maximum). Details can include information such as the collection\u0026rsquo;s taxonomic and geographic scope and strengths, as well as other major highlights (i.e. notable collectors, time intervals, and\/or field expeditions). Latitude\/Longitude - Submit coordinates using the globe icon. Category (if applicable) - e.g., bryophytes, lichens, fishes if more than one taxonomic group in a single portal Allow Public Edits - Checking \u0026ldquo;Allow Public Edits\u0026rdquo; will allow any user logged into the system to modify specimen records and resolve errors found within the collection. However, if the user does not have explicit authorization for the given collection, edits will not be applied until they are reviewed and approved by collection administrator. License - A legal document giving official permission to do something with the resource. This field can be limited to a set of values by modifying the portal\u0026rsquo;s central configuration file. For more details, see Darwin Core definition and the Creative Commons Licenses. Rights Holder - The organization or person managing or owning the rights of the resource. For more details, see Darwin Core definition. Access Rights - Information or a URL link to page with details explaining how one can use the data. See Darwin Core definition. Dataset Type - Preserved Specimens signify a collection type that contains physical samples that are available for inspection by researchers and taxonomic experts. Use Observations when the record is not based on a physical specimen. Personal Observation Management is a dataset where registered users can independently manage their own subset of records. Records entered into this dataset are explicitly linked to the user\u0026rsquo;s profile and can only be edited by them. This type of collection is typically used by field researchers to manage their collection data and print labels prior to depositing the physical material within a collection. Even though personal collections are represented by a physical sample, they are classified as \u0026ldquo;observations\u0026rdquo; until the physical material is publicly available within a collection. Management Type - Use \u0026ldquo;Snapshot\u0026rdquo; when there is a separate in-house database maintained in the collection and the dataset within the Symbiota portal is only a periodically updated snapshot of the central database. A \u0026ldquo;Live\u0026rdquo; dataset is when the data is managed directly within the portal and the central database is the portal data. Global Unique Identifier (GUID) Source - Occurrence Id is generally used for Snapshot datasets when a Global Unique Identifier (GUID) field is supplied by the source database (e.g. Specify database) and the GUID is mapped to the occurrenceIdfield. The use of the Occurrence Id as the GUID is not recommended for live datasets. Catalog Number can be used when the value within the catalog number field is globally unique. The Symbiota Generated GUID (UUID) option will trigger the Symbiota data portal to automatically generate UUID GUIDs for each record. This option is recommended for many for Live Datasets but not allowed for Snapshot collections that are managed in local management system. iDigBio\u0026rsquo;s GUID Guide can be found here. Publishing to GBIF - Activates GBIF publishing tools available within Darwin Core Archive Publishing menu option. Icon URL - Upload an icon image file or enter the URL of an image icon that represents the collection. If entering the URL of an image already located on a server, click on \u0026ldquo;Enter URL\u0026rdquo;. The URL path can be absolute or relative. The use of icons are optional. Sort Sequence - Leave this field empty if you want the collections to sort alphabetically (default) Collection ID - Global Unique Identifier for this collection (see dwc:collectionID): If your collection already has a previously assigned GUID, that identifier should be represented here. For physical specimens, the recommended best practice is to use an identifier from a collections registry such as the Global Registry of Biodiversity Repositories.    Collection Contacts  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). In the Administration Control Panel, click Contacts \u0026amp; Resources. In this tab you can change: In the Contacts \u0026amp; Resource tab, you can add links to other resources (e.g., homepages or lab pages), add or edit contact information, or add or edit the mailing address for your institution. Click a pencil icon to edit or an X icon to delete existing links\/resources. Listing at least two contacts is strongly recommended, and that, when available, one of these contacts includes a generic departmental email (e.g. yourherbarium@youruniversity.edu).          Contact information publicly displays at the top of your collection\u0026rsquo;s profile page.    </p><p>collection name</p><p>metadata</p><p>contact info</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/metadata\/"},{value:"[editor] Editing Records",label:"<p> This page provides instructions on how to search for and edit records.\n  To batch edit records (only available to collection administrators), visit this page. For an overview of fields in the occurrence editor form, visit this page. For an explanation of the record editor tabs, visit this page.  To edit individual records, select \u0026ldquo;Edit Existing Occurrence Records\u0026rdquo; from the Data Editor Control Panel (accessed via My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). From this page, you can search for specific records by entering search terms into the Record Search Form. You can search by multiple search terms using this form.\nTo search according to fields not explicitly stated in the Record Search Form, select the field from the dropdown menu after Custom Field 1. You can include up to 4 Custom fields in your search. The second dropdown lists after the Custom Fields will allow you to conduct more specific searches, such as for ranges or fields that are null. The options include:\n EQUALS: the field contains only the provided text NOT EQUALS: the field does not match the provided text exactly (does not only contain the provided text) STARTS WITH: the field starts with the provided text CONTAINS: the field contains the provided text anywhere in the field DOESN\u0026rsquo;T CONTAIN: the field does not contain the provided text anywhere in the field GREATER THAN LESS THAN IS NULL: the entire field contains no value IS NOT NULL: the field contains any value  To conduct a search, click the Display Editor (to view one record at a time) or Display Table (to view the first 1000 records at a time) buttons.\nWhen searching for the characters \u0026ldquo;_\u0026rdquo; or \u0026ldquo;%\u0026rdquo; in your record fields, you must precede this character with the backwards slash \u0026ldquo;escape\u0026rdquo; character (). E.g., when searching for the value \u0026ldquo;_1\u0026rdquo;, you should enter \u0026ldquo;_1\u0026rdquo;.\n To search by specimens entered by you (the current user), click the CU button.\nTo sort your search results, select a field from the dropdown menu after Sort By (bottom right of Record Search Form), then select whether you wish to sort in ascending or descending order.\nTo view a specific record from the Table Display, click the Symbiota ID number or the Open in New Window icon in the leftmost SymbiotaID column.\nTo re-open the record search form after you have conducted a search, click the magnifying glass icon to the right of the name of your collection at the top of the window.\n</p><p>edit</p><p>change</p><p>record search form</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/"},{value:"[coll_manager] Publishing Data to iDigBio",label:"<p> This page describes how to publish the data from your collection to the U.S. national aggregator iDigBio.\n  Contact iDigBio\u0026rsquo;s Biodiversity Informatics Coordinator (cchapman@floridamuseum.ufl.edu) requesting that your collection gets added to the iDigBio ingestion queue. Include a link to your collection profile page (e.g. http:\/\/swbiodiversity.org\/seinet\/collections\/misc\/collprofiles.php?collid=1) so that they have the necessary information for your collection and access to your data. For more information, see the iDigBio Data Ingestion Guide. Log in to your Administrator Control Panel in your Symbiota portal (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Edit Metadata and review the information and contacts associated with your collection. In addition to your institution\/collection codes and contact information, make sure to review data usage license and GUID (see below). Contact your portal manager if you are unclear about any of the available fields.   Data Usage License: This field determines the copyright guidelines that users must follow when using your data. Many portals have predefined options that follow the Creative Commons licenses, though that can vary by portal instance. See the Creative Commons website for more information about licenses. GUID source: This designates the field that will serve as the persistent globally unique identifier (GUID) for specimen records published from the portal. (More information about the importance of GUIDs can be found in this article. You will see the following options in your portal:  Occurrence ID: This option is for collections that create their own GUID using another protocol or manage their data in an external system and map the incoming GUID to the occurrenceID field at time of import. If you manage your central data within an update-to-date Specify database, you should use this option because the import data should automatically contain a Specify-generated UUID. Darwin Core definition: http:\/\/rs.tdwg.org\/dwc\/terms\/index.htm#occurrenceID. Catalog Number: You can use this option if your catalog number is configured as a globally unique identifier. See the Darwin Core occurrenceID definition above. Symbiota-generated UUID: If you are managing your data live within the Symbiota instance (e.g. the portal is your central database), this option is the easiest and most reliable method for assigning robust GUIDs to your specimen records. Even if your catalog number is configured as a unique identifier (e.g. institutionCode:collectionCode:catalogNumber format type), this is a more robust option when managing data within the portal. Do not select this option if you use the Symbiota portal to publish a snapshot of your data that is managed in another database system (e.g. Specify).    Return to your Administrator Control Panel and click “Darwin Core Archive Publishing”. Click the Create Darwin Core Archive button to build a new archive. iDigBio will periodically monitor the RSS feed that is associated with the DwC-Archive library within the portal. You should regularly refresh this archive to ensure the data within iDigBio remains current. Contact your portal manager to request automatic updates of your archive on a regular schedule.  </p><p>data publishing</p><p>idigbio</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/idigbio\/"},{value:"[coll_manager] Georeferencing",label:"<p>General instructions and resources for georeferencing can be found in the Editor Guide.\nCollection managers (i.e., users with administrator permissions) can additionally push data to the GEOLocate Collaborative Georeference client.\n</p><p>georeferencing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/georeference\/"},{value:"[editor] GEOLocate",label:"<p>GEOLocate is a software for georeferencing natural history data that was developed in 2005 and has continued to be improved and expanded. A GEOLocate module is available for Symbiota instances and can be accessed from the occurrence editor page or the Batch Georeferencing Tool. Below are some resources about the GEOLocate program:\n GEOLocate home page Georeferencing protocols from the California Phenology Network Georeferencing training course from the California Phenology Network GEOLocate 101 from the Milwaukee Public Museum (Note that MPM uses the content management system EMu, so some of the content in this guide is not relevant to Symbiota) iDigBio GEOLocate webinar recordings  </p><p>georeferencing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/georeference\/geolocate\/"},{value:"[editor] Georeferencing",label:"<p>Georeferencing is the process of assigning coordinates to generally textual descriptions of occurrence localities (e.g., the location where a specimen was collected). Some helpful resource about georeferencing can be found on the iDigBio website. Symbiota includes several tools for facilitating this process:\n Google Maps tool: view a Google Map, place a point on the occurrence location, and assign an error radius GEOLocate tool: view a map (many layers available, including USGS Topo Maps, Open Street Maps, Bing Maps, and Google Maps), view auto-generated potential georeference points, edit point location and error radius Coordinate cloning tool: search for records with similar or identical locality information and import any coordinates associated with that locality Duplicate specimen\/locality data import: search for records with identical or similar collector information or locality information and import any coordinates associated with the duplicate\u0026rsquo;s locality Batch georeferencing: use the Google Maps, GEOLocate, and coordinate cloning tool in batch for a collection of specimens with similar\/identical locality descriptions Integration with GEOLocate Collaborative Georeferencing (CoGe): export records to GEOLocate CoGe to be collaboratively georeferenced using the GEOLocate web client  </p><p>georeferencing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/georeference\/"},{value:"[user] Providing Feedback",label:"<p> Providing feedback on specimen records can improve the quality of data available in a given Symbiota portal.\n There are three main ways to provide feedback:\n Comment on a Record Contact a Curator Edit a Record (only available for some collections)  </p><p>crowdsourcing</p><p>annotations</p><p>comments</p><p>public feedback</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/public_feedback\/"},{value:"[editor] GEOLocate CoGe",label:"<p>The GEOLocate Collaborative Georeference Tool (CoGe) can be activated and integrated in a Symbiota portal to engage multiple users in a central georeferencing platform outside of the portal. More information about this platform can be found here: GeoLocate CoGe page.\nA training course for georeferencing using GEOLocate CoGe can be found here: GEOLocate CoGe Training Course, and a written protocol can be found here: GEOLocate CoGe Protocol.\nFor instructions on how to push specimens from your collection to CoGe, visit the collection manager CoGe page.\n</p><p>georeferencing</p><p>collaborative georeferencing</p><p>GEOLocate</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/georeference\/coge\/"},{value:"[coll_manager] Redacting \/ Obscuring Data",label:"<p> This page explains how data redaction functions in a Symbiota portal.\n Collection managers may wish to redact locality data for certain occurrences, for example, of rare or endangered species or for locations on private property. Locality data in Symbiota portals may be redacted in one of three ways: (a) individually (per occurrence), (b) globally (per taxon), or (c) by state.\nRedacting locality data in Symbiota portals is currently binary: an occurrence can have its locality redacted (Locality Security = 1, checked) or not (Locality Security = 0, unchecked). When the Locality Security box is checked on the Occurence Editor (or Locality Security is uploaded as 1) for a given occurrence, a user who does not have Rare Species Reader or Editor permissions will not be able to view that occurrence\u0026rsquo;s:\n Locality below the level of county Coordinates (if provided) Image  How redacting data affects various users Specimen occurrences with locality security applied affects portal users as follows:\n Administrators, Editors: all locality details are visible and can be edited on a per-collection basis Rare Species Readers: locality details are visible and can be downloaded, but not edited, on a per-collection basis All other users: no locality details are visible below county, if provided. On a record\u0026rsquo;s public view, any locality-related fields that contain obscured data will be listed in Information Withheld.  The complete list of fields that are redacted when locality security is applied includes: recordnumber, eventdate, verbatimeventdate, locality, locationid, decimallatitude, decimallongitude, verbatimcoordinates, locationremarks, georeferenceremarks, geodeticdatum, minimumelevationinmeters, maximumelevationinmeters, verbatimelevation, habitat, associatedtaxa\n Users with Administrator permissions can grant or remove access to their collections data through the Administation Control Panel. Learn how here.\n What taxa are protected in my portal? The master list of Protected Species in a given portal can be viewed by all portal users, including those who are not logged into the portal.\n        To view all protected taxa in a portal, navigate to Sitemap \u0026gt; Collections \u0026gt; Protected Species. This example comes from SEINet.    How to redact data Individually redacting locality data for certain occurrences The locality data can be redacted for individual occurrences by checking the Security box in the Occurrence Editor.\nBatch redacting locality data for certain occurrences If you wish to batch redact data, you can download a CSV file of all the specimen records you wish to redact using the Exporter tool, then add a column called \u0026ldquo;LocalitySecurity\u0026rdquo;. Enter \u0026ldquo;1\u0026rdquo; in this column for all specimens for which you wish to redact data (conversely, you would enter \u0026ldquo;0\u0026rdquo; to keep the data publicly visible, or leave this field blank). Use the Skeletal File Uploader to upload this spreadsheet into the portal, mapping the new column to localitySecurity. You may need to ask your portal manager to clear out any existing values in this field before importing via the Skeletal File Uploader.\nGlobally redacting locality data for certain taxa Locality data and images can be redacted for all occurrences by a specific taxon by someone with Super Administrator or Taxon Editor user permissions. To do this, find the species in the Taxonomic Tree Viewer or Taxonomy Explorer and open the editor (either by clicking on the taxon name or clicking the pencil next to the name). Change Locality Security from \u0026ldquo;show all locality data\u0026rdquo; to \u0026ldquo;hide locality data\u0026rdquo;.\nThis will hide locality data for all occurrences of that taxon throughout the portal, not just for your collection. Collections can opt out of this option by individually unchecking the Security box within the Occurrence Editor for individual specimen records or by contacting their portal manager for batch changes.\nRedacting data by state Finally, locality data and images can be redacted for occurrences of a given taxon that were collected in a certain state by managing a \u0026ldquo;Rare, threatened, protected species list\u0026rdquo;. User accounts with Rare Species Administrator permissions can create a species list specifically for managing sensitive species and then assigning editing rights to one of several appropriate users for populating and managing the state list. The addition of a species to the list will automatically protect locality details of all specimens collected within the designated state.\nThis will hide locality data for all occurrences of that taxon in the given state throughout the portal, not just for your collection. Collections can opt out of this option by individually unchecking the Security box within the Occurrence Editor for individual specimen records or by contacting their portal manager for batch changes.\nWill my redacted data be visible if published to GBIF? By default, no. Keep the box, \u0026ldquo;Redact Sensitive Localities\u0026rdquo;, in the Darwin Core Archive Publisher checked so that redacted data will remain obscured when a Darwin Core Archive file is sent to GBIF. To locate the Darwin Core Archive Publisher, navigate to Administration Control Panel \u0026gt; Darwin Core Archive Publishing. Scroll down to the \u0026ldquo;Create\/Refresh Darwin Core Archive\u0026rdquo; box.\nKeep in mind that the Security field must contain a value of \u0026ldquo;1\u0026rdquo; for your data to be successfully redacted within the portal, as well as during data publishing; this applies to both live-managed and snapshot collections. If Security is blank, your data will remain visible and can be published.\nInstructions for creating state-based redacted species lists   Create a new empty rare species checklist.\n Click “My Profile”, select the Species Checklists tab, and click the green plus sign. Change the Checklist Type to “Rare, threatened, protected species list”. If you don’t see the Checklist Type field located below the author field, then you do not have the necessary Rare Species Administrator permissions to create this type of checklist. In this case, you can continue creating the checklist (as normal) and ask a portal manager to change the checklist type at a later date. Enter the state name in the locality field. Do not abbreviate or add any other text other than the state name. The checklist can be private or public and made available to the general public.    Add one to several checklist editors to the checklist.\n From the new checklists, click on the checklist administration editing pencil located towards the user right of the page Checklist editors do not need Rare Species Administrator or any other special editing rights to manage the list    Checklist editors add species needing protection using the normal checklist editing tools.\n See checklist tutorials for help creating and managing checklists.    How users can request access to redacted data Individuals who require access to redacted data for legitimate reasons are encouraged to directly reach out to the contact(s) listed on collections profiles to acquire data access. However, if the request is complex and requires contacting numerous collections, individuals can contact the Symbiota Support Hub for assistance in reaching out to the relevant collections. Please keep your collection\u0026rsquo;s contact information up to date so that portal users and the Support Hub can contact you about these requests. It is also recommended that you add hub@symbiota.org to your own contacts so these messages are not blocked by an institutional firewall or routed to spam.\nRelated Resources   </p><p>rare species</p><p>data protection</p><p>redaction</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/redaction\/"},{value:"[user] Redacted and Protected Data",label:"<p>Some collections redact location information data (latitude\/longitude and locality description) for certain specimens from the general public. Individual specimens may have redacted location data because (a) the location is confidential (e.g., private or protected property), or (b) the taxon of the species is rare, threatened, endangered, or otherwise imperiled. Which taxa are selected for data redaction depends on the data portal, since some taxa are rare in certain places, but not in others.\nUsers may gain access to redacted location information by being granted Rare Species Reader (RSR) permissions, or through a one-time data export. To obtain these permissions, contact the curator or manager of the collection and provide information about yourself and why you would like access to their data.\nNot sure how to contact the manager of a collection? Visit the Contacting a Collection page.\n </p><p>rare species</p><p>redacted data</p><p>protected data</p><p>protected species</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/redaction\/"},{value:"[coll_manager] Images",label:"<p> This page describes the types of images that are available through Symbiota portals.\n Images may be associated with records in Symbiota portals by storing links to those images in the database tables. If a Symbiota portal is configured to accept images (i.e., image storage at the institution that hosts the portal is allowed), those image links may be generated by the portal. Storage of a large number of images on servers associated with a Symbiota portal (e.g., at Arizona State University) may require an image-hosting agreement and\/or incur image-hosting costs. Check with the portal administrator for more information about your portal\u0026rsquo;s image hosting allowances. Otherwise, tools are available to link external URLs to occurrences by matching on catalog numbers.\nThere are three categories of images that can be linked to a Symbiota portal:\n Images of physical specimens should be linked to their respective specimen records. Specimen images are available both from the Occurrence Details page and the Taxon Profile page for the taxon associated with the most recent determination. If the specimen identification is annotated, the identification of the image will be automatically re-mapped to that taxon. Specimen images can be added through the Occurrence Editor or in batch via multiple processes. See Uploading Images for more details.  Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed through Taxon Profile Pages, which have general information such as descriptions, distribution maps, synonyms, and common names. Field images are uploaded and managed through the Taxon Profile pages by users with Taxon Profile editing permissions. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see \u0022Field Images with Location\u0022 tab). See Uploading Images for more details.  Field images with specific locality information can adequately serve as vouchers for field observations of many birds, mammals, and easy-to-identify plants. In some instances, they may actually be preferable, for instance if the organism is rare or the observer does not have the appropriate collecting permit. These images are similar to georeferenced specimen records in that they are searchable occurrence records that place a dot on the species distribution maps. There are, however, some taxonomic groups that typically require microscopic or chemical analysis for accurate identification (e.g. lichens, bryophytes, fungi). For such organisms, field images will have limited value. For this reason, some data portals might not allow the submission of observations without a physical specimen. Image Vouchers are displayed on both the Occurrence Details and the Taxon Profile pages. Image-supported observations can be uploaded using the Observation Submission page. Up to three images can be submitted for a single observation. Data fields required for an image to qualify as a voucher include: observer name, observation date, country, state, locality description, latitude, longitude, and at least one image. Contact your data portal administrator for information on whether observations are allowed and the necessary protocols for submitting an observation. Some portals require the review of a tutorial before one is authorized to submit an observation. See Uploading Images for more details.    Related Video Tutorials      </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/images\/"},{value:"[user] Research Use Cases",label:"<p> This page contains user-contributed content related to research uses of Symbiota-mediated data. If you would like to contribute an example, please contact help@symbiota.org.\n It is incumbent upon the data user to appropriately cite the applicable data portals and data providers in publications, reports, theses, disserations, and similar works.\n Checklists Making Checklists in Symbiota Portals  Reference: Bell, M., E., \u0026amp; Landrum, L. R. (2021). Making Checklists with the SEINet Database\/Symbiota Portals. Canotia, 17, 1–24. PDF  Spatial Data Phytogeographic Analysis Using Georeferenced Specimen Data  NEARBY Software: serv.biokic.asu.edu\/dlafferty\/Nearby\/index.php Citation: Lafferty, D. L., \u0026amp; Landrum, L. R. (2021). NEARBY, a Computer Program for Phytogeographic Analysis Using Georeferenced Specimen Data. Canotia, 17, 25–45. PDF       </p><p>research</p><p>use case</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/research\/"},{value:"[editor] Images",label:"<p> This page describes the types of images that are available through Symbiota portals.\n There are three catefories of images that can be linked to a Symbiota portal:\n Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed through Taxon Profile Pages, which have general information such as descriptions, distribution maps, synonyms, and common names. Field images are uploaded and managed through the Taxon Profile pages by users with Taxon Profile editing permissions. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see \u0022Field Images with Location\u0022 tab). See Uploading Images for more details.  Field images with specific locality information can adequately serve as vouchers for field observations of many birds, mammals, and easy-to-identify plants. In some instances, they may actually be preferable, for instance if the organism is rare or the observer does not have the appropriate collecting permit. These images are similar to georeferenced specimen records in that they are searchable occurrence records that place a dot on the species distribution maps. There are, however, some taxonomic groups that typically require microscopic or chemical analysis for accurate identification (e.g. lichens, bryophytes, fungi). For such organisms, field images will have limited value. For this reason, some data portals might not allow the submission of observations without a physical specimen. Image Vouchers are displayed on both the Occurrence Details and the Taxon Profile pages. Image-supported observations can be uploaded using the Observation Submission page. Up to three images can be submitted for a single observation. Data fields required for an image to qualify as a voucher include: observer name, observation date, country, state, locality description, latitude, longitude, and at least one image. Contact your data portal administrator for information on whether observations are allowed and the necessary protocols for submitting an observation. Some portals require the review of a tutorial before one is authorized to submit an observation. See Uploading Images for more details.  Images of physical specimens should be linked to their respective specimen records. Specimen images are available both from the Occurrence Details page and the Taxon Profile page for the taxon associated with the most recent determination. If the specimen identification is annotated, the identification of the image will be automatically re-mapped to that taxon. Specimen images can be added through the Occurrence Editor or in batch via multiple processes. See Uploading Images for more details.      </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/"},{value:"[user] Searching for Records",label:"<p>Symbiota portals serve data from specimens and observations according to regional and taxonomic themes. To search and view these data, you can use one of the tools described below.\nTable of Contents  Basic Search (pre-Symbiota 3.1) Basic Search (Symbiota 3.1 and beyond) Map Search Quick Search  Basic Search  From the home page of the portal, click Search Collections from the top or side menu.  Pre-Symbiota 3.1 The following steps 2-5 apply to Symbiota portals that have codebases older than version 3.1. If you are using Symbiota 3.1, skip to Symbiota 3.1 and beyond\n If prompted on the next page, select or deselect collections from the provided list depending on which collections you would like to search.  If you do not wish to search specific collections, leave all the collections checked. There may be three tabs at the top of the page, \u0026ldquo;Specimens \u0026amp; Observations\u0026rdquo;, \u0026ldquo;Specimens\u0026rdquo;, and \u0026ldquo;Observations\u0026rdquo;. Specimens are physical collections of an organism, while observations are human observations or photos of organisms that are not supported by physical specimens. Use these tabs to select between these types of occurrences, if desired.    Notice that collections may be grouped together into categories. These categories may be minimized or maximized by clicking the small grey plus or minus box, respectively, to the bottom left of the name of the group.\n Click the Search button on the far right. On the next page, you can enter search criteria to find records of interest.  The criteria you can use will vary depending on the portal, but generally include the following categories: 1) Taxonomic Criteria, 2) Locality Criteria, 3) Latitude and Longitude, 4) Collector Criteria, 5) Specimen Criteria, and 6) Trait Criteria (if enabled). Any number of criteria can be entered and search at the same time. You can search for multiple values in a single field by separating the values by semicolons. For example, if you want to search by both Kern and Inyo counties, you should enter \u0026ldquo;Kern;Inyo\u0026rdquo; in the county field. Latitude and Longitude: To define a latitude\/longitude bounding box, polygon, or point with radius in which to search, enter the values in the provided fields or click the Mapping Aid icon to create the box, shape, or point radius in the Leaflet mapping interface. Specimen Criteria: If enabled for your portal, this search category may contain a dropdown list to query materialSampleType. Further documentation on the Material Sample module can be found here.   Click List Display to conduct the search and view results as a list, or click Table Display to conduct the search and view results in a table.  List Display\nTable Display\nTo return to the Search Criteria page and refine your search, click Search Criteria from the navigation menu at the top of the page.\n To view the results in a table or to sort the search results, click the Table Display button. In the Search Results box at the top of the page, select the field you would like to sort by, a second field you would like to sort by (if applicable), then whether you would like to sort results in ascending or descending order. Then click Sort.\n Symbiota 3.1 and beyond The following steps 2-3 apply to Symbiota portals that have codebases that are version 3.1 or higher. If you are using a version prior to Symbiota 3.1, return to to Pre-Symbiota 3.1 instructions.\n On the next page, you can enter search criteria to find records of interest. Click the Expand All Sections button if you would like to see all of the possible criteria you can search on.   The criteria you can use will vary depending on the portal, but generally include the following categories: 1) Taxonomic Criteria, 2) Locality Criteria, 3) Latitude and Longitude, 4) Collector Criteria, 5) Specimen Criteria, and 6) Trait Criteria (if enabled). Any number of criteria can be entered and search at the same time. The criteria you are searching by will show up as \u0026ldquo;chips\u0026rdquo; on the right side of the screen under \u0026ldquo;Criteria.\u0026rdquo; Click the X on any of these chips to remove them from your search criteria. You can search for multiple values in a single field by separating the values by semicolons. For example, if you want to search by both Kern and Inyo counties, you should enter \u0026ldquo;Kern;Inyo\u0026rdquo; in the county field. Latitude and Longitude: To define a latitude\/longitude bounding box, polygon, or point with radius in which to search, enter the values in the provided fields or click the appropriate button at the top of the Latitude \u0026amp; Longitude search criteria to create the box, shape, or point radius in the mapping interface. Sample Properties: Here you can search by catalog number or limit your search criteria to only include records that are types (i.e., have a value in the TypeStatus field), records that have media, records that have genetic data, records that are georeferenced, or records that have material samples (if enabled in your portal). You can also opt to include cultivated or captive records in this criterion category. Trait Criteria: Here you can limit your search to include only records with certain values of trait criteria. Note that the search will only be able to provide you with records that have been scored for those particular traits, and the absence of a certain trait value applied to a specimen does not necessarily indicate that a trait value does not apply to a record. Furthermore, the record is an \u0026ldquo;OR\u0026rdquo; search. Selecting multiple trait values will return all records with at least one of those traits.  Click the Search button on the far right to conduct your search.  To return to the Search Criteria page and refine your search, click the \u0026ldquo;back\u0026rdquo; button in your browser.\n To view the results in a table or to sort the search results, click the Table Display button. In the Search Results box at the top of the page, select the field you would like to sort by, a second field you would like to sort by (if applicable), then whether you would like to sort results in ascending or descending order. Then click Sort. To switch back to list view, click the List Display button.  Map Search Depending on the portal, the Map Search function may be under the Search Collections menu item, or listed as a separate menu item on the homepage.\nClick the Open Search Panel button in the top left corner. Once open, you can enter the same types of criteria into this search panel as were available in the regular search (described above). Then click the Search button.\nTo select specific collections from which you would like to search, select the Collections tab in the search panel, then check or uncheck boxes next to collections as desired.\nFurther customizations can be made in the Map Options tab of the search panel including grid size and min. cluster size. These will affect how many specimens will be clustered together on the map. You can also turn off clustering in this tab.\nOnce you have conducted a search, you can view a list of specimens by clicking the Open Search Panel button and viewing the Records and Taxa menu item. You can also download the specimen records, download a KML file of the specimen records, or generate a shareable link to these search results by clicking on the respective buttons on this page.\nQuick Search Using Quick Search forms, you can search for multiple taxa (scientificName values) or, where applicable, Catalog Number values by listing your search criteria in a semi-colon delimited list, e.g. \u0026ldquo;Rosa abietorum;Rosa alba\u0026rdquo; or \u0026ldquo;100;1000\u0026rdquo;.\n By portal Some Symbiota portals feature a portal-wide taxon quick search form on their homepage. Use this form to search for all specimen records in the portal that are linked to the portal\u0026rsquo;s central taxonomic thesaurus by typing directly into the form and then selecting the \u0026ldquo;Search\u0026rdquo; button. If you begin typing a scientific name in this box and a dropdown list does not appear (shown below), then the taxon you are searching for does not exist in the portal\u0026rsquo;s taxonomic thesaurus. By collection Similar to the portal-wide quick search forms described above, some Symbiota portals feature quick search forms on Collection Profiles to facilitate searching for records limited to individual datasets. Currently, these forms can be used to search within a given Collection Profile by Catalog Number or Scientific Name (Taxon).\n</p><p>search</p><p>specimens</p><p>observations</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/search\/"},{value:"[editor] Label Customization and Printing",label:"<p> This page describes how to access and use the label printing functions in a Symbiota portal.\n For instructions on printing specimen packets, see this page\n   Printing Specimen Labels To print specimen labels, navigate to the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection) and click “Print Labels\/Annotations.” Search for the desired specimens by entering search terms in one or many of the available fields in the “Define Specimen Recordset” box and clicking “Filter Specimen Records.” Select all the specimens for which you want to print labels by checking or unchecking the boxes in the leftmost column of the table. Use the “Select\/Deselect all Specimens” box to check or uncheck all the boxes as necessary.\nIn the Label Printing box, select the name of the “Label Profile” that you wish to apply to your printed labels. The label profile will specify which fields are printed and the aesthetics of the label. Default (simple) profiles can be found in the “Portal defined profiles” list, and you can select your own saved profile from the “User defined profiles” list. Instructions for defining a label profile are described in the next section. After selecting a profile, you can then enter a desired heading (e.g., “Robert F. Hoover Herbarium (OBI)”, heading mid-section (e.g., state or family), heading suffix (any additional text information), and label footer (e.g., the name of the project for which the specimen was collected) for all labels to be printed. You can also select from a number of other options in the “Label Printing” box including whether you want to print the specimen barcode.\nOnce you have selected your desired options, you can print the labels directly from your browser, export the specimen data to a CSV file (similar to an Excel file, this format can be used with mail merge to produce labels), or export the labels to a Word document (DOCX file) by clicking the appropriate button. Note that currently, Word (DOCX) output only generates the old static label format (the “label profile” settings will not be applied). If you click “Print in Browser”, you can manually edit the label content by clicking the Edit Labels Content button in the top right corner.\nCustomize Label Format To create a new label profile that will apply custom formatting to your printed labels, click the pencil icon to the right of Label Profiles (highlighted below).\nThen click the green plus sign in the User Profiles box. Enter the title of the user profile and any desired header and footer information. Select the desired options from the Options box.\nTo further customize the label, click “visual interface” to the right of the “JSON” header. The resulting page will allow you to tailor your label with more detail (see next screenshot).\nThe format of your label is represented in the grey “Label Content Area” box. This is where you will build the label format. To add fields to your label, drag and drop field names from the left hand list of buttons into the “Label Content Area”. As you do this, a preview of the label format will appear in the Label Preview box. Note that you can add as many lines as desired in the Label Content Area by clicking the Add Line button. By clicking on one of the fields that you have added to the Label Content Area, you can also add a prefix or suffix, change the font format, change the size, change the font, and move the position of the field by altering these elements under the Field Options section (highlighted in screenshot below). You can also change the options for an entire line by clicking the line in the Label Content Area (it will turn green) and then editing the options under the Line Options section.\n</p><p>labels</p><p>printing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/label\/"},{value:"[editor] Linking Records\/Resources",label:"<p> This page describes how to link single associated occurrences, checklists, duplicates, or genetic resources with occurrence records. For batch adding linked resources, see this page. Note that you must be a collection administrator to use the batch upload tool.\n To link an occurrence with an external (or internal) resource or record, navigate to that record\u0026rsquo;s occurrence editor page (see this page for instructions) and click the Linked Resources tab.\nLinking Associations (External\/Internal Resources or Occurrences) In the Associated Occurrences box of the Linked Resources tab, you can link an occurrence with a(n) (1) external resource (not another specimen or observation) or website (\u0026quot;Non-Occurrence Resource Link\u0026quot;), (2) Internal Occurrence (\u0026quot;Occurrence - Internal (this portal)\u0026quot;), (3) occurrence in another portal or database (\u0026quot;Occurrence - External Link\u0026quot;), and\/or (4) observational (non-vouchered) occurrence of a specific taxon (\u0026quot;Taxon Observation\u0026quot;).\nLinking to an external (non-occurrence) resource Non-occurrence Resource: a URL to an external resource that provides information or extended data relating to the occurrence, but is itself not an occurrence. Examples include field notes, a compiled dataset, etc.\n Click the plus sign icon in the Associated Occurrences box to add a new link. In the Association Type field, select \u0026ldquo;Non-Occurrence Resource Link\u0026rdquo; from the dropdown menu. In the Relationship field, select the term that defines the relationship between your occurrence and the link that you are adding to that occurrence. The term you select refers to the occurrence as the \u0026ldquo;subject\u0026rdquo; and the linked resource as the \u0026ldquo;object\u0026rdquo;. For example, if you select the relationship \u0026ldquo;siblingOf\u0026rdquo;, the inferred relationship is \u0026ldquo;My occurrence is the sibling of this external link.\u0026rdquo; (Optional) In the Relationship Subtype field, select the subtype of relationship between your occurrence and the link that you are adding. In the Resource URL field, enter the URL\/link to the external resource that you would like to link to your occurrence. (Optional) In the Additional Identifier of Object field, enter any unique identifier that belongs to the link you are adding to your occurrence. Click the Create Association button.  Linking to a record within the same portal Occurrence - Internal (this portal): a link to an occurrence (specimen\/observation) that exists in the same portal as the occurrence you are linking to; when creating associations within a portal, the portal will automatically update the corresponding occurrence with the reciprocal relationship\n Click the plus sign icon in the Associated Occurrences box to add a new link. In the Association Type field, select \u0026ldquo;Occurrence - Internal (this portal)\u0026rdquo; from the dropdown menu. In the Relationship field, select the term that defines the relationship between your occurrence and the link that you are adding to that occurrence. The term you select refers to the occurrence as the \u0026ldquo;subject\u0026rdquo; and the linked resource as the \u0026ldquo;object\u0026rdquo;. For example, if you select the relationship \u0026ldquo;siblingOf\u0026rdquo;, the inferred relationship is \u0026ldquo;My occurrence is the sibling of this external link.\u0026rdquo; (Optional) In the Relationship Subtype field, select the subtype of relationship between your occurrence and the link that you are adding. (Optional) In the Basis of Record field, select the basis of the evidence of a relationship between your occurrence and the link that you are adding. (Optional) In the Location on host field, enter the location of the occurrence on the host occurrence (e.g., \u0026ldquo;mid-gut\u0026rdquo;). (Optional) In the Notes field, enter any notes associated with the relationship. In the Identifier field, enter the catalog number or Symbiota number (\u0026ldquo;occid\u0026rdquo;) of the occurrence that exists in this same portal that you you would like to link to this occurrence, then select either Catalog Numbers or Occurrence PK (occid) from the Search Target dropdown menu, depending on what you entered. In the Search Collections dropdown menu, select which collection you would like to search through to find the occurrence you want to link to this occurrence, or leave at \u0026ldquo;All Collections\u0026rdquo; to search all collections in this portal. Click the Search button. In the Occurrence Matches Available to Link box, click the radio button to the left of the occurrence to which you would like to link your current occurrence. If there is a scientific name in that occurrence, it will automatically populate the Verbatim Scientific Name field. Click the Create Association button.  You can click the underlined portion of the occurrence search results to view the occurrence before linking it\n Linking to a record in a different portal\/database Occurrence - External Link: a link to an occurrence (specimen\/observation) that is available in another data portal \/ database (e.g., another Symbiota portal, GBIF, Arctos, etc.).\n Complete the same steps 1-7 from \u0026ldquo;Occurrence - Internal\u0026rdquo;, as applicable, but select \u0026ldquo;Occurrence - External Link\u0026rdquo; from the Association Type dropdown menu. In the Resource URL field, enter the URL\/link to the external resource that you would like to link to your occurrence. For example, here is a URL for a records from the Bryophyte Portal that could be entered: https:\/\/bryophyteportal.org\/portal\/collections\/individual\/index.php?occid=4595185. (Optional) In the Additional Identifier of Object field, enter any unique identifier that belongs to the occurrence object that you are linking to your occurrence subject. (Optional) In the Verbatim Scientific Name field, enter the name of the taxon represented by your external resource that you would like to link to your occurrence. Click the Create Association button.  Linking to an observation that lacks a record (\u0026ldquo;Taxon Observation\u0026rdquo;) Taxon Observation: the assertion of a taxon being associated with the occurrence you are linking to. This may be, for example, the host taxon of the occurrence, a parasite, a taxon sharing the same habitat, etc.\n Complete the same steps 1-7 from \u0026ldquo;Occurrence - Internal\u0026rdquo;, as applicable, but select \u0026ldquo;Taxon Observation\u0026rdquo; from the Association Type dropdown menu. In the Verbatim Scientific Name field, enter the name of the taxon that you would like to link to your occurrence. Click the Create Association button.  Linking to a Checklist  In the Checklist Voucher Linkages box of the Linked Resources tab, select the checklist to which you would like to link you occurrence from the dropdown menu. Note that you will only see checklists for which you have editor or administrator permissions.  To batch link vouchers to a checklist, see the Adding Vouchers to Checklist page.\n Linking to a Duplicate Specimen There are several ways to link an occurrence to duplicate specimens. You can:\n In the Linked Resource tab of the occurrence editor, click the Search for Records to Link button in the Specimen Duplicates box. This will open a search window that you can use to identify and link duplicate specimens. You can search according to collector name, collector number, date, catalog number, or occid (SymbiotaID) using this tool. In the main Occurrence Data tab of the occurrence editor, click the Duplicates button. This will search the portal for occurrences with the same collector last name, number, and date. If a potential identical duplicate is identified, you can check the box next to Link as Duplicate, then click Transfer All Fields or Transfer to Empty Fields Only to initiate the link. This will also transfer any data from that duplicate into your specimen record. Newly transferred data will be highlighted in blue. Batch link records to their duplicates using the Duplicate Clustering tool.  Linking to a Genetic Resource\/Sequence   In the Genetic Resources box of the Linked Resources tab, enter information about the genetic sequence associated with your occurrence in the provided fields. Be sure to provide a URL to the sequence. Here is an example of an acceptable URL from GenBank: https:\/\/www.ncbi.nlm.nih.gov\/nuccore\/BV165924.1. In addition to adding these links one-by-one through the Linked Resources tab, genetic data can also be associated with specimen records in batch by mapping these URLs to associatedSequences as a bulk data upload.\n    </p><p>genetic resources</p><p>DNA sequences</p><p>sequences</p><p>checklist</p><p>voucher</p><p>duplicate</p><p>associated occurrences</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/links\/"},{value:"[editor] Managing Loans",label:"<p> This page describes how to manage loans in a Symbiota portal.\n Navigate to the loan management tool by accessing the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection) and clicking “Loan Management.” You will see the following tabs:\nTo add a new outgoing loan, click the green plus sign near the right side of the “Outgoing Loans” page. You will be prompted to enter a Loan Identifier (a unique number to mark the identity of the loan) and select an institution to which the loan is being sent from the dropdown list. If the institution is not found in the dropdown list, click the green plus sign to the right of the “Send to Institution:” field and enter the required information. Click “Create Loan” when you have finished entering the required information in the “New Outgoing Loan” box. Once you click “Create Loan,” you will be taken to the Loan Out Details page. Here you can add information about the loan and generate loan paperwork using the appropriate boxes and buttons. Note that you can print an invoice, specimen list, mailing label, and envelope in the Generate Loan Paperwork box at the bottom of the page.\nTo link specimens to the loan, click the “Specimens” tab, click the green plus sign, enter the catalog number of the desired specimen in the field (preferably by scanning the barcode of the physical specimen), and click “Add Specimen.” Repeat as necessary. To edit an outgoing loan or link specimens to the loan in the future, navigate to the “Outgoing Loans” tab and click on the loan identifier (number or name given to the loan) directly to the right of the appropriate bullet point (circled in next screenshot).\nTo add a new incoming loan, repeat the procedure outlined above using the “Incoming Loans” tab. Note that specimens cannot be linked to incoming loans. To add a gift or exchange, repeat the procedure outlined above using the Gifts\/Exchanges tab. You will be able to enter the number of specimens in either the “gift” boxes or “exchange” box.\n  </p><p>loan</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/loan\/"},{value:"[user] Taxonomic Thesaurus",label:"<p> This page describes the use of backbone taxonomic thesauri in Symbiota portals.\n While providing foundational data to the science of taxonomy, Symbiota portals are not necessarily designed to be taxonomic resources. Other resources will likely be more up-to-date and carefully curated than Symbiota portals, which are designed for data discovery and curation rather than taxonomic precision.\nSymbiota portals are built with a taxonomic backbone so that:\n Users can perform searches for occurrences of a taxon, and occurrences of that taxon\u0026rsquo;s synonyms will be included in the search results. Taxa can be linked hierarchically so that searches for higher taxa (e.g., families) result in lists including lower taxa (e.g., species) Data entry personnel can select taxonomic names from a pick-list rather than typing entire Latin names. Family names and taxonomic authorships can be auto-populated for occurrences with provided taxonomic names.  Curation of Taxonomic Information The curation and accuracy of a portal\u0026rsquo;s taxonomic thesaurus depends largely on the community managing and using the portal. Some portal communities actively curate their taxonomic thesaurus, while others use it largely as a curational tool and update it sparingly. Users interested in helping to curate the taxonomic thesaurus of a portal should contact the portal manager or help@symbiota.org.\nThe taxonomic thesaurus can be edited by superadministrators and users with express Taxonomy Editor permissions (see the User Permissions page for more information).\nMultiple thesauri Some portals include several backbone thesauri in case of, e.g., conflicting taxonomic concepts or authorities. One thesaurus will be displayed by default, but users can select between thesauri when using the checklist or list-viewing functions.\n</p><p>thesaurus</p><p>scientific names</p><p>taxonomy</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/taxonomy\/"},{value:"[user] User Permissions",label:"<p> This page describes the possible levels of user permissions that can be granted for a given Symbiota portal.\n Once a user has created an account in a Symbiota portal, they can be granted permissions for one or many collections in that portal. Permissions can be granted by administrators of individual collections or by the portal manager or other superadministrator. Portals have different policies for granting permissions to users, so it is best to contact the portal manager or individual collections for which you would like user permissions.\nTypes of User Permissions Permissions that can be assigned by a collection manager or other administrator\n  Administrator: (generally a curator or manager of a collection) can use all tools in the Administration Control Panel. For example, this user can edit the contact information and description for your collection, view edits made to your records, batch upload data, and use data cleaning tools. Administrators can also delete records.\n  Editor: (generally a collector, technician, digitizer, or volunteer) can use all tools in the Data Editor Control Panel, including adding and editing records, printing labels, batch annotating specimens, batch georeferencing specimens, and managing loans. Editors cannot delete records.\n  Rare Species Reader: can view locality data for all specimens in a collection, even if the locality information is redacted from the general public (see Redacting \/ Obscuring Data).\n  Personal Observation\/Specimen Profile Manager: can add, edit, and manage data belonging to them in the \u0026ldquo;personal observations profile\u0026rdquo; (variously labeled the \u0026ldquo;specimens being processed\u0026rdquo; or \u0026ldquo;general research observations\u0026rdquo; profile, depending on the portal). The personal observations profile is a single collection in the portal that contains pooled data from all personal observation profile managers; however, a single user can only edit data that they have personally added to this collection through their user profile.\n  Create a Checklist: can create new checklists and datasets. See Creating a Checklist.\n  Permission types that can only be assigned by a superadministrator\n  Global Rare Species Reader: can view locality data for all specimens in a portal (if provided). See Redacting \/ Obscuring Data.\n  Glossary Editor: can add descriptions, images, or other information to the portal glossary (if enabled)\n  Identification Keys Administrator: can add, delete, and edit characters states used in the identification keys (if enabled)\n  Identification Keys Editor: can edit character states used in the identification keys (if enabled)\n  Taxonomy Editor: can edit the taxonomic thesaurus, including adding and deleting taxa, editing the acceptance of a taxon, and adding synonym or children taxa\n  Taxon Profile Editor: can add information to taxon profile pages, such as images and descriptions\n  </p><p>users</p><p>permissions</p><p>access</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/permissions\/"},{value:"[coll_manager] Importing \u0026 Uploading Data",label:"<p> This page provides instructions for uploading data into an existing collection in a Symbiota portal. Contact your portal manager if you do not already have a collection in your desired Symbiota portal.\n Upload Types There are several options for uploading data into a Symbiota portal:\n Full Text File Upload: Use this upload type if you will provide a comma-separated value (CSV) or tab-separated value (TSV) file containing ALL fields of your occurrence data. You can convert an Excel document into a CSV file by clicking Save As, then selecting comma-delimited (CSV) from the file types. Note that, if data exists in the portal for any of the occurrences you are uploading, those data will be overwritten by the incoming data. To upload partial records, use a Skeletal File Upload. Skeletal File Upload: Use this upload type if you will provide a CSV or TSV file containing data from only a few fields (e.g., georeferences or other ancillary data). Note that this upload type can only fill in fields that are currently empty in the portal. The data provided in a skeletal file upload will NOT overwrite existing data in the database, so any pre-existing data in the desired fields must be deleted if you wish to replace it with the data from the skeletal file. Darwin Core Archive Manual Upload: Use this upload type if the data you wish to upload is in the format of a Darwin Core Archive. A Darwin Core Archive (DwC-A) is a data standard that is commonly used to package species occurrence data into a single, self-contained dataset. A DwC-A includes metadata, a file of occurrence data, and, often, files for determinations (identifications) and images. IPT Resource \/ Darwin Core Archive Provider: Use this upload type if you will provide a URL to an existing Darwin Core Archive published on the web, such as one provided through an IPT. NfN File Upload: Use this upload type if you will provide a CSV file produced from Notes from Nature. Direct Database Mapping: Stored Procedure: Use this option if you are transferring from a source schema to a Symbiota database located on the same MySQL database server. Script Upload: Use this option if you are transferring from a MySQL source to Symbiota database that is located on a different server.  Initiating the Upload  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Import\/Update Specimen Records, then select \u0026ldquo;Create a new Import Profile\u0026rdquo;. Create a title for your upload in the Title field. Select the desired Upload Type from the dropdown menu (see Upload Types section above). Follow the directions below according to the Upload Type you have selected.  File Upload or Skeletal File Upload A Full Text File Upload will use the incoming data to overwrite all existing occurrence data in the database, even for fields that are not included in the upload file. For example, if your database contains a \u0026ldquo;country\u0026rdquo; field, but your input file does not have the \u0026ldquo;country\u0026rdquo; field, after upload, the \u0026ldquo;country\u0026rdquo; field will be blank. After a Full Text File Upload, the only data associated with your specimens in the occurrence editor will be the data that were in the upload file. Conversely, a Skeletal File Upload will only import data into fields that are empty. It will not replace existing values within fields.\n  If you or your portal manager have created a Stored Procedure with data cleaning or other checks, enter the name of the stored procedure in the provided field. Otherwise, ignore this step. Click the Create Profile button. On the next page, you will see a list of existing upload profiles. Select the profile that you wish to use (the one you just greated) and click Initialize Upload.  To edit your upload profile in the future, you can click the pencil icon on this page.   Click the Choose File button and navigate to the file that you wish to upload in your File Manager or Finder window. Select that file and click Open. Click the Analyze File button. If the collection to which you are uploading data is live managed (the portal is your database system), proceed to step 7. If the collection to which you are uploading data is a “snapshot” of a specimen database managed within the home institution, select the primary key for the source specimen record from the dropdown menu. The primary key is a required field for snapshot datasets that will serve as the primary record identifier (the permanent link between the source database and the portal records). This field must be populated for every record with unique values. These values must also be stable and not changed in the central database over time. Snapshots will typically use the catalog number (accession number), barcode, or database primary key from the source database specimen table for this field. You will then see a page that will look similar to the one shown below. The length and contents of the Source Field\/Target Field table will depend on what columns were included in the original CSV file.  Select which fields in your CSV file (Source Fields) will correspond to which fields in the Symbiota portal (Target Fields). Check the Symbiota Data Field Guide for definitions of each data field. Also see the Uploading Tips section below. Once you are satisfied with your field-to-field mapping (see next Notes), click the “Save Mapping” button. Select whether you would like the script to match the data in your file to existing data in the portal based on Catalog Number or Other Catalog Numbers. You will only need to do this if you are adding data to records that already exist in the portal. Otherwise, leave these unchecked. Select the Processing Status that you would like to apply to all your uploaded records (if desired) by selecting an option from the dropdown menu. Click the Start Upload button. This will upload your data into a temporary table so you can review it before committing the final upload. Verify that the correct number of records are being updated and\/or added by viewing the Pending Data Transport Report on the next page.  View the data that have been stored in the temporary table to ensure correct mapping and formatting of the fields you are uploading. You can either:  Click the small box icon to the right of \u0026ldquo;Records to be updated\u0026rdquo; or \u0026ldquo;New records\u0026rdquo; to view the records in a table in your browser. Click the multiple file icon to the right of the box icon to download a CSV file of the records to be updated or new records.   If anything is incorrect, fix your CSV file and re-upload it according to the steps you followed above, or return to your field mapping and fix the field mapping. If everything looks good, click the Transfer Records to Central Specimen Table button. Note that this step is final and is not possible to undo!  Darwin Core Archive Manual Upload Use this import profile when you are manually uploading a Darwin Core Archive. Rather than matching on catalog number or other catalog number, this import profile will match on the internal \u0026ldquo;id\u0026rdquo; field of the incoming Darwin Core Archive. This \u0026ldquo;id\u0026rdquo; field is then stored in the \u0026ldquo;dbpk\u0026rdquo; field in the occurrence table and will be used for future updating of the data. For this reason, it is very important that the \u0026ldquo;id\u0026rdquo; values do not change between subsequent uploads of your Darwin Core Archive. It is best practice to use a globally unique identifier (e.g., whatever is used as your occurrenceID) for this field.\nIf you have already created an import profile, or if you have been instructed to use an existing import profile, click \u0026ldquo;Saved Import Profiles\u0026rdquo; under the \u0026ldquo;Import\/Update Specimen Records\u0026rdquo; option, then skip to step 5.\n Enter a name for your upload in the Title field. If you or your portal manager have created a Stored Procedure with data cleaning or other checks, enter the name of the stored procedure in the provided field. Otherwise, ignore this step. Click Create Profile. Select the radio button for the name of the profile you would like to use for your import (if you just created an import profile, use that one). Click the Initialize Upload button. Click the Choose File button and select the Darwin Core Archive (packaged as a zip file) that you would like to upload. Then click the Analyze File button. On the resulting page, you will have the option to view and\/or verify the mappings of the incoming (1) occurrences file, (2) identification history file (if it exists), and (3) images file (if it exists). If one or more of these files does not exist in the incoming DwC-A, the checkbox to the left of it will be grayed out. To view or verify the mappings, click \u0026ldquo;view details\u0026rdquo; to the right of the file you wish to view.   Because the incoming file is aligned with Darwin Core, you will not likely need to change the mappings of this file. However, it is a good idea to check the fields. Check the Symbiota Data Field Guide for definitions of each data field. Also see the Uploading Tips section below.\n  Once you are satisfied with your field-to-field mapping, click the “Save Mapping” button.\n  If you want all the incoming data to be assigned a specific processing status, select that processing status value from the dropdown menu. If not, leave the processing status as \u0026ldquo;Leave as is \/ No Explicit Setting\u0026rdquo;.\n  Click the Start Upload button.\n  View the data that have been stored in the temporary table to ensure correct mapping and formatting of the fields you are uploading. You particularly want to make sure that the number of new records versus updated records matches your expectations. A large number of new records, if you did not expect them, may indicate that your \u0026ldquo;id\u0026rdquo; field changed since the last upload. To view the mapping of your fields, you can:\n Click the small box icon to the right of \u0026ldquo;Records to be updated\u0026rdquo; or \u0026ldquo;New records\u0026rdquo; to view the records in a table in your browser. Click the multiple file icon to the right of the box icon to download a CSV file of the records to be updated or new records.    You may also see that there are records that exist in your Symbiota portal that are not found in the incoming data. Note that these records will not automatically be deleted from the Symbiota portal. If you deleted these records in your home database or IPT provider, you will need to manually delete them in your Symbiota portal as well. For bulk deletions, contact your portal administrator.\nIf anything is incorrect, return to the Saved Import Profiles, re-initiate the upload, and fix your mappings. If everything looks good, click the Transfer Records to Central Specimen Table button. Note that this step is final and is not possible to undo!  IPT Resource \/ Darwin Core Archive Provider Use this import profile when you are importing from an IPT or other source that provides a URL to a Darwin Core Archive. Follow the steps outlined for Darwin Core Archive Manual Upload above, but instead of uploading a Darwin Core Archive as a zip file in Step 6, you will need to provide a URL to the IPT or Darwin Core Archive provider in the \u0026ldquo;path\u0026rdquo; field during Step 2. If you are bringing data in from an IPT, the URL will look something like this (with a different base URL): https:\/\/fmipt.fieldmuseum.org\/ipt\/archive.do?r=fmnh_seedplants. This URL is also sometimes referred to as the Darwin Core Archive \u0026ldquo;endpoint\u0026rdquo; for a dataset.\nThe same cautions as a manual Darwin Core Archive upload apply to this upload type.\nStored Procedure  Write a stored procedure used to transfer records (the collection cleanup scripts can be put in central stored procedure or kept separate) Set up the script to run as a regular cronjob.  Script Upload  Write a stored procedure used to transfer records. A sample Linux script is located here: SampleSystemUpload.sh. The cleanup scripts can be put in central stored procedure or kept separate. Set up the script to run as a regular cronjob.  Uploading Tips A list of fields that can be imported into a Symbiota data portal can be found here.\n If you\u0026rsquo;re new to using data ingestion tools in Symbiota portals, start by uploading a small number of records before ingesting larger datasets. You will be able to delete erroneous records one-by-one, but not in batch.\n  To take advantage of the Tag Name \u002b Identifier system (in which you can tag an identifier\/other catalog number with a specific title), enter the tag name followed by a colon and then the identifier value, e.g., \u0026ldquo;Old Accession Number: 12345\u0026rdquo;. For multiple identifiers, separate the tag name \u002b identifiers by semicolons, e.g., \u0026ldquo;NP #: 4321; Accession #: 9876\u0026rdquo; If the scientific names in your CSV file include taxonomic authorship (e.g., Acer circinatum Pursh), map this field to the Target Field “scientificname.” If the scientific names included in your CSV file do NOT include taxonomic authorship (e.g., Acer circinatum), map this field to “sciname.” Collection dates mapped to eventDate will be evaluated and validated. Illegal dates will be placed in the verbatimEventDate field. The majority of the standard date formats are accepted, including Gregorian dates and Excel numeric date format (US only). eventDate will be generated from separate year,month, and day field values. If month or day fields are left null, ’00’ values will be used (ex: 1954-03-00, 1965-00-00). Month field values can be numeric or text (English or Spanish). Scripts attempt to extract valid date values from verbatimEventDate field when the eventDate field is null. Values of ’00’ are used for missing month or day (ex: 1954-03-00, 1965-00-00) If your elevation field is not consistently in meters, map it to the verbatimElevation field. Elevations in feet will be converted to meters as long as the units are specified in the field. Coordinate values:  Upon upload, background scripts will attempt to extract lat\/long coordinate values from the verbatimCoordinates field. The field is evaluated for DMS and UTM formats, which are converted to decimal latitude and longitude. If you have lat\/long in a single field, you can map this field to verbatimCoordinates, and decimal latitude and longitude fields will be automatically parsed. If you have UTM coordinates in multiple fields, map the fields (northing, easting, zone) to their matching UTM fields (utmnorthing, utmeasting, utmzone). This will instigate conversion of UTM coordinates to decimal latitude and longitude. The values will additionally be stored in the verbatiumCoordinates field. If you have UTM coordinates in a single field, map this field to utmnorthing and leave other UTM fields null in order to direct scripts to parse using only the UTM parser. TRS coordinates (Public Lands Survey System) can be entered as a single field into verbatimCoordinates, or into separate fields (trstownship, trsrange, trssection, trssectiondetails); however, these coordinates will not be automatically converted into decimal degrees due to potential differences in interpretation. See the georeferencing section of this guide (coming soon) for information about converting TRS coordinates to decimal degrees.    Related Video Tutorials     </p><p>data upload</p><p>data import</p><p>file upload</p><p>IPT</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/upload\/"},{value:"[coll_manager] Paleo Data",label:"<p> The use of Symbiota portals for managing and sharing paleontological collections data is relatively new. Anyone with interest in advancing the functionality of Symbiota portals to manage, publish, or use fossil data is encouraged to contact the Symbiota Support Hub. Community discussions related the functionality of Symbiota portals for managing and publishing fossil collections data can be found here.\n Paleo Module The \u0026ldquo;Paleo Module\u0026rdquo; is a non-standard module available for collections in Symbiota portals that contain records of fossil specimen occurrences. This module is activated on a per-collection basis (i.e. not per portal) and appears as an additional form within the Occurrence Editor. Contact your portal administrator to have this module activated for your collection.\nGeochronologic\/Chronostratigraphic Data Controlled Vocabularies The values for the geochonologic\/chronostratigraphic terms in the paleo module rely on a suggested controlled vocabulary, which is in place to facilitate consistent data entry when records are manually created in your portal.\nIf your portal community has not yet selected a standard upon which to base this controlled vocabulary, recommended options include:\n The International Commission on Stratigraphy\u0026rsquo;s Chronostratigraphic Chart The Geological Society of America\u0026rsquo;s Geologic Time Scale The US Geological Survey\u0026rsquo;s Divisions of Geological Time  Bulk Importing \u0026amp; Manipulating Age-Related Values If the paleo module has been activated for your collection\u0026rsquo;s profile in a Symbiota portal, data can be imported, downloaded, and batch updated using the same methods for non-paleo collections. Refer to the documentation on Importing and Uploading Data for more information.\nWhen bulk ingesting data that includes values destined for the Eon, Era, Period, Epoch and\/or Stage fields, be aware of the idiosyncrasies illustrated below:\n        Only one hierarchy for age-related data can be specified within the Occurrence Editor. In other words, if a specimen\u0026rsquo;s possible age may span multiple divisions of geologic time\u0026ndash;e.g. Early Interval=\u0026ldquo;Campanian\u0026rdquo; and Late Interval=\u0026ldquo;Danian\u0026rdquo;\u0026ndash;at present, you can only specify one hierarchy using the Paleo Module form. In the example above, the hierachy for \u0026ldquo;Danian\u0026rdquo; is selected. However, full hierarchies for both Late Interval and Early Interval can be bulk uploaded after cataloging, if desired.            ⚠️ While a controlled vocabulary exists, it is only suggested; therefore, illogical age values can be keystroked and saved the Occurrence Editor. In the example above, because “Lower Cretaceous” exists as value for Epoch, the form accepted this value, even though it\u0026rsquo;s an illogical choice for Stage = “Maastrichtian”).            ❗A \u0026ldquo;mismatched term\u0026rdquo; error will appear if an entered value does not perfectly align with your portal\u0026rsquo;s suggested controlled vocabulary in the Eon, Era, Period, Epoch and Stage fields. In this example, the suggested vocabulary prefers \u0026ldquo;Upper Cretaceous\u0026rdquo; to \u0026ldquo;Late Cretaceous\u0026rdquo;; therefore, when \u0026ldquo;Late Cretaceous\u0026rdquo; is entered, an error appears. This error is only visible on the Occurence Editor interface and will not affect how your data are stored, exported, or displayed publicly.    Entering BasisOfRecord At present, the default value for basisOfRecord in Symbiota portals is \u0026ldquo;PreservedSpecimen\u0026rdquo;; however, the correct value for paleontological occurrences is \u0026ldquo;FossilSpecimen\u0026rdquo;. This value can be manually selected in the Occurrence Editor form during cataloging, or by batch editing to replace \u0026ldquo;PreservedSpecimen\u0026rdquo; with \u0026ldquo;FossilSpecimen\u0026rdquo; after cataloging is complete.\nFossil Localities\/Sites At present, locality (site) data are entered at the specimen occurence level\u0026ndash;i.e., on a record-by-record basis\u0026ndash;in Symbiota portals. Locality data are also publicly visible by default. Refer to the documentation on locality redaction for instructions on how to obscure this information for sensitive localities.\nExtinct Taxa Extinct taxa are supported in Symbiota. Refer to the Taxonomy Tree Viewer in Pteridoportal for examples.\nData Publishing Considerations At present, while Symbiota supports paleontological data, including the Darwin Core Class GeologicalContext, values in these fields do not export within Darwin Core archives when published to GBIF via Symbiota portals. However, this information can be exported as part of a backup file.\nPrinting Labels Similar to data publishing, values in the Darwin Core Class GeologicalContext are not currently available within the label editor\u0026rsquo;s visual interface. Contact the Symbiota Support Hub if you need assistance with creating customized labels that include paleo-related data fields.\nBest Practices Symbiota is compliant with the Darwin Core data standard, and this includes its application to paleontological data. However, because some of these terms are still in the process of becoming well-defined for use with fossil data, the Symbiota Support Hub recommendations following best practices recommended by the Paleo Data Working Group.\n📃 A complete list of fields used in Symbiota portals and their Darwin Core equivalents can be found here.\nNeed help? The Symbiota Support Hub encourages you to contact them for guidance if using Symbiota to manage paleontological data.\n</p><p>paleo</p><p>fossil</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/paleo\/"},{value:"[coll_manager] Restoring your Database",label:"<p>In the case of catastrophic database error (e.g., erroneous batch edit that cannot be easily reversed), you can replace your entire database by uploading a Darwin Core Archive (DwC-A). Note that you need to have recently downloaded a copy of your database with which you can replace your current database. To replace your database, navigate to the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection) and click “Restore Backup File” under the General Maintenance Tasks. You can then click Choose File and select a DwC-A with which to replace your dataset.\n If your DwC-A contains an “identifications” file, make sure that the \u0026ldquo;Restore Determination History\u0026rdquo; box is checked. If your DwC-A contains a “multimedia” file, make sure that the \u0026ldquo;Restore Images\u0026rdquo; box is checked. Click Analyze File.  The file will take some time to load and process. Once this is done, a report called “Final transfer” will be displayed at the bottom of the screen. This report will show you how many records will be updated, how many records are new, how many identifications\/determinations will be added, and how many records have images. Ensure that these numbers are as expected. You can preview these records by clicking the table icon to the immediate right of the desired dataset (circled on next screenshot), or you can download these records as a CSV by clicking the two boxes icon to the far right of the desired dataset (in a square on next screenshot). Once you are satisfied that the records will be properly uploaded, click the \u0026ldquo;Transfer Records to Central Specimen Table\u0026rdquo; button. Note that this change is permanent and cannot be undone!\n</p><p>restore</p><p>replace</p><p>upload</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/restore\/"},{value:"[coll_manager] Reviewing Edits",label:"<p> This page describes how to view, verify, and\/or revert edits that have been made to occurrence records for your collection.\n A user with administrator access can review edits that have been made to specimen records by selecting “Review\/Verify Occurrence Edits” from the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). The resulting dashboard will show a table of every edit made to any field of any record, which often means that the list is very long, depending on how much activity the database sees on a daily basis. You can limit the search using the Filter box in the top right corner. You can search by Applied Status, Review Status, Editor, or Date\/Date Range of editing.\nYou can select records by checking the boxes on the left side of the table and decide to approve (Apply Edits) or revert (Revert Edits) the edits made to each field of each record by clicking the appropriate radio button in the Action Panel and then the Update Selected Records button. You can also decide to change the Review Status of these records at this time. The Review Status is independent of and not related to the Processing Status.\nIf you click “Additional Action” in the Action Panel, you can delete the selected edits, download the selected records, download all records that were returned by the search (i.e., are shown in the resulting table), or print the page using one of the appropriate buttons.\nAll edits in the database, regardless of Review Status, are visible in the database as soon as they are made. This tool is meant to aid in reviewing of edits, but does not serve as a gateway to vetting edits before they are made public.\n </p><p>edits</p><p>review</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/review\/"},{value:"[editor] Traits",label:"<p>Occurrence traits can be scored and stored in some Symbiota portals (e.g., the CCH2, which contains phenological trait data for vascular plants).\nTraits are stored in a separate table than the main occurrence data and can be viewed for an occurrence in the Traits tab in the Occurrence Editor.\nTraits can often be downloaded along with occurrence data as an extended measurementOrFact file.\nSeveral tools have been developed to enable users to annotate specimen records for their traits, including the Trait Coding from Images tool and the Trait Coding from Text tool. The traits available to score vary between portals, and new traits can be developed as needed (contact your portal administrator for more information).\n</p><p>occurrence traits</p><p>attributes</p><p>phenology</p><p>phenological scoring</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/trait\/"},{value:"[coll_manager] User Permissions",label:"<p> This page defines each of the possible permission levels that can be assigned by an administrator of a collection and describes how to assign permissions to users.\n Permissions Definitions Administrator: can use all tools in the Administration Control Panel. For example, this user can edit the contact information and description for your collection, view edits made to your records, batch upload data, and use data cleaning tools. Administrators can also delete records.\nEditor: can use all tools in the Data Editor Control Panel, including adding and editing records, printing labels, batch annotating specimens, batch georeferencing specimens, and managing loans. Editors cannot delete records. Editors can view and modify all locality data, including for records with locality security applied.\nRare Species Reader: can view and download locality data for all specimens in your collection, even if the locality information is redacted from the general public (see Redacting \/ Obscuring Data).\nAdding \/ Changing User Permissions To assign user permissions, you must be an Administrator of the collection for which you will assign those permissions.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Manage Permissions. The resulting page will show a list of all Administrators, Editors, and Rare Species Readers assigned to your collection. To remove a user\u0026rsquo;s permissions, click the \u0026ldquo;X\u0026rdquo; icon next to their name:  To add or edit a user\u0026rsquo;s permissions, select their name from the dropdown list under \u0026ldquo;Add New Admin\/Editor\/Reader\u0026rdquo;, select the radio button next to the level of permissions you wish to assign, and click the \u0026ldquo;Add Permission for User\u0026rdquo; button:  If you don\u0026rsquo;t see someone\u0026rsquo;s name in the dropdown list, they might not have a profile yet. Contact the user and have them create a user profile.\n </p><p>users</p><p>permissions</p><p>access</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/users\/"},{value:"[] Symbiota Docs",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/"},{value:"[editor] Processing Status",label:"<p> This page explains how to automatically update Processing Status while editing occurrence records.\n Processing Status integrates with the Crowdsourcing module when Processing Status=\u0026ldquo;Unprocessed\u0026rdquo; and a record has at least one associated image. Additionally, this field can be used to track workflow- or project-specific needs as it is not publicly visible.\n Status Auto-Set is user-specific and relies on browser cookies to function.\n Set Processing Status Processing Status and Status Auto-Set are located under the Curation section of the Occurrence Editor form: Manually Processing Status can be manually updated from within the Curation box on the Occurrence Editor. This method is most useful when making on-off edits to records that do not requiore extensive workflow tracking.\nAutomatically The Status Auto-Set menu is used for making changes to Processing Status when multiple records are to be edited. If a user needs to edit numerous records, but they do not want to remember to manually update the Processing Status as they work, Status Auto-Set can be used to automatically update this field, for example, during image transcription. Once Status Auto-Set is set, any records a user subsequently edits will receive that new Processing Status value.\nQuery Records by Processing Status Processing Status values can be queried using the Data Editor Search Form (Data Editor Control Panel \u0026gt; Edit Existing Occurrence Records): </p><p>crowdsourcing</p><p>transcription</p><p>editing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/status\/"},{value:"[editor] Catalog Numbers",label:"<p>Catalog Numbers Catalog numbers are important to assign to each record because they act as unique identifiers for each record. This field should be used to store the barcode or the accession number (herbaria only). This field should be unique for each collection, for example, by including the collection acronym (e.g., ASU00012345, WIS-L-001456). Use of leading zeros is recommended to enable correct sorting in the Symbiota interface.\nTag Name \/ Additional Identifier Values (= Other Catalog Numbers) Because specimens may be associated with multiple catalog numbers \/ identifiers, Symbiota portals have a tool that allows multiple values for Other Catalog Number. In the Occurrence Editor, this tool is located to the right of the Catalog Number field, and it includes boxes for \u0026ldquo;Tag Names\u0026rdquo; and \u0026ldquo;Additional Identifier Values\u0026rdquo;.\nThe Tag Name field is an optional field that can be used to label the type of identifier being applied to the specimen. Tag Names are generally collection-specific (i.e., there are no recommended values). Common tags may include: Previous Catalog Number, Old Catalog #, Accession Number, NPS #, NPS Accession #, Secondary OccurrenceID, etc. The Tag Name field can be left blank if you do not wish to label the type of Other Catalog Number \/ Identifier.\nThe Additional Identifier Value field is where you can enter the alphanumeric value of the Other Catalog Number \/ Additional Identifier. For example, if you have an old accession number of 45678, you can enter \u0026ldquo;Accession #\u0026rdquo; in the Tag Name field, and 45678 into the Additional Identifier Value field. Alternatively, you can leave Tag Name blank and only enter a value into the Additional Identifier Value field.\nThese tools were released in March 2022 and are under active development. For this reason, some functionalities are still coming online. In particular, searching based on Other Catalog Numbers may produce incomplete results until all values of Other Catalog Number are transferred into the new Tag Name \u002b Additional Identifier Value system.\n How do I search for Other Catalog Numbers?    Values entered in the Tag Name \u002b Additional Identifier Value system are displayed in the Other Catalog Number field in the table display of a regular search. Because the Tag Name and Additional Identifier Value fields are separated, you can search for values of Other Catalog Number without having to include the tag name of the identifier. For example, if you search for Other Catalog Number \u0026ldquo;12345\u0026rdquo;, the results would include things like \u0026ldquo;Accession #: 12345\u0026rdquo; and \u0026ldquo;NPS #: 12345\u0026rdquo;. The search will be conducted based off of the number\/value rather than the tag name.   What is the difference between the Other Catalog Number field and the Tag Name \u002b Additional Identifier Value system?    On the technical side, the Other Catalog Number field was a single text field, and the new Tag Name \u002b Additional Identifier Value is a new table in the Symbiota schema (i.e., database structure). However, to a user, the function of these two systems are essentially the same. The Tag Name \u002b Additional Identifier Value system was created to facilitate smarter searching for values of Other Catalog Number. Values entered in the Tag Name \u002b Additional Identifier Value system are displayed in the Other Catalog Number field in the table display of a regular search. The Tag Name (if it exists) is listed first, followed by a colon, then the Additional Identifier Value (e.g., \u0026ldquo;NPS Accession #: 98765). If there are multiple identifiers, they are concatenated into the Other Catalog Number field and separated by semicolons (e.g., \u0026ldquo;NPS Accession #: 98765; Voucher Number: 12345).   How are Tag Names and Additional Identifier Values displayed in a search?    Values entered in the Tag Name \u002b Additional Identifier Value system are displayed in the Other Catalog Number field in the table display of a regular search. The Tag Name (if it exists) is listed first, followed by a colon, then the Additional Identifier Value (e.g., \u0026ldquo;NPS Accession #: 98765). If there are multiple identifiers, they are concatenated into the Other Catalog Number field and separated by semicolons (e.g., \u0026ldquo;NPS Accession #: 98765; Voucher Number: 12345).   How are Tag Names and Additional Identifier Values displayed in a data download?    Just like when you conduct a search on the Other Catalog Numbers field, values entered in the Tag Name \u002b Additional Identifier Value system are displayed in the Other Catalog Number field. The Tag Name (if it exists) is listed first, followed by a colon, then the Additional Identifier Value (e.g., \u0026ldquo;NPS Accession #: 98765). If there are multiple identifiers, they are concatenated into the Other Catalog Numbers field and separated by semicolons (e.g., \u0026ldquo;NPS Accession #: 98765; Voucher Number: 12345). This is also how the data are presented in a Darwin Core Archive.   Duplicate Catalog and Other Catalog Numbers Collection administrators can identify and resolve duplicate catalog numbers or other catalog numbers using the Data Cleaning Tools.\n</p><p>edit</p><p>fields</p><p>data fields</p><p>terms</p><p>dwc terms</p><p>catalog number</p><p>tag names</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/fields\/catno\/"},{value:"[coll_manager] Download a Subset of Data (Exporter)",label:"<p> This page describes how to find and use the Exporter tool, which allows you to download a subset of your data as a Darwin Core Archive.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Managment \u0026gt; name of collection). Click Processing Tools. Click the Exporter tab. Use the Processing Status and additional filters to define the dataset you would like to download from your collection. You can also select whether you would like to download a strict Darwin Core Archive (Darwin Core) or an archive containing all Symbiota fields (Symbiota Native); whether you would like to determination history (identifications), images, and\/or occurrence attributes (if enabled); whether you would like the results in a ZIP file; and the file format and character set (ISO-8859-1 or UTF-8) for your download. Click the Download Records button.  Downloading Specimens without Georeferences The Exporter tool also has the option to download all records without georeference data. To do this, select Georeference Export from the dropdown menu in the Export Type box at the top right of the Exporter tool. Select the search terms\/filters to apply to your download and click Download Records.\nDownloading Records that have been Batch Georeferenced For Snapshot collections (i.e., collections that do not manage their data live in the portal), there is also an option to download georeference data for specimens that have been batch georeferenced in the portal. To do this, select Georeference Export from the dropdown menu in the Export Type box at the top right of the Exporter tool. Select the search terms\/filters to apply to your download and click Download Records. The resulting file will include the following fields:\n institutionCode collectionCode catalogNumber occurrenceId decimalLatitude decimalLongitude geodeticDatum coordinateUncertaintyInMeters verbatimCoordinates georeferencedBy georeferenceProtocol georeferenceSources georeferenceVerificationStatus georeferenceRemarks minimumElevationInMeters maximumElevationInMeters verbatimElevation localitySecurity localitySecurityReason modified processingStatus collId sourcePrimaryKey occid recordID.  </p><p>download</p><p>exporter</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/download\/exporter\/"},{value:"[coll_manager] Batch OCR",label:"<p>Optical Character Recognition (OCR) is the use of computer algorithms to detect text information from images. If OCR tools are activated in your Symbiota portal, you can use OCR to attempt to parse textual information from pictures of, e.g., specimen labels and enter data into the appropriate fields from there. Instructions for using batch OCR processes are provided below. For instructions for using OCR in the Occurrence editor, see the OCR page in the Editor Guide.\nBatch Generating OCR for Records in a Portal To run OCR on many specimen images at a time\n Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Processing Toolbox. If OCR is enabled in your portal, click the OCR tab. From here, you can view how many specimens have a saved OCR output, and you can run OCR on a designated number of specimens using the Batch OCR Images using the Tesseract OCR Engine box. Indicate the processing status of the records for which you would like to run OCR in the \u0026ldquo;processing status\u0026rdquo; field, and indicate how many of these records you would like to run at one time using the \u0026ldquo;Number of records to process\u0026rdquo; field.  A video tutorial of this process is shown here:\n  Uploading Pre-Generated OCR Some portals also include an interface into which you can upload OCR text files generated outside of the portal environment. For instance, ABBYY FineReader has the ability to batch OCR specimen images and output the results as separate text files (.txt) named after the source image. OCR text files are linked to specimen records by matching catalog numbers extracted from the file name and comparing OCR and iamge file names.\nThis tool can also be found on the OCR tab in the Processing Toolbox.\nTo upload a file, make sure your collection and files meet the following requirements:\n OCR files must be in a text format with a .txt extension. When using ABBYY, use the setting: \u0026ldquo;Create a separate document for each file\u0026rdquo;, \u0026ldquo;Save as Text (*.txt)\u0026rdquo;, and \u0026ldquo;Name as source file\u0026rdquo; Compress multiple OCR text files into a single zip file to be uploaded into the portal Files must be named using the Catalog Number. The regular expression below will be used to extract catalog number from file name. Since OCR text needs to be linked to source image, images must have been previously uploaded into portal If there are more than one image linked to a specimen, the full file name will be used to determine which image to link the OCR  </p><p>batch</p><p>OCR</p><p>optical character recognition</p><p>automated transcription</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/edit\/ocr\/"},{value:"[coll_manager] Batch Editing",label:"<p> This page describes how to batch edit records.\n  For batch georeferencing instructions, visit this page. Scientific names can only be batch changed through the Batch Annotation tools or the Taxonomic Cleaning tools.  ⚠️ Exercise caution when using this tool. We strongly recommend downloading a copy of your database prior to making batch edits\n There are two options for batch editing specimen records: you can change a value for 1) the entire set of records in your collection or 2) change a value for a subset of records.\n Navigate to the Record Search Form (Administration Control Panel \u0026gt; Edit Existing Occurrence Records). Option A: To apply batch changes to all records, click on “Display Table” in the Record Search Form, then click the edit icon above the resulting table of all records from your collection. Continue to step 3. Option B: To apply batch changes to a subset of records, first search for the records of interest using the search form. Once you have the set of records to which you would like to make a batch edit, click on the edit icon at the top of your selected table of records.  From the dropdown menu next to \u0026ldquo;Field Name\u0026rdquo; in Batch Update, select the field you want to edit. In \u0026ldquo;Current Value:\u0026rdquo; enter the text that is presently in the field you want to edit. In \u0026ldquo;New Value\u0026rdquo; enter the text that you want to be in the edited field. Choose if you want to match the whole field or just match any part. Click “Batch Update Field” to enact your batch edit. A pop-up screen will tell you the number of records to be updated, warn you that the replace operation cannot be undone, and prompt you to OK the update. Proceed with caution and see examples below.  ⚠️ It is easy to inadvertently change text you did not intend to change. Generally, the more specific your \u0026ldquo;Current Value:\u0026rdquo; text, the less likely you are to run into unintended consequences. If possible, get an estimate from your record table of how many records should be affected, and check that against the number of records cited in the warning. If the numbers do not match, rethink your strategy.\nIt is not possible to check the record count when sorting by \u0026ldquo;Modified By\u0026rdquo; first. Doing so returns a count of the number of fields edited by the \u0026ldquo;Modified By\u0026rdquo; user for all records to be affected, rather than the number of records to be affected.\n Examples ✅ Example of a good use of batch edit: Standardizing plant collector names. If you are standardizing all Wherry’s collections to “Edgar T. Wherry,” then you could enter “E. T. Wherry” in Current Value and “Edgar T. Wherry” in New Value in batch edit for the collector field name, and update. Repeat for alternate forms of this name (e.g., “E T Wherry,” Edgar T Wherry”).\n❌ Example of a bad use of batch edit: Changing cardinal directions. For example, if you entered “n.” in Current Value and “N” in New Value in batch edit for the locality in an attempt to change lower case abbreviated instances of the cardinal direction north, you would accomplish that, but you would also change any other character string with “n.” such as a sentence ending in the letter n. This would result in batch misplacements of “N” where “n.” should be. For example, “Next to the station. East side” would be transformed into “Next to the statioN East side.”\n</p><p>batch</p><p>edit</p><p>change</p><p>record search form</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/edit\/batch\/"},{value:"[dev] Developer Guide",label:"<p>Coming soon!\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/dev\/"},{value:"[editor] Record Editor Tabs",label:"<p> This page explains the tabs at the top of the Occurrence Editor tool.\n Upon opening the Record Editor page, you will see 4-5 tabs, depending on your level of permissions. The image shown below is of the Record Editor page for a user with administrative permissions. A brief explanation of each tab is provided below.\nOccurrence Data – See Symbiota Data Fields\nDetermination History – where annotations are shown and can be entered. Annotations can also be batch uploaded from separate files. See Annotations\nImages – provides information on image(s) associated with the record. See Images\nMaterial Sample - allows for association of multiple material sample records per specimen occurrence. For this tab to be visible, it must be configured per portal and activated per collection. Contact your Collection Administrator for more information. See Material Sample\nLinked Resources – allows linkage of checklists, duplicates, and genetic resources to the record. See Linking Records\/Resources\nTraits - allows for scoring of trait data. For this tab to be visible, it must be configured per portal and activated per collection. Contact your Collection Administrator for more information. See Traits\nAdmin – tracks changes made to Edit History, allows for transfers of specimens between collections, and provides a means for deleting an occurrence record. Only users with administrative permissions can view this tab\n</p><p>edit</p><p>tabs</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/tabs\/"},{value:"[editor] Adding Full Records",label:"<p> This page describes where to add a new record by entering the data directly into the portal. For information about batch uploading label information, see this page.\n Navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Select Add New Occurrence Record. Enter the specimen information into the provided fields.\nTo learn more about the data fields available in Symbiota, visit this page.\n Before you click Add record to create the new record, you can select one of three options for the next action you will perform, shown below. Go to New Record and Carryover Locality Information is useful for entering data from a single collection event that will share locality information. Remain on Editing Page should be used when the user wishes to double-check the record or add supplementary information such as images or annotations.\n</p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/full\/"},{value:"[editor] Adding Records from Images",label:"<p> This page describes how to add a new record by uploading a single image.\n Using this tool, you can upload (or enter a URL to) an image that will establish a new occurrence record. Navigate to the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection), and select Create New Records Using Image (nested under Add New Occurrence Record). To upload an image, click the Choose File button and select the image you wish to upload. To use a URL, click the Enter URL link on the right side of the form.\nBefore upload, you can also define the processing status of the new record and enter any skeletal data for the record, including Catalog Number, Scientific Name, Country, State\/Province, and County. Checking the box next to OCR Text using Tesseract OCR engine (if applicable) will run optical character recognition on the image that will attempt to parse text from the image that can potentially be used to fill out the remaining fields in the future.\n</p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/image\/"},{value:"[editor] Adding Skeletal Records",label:"<p> This page describes how to use the Skeletal Data Entry tool to quickly add skeletal data to the portal, especially during the process of imaging. It can also be used to create a quick inventory of a collection.\n A video overview of using this tool is shown below:\n  This tool can be used to collect basic collection information, for example, during the imaging process. The imaging team can enter the basic collection information shared by the batch of specimens being processed, and each time they scan a barcode into the catalog number field, a record is added to the system with these skeletal data. Complete data can be entered by clicking on the catalog number, but a recommended workflow would be to process the full label data directly from the image of the specimen label at a later stage.\n To add a skeletal record, navigate to the Add Skeletal Records tool via the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). You will see a page like that below.  To control which fields are shown on the skeletal record entry form, click the Display Option symbol (a small, grey table) located above the Scientific Name field (circled above). In the dropdown menu of options, check or uncheck the boxes next to the desired fields to be shown. The following fields are commonly recommended for skeletal data entry: Scientific Name, Family, State\/Province, and County\/Parish. Select the preferred action that the data entry module will use when a record already exists for a scanned catalog (barcode) number.  Even if you choose the “Append values to existing records” option, skeletal data will not copy over existing field values.\n If using the fields recommended above, enter the scientific name by beginning to type the name from the specimen label, then selecting the appropriate name from the dropdown list. If desired and\/or possible (given the data provided on the specimen sheet), enter the State\/Province and the County\/Parish of the specimen if the specimen was collected in the U.S. or Canada. The authorship and family of the name of the taxon will automatically populate if the taxon is in the portal’s taxon table. Enter the barcode number into the Catalog Number field by clicking inside the Catalog Number field and scanning the barcode of the specimen. Once all fields are filled, click Add Record.  Important: After Add Record is clicked, the data in the Skeletal Data box fields will NOT be cleared. Only the Catalog Number field will be cleared. This is designed to facilitate rapid skeletal data entry for folders of similar specimens (i.e., specimens of the same species). Close attention should be paid to these fields when transitioning between species or specimen localities. It is easy to forget to change the data in these fields.\n In the event that a user needs to edit a previously entered skeletal record, you can click the catalog number of the record as it is listed in the Records box below the Skeletal Data box. Note that this list will disappear if the page is refreshed. You can also add an image to the record from this page by clicking the icon to the right of the listed catalog number.\n Further clarification about the Skeletal Data Entry tool can be found here: SERNEC Skeletal Data Entry Guide\n</p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/skeletal\/"},{value:"[editor] Downloading a Darwin Core Archive",label:"<p> This page describes how you can download Darwin Core Archive files (DwC-A; standard format for data publishing and sharing) from a Symbiota portal.\n If you are an administrator of a collection, you can download a Darwin Core Archive of your collection in one of two ways:\n Navigate to your Administration Contol Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection) and click Darwin Core Archive Publishing. If you have already created a DwC-A, click the link next to \u0026ldquo;DwC-Archive File:\u0026rdquo;. If you have not yet created a DwC-A, create one. Navigate to your Administration Contol Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection) and click Download Data Backup file. The backup file will be in the format of a Darwin Core Archive, but all specimens will be included in the file, even if they don\u0026rsquo;t have unique GUIDs.  If you are not an administrator of a collection, or you would like to download a DwC-A of another collection:\n Navigate to the Sitemap of the portal (generally an option on the menu). Under \u0026ldquo;Collections\u0026rdquo;, click Darwin Core Archives (DwC-A). Find the collection for which you would like to download a DwC-A and click the link in the DwC-A column. The size of the file will be displayed in parentheses.  </p><p>data publishing</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/download\/dwc\/"},{value:"[editor] Alphabetically Sorted Index of Symbiota Data Fields (by verbatim field name)",label:"<p>Click each field to see a full definition and examples.\n Absolute Age Associated Collectors Associated Taxa Author Basis of Record Bed Biostratigraphy (Biozone) Biota (Flora\/Fauna) Catalog Number Collection Code Collector Continent Country County Cultivated Checkbox Data Generalizations Date (start) Date Identified Datum Day Day of year range Depth in Meters Description Disposition Duplicate Quantity Dynamic Properties Early Interval Element Elevation in Meters End Date Eon Epoch Era Establishment Means Family Field Number Formation Geological Context ID Georef Verificiation Status Georeference Protocol Georeference Remarks Georeference Sources Georeferenced By Group Habitat ID References ID Remarks Identification Qualifier Identified By Individual Count institutionCode island islandBody Label Project Language Late Interval Latitude and Longitude Life Stage Lithology Local Stage Locality Locality Security Location ID Location Remarks Member Month Municipality Notes Number Occurrence Id Other Catalog Numbers Owner Code Period Phenology (Reproductive Condition) Preparations Processing Status Sampling Protocol Scientific Name Sex Slide Properties Stage State\/Province Storage Age Strat Remarks Substrate Taxon Environment Taxon Remarks Type Status Uncertainty (meters) Verbatim Coordinates Verbatim Date Verbatim Depth Verbatim Elevation Water Body Year, Month, Day  </p><p>fields</p><p>data fields</p><p>terms</p><p>dwc terms</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/fields\/list_az\/"},{value:"[editor] Symbiota Data Fields",label:"<p>The Symbiota data schema is strongly aligned to the Darwin Core data exchange standard. For more details, links to the Darwin Core definitions are supplied for each term. Learn more about Darwin Core terms in the following TDWG pages:\n TDWG - Darwin Core quick reference guide TDWG - List of Darwin Core terms  Fields listed here differ from the fields visible in the data uploading tools. For field information specific to the data upload tools, see the Data Import fields page.\n Table of Contents  Standard Fields Material Sample Fields Paleontology Fields  Since portals have the ability to customize the field names found on their data entry form, field names may differ from the core field definition and how it is mapped to Darwin Core export tools.\n Download full content as a CSV See full content as a CSV\nStandard Fields  Catalog Number:  The unique identifier (primary key) for the specimen record. This field should be used to store the barcode or the accession number (herbaria only). This field is enforced to be unique per collection\nEx: WIS-L-0123456, ASU0012345, 12345\nSee Darwin Core\u0027s catalogNumber Additional Identifier Values: Any other identifier for a specimen record that is not the central catalog number. This field is typically used to store the old catalog numbers, accession numbers, National Park identifiers, etc. Identifiers can be assigned a tag name to distinguish it from other identifiers (e.g. old accession #, NPS#, etc). These identifiers map best to dwc:otherCatalogNumbers definition, and thus included in the exports under this field. More information about this system can be found on the Catalog Numbers documentation page.\nEx: 12345, TUZI 3082, NPS Acc #: GUIS-M-00126.\nSee Darwin Core\u0026rsquo;s otherCatalogNumbers\n Collector:  The name of the person who collected the specimen or made the observation.\nEx: C.G. Pringle, Goodding, L.N.\nSee Darwin Core\u0027s recordedBy  Associated Collectors:  Other collectors that were present at the time of collection.\nEx: John R. Reeder, A. Nelson\nThis field is not defined by the Darwin Core standard, which places primary and secondary collectors concatenated the recordedBy field.  Number:  The collection number assigned to the specimen by the collector.\nEx: 1294, 12490b, 94-132\nSee Darwin Core\u0027s recordNumber  Date:  The date the specimen was collected or, if a range of dates is indicated, the first day in the range of collecting dates. While dates can be entered using your preferred format, the value will be converted and stored as an ISO-8601 numeric format (YYYY-MM-DD). Note that unknown month and days can be entered as \u0026#34;00\u0026#34;. For example, a collection with a date of \u0026#34;March 1956\u0026#34; can be entered as \u0026#34;1956-03-00\u0026#34;.\nEx: 1983-09-15, 1983-07-00, 1934-00-00\nSee Darwin Core\u0027s eventDate End Date: The last date of collection, in the case of a range of collecting dates. While dates can be entered using your preferred format, the value will be converted and stored as an ISO-8601 numeric format (YYYY-MM-DD). Note that unknown month and days can be entered as \u0026quot;00\u0026quot;. For example, a collection with a date of \u0026quot;March 1956\u0026quot; can be entered as \u0026quot;1956-03-00\u0026quot;.\nEx: 1983-09-15, 1983-07-00, 1934-00-00  See Darwin Core\u0026rsquo;s eventDate\n Verbatim Date:  Can be used to record date exactly as entered on label. Particularly useful for non-standard date formats or date ranges.\nEx: Spring 1901, March-April 1952, late Sept. 1909\nSee Darwin Core\u0027s verbatimEventDate  Year:  The numeric value of the year at the time of collection. This field (along with month and day) is automatically populated when the date is entered.\nEx: 1959\nSee Darwin Core\u0027s year  Month:  The numeric value of the month at the time of collection. This field (along with year and day) is automatically populated when the date is entered.\nEx: 10\nSee Darwin Core\u0027s month  Day:  The numeric value of the day at the time of collection. This field (along with year and month) is automatically populated when the date is entered.\nEx: 28\nSee Darwin Core\u0027s day Day of year range: A range of collection dates can be represented here as numeric day of year values. These values will be automatically calculated if you enter a date range in the verbatim date field (e.g. 12 Sept 1968 to 19 Sept 1968, 1968-09-12 to 1968-09-19) See Darwin Core\u0026rsquo;s startDayOfYear, endDayOfYear.\n Scientific Name:  The Latin name of the specimen without the author. Could be anything from kingdom down to subspecies or variety, depending on the level of the identification.\nEx: Pinaceae, Pinus, Pinus edulis, Pinus edulis var. fallax\nSee Darwin Core\u0027s scientificName  Author:  The name of the person who first named the taxa. This field autofills after entering the scientific name.\nEx: L., Asa A. Gray\nSee Darwin Core\u0027s scientificNameAuthorship  Identification Qualifier:  The determiner\u0026#39;s expression of uncertainty in their identification. This will be listed on the label along with the scientific name.\nEx: cf., aff.\nSee Darwin Core\u0027s identificationQualifier  Family:  The family to which the taxa belongs. This field autofills after entering the scientific name.\nEx: Pinaceae\nSee Darwin Core\u0027s family  Identified By:  The name of the person who identified the specimen. Also called a determiner.\nEx: L. R. Landrum\nSee Darwin Core\u0027s identifiedBy  Date Identified:  The date the identification was made. Date can be entered as free form text and do not need to be in a standard date format.\nEx: 1992, May 1992, 2 May 1992\nSee Darwin Core\u0027s dateIdentified  ID References:  The reference source used to make the identification.\nEx: Nesom, Guy L. 2006. Flora of North America - Asteraceae. vol. 20\nSee Darwin Core\u0027s identificationReferences  ID Remarks:  Any additional notes regarding the identification of the specimen.\nSee Darwin Core\u0027s identificationRemarks  Taxon Remarks:  Any additional notes regarding the taxonomic name to which the specimen was identify.\nSee Darwin Core\u0027s taxonRemarks  Continent:  The name of the continent in which the specimen was collected.\nEx: North America, Africa\nSee Darwin Core\u0027s continent  Water Body:  The name of the water body in which the specimen was collected.\nEx: Pacific Ocean, Black Sea\nSee Darwin Core\u0027s waterBody  Island Group:  The name of the island group in which the specimen was collected.\nEx: Hawaiian Islands, Alexander Archipelago\nSee Darwin Core\u0027s islandGroup  Island:  The name of the island on which the specimen was collected.\nEx: Cayo Coco, Maui\nSee Darwin Core\u0027s island  Country:  The name of the country in which the specimen was collected. To aid data entry, a drop down menu will appear as one types, though names outside of the list can still be entered.\nEx: USA, Canada, Mexico\nSee Darwin Core\u0027s country  State\/Province:  The name of the state or province in which the specimen was collected. As one types, a selection list will appear for the given country.\nEx: New York, Arizona, Sonora\nSee Darwin Core\u0027s stateProvince  County:  The name of the county in which the specimen was collected. Choose one from the drop down menu. For specimens collected outside of the United States, enter the next geographic region below state\/province.\nEx: Maricopa\nSee Darwin Core\u0027s county  Municipality:  The name of the municipality in which the specimen was collected. For specimens collected outside of the United States, enter the 4th level geographic designation.\nEx: Paradise Valley\nSee Darwin Core\u0027s municipality  Locality:  The detailed location in which the specimen was collected.\nEx: 9.5 miles NW of Sedona along Boynton Pass Rd.\nSee Darwin Core\u0027s locality  Location ID:  An identifier for the set of location information (data associated with dcterms:Location). May be a global unique identifier or an identifier specific to the dataset.\nEx: https:\/\/opencontext.org\/subjects\/768A875F-E205-4D0B-DE55-BAB7598D0FD1\nSee Darwin Core\u0027s locationID Locality Security: Checking the Locality Security checkbox will hide locality details below the level of county from unauthorized users. This is typically done because the species is rare or threatened. Images are also hidden to protect locality details that might be viewable from the label. Users that are logged into the system and have the necessary permission to view locality details (e.g. collection managers) will continue to have access to all data. This box will automatically be checked if the species name is on any of the rare species lists (see sitemap). If one wishes to lock protection (on or off), click the Lock Security checkbox and\/or enter a reason for security override in the text field. Leaving the locality security unlocked will allow default settings to be applied as determined by the sensitive species administrators, which is the recommendation for most records. For more information on sensitive species protection, see the page on Sensitive Species Protection. This field is not defined by the Darwin Core standard.\n Location Remarks:  Comments or notes about the locality\nEx: Previously known as Mt. Evans\nSee Darwin Core\u0027s locationRemarks  Latitude and Longitude (decimal format):  The geographic latitude and longitude in decimal degrees. Latitudes from the southern hemisphere and longitudes in the western hemisphere (e.g. USA) should be entered as negative values. Click on the \u0026#34;Tools\u0026#34; button to enter the coordinates in the degree, minute, seconds (DMS) or the UTM formats. Decimal degrees are the preferred coordinate standard as defined by Darwin Core. See below for more information on using this tool.\nEx: 34.874022, -111.75774\nSee Darwin Core\u0027s decimalLatitude  Uncertainty (meters):  The accuracy of the georeference coordinates in meters (numeric value only). This is measured as the radius of a circle where the true point would be found if known. If coordinates are collected using a GPS, than the accuracy would be the error found within the GPS unit (usually around 10m). While previously collected specimens that have coordinates on the label recorded by the collector typically do not state the source of the coordinates (GPS, map, etc), it is typically a good assumption that the coordinates are accurate within one to two hundred meters. If the locality details are vague such as just \u0026#34;Grand Canyon\u0026#34;, then the coordinates should be the centroid within the uncertainty encompassing the greater area where the specimen may have been collected. If the locality is \u0026#34;Boynton Canyon, Sedona\u0026#34;, the uncertainty would be about 1500 m. This field autofills when using GeoLocate for georeferencing.\nEx: 42000 for Phoenix, 20000 for Salt Lake City\nSee Darwin Core\u0027s coordinateUncertaintyInMeters  Datum:  The geographic system that was used to get the coordinates. This field autofills when using [http:\/\/www.museum.tulane.edu\/geolocate\/|GeoLocate] or the Google Maps tool for georeferencing.\nEx: NAD27, NAD83, WGS84\nSee Darwin Core\u0027s geodeticDatum Verbatim Coordinates: If the coordinates recorded on the specimen label are in a format other than decimal degrees, enter them here. When decimal lat\/long fields are blank and one enters UTM or DMS using one of the formats displayed in the example below, decimal lat\/long values will be automatically generated. Click the \u0026ldquo;\u0026lt;\u0026lt;\u0026rdquo; symbol to replace existing decimal values. This field autofills when using the DMS, UTM, and TRS georeferencing tools.\nEx: 34° 13.940\u0027 N 112° 2.370\u0027 W, 12 420944E 4064025N, TRS: T40N R32E S29 See Darwin Core\u0026rsquo;s verbatimCoordinates.\n Elevation in Meters:  The elevation in meters at which the specimen was collected. Also called altitude. Use only the left field with the right field blank when a single elevation exists.\nEx: 1400, 2000-2200\nSee Darwin Core\u0027s minimumElevationInMeters Verbatim Elevation: The verbatim elevation at which the specimen was collected. This is typically used to record an elevation measurement that was recorded in feet or an uncertainty designation. When the elevation in meters field is left blank, the value will automatically be converted to meters. Click the \u0026ldquo;\u0026lt;\u0026lt;\u0026rdquo; symbol to replace the previously entered meters values.\nEx: 4500ft, 4500 feet, ca 4500\u0027, ca 2000m, 4500\u0027 \u002b-300\u0027\nSee Darwin Core\u0026rsquo;s verbatimElevation.\nDepth in Meters: The range of depth below the local surface, in meters.\nEx: 100, 150-200\nSee Darwin Core\u0026rsquo;s minimumDepthInMeters, maximumDepthInMeters.\n Verbatim Depth:  The original verbatim description of the depth below the local surface at which the specimen was collected.\nEx: 100ft, 100 feet, ca 100\u0026#39;, ca 30m, 100\u0026#39; \u0026#43;-10\u0026#39;\nSee Darwin Core\u0027s verbatimDepth  Georeferenced By:  The name of the person who georeferenced the specimen record. This field autofills when using GeoLocate for georeferencing.\nEx: A. Gonzales, emakings, acbarber\nSee Darwin Core\u0027s georeferencedBy  Georeference Protocol:  The source of the standards used to georeference.\nEx: Georeferencing Quick Guide. Zermoglio et al. 2020\nSee Darwin Core\u0027s georeferenceProtocol  Georeference Sources:  The tool or tools used to georeference.\nEx: GeoLocate, Google Earth, USGS map\nSee Darwin Core\u0027s georeferenceSources  Georef Verificiation Status:  Says whether or not the georeference has been reviewed or verified.\nEx: reviewed, not reviewed\nSee Darwin Core\u0027s georeferenceVerificationStatus  Georeference Remarks:  Any additional notes regarding the georeferencing of the specimen.\nSee Darwin Core\u0027s georeferenceRemarks  Habitat:  The description of the habitat in which the specimen was collected.\nEx: Wet areas along a small stream in chaparral.\nSee Darwin Core\u0027s habitat  Substrate:  The substrate on which the specimen was collected. Mostly used for lichen and bryophyte specimens.\nEx: On basalt, trunk of oak\nDarwin Core lumps this information into habitat.  Associated Taxa:  A list of the names of other species occurring with the collected specimen.\nEx: Quercus, Arctostaphylos, Ceanothus, Rhus, Eriogonum, Salvia\nSee Darwin Core\u0027s associatedTaxa  In MycoPortal, this is also the field where \u0026ldquo;host\u0026rdquo; information is stored. See this comment for instructions.   Description:  A physical description of the specimen at the time of collection. This often includes information that can be lost or difficult to observe after the collection and preservation process.\nEx: Shrub 3 m tall, corolla yellow\n Notes:  Any additional notes regarding the specimen.\nSee Darwin Core\u0027s occurrenceRemarks  Dynamic Properties:  A list of additional measurements, facts, characteristics, or assertions about the specimen in a format that allows programmatic parsing of the data. See the Darwin Core link below for further details.\nEx: awnLengthInMeters=0.014, heightInMeters=1.5, relativeHumidity=28, airTemperatureInC=22\nSee Darwin Core\u0027s dynamicProperties  Life Stage:  The age or stage of the organism at the time of collection\/observation. Typically used for zoological collections.\nEx: larva, juvenile\nSee Darwin Core\u0027s lifeStage  Sex:  The biological sex of the occurrence.\nEx: female, male\nSee Darwin Core\u0027s sex  Individual Count:  The number of individuals represented by the occurrence\nEx: 2, 15\nSee Darwin Core\u0027s individualCount  Sampling Protocol:  The names and references to methods used to collect or sample an occurrence\nEx: UV light trap, mist net, Takats et al. 2001. Guidelines for Nocturnal Owl Monitoring in North America. Beaverhill Bird Observatory and Bird Studies Canada, Edmonton, Alberta. 32 pp., http:\/\/www.bsc-eoc.org\/download\/Owl.pdf\nSee Darwin Core\u0027s samplingProtocol  Preparations:  Preparation or preservation method for a specimen\nEx: in ethanol, skeleton\nSee Darwin Core\u0027s preparations  Phenology (Reproductive Condition):  The reproductive stage the specimen is in. Typically used for plant and fungal collections.\nEx: flower, fruit, sterile\nSee Darwin Core\u0027s reproductiveCondition  Behavior:  The behavior exhibited by the organism\/occurrence at the time of collection\/observation.\nEx: flying, sitting on eggs\nSee Darwin Core\u0027s behavior  Vitality:  An indication of whether the organism was alive or dead at the time of collection or observation.\nEx: live, dead\nSee Darwin Core\u0027s vitality  Establishment Means:  The state of establishment at the time of collection.\nEx: cultivated, invasive, native\nSee Darwin Core\u0027s establishmentMeans  Cultivated Checkbox:  Check when the organism was established with the aid of humans and would not be able to exist on their own. This true\/false field enables the ability to filter non-native or naturalized species.\nNot currently exported in DwC format.  Type Status:  The type designation of a specimen, if it is a type specimen\nEx: HOLOTYPE, ISOTYPE, PARATYPE\nSee Darwin Core\u0027s typeStatus  Disposition:  The location or status of the physical specimen.\nEx: missing, on loan, in collection\nSee Darwin Core\u0027s disposition  Occurrence ID:  This is the Global Unique Identification (GUID) for the specimen. This identification code should be stable and uniquely identify the specimen relative to all other specimens within the world. If your collection is set to have occurrenceIDs\/GUIDs generated by the portal (this is the suggested setting for all live-managed collections), this field will appear blank in the occurrence editor form. To view the occurrenceID value associated with your specimen, click the Public Display link at the top of the page.\nEx: 000866d2-c177-4648-a200-ead4007051b9, urn:catalog:UWBM:Bird:89776\nSee Darwin Core\u0027s occurrenceID  Field Number:  An identifier given to the collecting event in the field. This number often serves as a link between the event indicated in the field notes and the specimen record.\nEx: 2024-04-05-00045, JOSHUATREE_35\nSee Darwin Core\u0027s fieldNumber  Language:  The language of the label information\nEx: English, Spanish, Portuguese\nSee Darwin Core\u0027s language  Label Project:  Used for printing labels. You can create a label project and print that set of labels after you\u0026#39;ve entered the data.\nEx: Plants of Sedona 2012\n Duplicate Quantity:  The number of duplicate specimens created. This will dictate the number of labels printed for specimen.\nEx: 10\n Institution Code:  The acronym of the institution that stewards this occurrence. Only enter a value if the institution is different than what was entered when the metadata for the institution was added to the portal.\nEx: NMNH, FLMNH\nSee Darwin Core\u0027s institutionCode  Collection Code:  The acronym of the collection that stewards this occurrence. Only enter a value if the collection is different than what was entered when the metadata for the collection was added to the portal.\nEx: Herps, F\nSee Darwin Core\u0027s collectionCode  Owner Code:  The name (or acronym) in use by the institution having ownership of the object(s) or information referred to in the record.\nEx: NPS\nSee Darwin Core\u0027s ownerInstitutionCode  Basis of Record:  The type of record the specimen is classified as. For physical collections, this field defaults to “PreservedSpecimen” (aka herbarium specimen) and for observation projects, the default is “Observation”.\nEx: PreservedSpecimen, LivingSpecimen, Observation\nSee Darwin Core\u0027s basisOfRecord  Processing Status:  The status of the digital record. This field is used for internal data management and review. The values used are dictated by the specific workflow of each institution.\n Data Generalizations:  Internal notes associated with the occurrence record. Data entered in this field is not visible to the public and is not downloaded in a Darwin Core Archive.\nMaterial Sample Fields         Material Sample tab    Controlled vocabularies for Material Sample data fields are managed per portal, and the suggested examples provided below are derived from vocabularies used for the NEON Biorepository. These vocabularies vary by portal, and modifications may require community input. Contact your Portal Administrator for more information.\n  Sample Type:  Controlled vocabulary defining the sample type, which is often anatomical in nature.\nEx: skull, liver, gastrointestinal tract, ectoparasite\n Catalog Number \/ Barcode:  A unique identifier for the material sample, analagous to _catalogNumber_ for specimen occurrences.\nEx: WIS-L-0123456, ASU0012345\nSee Darwin Core\u0027s catalogNumber  Material Sample ID (GUID):  A globally unique identifier for the material sample. In the absence of a persistent global unique identifier, construct one from a combination of identifiers in the record that will make this identifier globally unique.\nEx: 06809dc5-f143-459a-be1a-6f03e63fc083\nSee Darwin Core\u0027s materialSampleID  Condition:  Free text field to describe the physical condition of the sample. Use of a controlled vocabulary is recommended but not forced.\nEx: very poor, poor, fair, good, unknown\n Disposition:  Controlled vocabulary describing the current state of a sample with respect to its collection.\nEx: in collection, being processed, consumed, on loan\nSee Darwin Core\u0027s disposition  Preservation type:  Controlled vocabulary defining the physical storage\/preservation method of a sample.\nEx: dry, ethanol, frozen, pinned\n Preparation date:  The date of a sample\u0026#39;s physical preparation. Dates in this field visually conform to MM\/DD\/YYYY formatting. Manual data entry into this field is validated using a calendar form.\nEx: 08\/01\/2022\n Prepared by:  Name of the individual who prepared a sample. The individual must have an user account in the portal to be recorded in this field.\nEx: Liao, Rosie; Johnston, Andrew\n Preparation details:  Free text field to record notes providing more context about the physical preparation and condition of the sample.\nEx: upper and lower GI tract; kidney, left, whole; prepared with borax\n Individual count:  The number of loanable objects associated with the sample, i.e. all pieces of the sample assigned to the same unique materialSampleID (see above).\nEx: 0, 1, 100\nSee Darwin Core\u0027s individualCount  Sample Size:  Free text field to quantify the sample beyond counted number of objects, e.g. dry weight.\nEx: 200 uL\n Storage Location:  Free text field to describe a sample\u0026#39;s permanent physical storage location.\nEx: Freezer 3; Oversize Storage; Cab011, Dwr002\n Remarks:  Free text field to provide additional notes, comments, and context unique to a sample that cannot be captured by other existing data fields. Limited to 250 characters.\nEx: genotype sampling; left jaw consumed in research; with post-cranial skeleton\nPaleontology Fields         Paleo Module on the Occurrence Data tab    Controlled vocabularies for the following data fields are managed per portal. Modifications to these values may require community discussion. Contact your Portal Administrator for more information.\n Eon: The longest geologic time intervals. Ex: Archean, Proterozoic, Phanerozoic See Darwin Core\u0026rsquo;s earliestEonOrLowestEonothem, latestEonOrHighestEonothem\nEra: A subdivision of an eon that is a shorter interval of geologic time.\nEx: Paleozoic, Mesozoic, Cenozoic.\nSee Darwin Core\u0026rsquo;s earliestEraOrLowestErathem, latestEraOrHighestErathem\nPeriod: A subdivision of an era that is a shorter interval of geologic time.\nEx: Ordovician, Silurian, Devonian, Carboniferous, Mississippian, Pennsylvanian, Permian, Triassic, Jurassic, Cretaceous, Paleogene, Neogene, Quaternary.\nSee Darwin Core\u0026rsquo;s earliestPeriodOrLowestSystem, latestPeriodOrHighestSystem\nEpoch: A subdivision of a period that is a shorter interval of geologic time.\nEx: Lower, Middle, Upper Ordovician; Wenlock; Pridoli; Lower, Middle, Upper Devonian; Lower, Middle, Upper Mississippian; Lower, Middle, Upper Pennsylvanian; Cisuralian; Lower, Middle, Upper Jurassic; Lower, Upper Cretaceous; Paleocene; Eocene; Oligocene; Miocene; Pliocene; Pleistocene; Holocene.\nThe Epoch field is currently being recorded using Series, the chronostratigraphic units.\nSee Darwin Core\u0026rsquo;s earliestEpochOrLowestSeries, latestEpochOrHighestSeries\nStage: The chronostratigraphic term given to the succession of rock strata laid down in a single geochronologic age.\nEx: Lochkovian, Emsian, Eifelian, Givetian, Frasnian, Tournaisian, Serpukhovian, Moscovian, Changhsingian, Norian, Oxfordian, Hauterivian, Albian, Maastrichtian, Thanetian, Messinian, etc.\nSee Darwin Core\u0026rsquo;s earliestAgeOrLowestStage, latestAgeOrHighestStage\n Local Stage:  A local name for a stage that was applied to this specimen.\nEx: Ulatsian, Helvetian.\n Early Interval:  Name of the earliest possible geochronologic eon, era, period, epoch or age, or the lowest chronostratigraphic eonothem, erathem, system, series, or stage attributable to the stratigraphic horizon from which the cataloged specimen was collected.\nEx: Aalenian, Aeronian, Albian, Anisian, Aptian, Aquitanian, Archean, Artinskian, Asselian, Bajocian, Barremian, Bartonian, etc.\n Late Interval:  Name of the latest possible geochronologic eon, era, period, epoch or age, or the highest chronostratigraphic eonothem, erathem, system, series or stage attributable to the stratigraphic horizon from which the cataloged specimen was collected.\nEx: Aalenian, Aeronian, Albian, Anisian, Aptian, Aquitanian, Archean, Artinskian, Asselian, Bajocian, Barremian, Bartonian, etc.\n Absolute Age:  Field to record the age of specimen\/rock in years determined using radioactive decay of isotopes (Carbon-14, argon-argon, potassium-argon, uranium-lead, etc.) and other quantitative dating methods.\nEx: 20 Ma, 75 ka, 10.5 – 12.7 \u0026#43;\/- 0.5 Ma, etc.\n Storage Age:  Field for institutions that arrange collections by geologic time or biostratigraphy. The physical location of a specimen within the collection space.\nEx: Miocene, Wasatchian, Paleocene, Bridgerian, etc.\n Biota (Flora\/Fauna):  Name given to collections of fossils of the same age from a single locality or multiple localities within a specific geographic area.\nEx: Chalk Bluffs, Stewart Valley, Bridge Creek, Mazon Creek, etc.\nBiostratigraphy (Biozone): The name of the lowest possible geological biostratigraphic zone of the stratigraphic horizon from which the cataloged item was collected.\nEx: “Wa0”, “Uvigerinella sparsicostata Zone”, “Ogygiocaris”\nSee Darwin Core\u0026rsquo;s lowestBiostratigraphicZone, highestBiostratigraphicZone\nGroup: The name of the lithostratigraphic group from which the cataloged specimen was collected. The National Geologic Map Database Geolex Search is a great resource for the named lithostratigraphic units accepted by the USGS.\nEx: Bathurst, Lower Wealden, Monte Cristo, Contra Costa, Panoche, etc.\nSee Darwin Core\u0026rsquo;s group\nFormation: The name of the lithostratigraphic formation from which the cataloged specimen was collected. The National Geologic Map Database Geolex Search is a great resource for the named lithostratigraphic units accepted by the USGS.\nEx: Notch Peak, House Limestone, Fillmore, Chinle, etc.\nSee Darwin Core\u0026rsquo;s formation\nMember: The name of the lithostratigraphic member from which the cataloged item was collected. The National Geologic Map Database Geolex Search is a great resource for the named lithostratigraphic units accepted by the USGS.\nEx: Lava Dam, Hellnmaria, Brown Mountain Sandstone\nSee Darwin Core\u0026rsquo;s member\nBed: The name of the lithostratigraphic bed from which the cataloged item was collected. The National Geologic Map Database Geolex Search is a great resource for the named lithostratigraphic units accepted by the USGS.\nEx: Harlem coal\nSee Darwin Core\u0026rsquo;s bed\n Taxon Environment:  The depositional environment of the rock unit from which cataloged specimen was collected.\nEx: marine, lacustrine, non-marine, marine-non-marine\n Lithology:  Field for terms to describe the types of rock\/sediment from which the cataloged specimen was collected.\nEx: sandstone, mudstone, siltstone, shale, etc.\nSee Darwin Core\u0027s lithostratigraphicTerms  Strat Remarks:  Field to record additional details about the geology, stratigraphy, more detailed lithology description, palynological sampling info, core data, etc.\n Element:  Field to record type of plant organ cataloged specimen represents.\nEx: stem, strobilus, sterile leaf, fertile leaf, pinnule(s), rooting organ, rootlet, megasporangium, sporangium, spore, sterile axis, fertile axis, root, etc.\n Slide Properties:  Field to record types of prepared slides of specimens, noting the type of preparation and mounting medium, and to provide England Finder coordinates for palynomorph slides.\nEx: strewn, petrographic thin-section, mounted peel\n Geological Context ID:  An identifier for the set of information associated with a GeologicalContext (the location within a geological context, such as stratigraphy). May be a global unique identifier or an identifier specific to the data set.\nEx: https:\/\/opencontext.org\/subjects\/e54377f7-4452-4315-b676-40679b10c4d9\nSee Darwin Core\u0027s geologicalContextID </p><p>edit</p><p>fields</p><p>data fields</p><p>terms</p><p>dwc terms</p><p>paleontology fields</p><p>material sample fields</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/fields\/"},{value:"[about] About",label:"<p>Symbiota Docs is a central repository for documentation regarding Symbiota-based data portals. This site is maintained by the Symbiota Support Hub, but all Symbiota users are encouraged to contribute. Contact Us to contribute or provide feedback on this site.\nSymbiota is an open-source software for managing and mobilizing biodiversity data that serves over 1,400 natural history collections and publishes over 70 million occurrence (specimen or observation) records. Visit Symbiota.org to learn more. The core Symbiota code is developed at the Arizona State University Biodiversity Knowledge Integration Center. The central code for this version of Symbiota can be found in GitHub.\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/about\/"},{value:"[categories] Categories",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/categories\/"},{value:"[citing] Citing This Website",label:"<p>To cite the entire website use this snippet: Symbiota Support Hub (2021). Symbiota Documentation. https:\/\/biokic.github.io\/symbiota-docs\/\nTo cite one particular page, you can use this template, or navigate to the bottom of the desired page and copy it from there: [Content authors, editors and translators full names]. [Page Title]. In: Symbiota Support Hub (2021). Symbiota Documentation. [Permalink]. Created on [DD month YYYY], last edited on [DD month YYYY].\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/citing\/"},{value:"[faq] Frequently Asked Questions",label:"<p> What is Symbiota?    Symbiota is an open source software that provides content management for biodiversity data while serving as a data access portal and aggregator. Learn more about Symbiota on the project website.   I would like to use Symbiota for my collection. What should I do?      Visit the list of existing Symbiota portals and determine whether your collection fits within the scope of any of these portals.\n  Contact the portal administrator or help desk email of the portal you would like to join. If you can\u0026rsquo;t find this, fill out this form to request to join an existing Symbiota portal.\n     How do I get my data into a Symbiota portal?    Once you\u0026rsquo;ve joined a portal (see FAQ question above), you can import your data as a spreadsheet (converted into CSV format) or as a Darwin Core Archive. Instructions for uploading your data can be found on this page; however, it\u0026rsquo;s advised to get help the first time you upload data. For assistance, email the Symbiota Support Hub.   How do I upload images into a Symbiota portal?    Free image storage in Symbiota portals is not unlimited, and therefore, there are no batch uploading tools in the portals. Please contact the Symbiota Support Hub for image hosting options.\nIf you host images on your own web-accessible servers (i.e., all of your images have public URLs), you can use the Image URL Mapping tool to associate these images with your occurrence records.\nFor a more comprehensive description of how images are handled in Symbiota portals and your options for image hosting, watch our recorded webinar on this topic.\n   How do I change the description and contact information for my collection?    Instructions for editing your collection metadata can be found on this page.   How do I request access to redacted data in a Symbiota portal?    Depending on the portal, the Symbiota Support Hub may be able to help you get the right permissions for your request. Please fill out this form, providing as much information as possible, to request access.   How do I add taxonomic names to a Symbiota portal?    If the taxonomic name(s) you\u0026rsquo;d like to add are associated with specimens in your collection, you may be able to automatically add those names using the Taxonomy Cleaning Tool. This tool will auto-import any names that it finds in the Catalog of Life that match names in your records.\nWhen this tool does not prevail, contact your portal administrator or the Symbiota Support Hub about adding names to the taxonomic thesaurus. It helps to provide a list of references (e.g., publications, online articles) associated with the names you\u0026rsquo;d like to add.\n   What do I do if I don’t remember my username or password?    Locked out of your account? Don\u0026rsquo;t create a new one! Click the Reset Password or Retreive Login link on the login page to your portal.\nIf you don\u0026rsquo;t receive the password reset email shortly, check your spam folder. If you still don\u0026rsquo;t see it, contact the Symbiota Support Hub to hard reset your password.\n   </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/faq\/"},{value:"[search] Site Search",label:"<p>    let params = { json_src: \u0027..\/index.json\u0027, minlength: 3, defaultsearch: \u0027AND\u0027, sort_date: \u0027DESC\u0027, autocomplete: 1, section_search: 0, badwords: \u0027and,there,the,this,that,you\u0027, json_wait: \u0027Loading\n\u0027, json_ready: \u0027\n\u0027, extern_icon: \u0027 (external link)\u0027, err_badstring: \u0027Too short!\n\u0027, err_noresult: \u0027Unfortunately no search results were found. Please try again with different terms.\n\u0027, err_norequest: \u0027The full-text search is currently not available. Please try again later.\n\u0027, err_filefailed: \u0027Search file could not be retrieved.\n\u0027, res_one_item: \u0027[CNT] result\n\u0027, res_more_items: \u0027[CNT] results\n\u0027, res_out_top: \u0027\u0027, res_out_bottom: \u0027\u0027, res_item_tpl: \u0027[TITLE]\n[SUMMARY]\n[SECTION][TAGS] ([DATE])\u0027, add_searchlink: \u0027Not satisfied with these results? Try the external search via DuckDuckGo...\n\u0027 }; let section_trans = { \u0022post\u0022: \u0022Blog\u0022, }; let searchfield_weight = { \u0022title\u0022: 5, \u0022tags\u0022: 5, \u0022summary\u0022: 2, \u0022content\u0022: 1 };   </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/search\/"},{value:"[contact] Still Have Questions?",label:"<p>Can\u0026rsquo;t find the answer to your question or have a specific problem? We\u0026rsquo;re here to help!\nPlease submit your question as a new Discussion on our Discussion Page.\nGo to the button highlighted here in yellow to create a new topic.\nNote that you can select a category to which the discussion belongs. Categories are highlighted in yellow in this image above. For example, if you are experiencing a specific problem, you can use the category \u0026ldquo;Problems\/Bugs\u0026rdquo;. If you are asking a question, you can use the \u0026ldquo;Q\u0026amp;A\u0026rdquo; category.\nAlternatively, you can always email the Symbiota Support Hub!\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contact\/"},{value:"[tags] Tags",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/tags\/"},];$("#search").autocomplete({source:projects}).data("ui-autocomplete")._renderItem=function(ul,item){return $("<li>").append("<a href="+item.url+" + \" &quot;\" +  >"+item.value+"</a>"+item.label).appendTo(ul);};var searchUrl="search/?query=";$('#search').keypress(function(e){if(e.which==13){window.location.href=searchUrl+$('#search').val();}});});</script></div></div></div></div></header><section class=section><div class=container><div class="row justify-content-center"><div class="col-12 text-center"><h2 class=section-title>Browse articles by subject</h2></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/user/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-search icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">User Guide</h3><p class=mb-0>Learn to access, view, and download data and checklists.</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/editor/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-marker-alt icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Editor Guide</h3><p class=mb-0>Learn to edit, manage, import, and export data in Symbiota portals</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/coll_manager/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-panel icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Collection Manager Guide</h3><p class=mb-0>Learn to clean data, edit permissions, upload data, and perform other administrator functions.</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/col_obs/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-pencil-alt icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Collector/Observer Guide</h3><p class=mb-0>For collectors of data and/or specimens: learn to add, edit, and transfer your own collections in a Symbiota portal</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/portal_manager/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-server icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Portal Manager Guide</h3><p class=mb-0>Learn to change configuration files, access the Symbiota backend, and perform other advanced functions</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/dev/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-harddrive icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Developer Guide</h3><p class=mb-0>Learn to edit and develop Symbiota code</p></a></div></div></div></section><section><div class=container><div class=row><div class=col-12><div class="section px-3 bg-white shadow text-center"><h2 class=mb-4>Still need help?</h2><p class=mb-4></p><a href=/symbiota-docs/contact class="btn btn-primary">contact us</a></div></div></div></div></section><footer class="section pb-4"><div class=container><div class="row align-items-center"><div class="col-md-8 text-md-left text-center"><p class="mb-md-0 mb-4">© 2024 <a href=https://symbiota.org>Symbiota Support Hub</a></p></div><div class="col-md-4 text-md-right text-center"><ul class=list-inline><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li></ul></div></div></div></footer><script src=https://biokic.github.io/symbiota-docs/js/script.min.js></script></body></html>