<!doctype html><html lang=en-us><head><meta charset=utf-8><title>Symbiota Docs</title><meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=1"><meta name=generator content="Hugo 0.79.1"><link rel=stylesheet href=/symbiota-docs/plugins/bootstrap/bootstrap.min.css><link rel=stylesheet href=/symbiota-docs/plugins/themify-icons/themify-icons.css><link rel=icon href=/symbiota-docs/images/favicon.png type=image/x-icon><link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel=stylesheet><style>:root{--primary-color:#428500;--body-color:#f9f9f9;--text-color:#636363;--text-color-dark:#242738;--white-color:#ffffff;--light-color:#f8f9fa;--font-family:Lato}</style><link href=https://biokic.github.io/symbiota-docs/css/style.min.css rel=stylesheet media=screen><link href=https://biokic.github.io/symbiota-docs/css/custom.min.css rel=stylesheet media=screen><script src=/symbiota-docs/plugins/jquery/jquery-1.12.4.js></script><script src=/symbiota-docs/plugins/jquery/jquery-ui.js></script><script src=/symbiota-docs/plugins/bootstrap/bootstrap.min.js></script><script src=/symbiota-docs/plugins/match-height/jquery.matchHeight-min.js></script></head><body><header class="banner overlay bg-cover" data-background=https://biokic.github.io/symbiota-docs/images/annie-spratt-5cFwQ-WMcJU-unsplash.jpg><nav class="navbar navbar-expand-md navbar-dark"><div class=container><a class="navbar-brand px-2" href=/symbiota-docs>Symbiota Docs</a>
<button class="navbar-toggler border-0" type=button data-toggle=collapse data-target=#navigation aria-controls=navigation aria-expanded=false aria-label="Toggle navigation">
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse text-center" id=navigation><ul class="navbar-nav ml-auto"><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/>Home</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/about>About</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/faq>Faq</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/citing>Citing The Docs</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/contact>contact</a></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle text-dark" href=# role=button data-toggle=dropdown aria-haspopup=true aria-expanded=false>docs</a><div class=dropdown-menu><a class=dropdown-item href=https://biokic.github.io/symbiota-docs/user>User Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/editor>Editor Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/coll_manager>Collection Manager Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/coll_manager>Portal Manager Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/dev>Developer Guide</a></div></li></ul><select class=lang-list id=select-language onchange="location=this.value;">
<option id=en value=https://biokic.github.io/symbiota-docs/ selected>En</option>
<option id=es value=https://biokic.github.io/symbiota-docs/es/>Es</option>
<option id=fr value=https://biokic.github.io/symbiota-docs/fr/>Fr</option></select></div></div></nav><div class="container section"><div class=row><div class="col-lg-8 text-center mx-auto"><h1 class="text-white mb-3">Symbiota Documentation</h1><p class="text-white mb-4">Growing archive of help files for one of the most popular Natural History content management systems</p><div class=position-relative><input id=search class=form-control placeholder="Try searching for terms here">
<i class="ti-search search-icon"></i><script>$(function(){var projects=[{value:"Compare Duplicate IDs Across Collections",label:"<p> This page describes how you can identify specimens in your collection that have different identifications (i.e., scientific names) than duplicates of that specimen housed at another collection.\n Navigate to this tool through the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Duplicate Cleaning Tools. In the bottom box under Taxonomy, click \u0026ldquo;Duplicate specimens with potential identification conflicts\u0026hellip;\u0026rdquo;.\nThis tool lists the duplicate clusters of specimens (specimens that have the same collector, number, and date, even across different collections) for which one or more of the specimens does not have the same taxonomic name. The tool can be used to identify whether there are specimens in your collection that should be annotated to the most recently identified taxonomic name. For example, in the cluster below, OBI and UCR may consider annotating their specimens according to the most recent identification of the SBBG duplicate. The specimen records associated with each specimen can be viewed by clicking on the catalog number of the desired record.\n</p><p>data cleaning</p><p>taxonomy</p><p>duplicates</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/id_conflict\/"},{value:"User Guide",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/"},{value:"Batch Uploading\/Linking Images",label:"<p> This page describes how to upload multiple images at a time or link multiple images from an external location.\n There are several workflows used to batch upload\/link specimen images. Batch processing typically consists of two separate steps: 1) Batch loading images onto a web server, and 2) Mapping image URLs to specimen records residing within the Symbiota portal. Technically, images are not stored within a Symbiota database. Instead, the URLs of the images are mapped to specimen records. Contact the portal manager to inquire about the workflow details supported by a given portal.\nImages uploaded or linked to specimens in a Symbiota portal should be JPGs or in another web-friendly format. It is recommended that individual files are no larger than 8 MB to allow users with slower connectivity to view them within a reasonable timeframe.\n Common Workflows   Image Drop Folder: Using a mapped network drive, FTP, SFTP, Dropbox, or other cloud storage, a drop folder is made available so that imaging team can regularly deposit new images. This folder is referred to as the \u0026ldquo;source folder\u0026rdquo;. Once images are uploaded to the source folder, you can use the Processing Toolbox to create web-ready derivatives and move the images to a local server. The derivatives are then mapped to specimen records through image URLs.\n  Local Storage: If images are stored on a local server that is write-accessible to the portal code, image processing can be performed directly through the portal interface. Image URLs stored locally can be mapped into the database using relative links without the domain name. In this case, the the web server user (e.g. Apache user) must have write access to both the source and target folders.\n  Remote Image Storage: Images can also be processed and stored on a remote server and mapped to the specimen image through the full image URL. Standalone image processing scripts will be needed to process images and map image URLs to the portal database. Scripts can be configured to write the image URL directly to the database or image metadata can be written to a log file, which is loaded into the database afterwards. Remote images must be mapped into the database using the full image URL with the domain name.\n If only a large image is made available from the remote server, the image URL can be mapped to the urloriginal field, and then portal will then create local web and thumbnail dirivatives. If the images are named using the catalog number, and the web server is configured to display directory contents, the Processing Toolbox within the collection\u0026rsquo;s management menu contains a method to harvest image links from the directory listing. Alternatively, one can perform a \u0026ldquo;skeletal record import\u0026rdquo; that contains a column with the catalogNumber and another associatedMedia column with the image URL.    CyVerse Image Storage: This method requires an agreement of a collection with CyVerse. CyVerse can store images as Community Data files and use BisQue to create web-accessible URLs for these files. You can then use the Processing Toolbox to map CyVerse URLs to specimen records in your portal.\n  </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/images\/batch\/"},{value:"Duplicate Catalog Numbers",label:"<p> This page describes how to identify and merge (when necessary) specimens with duplicate catalog nubmers or other catalog numbers.\n This tool can be found in the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection).\nTo use the tool, first select whether you would like to list duplicates based on Catalog Numbers or Other Catalog Numbers from the “List Duplicates based on\u0026hellip;” box. If duplicates exist, a table of duplicate records will appear on the resulting page.\nFrom here, you can evaluate whether the duplicate records should be edited, merged, or ignored. To view and\/or edit a record, click the number in the ID column next to the record you wish to view. This will open that specimen’s Record Editor page.\nIf you decide that the two records should be merged, check the boxes next to both duplicates, then select the radio button that corresponds to the record you wish to make the primary record. If both records have a value in a given field, Symbiota will keep the value that belongs to this primary record and discard the other value. When you are satisfied that the correct information will be retained, click the Merge Duplicate Records button.\nIf there are many duplicate records to evaluate, you can click the checkbox at the top of the checkbox column. This will check all of the boxes in the entire table. You can also select the radio button column header, which will allow Symbiota to auto-select a record to retain for each pair. You will want to vet this list to ensure that the correct record is selected for retention.\n </p><p>data cleaning</p><p>duplicates</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/dupes\/"},{value:"Linking New Images to Old Records",label:"<p> This page addresses a common digitization workflow challenge: how do you link images of specimens (with new barcode numbers) to specimen records that already exist in the portal if the barcode number isn\u0026rsquo;t associated with the old data yet? Here we describe two options to solve this problem.\n The following instructions assume that the barcode number is stored in the Catalog Number field, and the accession number (usually a stamped number) is stored in the Other Catalog Numbers field.\n Option 1: Add catalog (barcode) numbers to occurrence records while imaging This protocol describes how to add a barcode number to each of the specimens that you image during the imaging process.\n (Example instructions for an herbarium collection)\n Acquire a specimen. Log in to your account and navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Edit Existing Occurrence Records. Place a barcode on your specimen. Look for a stamped number (“accession number”) on the specimen.  If a stamped number does not exist, proceed to step 8.   In the Record Search Form, enter the stamped number into the Other Catalog Numbers field. Do not include leading zeros (e.g., if the stamp is 01499, enter 1499). Click Display Editor.  If no record is returned, double check that you entered the accession number correctly into the Other Catalog Numbers field and that there are no extra spaces before or after the numbers you typed. If the number is correct, contact a supervisor. If a record is returned, check that the data on the specimen record match the data on the label of the specimen in front of you, particularly collector, collector number, and date. i. If these data match, place your cursor in the Catalog Number field (top left) and scan the barcode of the specimen in front of you. The barcode number will then appear in this field. Press Tab on your keyboard, followed by Enter. Proceed with imaging the specimen and return to step 1. ii. If these data do NOT match, contact a supervisor.    To re-open the Record Search Form after you have performed a search, click the magnifying glass icon at the top right side of the specimen record.\n In the Record Search Form, enter the last name of the collector preceded by a percent sign (%), the collector number, and the date (in YYYY-MM-DD format) into the top three fields (see example below) and click Display Editor.  If no record is returned, double check that you entered data correctly into the appropriate fields. If everything looks correct, contact a supervisor. If a record is returned, check that the data on the specimen record match the data on the label of the specimen in front of you, particularly collector, collector number, and date. i.\tIf these data match, place your cursor in the Catalog Number field (top right) and scan the barcode of the specimen in front of you. The barcode number will then appear in this field. Press Tab on your keyboard, followed by Enter. Proceed with imaging the specimen and return to step 1. ii.\tIf these data do NOT match, contact a supervisor.    Option 2: View new images in the portal and link by manually entering the accession number This protocol describes how to update your old accession number with the specimen’s new barcode number. You can do this only after you have uploaded specimen images into your portal through a batch process that has created new \u0026ldquo;unprocessed\u0026rdquo; records consisting of only a barcode number and an image.\n (Example instructions for an herbarium collection)\n Log in to your account and navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Edit Existing Occurrence Records. In the Record Search Form, click the Processing Status field and select “Unprocessed” from the dropdown menu. In the Custom Field 1 field, select Other Catalog Number from the first dropdown menu and IS NULL from the second dropdown menu. Click the “Display Editor” button. Locate the accession number (the stamped number, rather than the barcoded number or the collector number) on the specimen in the image to the right of the Occurrence Editor form.  You can zoom in by pressing Command (Mac) or Control (Windows) and clicking on the image where you want to zoom in. Alternatively, you can hold Shift, click on the area where you want to zoom in, and move the mouse up (to zoom in) or down (to zoom out).   Enter the accession number, without leading zeros (e.g., if the number reads “0145”, you will enter “145”), into the Other Cat. #s field (circled in screenshot below).  Click outside of the Other Cat. #s field (or press the Tab button). A message should show up under the “Dupes?” button. Make sure that your browser’s pop-up blocker is disabled so you can see these messages.  If the message shows “No Dupes Found”:  Check that the specimen sheet is not stamped “Databased” or has any other indication that it SHOULD have a record in the database. If it is, try adding a leading zero to the accession number and repeat step 9. Press the Save Edits button (or press Tab and Enter on your keyboard) Move on to step 9.   If a pop-up window shows up that says “Record(s) using the same identifier already exists. Do you want to view this record?”:  Click OK. Check that the identified duplicate does not already have an image associated with it (if it does, you will see a bold barcode number on that record). If the duplicate does have an image already, close the window, scroll down to the Processing status field in the Curation box, select “Expert Required” from the dropdown menu, and then click the Save Edits button. Otherwise, move to step c. Check that the data of the identified duplicate matches the data on the label of the specimen image you were viewing. You need not check every field, just two or three important fields such as Scientific Name, Collector, and Locality. If the data match up, click “Merge Records” and exit the duplicate window. If the data do not match up, close the window, scroll down to the Processing status field in the Curation box, select “Expert Required” from the dropdown menu, and then click the Save Edits button.     Click the double arrow icon at the top right corner of the occurrence editor. Repeat steps 6-9.  </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/link\/"},{value:"Editor Guide",label:"<p>Welcome to the documentation pages of Symbiota Docs! This site is under construction.\nThe following Symbiota guides and websites will help you get started:\n Symbiota Guide (developed for the Consortium of California Herbaria CCH2 data portal) Symbiota YouTube Channel  </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/"},{value:"Collection Manager Guide",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/"},{value:"Geography Cleaning Tools",label:"<p> This page describes how to use the two geographic cleaning tools in Symbiota portals to batch clean geographic data for occurrences.\n The two geographic cleaning tools are the Geographic Distribution viewer and the Geography Cleaning Tool.\nGeography Distribution Viewer Navigate to this tool through the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Data Cleaning Tools, then Geographic Distributions.\nThe Geography Distribution viewer can be used to examine the countries, states, and counties that exist in your database. Any misspellings, non-standardized entries (e.g., “USA” rather than “United States”), or suspected mistakes (e.g., “United Arab Emirates” rather than “United States”) can be detected using this tool. To view the state\/province values for each country, click the name of the country. Then, to view the county values for each state\/province, click the name of the state\/province.\nA user with administrator permissions can correct errors in country, state\/province, and\/or county individually by clicking the number next to the place name (circled on next screenshot), or you can search for those records using the record search form and batch edit them.\nGeography Cleaning Tool Navigate to this tool through the Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Data Cleaning Tools, then Geography Cleaning Tools.\nThe Geography Cleaning Tool will search your database for non-standardized geographical terms (countries, states\/provinces, and counties) used in your collection. These will be listed as “questionable.” To view and potentially edit these records, you can click the “List [terms]\u0026hellip;” link (one example circled below).\nThis tool will also check whether there are records that lack data in the country, state\/province, or county fields, yet have geographic data in other fields. For example, the line “Null country with non-Null state\/province” lists all records that do not have a country value in the specimen record even though there is a state or province listed in the state\/province field of that record. You can click “List records\u0026hellip;” and assign a higher geographic value to those records (see example below).\nSimilar lists are provided for records with empty state\/province fields but filled county fields and for records with empty county fields but filled locality fields.\n</p><p>geography</p><p>data cleaning</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/geography\/"},{value:"Adding Specimens to Crowdsourcing Queue",label:"<p> This page describes how a collection manager or administrator can add specimen records to the queue to be crowdsourced and how to review crowdsourced transcriptions.\n A webinar demonstrating this process can be found here: https:\/\/youtu.be\/1IZhUMqCGvs\n Specimens can be crowdsourced for transcription if they (1) have an associated image, and (2) have a processing status of \u0026ldquo;unprocessed.\u0026rdquo; To manage the processing status of specimens, you can use the Batch Editor Tool if necessary. Specimen images that are added to a portal without associated data are automatically assigned a processing status of \u0026ldquo;unprocessed\u0026rdquo;; however, if skeletal data were uploaded with the images, the processing status was defined by the person who uploaded the data.\n Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). In the Administration Control Panel, click Processing Toolbox. Click the Crowdsourcing tab.  Don\u0026rsquo;t see a Crowdsourcing tab? Crowdsourcing may not be enabled in your portal. Contact your portal administrator for more information.\n On the resulting page, you will see tallies of how many records are in the queue, how many have been processed, and how many could be added to the queue. Click the Add to Queue button to see additional options for refining which specimens you will include in the queue (shown in screenshot below).  You can\u0026rsquo;t conduct a search for values that don\u0026rsquo;t exist! Many \u0026ldquo;unprocessed\u0026rdquo; records don\u0026rsquo;t have data for any fields; therefore, they might not be included if you narrow your search by one of the additional fields shown above.\n Define how many specimens you will add to the queue at one time by modifying the number in the Record Limit field.  It is recommended to not add many thousands of records to the queue at one time. Too many records may leave your crowdsourcers feeling hopeless to finish the task!   Click the Add to Queue button.  </p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/crowdsource\/edit\/"},{value:"Removing Specimens from Crowdsourcing Queue",label:"<p>Specimens will remain in the crowdsourcing queue until their processing status has been changed from \u0026ldquo;unprocessed\u0026rdquo;. Use the occurrence editor or the Batch Editor tool to change the processing status of the specimen or specimens that you would like removed from the queue.\n</p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/crowdsource\/remove\/"},{value:"Reviewing and Approving Crowdsourcing",label:"<p> This page describes how to review and approve edits that have been made to records in the crowdsourcing module.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). In the Administration Control Panel, click Processing Toolbox. Click the Crowdsourcing tab.  Don\u0026rsquo;t see a Crowdsourcing tab? Crowdsourcing may not be enabled in your portal. Contact your portal administrator for more information.\n If there are records that need approval, they will show up after Pending Approval. Click Review to review them.  Alternatively, you can review records edited by individual users by clicking Review in the Pending Review column in the tables at the bottom of the page.   Review records in the provided table, or click the pencil icon to view individual records along with their images. To approve records, click the check box next to the records you wish to approve, scroll down to the bottom of the page, and click Submit Reviews.  Click \u0026ldquo;Additional Options\u0026rdquo; at the bottom of the page for more options. From here you can, instead of reviewing a record, \u0026ldquo;Remove Points and change to Not Reviewed\u0026rdquo; (if the record had previously been reviewed\u0026quot; or \u0026ldquo;Move back into crowdsourcing queue as Open Records\u0026rdquo;\n </p><p>crowdsourcing</p><p>volunteer</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/crowdsource\/review\/"},{value:"Crowdsourcing (Volunteer Instructions)",label:"<p> This page provides basic instructions that you can give to your crowdsourcers to get them started in the crowdsourcing module. Note that you will want to add specific transcription instructions for your collection.\n Creating an Account First, you’ll need to create an account in the portal for which you will be crowdsourcing.\n Navigate to the portal in which you will be georeferencing. On the right side of the grey menu, click New Account. Enter the requested information, check the I’m not a robot box, and click Create Login.  After you’ve created an account once, you will use this login information for all future sessions. Just click Login and enter your username and password.\n Navigating to Records to Transcribe  Once you are logged in, click the Crowdsourcing tab from the main menu. Your stats will be shown in the top right side of the page under Your User Status. Here you can see how many specimens you have transcribed, as well as how many points are awaiting approval and how many have been approved.  Pending points are those that have not yet been reviewed by a curator  Curators are very busy, please be patient if your pending points don’t turn into approved points very quickly!   Approved points are those that have been reviewed and approved by a curator   In the bottom table that reads Your Stats by Collections, decide which herbarium you would like to work on (or select the herbarium instructed by your event leader). Click the number in the Open Records column that belongs to that collection’s row. On the resulting page, you’ll see a table of specimen records that require transcription. Randomly select one of the specimens by clicking on its Symbiota ID number (leftmost column).  If you see a message that says Record Locked!, go to the next specimen by clicking the double arrows at the top of the form, just under the magnifying glass   Transcribe the record as instructed.  This material is based upon work supported by the National Science Foundation under Grant Numbers 2001500, 2001509, 2001394, 2001422, 2001427, 2001431, 2001299, 2001354, 2001288, and 2001345. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation.\n</p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/crowdsource\/individual\/"},{value:"Contribute",label:"<p>How to Contribute to Symbiota Docs Symbiota Docs is a collaborative effort of the Symbiota community, coordinated by the Symbiota Support Hub. We welcome and encourage all contributions.\nContent on this site managed through GitHub.\nIf you are famililar with GitHub or willing to learn to manage content in GitHub:\n Set up a user account in GitHub. Familiarize yourself with GitHub Markdown. Contact the Symbiota Support Hub indicating your intent to contribute via GitHub. We will meet with you to discuss your contributions.  If you are not familiar with GitHub or unwilling to learn (don\u0026rsquo;t be afraid to learn new things!).\nSend your contributions as Word or text documents to the Symbiota Support Hub. Make sure you include your full name so we can credit you as the author! Please include any images as separate files.\n</p><p>contribute</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contribute\/"},{value:"Uploading Images",label:"<p> This page describes how to upload individual field and specimen images directly into a Symbiota portal. It is also possible to link images that are stored in external servers. For information about the latter option, visit the Batch Adding\/Linking Images page.\n There are three catefories of images that can be linked to a Symbiota portal. Instructions for uploading each of these types are provided below.\nSpecimen images To add images individually:  Log in to your account in the portal. Navigate to the Occurrence Editor page of the specimen to which you would like to add an image (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection \u0026gt; Edit Existing Occurrence Records \u0026gt; conduct search for specimen) Click the Images tab.  If an image has already been added to the specimen, and you wish to add another click the green plus sign at the top right of the window. Otherwise, the image uploading interface will be highlighted in yellow on this page. Click the Choose File button and navigate to the image file you wish to upload. If you instead have a URL to the image, you can click Enter URL and paste the URL into the resulting field instead. Enter any additional information, if desired, in the other image uploading fields.  You can enter a number into the Sort field if you wish to define the order in which mutliple images will appear on the occurrence page. If you enter a number greater than 500 in the Sort field, that image will be displayed on the Occurrence Details page but on the Taxon Profile page for that taxon. This is ideal for poor-quality or sensitive images (e.g. road kills).\n Click the Submit New Image button.  To batch add images: Batch adding images can be facilitated in multiple ways but requires discussion with a portal manager. Contact the manager of your Symbiota portal to establish an agreeable workflow.\nField image without location information Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed on the Taxon Profile Page, which has general information such as descriptions, distribution maps, synonyms, and common names. To upload an image:\n Log in to your account in the portal. You must have Taxon Profile Editor permissions to do the following\n  Navigate to the page of the taxon you wish to edit. To do so, you may be able to:  Click Sitemap, then Taxonomic Tree Viewer or Taxonomy Explorer. Search for the taxon of interest and click its name. Perform a quick search on the home page for the taxon of interest.   Click the pencil icon at the top right corner of the taxon page. Click the Add Image tab. Select the image file you would like to upload from Choose File, then enter any additional information in the provided fields.  The Sort Sequence field allows you to determine the order of the images that will show up on the taxon profile. The higher the number, the further down the priority list the image will be.   Click the Upload Image button. Field images are uploaded and managed through the Taxon Profile Editing interface. Users with Taxon Profile editing permissions can submit an image by clicking on the editing symbol located in the upper right of any Taxon Profile page, or through the image submission links available on the sitemap page. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see below).  Image Vouchers (field images with location information) Contribute to this section!\n</p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/upload\/"},{value:"Joining an Existing Portal",label:"<p>You can find a list of active Symbiota portals at this site. You will want to select a Symbiota portal that matches the scope of your data.\nOnce you have identified an existing Symbiota portal to which you want to publish specimen data, contact the project or portal managers to become a data provider. Portal contacts are typically listed on the homepage or within the left menu of the data portal. Contact symbiota@asu.edu for help if you are unable to find the portal contact.\nYou may publish data to more than one portal; however, it is recommended to only manage a particular dataset or collection in one Symbiota portal, and all other portals would host only a snapshot of those data. Different collections (e.g., vascular plants vs. lichens vs. bryophytes) can be live-managed on their respective portals.\nTypically, the portal manager will need the information in the following list to initiate your collection within the portal. Once the collection has been added, data can be loaded. See the page on Uploading Specimen Data for details on methods and acceptable data formats for upload specimen.\nInformation needed to create a new collection within a portal:\n Collection title that will be listed on the search page Institution and collection codes (e.g. acronyms) Short description of collection holdings and preparation types Contact(s): name, title, email, and ORCID, if available Link(s) to homepage and\/or other external resources Collection or institution icon – a small image that visually represents the collection, icons that are roughly the same height and width work best Preferred usage rights. The portal manager should be able to advice you on this. It may also be useful to read the following Creative Commons links: https:\/\/creativecommons.org\/licenses\/, http:\/\/wiki.creativecommons.org\/Data Whether you are publishing a snapshot of your data that will be periodically updated, or you plan to manage the data as a live dataset directly within the portal Source of the Global Unique identifier (GUID) for the specimen records. If the specimen records are managed directly within the portal, we recommend letting the portal automatically generate a GUID (UUID) when records are created. If a data snapshot is published from a local in-house data management system, GUIDs need to be generated locally and then included in the data published to the portal.  </p><p>join</p><p>contribute</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/join\/"},{value:"Taxonomic Cleaning Tools",label:"<p> This page describes how to use the two taxonomic cleaning tools in Symbiota portals. For more information about the taxonomic thesaurus, visit the Taxonomy page.\n The taxonomic cleaning tools are great resources for cleaning up misspellings and other taxonomic name entries that do not link to names in the central taxonomic thesaurus. They are meant to aid in locating and fixing taxonomic errors and inconsistencies.\nAnalyze Taxonomic Names   This function analyzes the values that exist in the Scientific Name field in your database and flags the values that are not linked to the taxonomic thesaurus (i.e., are not linked to a recognized name). Note that this tool does NOT evaluate whether that name is accepted according to current taxonomy. The number of unrecognized scientific names is listed at the top of the Action Menu box.\nYou can evaluate the unrecognized scientific names and link them to the names of the correct taxa by clicking the name of the Taxonomic Resource to which you want to compare the names, selecting the target kingdom, and clicking the Analyze Taxonomic Names button. If you wish to begin at a certain name (e.g., you have already checked names up to Mentzelia), you can enter this name in the Start Index field. You can also alter the number of names you would like the portal to analyze per run in the Names Processed per Run field. There are multiple types of results from running this tool. In general, the tool will list the name that it is attempting to index (i.e., match to an existing taxonomic name), search your defined taxonomic resource for that name, and, if that name is not found in the taxonomic resource, check the taxonomic thesaurus for similar names to the unrecognized name. You then have several options for resolving the unrecognized name, depending on the type of result. To view the specimen(s) associated with any unrecognized name, click the pencil icon to the right of the name and number of specimens, in brackets. The types of results are explained in more detail below.\nExample 1: Misspelling, incomplete name, or orthographic variant This is the most common type of result. In this case, you discover that the unrecognized name was misspelled or spelled differently than the recognized name. You can click “remap to this taxon” to change the name of the specimen(s) to this recognized name.\nExample 2: Unpublished or unrecognized taxon name In this case, you find that the name listed on the specimen was not published or is not currently listed in the taxonomic thesaurus. You must then use taxonomic expertise to decide whether this specimen should be remapped to a different taxonomic name (if you are confident that they are synonymous) and\/or annotated, or if the taxonomic name should be left as is and it should be included in the taxonomic thesaurus. If you believe that a name should be found in the taxonomic thesaurus and you do not find it with a manual search, contact the portal manager.\nExample 3: Name exists but is not found in taxonomic thesaurus When the portal does not find a name in the taxonomic thesaurus, but it does find the name in the taxonomic resource, it will import the taxonomic name into the taxonomic thesaurus. This will automatically map the specimen’s taxonomic name to this newly added entry in the taxonomic thesaurus as well.\nIf you have not analyzed all the taxonomic names in one pass, you can click the Continue Analyzing Names button to have Symbiota check the next 20 (or whatever number is defined by the user) names.\nTaxonomic Distributions Much like the Geographic Distribution viewer, the Taxonomic Distribution viewer can be used to examine the families, genera, species, and infraspecific taxa that exist in your database. Misspellings, non-standardized entries, or suspected mistakes can be detected using this tool. To view the genera for each family, click the name of the family. Then, to view the species for each genus, click the name of the genus, and so on. A user with administrator permissions can correct errors in taxonomic names individually by clicking the number next to the taxonomic name (circled below), or the user can search for those records using the record search form and batch edit them.\n</p><p>taxonomy</p><p>data cleaning</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/taxonomy\/"},{value:"Adding Records",label:"<p>There are several ways to add occurrence records, depending on your level of permissions. Editors can add specimens as skeletal records, as full records, and by uploading an image.\n</p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/"},{value:"Annotations \/ Determinations",label:"<p> This page describes how to add annotations individually or in batch.\n Annotations (also called Determinations) in Symbiota portals are taxonomic identifications provided for an occurrence.\nAdding Individual Determinations\/Annotations To annotate an individual record or add previous determinations, navigate to the specimen in the Occurrence Editor (My Profile \u0026gt; Occurrence Records \u0026gt; name of collection \u0026gt; Edit Existing Records) and click the Determination History tab. If no previous annotations exist, you will see a message that says “There are no historic annotations for this specimen” and a box labeled Add a New Determination. If an annotation already exists on the specimen, you will see that annotation. In the latter situation, click the green plus sign to add a new annotation.\nIn the Add a New Determination Box, enter, at minimum, the scientific name, determiner (person who identified the specimen, if known. Enter “unknown” if not known), and the date (or “n.d.” if not known), at minimum. You can also add an identification qualifier (e.g., “cf.” or “aff.”), a reference, and notes, and you can select a confidence of determination, if desired. By default, if the date of determination is newer than that previous date of determination, the Make this the current determination box will be checked once you move your cursor out of the Date box. If you want this determination to be the most up-to-date identification for the specimen, keep this box checked (or check it, if it was not auto-checked). If you wish to print this annotation label now or in the future, also check the Add to Annotation Print Cue box. Click Submit Determination.\nIf the specimen previously had an identification and you have checked the box next to “Make this the current determination,” a historic annotation will be added to the record automatically as shown in the next screenshot. For this reason, you do not need to manually add a historic determination before you add a new determination.\n Batch Adding Annotations\/Determinations You can add many annotations at once using the Add Batch Determinations\/Nomenclatural Adjustments tool. Navigate to this tool by accessing the Data Editor Control Panel (My Profile \u0026gt; Occurrence Records \u0026gt; name of collection) and clicking “Add Batch Determinations\/Nomenclatural Adjustments.” To select the specimens to which determination data will be added (“Define Specimen Recordset”), either enter a list of catalog numbers (separated by commas) in the “Catalog Number:” field or select a taxon to evaluate by entering its name in the “Taxon:” field. Click the Add Record(s) to List button.\nA table of specimens of the indicated taxon will appear below the “Define Specimen Recordset” box (see screenshot below). You can then select all the specimens that you wish to annotate by checking or unchecking the boxes in the leftmost column of the table. Use the “Select\/Deselect all Specimens” box to uncheck or check all the boxes if you would prefer to select specimens one by one.\nAdd information about the name change in the “New Determination Details” box. Here you can indicate whether the change is due to a new identification (“Identification Adjustment\/Verification”) or a nomenclatural change (“Nomenclatural Adjustment”). An identification qualifier (e.g., “aff.” or “cf.”) can be added in the “Identification Qualifier:” field. Enter the new scientific name in the “Scientific Name” field, and the “Author:” field will automatically populate if the taxon is already in the taxon table. If the taxon is not in the taxon table, you will have to manually enter the author of the scientific name. You should also enter the name of the determiner in the “Determiner:” field, as well as the full date of determination in the “Date:” field. You can indicate the confidence in the determination (Low, Medium, or High) in the “Confidence of Determination:” field, list your determination references in the “Reference:” field, or include any other notes in the “Notes:” field. Finally, if you check the “Make this the current determination” box, the scientific name will be updated for the selected records. Otherwise, the determination will be added to the specimens’ records, but the most current ID name will not be changed.\nIf you wish to print the annotation labels in the future, check the “Add to Annotation Print Queue” box.\n</p><p>edit</p><p>annotation</p><p>identification</p><p>determination</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/annotations\/"},{value:"Creating a Darwin Core Archive",label:"<p> This page describes how you can package your data as a Darwin Core Archive (standard format for data publishing and sharing) in a Symbiota portal.\n A Darwin Core Archive (DwC-A) is a data standard that is commonly used to package species occurrence data into a single, self-contained dataset. A DwC-A includes metadata, a file of occurrence data, and, often, files for determinations (identifications), images, and any other data extensions.\nYou can only publish a Darwin Core Archive of occurrences that have unique occurrence IDs and GUIDs. In your collection metadata page, make sure that you have selected a source for your GUID and that this source contains unique values for each occurrence.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Darwin Core Archive Publishing. Click the Create\/Refresh Darwin Core Archive button.  </p><p>data publishing</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/dwc\/"},{value:"Data Cleaning",label:"<p>Symbiota has a number of useful built-in tools for data cleaning processes. Each of the tools described in this section can be accessed through the Administration Control Panel under Data Cleaning Tools.\n</p><p>data cleaning</p><p>taxonomy</p><p>geography</p><p>duplicates</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/"},{value:"Crowdsourcing",label:"<p>Symbiota has a built-in crowdsourcing module that can enable any logged-in user to transcribe specimen data from provided images. These specimens must have a processing status of \u0026ldquo;unprocessed\u0026rdquo; and have an associated image.\nOnce a crowdsourcer has transcribed a specimen record and saved the edits, the processing status of the record will be automatically changed to Pending Review and the record will be removed from the queue. All edits that have been made to the record will be instantly publicly visible but can be reverted by a collection administrator. The transcribed record is now considered \u0026ldquo;Pending\u0026rdquo;, and the crowdsourcer will have \u0026ldquo;pending\u0026rdquo; points. Once the record is reviewed and approved by a collection administrator (i.e., processing status changed to \u0026ldquo;Reviewed\u0026rdquo;), these points will become \u0026ldquo;approved\u0026rdquo; points.\n</p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/crowdsource\/"},{value:"Data Publishing",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/"},{value:"Downloading Data",label:"<p></p><p>download</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/download\/"},{value:"Publishing Data to GBIF",label:"<p> This document describes how to publish the data from your collection to global aggregator GBIF.\n Collections managed as \u0026ldquo;live datasets\u0026rdquo; within a Symbiota portal can immediately publish to GBIF without issues. Collections that make use of an in-house management system (e.g. Specify, Ke-Emu, etc.) and only publish a snapshot of their data to a Symbiota instance can also use the portal to publish their data to GBIF, but only if: 1) they are not publishing their data through another means (e.g. IPT installation, VertNet, etc.), and 2) an occurrenceID GUID is included in the data being pushed from their in-house database to the Symbiota dataset. If the collection is using the Symbiota publishing tool built into Specify, the occurrenceID GUID will be automatically included in the data upload from Specify.\nYour portal must be set up as a GBIF Publishing Installation to publishing your data to GBIF. This can be done by your portal manager.\n  Set up an institutional account with GBIF so that there is a direct publishing agreement established between GBIF and the institution. Since the institutional account will be used to list multiple collection datasets associated with that institution (https:\/\/www.gbif.org\/publisher\/4c0e9f60-c489-11d8-bf60-b8a03c50a862), you should coordinate with other collections within your institution, if applicable. Note that the institutional datasets can be published to GBIF using different publishing resources. For instance, the zoological collections could import their data from VertNet IPT (http:\/\/ipt.vertnet.org) or their institutional IPT, vascular plant data from the SEINet https:\/\/swbiodiversity.org, and lichens from CNALH (https:\/\/lichenportal.org). Use the GBIF Endorsement Request page (https:\/\/www.gbif.org\/become-a-publisher) to register your institution. Use the organization lookup on that page to make sure your institution is not already registered.  If you are sure your institution is not yet registered, complete the registration form and follow the instructions provided by GBIF. If your institution is already registered, review the GBIF metadata for your organization and existing datasets and contact GBIF to make any necessary changes. Be sure that none of the existing datasets contain the same data you are trying to publish. If they do, make the appropriate arrangements with GBIF so that the old dataset can be archived BEFORE re-publishing the new dataset.   Login to your Symbiota portal, go to your Administrator Control Panel (via My Profile =\u0026gt; Specimen Management tab =\u0026gt; click your collection name), and click on \u0026ldquo;Edit Metadata and Contact Information\u0026rdquo; link in the Administration Control Panel. Verify your collection name and description (these will be used within the GBIF page), check the GBIF box to the right of \u0026ldquo;Publish to Aggregators\u0026rdquo;, and click the \u0026ldquo;Save Edits\u0026rdquo; button. If you do not see a GBIF publishing checkbox, contact your portal manager and ask them to configure the portal for GBIF publishing. Return to the Administrator Contol Panel and click on the \u0026ldquo;Darwin Core Archive Publishing\u0026rdquo; link in the Administration Control Panel. Click \u0026ldquo;Create\/Refresh Darwin Core Archive\u0026rdquo; button to package up your data within a Darwin Core Archive data package. Enter your institution\u0026rsquo;s GBIF publication key and click the Validate Key button. If you key validates, more instructions will be displayed along with a Submit Data button. The GBIF Publisher Key should have the following format: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx (e.g. 4c0e9f60-c489-11d8-bf60-b8a03c50a872). You can also enter the full URL to your GBIF publishing page, and the key will be automatically extracted. Before you can submit data, you will need to contact GBIF help desk (helpdesk@gbif.org) and request for the portal\u0026rsquo;s GBIF user account to be given permission to create and update datasets within your institution\u0026rsquo;s publishing instance. The GBIF username associated with the Symbiota portal installation is displayed in paragraph above the Submit Data button. Click on the GBIF email address to automatically generate a message within your email client. Once you hear back from GBIF affirming that the portal has permission to submit data to your publisher, click the Submit Data button. A link to your GBIF dataset will be immediately displayed, though it may take an hour or so for your data to be loaded, indexed, and available.  </p><p>aggregator</p><p>gbif</p><p>data publishing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/gbif\/"},{value:"Editing Records",label:"<p> This page provides instructions on how to search for and edit records.\n  To batch edit records, visit this page. For an overview of fields in the occurrence editor form, visit this page. For an explanation of the record editor tabs, visit this page.  To edit individual records, select \u0026ldquo;Edit Existing Occurrence Records\u0026rdquo; from the Data Editor Control Panel (accessed via My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). From this page, you can search for specific records by entering search terms into the Record Search Form. You can search by multiple search terms using this form.\nTo search according to fields not explicitly stated in the Record Search Form, select the field from the dropdown menu after Custom Field 1. You can include up to 4 Custom fields in your search. The second dropdown lists after the Custom Fields will allow you to conduct more specific searches, such as for ranges or fields that are null. The options include: * EQUALS: the field contains only the provided text * NOT EQUALS: the field does not match the provided text exactly (does not only contain the provided text) * STARTS WITH: the field starts with the provided text * CONTAINS: the field contains the provided text anywhere in the field * DOESN\u0026rsquo;T CONTAIN: the field does not contain the provided text anywhere in the field * GREATER THAN * LESS THAN * IS NULL: the entire field contains no value * IS NOT NULL: the field contains any value\nTo conduct a search, click the Display Editor (to view one record at a time) or Display Table (to view the first 1000 records at a time) buttons.\nTo search by specimens entered by you (the current user), click the CU button.\nTo sort your search results, select a field from the dropdown menu after Sort By (bottom right of Record Search Form), then select whether you wish to sort in ascending or descending order.\nTo view a specific record from the Table Display, click the Symbiota ID number or the Open in New Window icon in the leftmost SymbiotaID column.\nTo re-open the record search form after you have conducted a search, click the magnifying glass icon to the right of the name of your collection at the top of the window.\n</p><p>edit</p><p>change</p><p>record search form</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/"},{value:"Publishing Data to iDigBio",label:"<p> This page describes how to publish the data from your collection to the U.S. national aggregator iDigBio.\n  Contact iDigBio (data@idigbio.org) requesting that your collection gets added to the iDigBio ingestion queue. Include a link to your collection profile page (e.g. http:\/\/swbiodiversity.org\/seinet\/collections\/misc\/collprofiles.php?collid=1) so that they have the necessary information for your collection and access to your data. For more information, see the iDigBio Data Ingestion Guide. Log in to your Administrator Control Panel in your Symbiota portal (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Edit Metadata and review the information and contacts associated with your collection. In addition to your institution\/collection codes and contact information, make sure to review data usage license and GUID (see below). Contact your portal manager if you are unclear about any of the available fields.   Data Usage License: This field determines the copyright guidelines that users must follow when using your data. Many portals have predefined options that follow the Creative Commons licenses, though that can vary by portal instance. See the Creative Commons website for more information about licenses. GUID source: This designates the field that will serve as the persistent globally unique identifier (GUID) for specimen records published from the portal. (More information about the importance of GUIDs can be found in this article. You will see the following options in your portal:  Occurrence ID: This option is for collections that create their own GUID using another protocol or manage their data in an external system and map the incoming GUID to the occurrenceID field at time of import. If you manage your central data within an update-to-date Specify database, you should use this option because the import data should automatically contain a Specify-generated UUID. Darwin Core definition: http:\/\/rs.tdwg.org\/dwc\/terms\/index.htm#occurrenceID. Catalog Number: You can use this option if your catalog number is configured as a globally unique identifier. See the Darwin Core occurrenceID definition above. Symbiota-generated UUID: If you are managing your data live within the Symbiota instance (e.g. the portal is your central database), this option is the easiest and most reliable method for assigning robust GUIDs to your specimen records. Even if your catalog number is configured as a unique identifier (e.g. institutionCode:collectionCode:catalogNumber format type), this is a more robust option when managing data within the portal. Do not select this option if you use the Symbiota portal to publish a snapshot of your data that is managed in another database system (e.g. Specify).    Return to your Administrator Control Panel and click “Darwin Core Archive Publishing”. Click the Create Darwin Core Archive button to build a new archive. iDigBio will periodically monitor the RSS feed that is associated with the DwC-Archive library within the portal. You should regularly refresh this archive to ensure the data within iDigBio remains current. Contact your portal manager to request automatic updates of your archive on a regular schedule.  </p><p>data publishing</p><p>idigbio</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/idigbio\/"},{value:"Redacting \/ Obscuring Data",label:"<p> This page explains how data redaction functions in a Symbiota portal.\n Collection managers may wish to redact locality data for certain occurrences, for example, of rare or endangered species or for locations on private property. Locality data in Symbiota portals may be redacted in one of two ways: (a) individually (per occurrence) or (b) globablly (per taxon).\nRedacting locality data in Symbiota portals is currently binary: an occurrence can have its locality redacted (Locality Security = 1, checked) or not (Locality Security = 0, unchecked). When the Locality Security box is checked (or Locality Security is uploaded as 1) for a given occurrence, a user who does not have rare species reader or editor permissions will not be able to view that occurrence\u0026rsquo;s:\n Locality below the level of county Coordinates (if provided) Image  Individually redacting locality data for certain specimens The locality data can be redacted for individual occurrences by checking the Locality Security box in the Occurrence Editor.\nIf you wish to batch redact data, you can download a CSV file of all the specimen records you wish to redact using the Exporter tool, then add a column called LocalitySecurity. Enter a 1 in this column for all specimens for which you wish to redact data. Use the Skeletal File Uploader to upload this spreadsheet into the portal, mapping the new column to localitysecurity.\nGlobally redacting locality data for certain taxa In addition, locality data and images can be redacted on a per-species level by someone with superadministrator or taxon editor permissions. To do this, find the species in the Taxonomic Tree Viewer or Taxonomy Explorer and open the editor (either by clicking on the taxon name or clicking the pencil next to the name). Change the Locality Security field to \u0026ldquo;hide locality data\u0026rdquo;.\nThis will hide locality data for all occurrences of that taxon throughout the portal. Collections can opt out of this option by individually unchecking the Locality Security box or contacting their portal administrator for batch changes.\n</p><p>rare species</p><p>data protection</p><p>redaction</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_publishing\/redaction\/"},{value:"Images",label:"<p> This page describes the types of images that are available through Symbiota portals.\n There are three catefories of images that can be linked to a Symbiota portal:\n Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed through Taxon Profile Pages, which have general information such as descriptions, distribution maps, synonyms, and common names. Field images are uploaded and managed through the Taxon Profile pages by users with Taxon Profile editing permissions. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see \u0022Field Images with Location\u0022 tab). See Uploading Images for more details.  Field images with specific locality information can adequately serve as vouchers for field observations of many birds, mammals, and easy-to-identify plants. In some instances, they may actually be preferable, for instance if the organism is rare or the observer does not have the appropriate collecting permit. These images are similar to georeferenced specimen records in that they are searchable occurrence records that place a dot on the species distribution maps. There are, however, some taxonomic groups that typically require microscopic or chemical analysis for accurate identification (e.g. lichens, bryophytes, fungi). For such organisms, field images will have limited value. For this reason, some data portals might not allow the submission of observations without a physical specimen. Image Vouchers are displayed on both the Occurrence Details and the Taxon Profile pages. Image-supported observations can be uploaded using the Observation Submission page. Up to three images can be submitted for a single observation. Data fields required for an image to qualify as a voucher include: observer name, observation date, country, state, locality description, latitude, longitude, and at least one image. Contact your data portal administrator for information on whether observations are allowed and the necessary protocols for submitting an observation. Some portals require the review of a tutorial before one is authorized to submit an observation. See Uploading Images for more details.  Images of physical specimens should be linked to their respective specimen records. Specimen images are available both from the Occurrence Details page and the Taxon Profile page for the taxon associated with the most recent determination. If the specimen identification is annotated, the identification of the image will be automatically re-mapped to that taxon. Specimen images can be added through the Occurrence Editor or in batch via multiple processes. See Uploading Images for more details.    </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/images\/"},{value:"Images",label:"<p> This page describes the types of images that are available through Symbiota portals.\n There are three catefories of images that can be linked to a Symbiota portal:\n Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed through Taxon Profile Pages, which have general information such as descriptions, distribution maps, synonyms, and common names. Field images are uploaded and managed through the Taxon Profile pages by users with Taxon Profile editing permissions. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see \u0022Field Images with Location\u0022 tab). See Uploading Images for more details.  Field images with specific locality information can adequately serve as vouchers for field observations of many birds, mammals, and easy-to-identify plants. In some instances, they may actually be preferable, for instance if the organism is rare or the observer does not have the appropriate collecting permit. These images are similar to georeferenced specimen records in that they are searchable occurrence records that place a dot on the species distribution maps. There are, however, some taxonomic groups that typically require microscopic or chemical analysis for accurate identification (e.g. lichens, bryophytes, fungi). For such organisms, field images will have limited value. For this reason, some data portals might not allow the submission of observations without a physical specimen. Image Vouchers are displayed on both the Occurrence Details and the Taxon Profile pages. Image-supported observations can be uploaded using the Observation Submission page. Up to three images can be submitted for a single observation. Data fields required for an image to qualify as a voucher include: observer name, observation date, country, state, locality description, latitude, longitude, and at least one image. Contact your data portal administrator for information on whether observations are allowed and the necessary protocols for submitting an observation. Some portals require the review of a tutorial before one is authorized to submit an observation. See Uploading Images for more details.  Images of physical specimens should be linked to their respective specimen records. Specimen images are available both from the Occurrence Details page and the Taxon Profile page for the taxon associated with the most recent determination. If the specimen identification is annotated, the identification of the image will be automatically re-mapped to that taxon. Specimen images can be added through the Occurrence Editor or in batch via multiple processes. See Uploading Images for more details.    </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/images\/"},{value:"Importing \u0026 Uploading Data",label:"<p> This page provides instructions for uploading data into an existing collection in a Symbiota portal. Contact your portal manager if you do not already have a collection in your desired Symbiota portal.\n Initiating the Upload   Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection).\n  Click Import\/Update Specimen Records, then select \u0026ldquo;Create a new Import Profile\u0026rdquo;.\n  Create a title for your upload in the Title field.\n  Select the desired Upload Type from the dropdown menu.\n Darwin Core Archive Manual Upload: Use this upload type if the data you wish to upload is in the format of a Darwin Core Archive. A Darwin Core Archive (DwC-A) is a data standard that is commonly used to package species occurrence data into a single, self-contained dataset. A DwC-A includes metadata, a file of occurrence data, and, often, files for determinations (identifications) and images. IPT Resource \/ Darwin Core Archive Provider: Use this upload type if you will provide a URL to an existing Darwin Core Archive published on the web, such as one provided through an IPT. File Upload: Use this upload type if you will provide a comma-separated value (CSV) or tab-separated value (TSV) file containing ALL fields of your occurrence data. You can convert an Excel document into a CSV file by clicking Save As, then selecting comma-delimited (CSV) from the file types. Note that, if data exists in the portal for any of the occurrences you are uploading, those data will be overwritten by the incoming data. To upload partial records, use a Skeletal File Upload. Skeletal File Upload: Use this upload type if you will provide a CSV or TSV file containing data from only a few fields (e.g., georeferences or other ancillary data). Note that any data provided in a skeletal file upload will NOT overwrite existing data in the database, so any pre-existing data in the desired fields must be deleted if you wish to replace it with the data from the skeletal file. NfN File Upload: Use this upload type if you will provide a CSV file produced from Notes from Nature. Direct Database Mapping: Stored Procedure: Use this option if you are transferring from a source schema to a Symbiota database located on the same MySQL database server. Script Upload: Use this option if you are transferring from a MySQL source to Symbiota database that is located on a different server.    Follow the directions below according to the Upload Type you have selected.\n  Darwin Core Archive Manual Upload Contribute to this section!\nIPT Resource \/ Darwin Core Archive Provider Contribute to this section!\nFile Upload or Skeletal File Upload  If you or your portal manager have created a Stored Procedure with data cleaning or other checks, enter the name of the stored procedure in the provided field. Otherwise, ignore this step. Click the Create Profile button. On the next page, you will see a list of existing upload profiles. Select the profile that you wish to use (the one you just greated) and click Initialize Upload.  To edit your upload profile in the future, you can click the pencil icon on this page.   Click the Choose File button and navigate to the file that you wish to upload in your File Manager or Finder window. Select that file and click Open. Click the Analyze File button. If the collection to which you are uploading data is live managed (the portal is your database system), proceed to step 7. If the collection to which you are uploading data is a “snapshot” of a specimen database managed within the home institution, select the primary key for the source specimen record from the dropdown menu. The primary key is a required field for snapshot datasets that will serve as the primary record identifier (the permanent link between the source database and the portal records). This field must be populated for every record with unique values. These values must also be stable and not changed in the central database over time. Snapshots will typically use the catalog number (accession number), barcode, or database primary key from the source database specimen table for this field. You will then see a page that will look similar to the one shown below. The length and contents of the Source Field\/Target Field table will depend on what columns were included in the original CSV file.  Select which fields in your CSV file (Source Fields) will correspond to which fields in the Symbiota portal (Target Fields). Check the Symbiota Data Field Guide for definitions of each data field. Also see the Uploading Tips section below. Once you are satisfied with your field-to-field mapping (see next Notes), click the “Save Mapping” button. Select whether you would like the script to match the data in your file to existing data in the portal based on Catalog Number or Other Catalog Numbers. You will only need to do this if you are adding data to records that already exist in the portal. Otherwise, leave these unchecked. Select the Processing Status that you would like to apply to all your uploaded records (if desired) by selecting an option from the dropdown menu. Click the Start Upload button. This will upload your data into a temporary table so you can review it before committing the final upload. Verify that the correct number of records are being updated and\/or added by viewing the Pending Data Transport Report on the next page.  View the data that have been stored in the temporary table to ensure correct mapping and formatting of the fields you are uploading. You can either:  Click the small box icon to the right of \u0026ldquo;Records to be updated\u0026rdquo; or \u0026ldquo;New records\u0026rdquo; to view the records in a table in your browser. Click the multiple file icon to the right of the box icon to download a CSV file of the records to be updated or new records.   If anything is incorrect, fix your CSV file and re-upload it according to the steps you followed above, or return to your field mapping and fix the field mapping. If everything looks good, click the Transfer Records to Central Specimen Table button. Note that this step is final and is not possible to undo!  Stored Procedure  Write a stored procedure used to transfer records (the collection cleanup scripts can be put in central stored procedure or kept separate) Set up the script to run as a regular cronjob.  Script Upload  Write a stored procedure used to transfer records. A sample Linux script is located here: SampleSystemUpload.sh. The cleanup scripts can be put in central stored procedure or kept separate. Set up the script to run as a regular cronjob.  Uploading Tips A list of fields that can be imported into a Symbiota data portal can be found here.\n  If the scientific names in your CSV file include taxonomic authorship (e.g., Acer circinatum Pursh), map this field to the Target Field “scientificname.” If the scientific names included in your CSV file do NOT include taxonomic authorship (e.g., Acer circinatum), map this field to “sciname.” Collection dates mapped to eventDate will be evaluated and validated. Illegal dates will be placed in the verbatimEventDate field. The majority of the standard date formats are accepted, including Gregorian dates and Excel numeric date format (US only). eventDate will be generated from separate year,month, and day field values. If month or day fields are left null, ’00’ values will be used (ex: 1954-03-00, 1965-00-00). Month field values can be numeric or text (English or Spanish). Scripts attempt to extract valid date values from verbatimEventDate field when the eventDate field is null. Values of ’00’ are used for missing month or day (ex: 1954-03-00, 1965-00-00) If your elevation field is not consistently in meters, map it to the verbatimElevation field. Elevations in feet will be converted to meters as long as the units are specified in the field. Coordinate values:  Upon upload, background scripts will attempt to extract lat\/long coordinate values from the verbatimCoordinates field. The field is evaluated for DMS and UTM formats, which are converted to decimal latitude and longitude. If you have lat\/long in a single field, you can map this field to verbatimCoordinates, and decimal latitude and longitude fields will be automatically parsed. If you have UTM coordinates in multiple fields, map the fields (northing, easting, zone) to their matching UTM fields (utmnorthing, utmeasting, utmzone). This will instigate conversion of UTM coordinates to decimal latitude and longitude. The values will additionally be stored in the verbatiumCoordinates field. If you have UTM coordinates in a single field, map this field to utmnorthing and leave other UTM fields null in order to direct scripts to parse using only the UTM parser. TRS coordinates (Public Lands Survey System) can be entered as a single field into verbatimCoordinates, or into separate fields (trstownship, trsrange, trssection, trssectiondetails); however, these coordinates will not be automatically converted into decimal degrees due to potential differences in interpretation. See the georeferencing section of this guide (coming soon) for information about converting TRS coordinates to decimal degrees.    </p><p>data upload</p><p>data import</p><p>file upload</p><p>IPT</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/upload\/"},{value:"User Permissions",label:"<p> This page defines each of the possible permission levels that can be assigned by an administrator of a collection and describes how to assign permissions to users.\n Permissions Definitions Administrator: can use all tools in the Administration Control Panel. For example, this user can edit the contact information and description for your collection, view edits made to your records, batch upload data, and use data cleaning tools. Administrators can also delete records.\nEditor: can use all tools in the Data Editor Control Panel, including adding and editing records, printing labels, batch annotating specimens, batch georeferencing specimens, and managing loans. Editors cannot delete records.\nRare Species Reader: can view locality data for all specimens in your collection, even if the locality information is redacted from the general public (see Redacting \/ Obscuring Data).\nAdding \/ Changing User Permissions To assign user permissions, you must be an administrator of the collection for which you will assign those permissions.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Manage Permissions. The resulting page will show a list of all Administrators, Editors, and Rare Species Readers assigned to your collection. To remove a user\u0026rsquo;s permissions, click the X icon next to their name. To add or edit a user\u0026rsquo;s permissions, select their name from the dropdown list under Add New Admin\/Editor\/Reader, select the radio button next to the level of permissions you wish to assign, and click the Add Permission for User button.  If you don\u0026rsquo;t see someone\u0026rsquo;s name in the dropdown list, they might not have a profile yet. Contact the user and have them create a user profile.\n </p><p>users</p><p>permissions</p><p>access</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/users\/"},{value:"Symbiota Docs",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/"},{value:"User Permissions",label:"<p> This page describes the possible levels of user permissions that can be granted for a given Symbiota portal.\n Once a user has created an account in a Symbiota portal, they can be granted permissions for one or many collections in that portal. Permissions can be granted by administrators of individual collections or by the portal manager or other superadministrator. Portals have different policies for granting permissions to users, so it is best to contact the portal manager or individual collections for which you would like user permissions.\nTypes of User Permissions Permissions that can be assigned by a collection manager or other administrator\n  Administrator: (generally a curator or manager of a collection) can use all tools in the Administration Control Panel. For example, this user can edit the contact information and description for your collection, view edits made to your records, batch upload data, and use data cleaning tools. Administrators can also delete records.\n  Editor: (generally a collector, technician, digitizer, or volunteer) can use all tools in the Data Editor Control Panel, including adding and editing records, printing labels, batch annotating specimens, batch georeferencing specimens, and managing loans. Editors cannot delete records.\n  Rare Species Reader: can view locality data for all specimens in a collection, even if the locality information is redacted from the general public (see Redacting \/ Obscuring Data).\n  Permission types that can only be assigned by a superadministrator\n  Global Rare Species Reader: can view locality data for all specimens in a portal (if provided). See Redacting \/ Obscuring Data.\n  Glossary Editor: can add descriptions, images, or other information to the portal glossary (if enabled)\n  Identification Keys Administrator: can add, delete, and edit characters states used in the identification keys (if enabled)\n  Identification Keys Editor: can edit character states used in the identification keys (if enabled)\n  Taxonomy Editor: can edit the taxonomic thesaurus, including adding and deleting taxa, editing the acceptance of a taxon, and adding synonym or children taxa\n  Taxon Profile Editor: can add information to taxon profile pages, such as images and descriptions\n  </p><p>users</p><p>permissions</p><p>access</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/permissions\/"},{value:"Batch Editing",label:"<p> This page describes how to batch edit records.\n  For batch georeferenceing instructions, visit this page. Scientific names can only be batch changed through the Batch Annotation tools or the [Taxonomic Cleaning tools]((https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/data_cleaning\/taxonomy).  Exercise caution when using this tool. Only experienced Symbiota users should perform batch edits. We also recommend downloading a copy of your database prior to making batch edits\n  There are two options for batch editing specimen records: you can change a value for the entire set of records in your collection or change a value for a selected set of records. To change all records, click on “Display Table” in the Record Search Form, then click the edit icon above the resulting table of all records from your collection. Continue to step 3. To change a value for a selected set of records, first search for the records of interest using the search form. Once you have the set of records to which you would like to make a batch edit, click on the edit icon at the top of your selected table of records. From the dropdown menu next to “Field Name” in Batch Update, select the field you want to edit. In “Current Value:” enter the text that is presently in the field you want to edit. In “New Value:” enter the text that you want to be in the edited field. Choose if you want to match the whole field or just match any part. Click “Batch Update Field” to enact your batch edit. A pop-up screen will tell you the number of records to be updated, warn you that the replace operation cannot be undone, and prompt you to OK the update. Proceed with caution and see examples below.  It is easy to inadvertently change text you did not intend to change. Generally, the more specific your “Current Value:” text, the less likely you are to run into unintended consequences. If possible, get an estimate from your record table of how many records should be affected, and check that against the number of records cited in the warning. If the numbers do not match, rethink your strategy.\nIt is not possible to check the record count when sorting by “Modified By” first. Doing so returns a count of the number of fields edited by the “Modified By” user for all records to be affected, rather than the number of records to be affected.\n Examples Example of a good use of batch edit: Standardizing plant collector names. If you are standardizing all Wherry’s collections to “Edgar T. Wherry,” then you could enter “E. T. Wherry” in Current Value and “Edgar T. Wherry” in New Value in batch edit for the collector field name, and update. Repeat for alternate forms of this name (e.g., “E T Wherry,” Edgar T Wherry”).\nExample of a bad use of batch edit: Changing cardinal directions. For example, if you entered “n.” in Current Value and “N” in New Value in batch edit for the locality in an attempt to change lower case abbreviated instances of the cardinal direction north, you would accomplish that, but you would also change any other character string with “n.” such as a sentence ending in the letter n. This would result in batch misplacements of “N” where “n.” should be. For example, “Next to the station. East side” would be transformed into “Next to the statioN East side.”\n</p><p>batch</p><p>edit</p><p>change</p><p>record search form</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/batch\/"},{value:"Developer Guide",label:"<p>Coming soon!\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/dev\/"},{value:"Record Editor Tabs",label:"<p> This page explains the tabs at the top of the Occurrence Editor tool.\n Upon opening the Record Editor page, you will see 4-5 tabs, depending on your level of permissions. The image shown below is of the Record Editor page for a user with administrative permissions. A brief explanation of each tab is provided below.\nOccurrence Data – See Symbiota Data Fields Determination History – where annotations are shown and can be entered. Annotations can also be batch uploaded from separate files. See Annotations Images – provides information on image(s) associated with the record. See Images Linked Resources – allows linkage of checklists, duplicates, and genetic resources to the record Admin – tracks changes made to Edit History, allows for transfers of specimens between collections, and provides a means for deleting an occurrence record. Only users with administrative permissions can view this tab\n</p><p>edit</p><p>tabs</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/tabs\/"},{value:"Adding Full Records",label:"<p> This page describes where to add a new record by entering the data directly into the portal. For information about batch uploading label information, see this page.\n Navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Select Add New Occurrence Record. Enter the specimen information into the provided fields.\nTo learn more about the data fields available in Symbiota, visit this page.\n Before you click Add record to create the new record, you can select one of three options for the next action you will perform, shown below. Go to New Record and Carryover Locality Information is useful for entering data from a single collection event that will share locality information. Remain on Editing Page should be used when the user wishes to double-check the record or add supplementary information such as images or annotations.\n</p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/full\/"},{value:"Adding Records from Images",label:"<p> This page describes how to add a new record by uploading a single image.\n Using this tool, you can upload (or enter a URL to) an image that will establish a new occurrence record. Navigate to the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection), and select Create New Records Using Image (nested under Add New Occurrence Record). To upload an image, click the Choose File button and select the image you wish to upload. To use a URL, click the Enter URL link on the right side of the form.\nBefore upload, you can also define the processing status of the new record and enter any skeletal data for the record, including Catalog Number, Scientific Name, Country, State\/Province, and County. Checking the box next to OCR Text using Tesseract OCR engine (if applicable) will run optical character recognition on the image that will attempt to parse text from the image that can potentially be used to fill out the remaining fields in the future.\n</p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/image\/"},{value:"Adding Skeletal Records",label:"<p> This page describes how to use the Skeletal Data Entry tool to quickly add skeletal data to the portal, especially during the process of imaging. It can also be used to create a quick inventory of a collection.\n This tool can be used to collect basic collection information, for example, during the imaging process. The imaging team can enter the basic collection information shared by the batch of specimens being processed, and each time they scan a barcode into the catalog number field, a record is added to the system with these skeletal data. Complete data can be entered by clicking on the catalog number, but a recommended workflow would be to process the full label data directly from the image of the specimen label at a later stage.\n To add a skeletal record, navigate to the Add Skeletal Records tool via the Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). You will see a page like that below.  To control which fields are shown on the skeletal record entry form, click the Display Option symbol (a small, grey table) located above the Scientific Name field (circled above). In the dropdown menu of options, check or uncheck the boxes next to the desired fields to be shown. The following fields are commonly recommended for skeletal data entry: Scientific Name, Family, State\/Province, and County\/Parish. Select the preferred action that the data entry module will use when a record already exists for a scanned catalog (barcode) number.  Even if you choose the “Append values to existing records” option, skeletal data will not copy over existing field values.\n If using the fields recommended above, enter the scientific name by beginning to type the name from the specimen label, then selecting the appropriate name from the dropdown list. If desired and\/or possible (given the data provided on the specimen sheet), enter the State\/Province and the County\/Parish of the specimen if the specimen was collected in the U.S. or Canada. The authorship and family of the name of the taxon will automatically populate if the taxon is in the portal’s taxon table. Enter the barcode number into the Catalog Number field by clicking inside the Catalog Number field and scanning the barcode of the specimen. Once all fields are filled, click Add Record.  Important: After Add Record is clicked, the data in the Skeletal Data box fields will NOT be cleared. Only the Catalog Number field will be cleared. This is designed to facilitate rapid skeletal data entry for folders of similar specimens (i.e., specimens of the same species). Close attention should be paid to these fields when transitioning between species or specimen localities. It is easy to forget to change the data in these fields.\n In the event that a user needs to edit a previously entered skeletal record, you can click the catalog number of the record as it is listed in the Records box below the Skeletal Data box. Note that this list will disappear if the page is refreshed. You can also add an image to the record from this page by clicking the icon to the right of the listed catalog number.\n </p><p>add</p><p>create</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/add\/skeletal\/"},{value:"Data Import Fields",label:"<p> This page lists the possible specimen fields that can be imported into a Symbiota data portal. The type of field (text, number, etc.) is listed in the Type field. The number of characters allowed in that field is displayed in parentheses.\n If the data portal is being used to displaying a “snapshot” of your data stored in your local central database, the upload file MUST have a field that serves as the unique identifier for each incoming specimen record (dbpk). This field serves a link between the source record and the snapshot record within the portal. If import is a CSV file, the first row must contain field names. Note that field names do not have to match the name used below, but there cannot be any characters ($#@\u0026amp;%) in the column names.\nFor more information about data fields, also see our Symbiota Data Fields page.\nItalic = Darwin Core fields Bold = strongly encouraged fields, though none of the fields are technically required * = for import only, merged into another field after import\n   Name Type Notes     dbpk Text (45) Specimen record unique identifier (primary key) of source database (record id). Barcode, occurrenceID (GUID), or a database Primary Key is ideal. Can also be catalogNumber, given that it is populated for each record and enforced as unique. Required if collection is “snapshot” of a central database. Not needed if collection is managed directly within portal.   occurrenceId Text (255) Occurrence Global Unique Identifier (GUID)   catalogNumber Text (32) Barcode or Accession number   otherCatalogNumbers Text (255)    family Text (255)    scientificname Text (255) Scientific name w\/ author; author is parsed from name   sciname Text (255) Scientific name without author   genus Text (255)    specificEpithet Text (255)    taxonRank Text (32) Rank name of infraspecific abbreviation allowed   infraspecificEpithet Text (255)    scientificNameAuthorship Text (255) Author of scientific name   identifiedBy Text (255)    dateIdentified Text (45)    identificationReferences Text    identificationRemarks Text    identificationQualifier Text (255) cf, aff. etc   typeStatus Text (255)    recordedBy Text (255) Collector name   recordNumber Text (45) Collector number   *recordNumberPrefix Text (45) Merged into recordNumber   *recordNumberSuffix Text (45) Merged into recordNumber   associatedCollectors Text (255)    eventDate Date\/Time Date collected, or earliest date when range provided   eventDate2 Date\/Time Latest date collected, when a range is provided   year Integer (4) If eventDate is null, year-month-day will be used to   month Integer (4) build eventDate   day Integer (4)    verbatimEventDate Text (255)    habitat Text    substrate Text (500)    host     occurrenceRemarks Text General notes or remarks   dataGeneralizations Text (250)    informationWithheld Text (250)    associatedTaxa Text Associated species delimited by commas or semicolons   associatedMedia Text URL to jpg images, delimited by commas or semicolons   associatedOccurrences     associatedSequences     associatedReferences     dynamicProperties Text    verbatimAttributes Text Verbatim description of organism (e.g. 1.5m tall, flowers white with purple tips, etc)   behavior Text (500)    reproductiveCondition Text (255) e.g. sterile, flw, frt, asci, etc. Use of controlled vocabulary is preferred.   cultivationStatus Integer 0 = wild, 1 = cultivated   establishmentMeans Text (45) e.g. cultivated, garden escape, etc   lifeStage Text (45)    sex Text (45)    individualCount Text (45)    samplingProtocol Text (100)    samplingEffort Text (200)    preparations Text (100)    locationID     parentLocationID     continent     waterBody     islandGroup     island     countryCode     country Text (64)    stateProvince Text (255)    county Text (255)    municipality Text (255)    locality Text    locationRemarks Text    localitySecurity Integer 0=don’t hide locality details from general public, 1=hide locality   localitySecurityReason Text (100)    decimalLatitude Double (8)    decimalLongitude Double (8) For USA records, value is negative   geodeticDatum Text (255 WGS84, NAD83, NAD27, etc   coordinateUncertaintyInMeters Integer    footprintWKT Text    verbatimCoordinates Text (255) e.g. UTM: 12N 334543 5463754; 34° 25’N 113° 43’W   *verbatimLatittude Text (255) These and following fields are used to generate decimal lat\/lng; merged into verbCoor.   *verbatimLongitude Text (255)    *latDeg Integer Latitude degrees   *latMin Double Latitude minutes   *latSec Double Latitude seconds   *latNS Text (3) North\/south hemisphere   *lngDeg INT Longitude degrees   *lngMin Double Longitude minutes   *lngSec Double Longitude seconds   *lngEW Text (3) East\/west hemisphere   *UtmNorthing Text (45)    *UtmEasting Text (45)    *UtmZoning Text (45)    *trsTownship, Text (45) TRS values are merge into verbCoor.   *trsRange, Text (45)    *trsSection, Text (45)    *trsSectionDetails Text (45)    georeferenceProtocol Text (255)    georeferenceSources Text (255)    georeferenceRemarks Text (255)    georeferencedBy Text (255)    georeferencedDate     georeferenceVerificationStatus Text (32)    minimumElevationInMeters Integer If elevation is single value, just use this field   maximumElevationInMeters Integer    *elevationNumber Integer Elevation number and units are concatenated into   *elevationUnits Text (45) verbatimElevation and translated into min. elev. in m.   verbatimElevation Text (255)    minimumDepthInMeters Integer    maximumDepthInMeters Integer    verbatimDepth Text (50)    storageLocation Text (100)    disposition Text (32)    basisOfRecord Text (32)    institutionCode Text (32) Populate only if different than source collection   collectionCode Text (32) Populate only if different than source collection   ownerInstitutionCode     organismID     materialSampleID     exsiccatiIdentifier Integer Identifier from exsiccati indexing table   exsiccatiNumber     exsiccatiNotes     duplicateQuantity Used for printing labels    labelProject Used for printing labels    processingStatus Processing status for digitization tasks    recordEnteredBy Text (250) Data entry personnel   modified Date\/Time Date last modified within source database    </p><p>data upload</p><p>data import</p><p>file upload</p><p>IPT</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/coll_manager\/upload\/fields\/"},{value:"Adding Taxa to Checklist",label:"<p> This page describes how to add taxa (species) to a checklist in a Symbiota portal.\n There are two ways to add taxa to a checklist, either individually or via batch upload. You may want to start by adding them individually so you can become familiar with the information that can be uploaded and the care one needs to take in doing so.\nAdding Individual Taxa    On the checklist page, click the rightmost pencil icon (labeled spp.) at the top right of the page. This will bring up a panel on the lower right side of the screen. Add the name of a taxon you wish to add. Do NOT include the authorship of the taxon. You can start typing, and a dropdown list will allow you to select the taxon from available options.  You cannot add a name that is not present in the taxonomic thesaurus (it will tell you if this is the case). If you find that the taxon you wish to add is not present:  Check that your spelling is correct. Use a vetted taxonomic resources, such as Tropicos or IPNI to check. If your spelling is correct, contact your portal manager with the name, authorship, and a reference or link to that name in another taxonomic database, and it will be added for you.     Fill in the other fields if desired. All of these fields can be left blank:  Family Override: If you enter the family name here, you will override the family provided by the taxonomic thesaurus. It is advised not to enter anything in this field. Habitat: If the taxon is usually found in a particular habitat in the area circumscribed by the checklist, you can include this information here. Abundance: Here you can note how frequently\/abundantly this taxon is found in the area circumscribed by the checklist. For example, you can use terms like “common,” “frequent,” or “rare.” Notes: Any other comments about this taxon pertaining to your checklist area can be put here, for example: “Cultivated ornamental” or “favored by honeybees” Internal notes: This field, unlike the others, will be visible only to people authorized to edit the checklist. You might make a note “need to check the species” or “no picture available” for example. Source: This field is more likely to be used for research and teaching checklists. For such checklists, it is useful to know whether the information came from a flora, some other publication, or a report by some   Click the “Add species to List” button.  Batch Adding Taxa   If you have several taxa to add at one time, you can upload them in a CSV (comma separated value) file. The easiest way to create the desired listing is to use Microsoft Excel or OpenOffice Calc and then save the file as a CSV file. In your CSV file, one column in the file should be labeled sciname. This should contain the scientific name of the taxon WITHOUT the authorship included. In this CSV file, you must be very careful to spell the scientific names correctly. If you have an error in your spelling, you will not be able to load that name into your checklist. You can also include “family”, “habitat”, “abundance”, and “notes” columns (see above for descriptions of these fields) in this file. An example is shown below.\n   Sciname Habitat Abundance Notes     Asystasia gangetica Middle of the field Rare Demonstration; correct spelling   Asystasia gangetca field unknown Demonstration; wrong spelling    Below is a screenshot of uploading the previous example “file”. Note that only one name, the one that was correctly spelled, was uploaded.\nTo return to the Checklist, click “Return to Checklist” at the top left of the page (underlined below).\nYou can then view the taxa that were added to the checklist.\n</p><p>checklist</p><p>inventory</p><p>flora</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/add\/"},{value:"Adding Vouchers to Checklist",label:"<p> This page describes how to add voucher specimens (physical evidence of the existence of a taxon in a checklist) to a checklist in a Symbiota portal.\n There are two main ways to add voucher specimens to a research checklist. You can add specific, individual specimens from your own collection to the checklist through the occurrence editor (the page of the specimen record) or add specimens from any collection through the Manage Linked Voucher tool on the checklist page.\nAdding Specimens from Any Collection (Manage Linked Voucher Tool)    Navigate to the homepage of your checklist. Click the pencil icon that has a V next to it (the center icon in the top right corner of the page, circled below).  If this is the first time you have used this tool, you will be asked to first define the search statement (i.e., search parameters) that will be used to find the specimens that might pertain to your checklist. You can use a variety of fields to define your search, as described below.  You will be able to change the search statement in the future, so you are not necessarily defining the bounds forever.\n  You can enter search terms for a number of fields on the left side of the form. You can use the polygon footprint that you defined when creating the checklist (see 29:1 Creating a checklist, step 3c) to search for specimen records by checking the “Search based on polygon defining checklist research boundaries” box. If you did not define a polygon previously, you can also do so from here by clicking the pencil icon to the right of this option and following the steps outlined in 29:1 Creating a checklist, step 3c. You can define a specific bounding box by entering the latitude and longitude values that delimit the box, or by defining the lat\/long values on a map. To do the latter:  Click the globe icon to the right of the Lat North field In the pop-up window, pan and zoom to the location where you want to place the bounding box. Click on the approximate center where you want the bounding box to be located. Adjust the location and size of the bounding box as desired by clicking and dragging inside the box (to move the box) or clicking and dragging one of the circles on the sides or corners of the box (to resize the box). To save the bounding box, click Save and Close.     There are a number of important checkboxes below the Lat\/Long fields that you can use to refine your search. For example, you can allow the search to return specimens with the desired search terms OR within certain latitude and longitude coordinates. You can use any combination of the above options to search for voucher specimens.  Once you have entered the desired search terms, click the Save Search Terms button. There are a number of ways to add voucher specimens from this point.   If you have already defined a taxon list for your checklist, you can add voucher specimens from the New Vouchers tab. From this tab, you can:  View taxa that do not currently have voucher specimens in your checklist (select Non-vouchered taxa list from the Display Mode dropdown list). View specimen records that match your search criteria and are one of the taxa defined in your taxon list that is NOT currently vouchered (select Occurrences for non-vouchered taxa from the Display Mode dropdown list).  To add voucher specimens from this list, click the box to the left of the specimen record(s) that you wish to add and click the Add Vouchers button.   View specimen records that match your search criteria and are one of the taxa defined in your taxon list, regardless of whether that taxon has been vouchered previously (select New occurrences for all taxa from the Display Mode dropdown list).   To add voucher specimens from this list, click the box to the left of the specimen record(s) that you wish to add and click the Add Vouchers button. Regardless of whether you have previously defined a taxon list for your checklist, you can add voucher specimens through the Missing Taxa tab. From this tab, you can:  View the taxa that are not found in the checklist currently but are represented by one or more specimens that align with your search criteria (select Species List from the Display Mode dropdown list).  You can then view a list of the specimens associated with each taxon by clicking the page and link icon . From there, you can link the desired specimens individually by clicking the add voucher icon to the right of the specimen record in the list.   View the specimens that are not yet added as voucher specimens for your checklist but do align with your search criteria (select Batch Linking from the Display Mode dropdown menu).  To add voucher specimens from this list, click the box to the left of the specimen record(s) that you wish to add and click the Add Vouchers button.   View specimen records that align with your search criteria, but have scientific names that are not linked to a particular name in the taxonomic thesaurus (select Problem Taxa from the Display Mode dropdown menu).  To add voucher specimens from this list, enter the name of the taxon to which this specimen belongs into the text box to the right of the specimen’s scientific name. Note that the name you enter in this box must already be found in the list of taxa in your checklist. Click the Link Voucher button.      Adding Individual Specimens from Your Own Collection You must be an editor of a collection to assign specimens in this way.\n  Look up a specific specimen in your own collection (e.g., by clicking Edit Existing Occurrence Records in your Data Editor Control Panel and searching by a specific catalog number or by using other search terms). On the resulting page, click the Linked Resources tab. In the Checklist Voucher Linkages box (see below), select the desired Checklist from the dropdown menu and click Link to Checklist as Voucher. A new window will likely pop up as a result, but you can ignore this window and close the editor. The specimen has been successfully linked as a voucher to the desired checklist.  </p><p>checklist</p><p>inventory</p><p>flora</p><p>voucher</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/voucher\/"},{value:"Checklists",label:"<p>A research checklist is a list of taxa for a certain region. Checklists may be documented by voucher specimens, which may be in several different collections. The checklist tool enables linking to voucher specimens, if they are available online, or simply listing their catalog number if not. The voucher tool also enables tracking new potential voucher specimens and re-identification of existing vouchers.\n  </p><p>checklist</p><p>inventory</p><p>flora</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/"},{value:"Creating a Checklist",label:"<p> This page describes how to create a checklist or biotic inventory in a Symbiota portal.\n  Log in to the Symbiota profile (if you don\u0026rsquo;t already have an account, click New Account to create one). Click “My Profile” Under the “Species Checklists” tab, click the green plus sign.  Fill out the “Create New Checklist” box with the desired information.  Most checklists will be a General Checklist. Alternatively, you can another list type from the Checklist Type dropdown menu. A Species Exclusion List is used to curate a subset of taxa that are purposely excluded from another list, and a Rare, threatened, protected species list is used to compile a state-level list of rare, threatened, and protected taxa for which locality data will be redacted for specimens in that state. To indicate that this checklist is a subset within a larger checklist, select the “parent” (larger) checklist from the More Inclusive Reference Checklist dropdown menu. You can define the area included in the checklist by describing it in the Locality and Abstract fields, including Latitude and Longitude coordinates, and\/or you can define the boundaries of that area geographically by setting a “footprint polygon”. To define a “footprint polygon”:  Click the globe icon inside the Polygon Footprint box. A Google map will appear a new window. Drag the map until it shows the region covered by your checklist, then zoom in until you can see the boundaries of the area. Click the polygon icon in the top center of the window then click around the boundaries of your area. When you get back to the point where you started, double click the starting point. This will lock the coordinates into the text box at the bottom of the window. Your defined area will shaded in. If you are dissatisfied with your polygon, you can delete or redraw it by clicking the appropriate button on the bottom right. When you are satisifed with your polygon, click the Save Polygon button.   You can decide whether you want to keep this checklist private, or if you want it to be available to the public by selecting Private or Public from the Access dropdown menu.    To add taxa (species) to your list, see the section Add Taxa to Checklist.\n</p><p>checklist</p><p>inventory</p><p>flora</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/create\/"},{value:"Dynamic Checklists",label:"<p>This page will describe how to create and use dynamic checklists.\nWant this page faster? Contribute!\n</p><p>checklists</p><p>dynamic</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/checklist\/dynamic\/"},{value:"Searching for Records",label:"<p>Symbiota portals serve data from specimens and observations according to regional and taxonomic themes. To search and view these data, you can use one of the tools described below.\nBasic Search  From the home page of the portal, click Search Collections from the top or side menu. If prompted on the next page, select or deselect collections from the provided list depending on which collections you would like to search.  If you do not wish to search specific collections, leave all the collections checked.    There may be three tabs at the top of the page, \u0026ldquo;Specimens \u0026amp; Observations\u0026rdquo;, \u0026ldquo;Specimens\u0026rdquo;, and \u0026ldquo;Observations\u0026rdquo;. Specimens are phyiscal collections of an organism, while observations are human observations or photos of organisms that are not supported by physical specimens. Use these tabs to select between these types of occurrences, if desired.\n Notice that collections may be grouped together into categories. These categories may be minimized or maximized by clicking the small grey plus or minus box, respectively, to the bottom left of the name of the group.\n Click the Search button on the far right. On the next page, you can enter search criteria to find records of interest.  The criteria you can use will vary depending on the portal, but generally include taxonomic criteria, locality criteria, collector criteria, and specimen criteria. Any number of criteria can be entered and search at the same time. To define a latitude\/longitude bounding box, polygon, or point with radius in which to search, enter the values in the provided fields or click the Google Maps icon to create the box, shape, or point radius in a Google Maps interface. You can search for multiple values in a single field by separating the values by semicolons. For example, if you want to search by both Kern and Inyo counties, you should enter \u0026ldquo;Kern;Inyo\u0026rdquo; in the county field.   Click List Display to conduct the search and view results as a list, or click Table Display to conduct the search and view results in a table.  List Display\nTable Display\nTo return to the Search Criteria page and refine your search, click Search Criteria from the navigation menu at the top of the page.\n To sort the search results, click Display Table (or the Table Display button In the Search Results box at the top of the page, select the field you would like to sort by, a second field you would like to sort by (if applicable), then whether you would like to sort results in ascending or descending order. Then click Sort.\n Map Search Depending on the portal, the Map Search function may be under the Search Collections menu item, or listed as a separate menu item on the homepage.\nClick the Open Search Panel button in the top left corner. Once open, you can enter the same types of criteria into this search panel as were available in the regular search (described above). Then click the Search button.\nTo select specific collections from which you would like to search, select the Collections tab in the search panel, then check or uncheck boxes next to collections as desired.\nFurther customizations can be made in the Map Options tab of the search panel including grid size and min. cluster size. These will affect how many specimens will be clustered together on the map. You can also turn off clustering in this tab.\nOnce you have conducted a search, you can view a list of specimens by clicking the Open Search Panel button and viewing the Records and Taxa menu item. You can also download the specimen records, download a KML file of the specimen records, or generate a shareable link to these search results by clicking on the respective buttons on this page.\n</p><p>search</p><p>specimens</p><p>observations</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/search\/"},{value:"Contacting a Collection",label:"<p>To contact the collections manager or curator of a collection, visit the Collection Details page of that collection by doing the following:\n Select Search Collections from the portal menu. Click \u0026ldquo;more info\u0026rdquo; after the name of the collection you are interested in. Contact information for the person or people responsible for the collection will be listed at the top of this page.  </p><p>contact</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/contact\/"},{value:"Redacted and Protected Data",label:"<p>Some collections redact location information data (latitude\/longitude and locality description) for certain specimens from the general public. Individual specimens may have redacted location data because (a) the location is confidential (e.g., private or protected property), or (b) the taxon of the species is rare, threatened, endangered, or otherwise imperiled. Which taxa are selected for data redaction depends on the data portal, since some taxa are rare in certain places, but not in others.\nUsers may gain access to redacted location information by being granted Rare Species Reader permissions. To obtain these permissions, contact the curator or manager of the collection and provide information about yourself and why you would like access to their data.\nNot sure how to contact the manager of a collection? Visit the Contacting a Collection page.\n </p><p>rare species</p><p>redacted data</p><p>protected data</p><p>protected species</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/redaction\/"},{value:"Downloading a Darwin Core Archive",label:"<p> This page describes how you can download Darwin Core Archive files (DwC-A; standard format for data publishing and sharing) from a Symbiota portal.\n If you are an administrator of a collection, you can download a Darwin Core Archive of your collection in one of two ways:\n Navigate to your Administration Contol Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection) and click Darwin Core Archive Publishing. If you have already created a DwC-A, click the link next to \u0026ldquo;DwC-Archive File:\u0026rdquo;. If you have not yet created a DwC-A, create one. Navigate to your Administration Contol Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection) and click Download Data Backup file. The backup file will be in the format of a Darwin Core Archive, but all specimens will be included in the file, even if they don\u0026rsquo;t have unique GUIDs.  If you are not an administrator of a collection, or you would like to download a DwC-A of another collection:\n Navigate to the Sitemap of the portal (generally an option on the menu). Under \u0026ldquo;Collections\u0026rdquo;, click Darwin Core Archives (DwC-A). Find the collection for which you would like to download a DwC-A and click the link in the DwC-A column. The size of the file will be displayed in parentheses.  </p><p>data publishing</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/download\/dwc\/"},{value:"Symbiota Data Fields",label:"<p> This page describes the fields available in Symbiota. The Symbiota data schema is strongly aligned to the Darwin Core data exchange standard. For more details, links to the Darwin Core definitions are supplied for each term.\n Since portals have the ability to customize the field names found on their data entry form, field names may differ from the core field definition and how it is mapped to Darwin Core export tools.\n Standard Fields    Field Description Example(s) Darwin Core field     Catalog Number The unique identifier (primary key) for the specimen record. This field should be used to store the barcode or the accession number (herbaria only). This field is enforced to be unique per collection. WIS-L-0123456, ASU0012345, 12345 catalogNumber   Other Catalog Numbers Any other identifier for a specimen record that is not the catalog number. This field is typically used to store the old catalog number for collections that are in the process of switching from one cataloging system to another (e.g. barcode system). 12345 otherCatalogNumbers   Collector The name of the person who collected the specimen or made the observation. C.G. Pringle, Goodding, L.N. recordedBy   Associated Collectors Other collectors that were present at the time of collection John R. Reeder, A. Nelson This field is not defined by the Darwin Core standard, which places primary and secondary collectors concatenated the recordedBy field.   Number The collection number assigned to the specimen by the collector. 1294, 12490b, 94-132 recordNumber   Date The date the specimen was collected. While dates can be entered using your preferred format, the value will be converted and stored as an ISO-8601 numeric format (YYYY-MM-DD). Note that unknown month and days can be entered as “00”. For example, a collection with a date of “March 1956” can be entered as “1956-03-00”. 1983-09-15, 1983-07-00, 1934-00-00 eventDate   Verbatim Date Can be used to record date exactly as entered on label. Particularly useful for non-standard date formats or date ranges. Spring 1901, March-April 1952, late Sept. 1909 verbatimEventDate   Year, Month, Day The numeric value of the year, month, and day at the time of collection. These fields are automatically populated when the date is entered.  year, month, day   Day of year range A range of collection dates can be represented here as numeric day of year values. These values will be automatically calculated if you enter a date range in the verbatim date field 12 Sept 1968 to 19 Sept 1968, 1968-09-12 to 1968-09-19) startDayOfYear, endDayOfYear   Scientific Name The Latin name of the specimen without the author. Could be anything from kingdom down to subspecies or variety, depending on the level of the identification. Pinaceae, Pinus, Pinus edulis, Pinus edulis var. fallax scientificName   Author The name of the person who first named the taxon. This field autofills after entering the scientific name as long as that taxon is in the taxonomic thesaurus. L., Asa A. Gray scientificNameAuthorship   Identification Qualifier The determiner’s expression of uncertainty in their identification. This will be listed on the label along with the scientific name. cf., aff. identificationQualifier   Family The family to which the taxa belongs. This field autofills after entering the scientific name as long as the taxon exists in the taxonomic thesaurus. Pinaceae, Asteraceae family   Identified By The name of the person who identified the specimen. Also called a determiner. L. R. Landrum identifiedBy   Date Identified The date the identification was made. Date can be entered as free form text and do not need to be in a standard date format. 1992, May 1992, 2 May 1992 dateIdentified   ID References The reference source used to make the identification. Nesom, Guy L. 2006. Flora of North America – Asteraceae. vol. 20 identificationReferences   ID Remarks Any additional notes regarding the identification of the specimen.  identificationRemarks   Taxon Remarks Any additional notes regarding the taxonomic name to which the specimen was identify.  taxonRemarks   Country The name of the country in which the specimen was collected. To aid data entry, a drop down menu will appear as one types, though names outside of the list can still be entered. United States, Canada, Mexico country   State\/Province The name of the state or province in which the specimen was collected. As one types, a selection list will appear for the given country. New York, Arizona, Sonora stateProvince   County The name of the county in which the specimen was collected. Choose one from the drop down menu. For specimens collected outside of the United States, enter the next geographic region below state\/province. Maricopa county   Municipality The name of the municipality in which the specimen was collected. For specimens collected outside of the United States, enter the 4th level geographic designation. Paradise Valley, Tucson municipality   Locality The detailed location in which the specimen was collected. 9.5 miles NW of Sedona along Boynton Pass Rd. Locality   Location Remarks Comments or notes about the Location. under water since 2005 locationRemarks   Location ID An identifier for the set of location information (data associated with dcterms:Location). May be a global unique identifier or an identifier specific to the data set. https:\/\/opencontext.org\/subjects\/ 768A875F-E205-4D0B-DE55-BAB7598D0FD1 locationID   Locality Security Checking the Locality Security checkbox will hide images and locality details below the level of county from unauthorized users. For more information about locality security in Symbiota, visit our Redacting Data page.  This field is not defined by the Darwin Core standard.   Latitude and Longitude The geographic latitude and longitude in decimal degrees. Latitudes from the southern hemisphere and longitudes in the western hemisphere (e.g. USA) should be entered as negative values. Click on the “Tools” button to enter the coordinates in the degree, minute, seconds (DMS) or the UTM formats. Decimal degrees are the preferred coordinate standard as defined by Darwin Core. 34.874022, -111.75774 decimalLatitude, decimalLongitude   Uncertainty (meters) The accuracy of the georeference coordinates in meters (numeric value only). This is measured as the radius of a circle where the true point would be found if known. Protocols for assigning error radii to georeference points can be found in this Georeference Quick Reference Guide. 1500, 10000 coordinateUncertaintyInMeters   Datum The geographic system that was used to get the coordinates. This field autofills when using GeoLocate or the Google Maps tool for georeferencing. NAD27, NAD83, WGS84 geodeticDatum   Verbatim Coordinates If the coordinates recorded on the specimen label are in a format other than decimal degrees, enter them here. Click the F (format) button to enter non-decimal formats (e.g., UTMs, degrees\/minutes\/seconds) and click Insert Coordinates. This will place the original format in Verbatim Coordinates and will auto-convert to decimal degrees in the Latitude and Longitude fields if these fields are blank. Click the “\u0026laquo;” symbol to replace existing decimal values. 34° 13.940′ N 112° 2.370′ W, 34d 13m 12.940s N 112d 20m 46.370s W, 12 420944E 4064025N, TRS: T40N R32E S29 verbatimCoordinates   Elevation in Meters The elevation in meters at which the specimen was collected. Also called altitude. Use only the left field with the right field blank when a single elevation exists. 1400, 2000-2200 minimumElevationInMeters, maximumElevationInMeters   Verbatim Elevation The verbatim elevation at which the specimen was collected. This is typically used to record an elevation measurement that was recorded in feet or an uncertainty designation. When the elevation in meters field is left blank, the value will automatically be converted to meters if you press Tab after entering a value in this field. Click the “\u0026laquo;” symbol to replace the previously entered meters values. 4500ft, 4500 feet, ca 4500′, ca 2000m, 4500′ \u002b-300′ verbatimElevation   Depth in Meters The range of depth below the local surface, in meters. 100, 150-200 minimumDepthInMeters, maximumDepthInMeters   Verbatim Depth The original verbatim description of the depth below the local surface at which the specimen was collected. 100ft, 100 feet, ca 100′, ca 30m, 100′ \u002b-10′ verbatimDepth   Georeferenced By The name of the person who georeferenced the specimen record. This field autofills when using GeoLocate for georeferencing. A. Gonzales, emakings, acbarber georeferencedBy   Georeference Protocol The source of the standards used to georeference. Georeferencing Quick Guide. Zermoglio et al. 2020 georeferenceProtocol   Georeference Sources The tool or tools used to georeference. GeoLocate, Google Earth, USGS map georeferenceSources   Georef Verificiation Status Indicates whether or not the georeference has been reviewed or verified. reviewed, not reviewed georeferenceVerificationStatus   Georeference Remarks Any additional notes regarding the georeferencing of the specimen.  georeferenceRemarks   Habitat The description of the habitat in which the specimen was collected. Wet areas along a small stream in chaparral, roadside habitat   Substrate The substrate on which the specimen was collected. Mostly used for lichen and bryophyte specimens. On basalt, trunk of oak Darwin Core lumps this information into habitat.   Associated Taxa A list of the names of other species occurring with the collected specimen. Quercus, Arctostaphylos, Ceanothus, Rhus, Eriogonum, Salvia associatedTaxa   Description A physical description of the specimen at the time of collection. This often includes information that can be lost or difficult to observe after the collection and preservation process. Shrub 3 m tall, corolla yellow In a Darwin Core export, this is concatenated with occurrenceRemarks   Notes Any additional notes regarding the specimen. This field often includes characteristics of the population, such as its frequency or rarity.  occurrenceRemarks   Dynamic Properties A list of additional measurements, facts, characteristics, or assertions about the specimen in a format that allows programmatic parsing of the data awnLengthInMeters=0.014, heightInMeters=1.5, relativeHumidity=28, airTemperatureInC=22 dynamicProperties   Life Stage The age or stage of the organism at the time of collection\/observation. Typically used for zoological collections. larva, juvenile lifeStage   Sex The biological sex of the occurrence. female, male sex   Individual Count The number of individuals in the occurrence 0, 2, 15 individualCount   Sampling Protocol The names and references to methods used to collect or sample an occurrence UV light trap, mist net, Takats et al. 2001. Guidelines for Nocturnal Owl Monitoring in North America. Beaverhill Bird Observatory and Bird Studies Canada, Edmonton, Alberta. 32 pp., http:\/\/www.bsc-eoc.org\/download\/Owl.pdf samplingProtocol   Preparations Preparation or preservation method for a specimen in ethanol, skeleton preparations   Phenology (Reproductive Condition) The reproductive stage the specimen is in. Typically used for plant and fungal collections. flower, fruit, sterile reproductiveCondition   Establishment Means The state of establishment at the time of collection. cultivated, invasive, native establishmentMeans   Cultivated Checkbox Check when the organism was established with the aid of humans and would not be able to exist on their own. This true\/false field enables the ability to filter non-native or naturalized species.  Not currently exported in DwC format, but see degreeOfEstablishment   Type Status The type designation of a specimen, if it is a type specimen HOLOTYPE, ISOTYPE, PARATYPE typeStatus   Disposition The location or status of the physical specimen. missing, on loan, cone collection disposition   Occurrence Id This is the Global Unique Identification (GUID) for the specimen. This identification code should be stable and uniquely identify the specimen relative to all other specimens within the world. 000866d2-c177-4648-a200-ead4007051b9, urn:catalog:UWBM:Bird:89776 occurrenceID   Field Number   fieldNumber   Owner Code The acronym of the owning institution. Only enter a value if the owning institution is different than what was entered when the metadata for the collection institution was added to the portal. NPS, Forest Service ownerInstitutionCode   Basis of Record The type of record the specimen is classified as. For physical collections, this field defaults to “PreservedSpecimen” (aka herbarium specimen) and for observation projects, the default is “Observation”. PreservedSpecimen, LivingSpecimen, Observation basisOfRecord   Language The language of the label information. English, Spanish, Portuguese language   Processing Status The status of the digital record. This field is used for internal data management and review. The values used are dictated by the specific workflow of each institution. Expert Required, Pending Review    Label Project Used for printing labels. You can create a label project and print that set of labels after you’ve entered the data. Plants of Sedona 2012    Duplicate Quantity The number of duplicate specimens created. This will dictate the number of labels printed for specimen. 10     Paleo Terms    Field Description Example(s) Darwin Core field     Eon The longest geologic time intervals. Archean, Proterozoic, Phanerozoic earliestEonOrLowestEonothem, latestEonOrHighestEonothem   Era A subdivision of an eon that is a shorter interval of geologic time. Paleozoic, Mesozoic, Cenozoic earliestEraOrLowestErathem, latestEraOrHighestErathem   Period A subdivision of an era that is a shorter interval of geologic time. Ordovician, Silurian, Devonian, Carboniferous, Mississippian, Pennsylvanian, Permian, Triassic, Jurassic, Cretaceous, Paleogene, Neogene, Quaternary. earliestPeriodOrLowestSystem, latestPeriodOrHighestSystem   Epoch A subdivision of a period that is a shorter interval of geologic time. The Epoch field is currently being recorded using Series, the chronostratigraphic units. Lower, Middle, Upper Ordovician; Wenlock earliestEpochOrLowestSeries, latestEpochOrHighestSeries   Stage The chronostratigraphic term given to the succession of rock strata laid down in a single geochronologic age. Lochkovian, Emsian, Eifelian, Givetian, Frasnian earliestAgeOrLowestStage, latestAgeOrHighestStage   Local Stage A local name for a stage that was applied to this specimen. “Ulatsian”, “Helvetian.”    Early Interval Name of the earliest possible geochronologic eon, era, period, epoch or age, or the lowest chronostratigraphic eonothem, erathem, system, series, or stage attributable to the stratigraphic horizon from which the cataloged specimen was collected. Aalenian, Aeronian, Albian, Anisian    Late Interval Name of the latest possible geochronologic eon, era, period, epoch or age, or the highest chronostratigraphic eonothem, erathem, system, series or stage attributable to the stratigraphic horizon from which the cataloged specimen was collected. Aalenian, Aeronian, Albian, Anisian    Absolute Age Age of specimen\/rock in years determined using radioactive decay of isotopes (Carbon-14, argon-argon, potassium-argon, uranium-lead, etc.) and other quantitative dating methods. 20 Ma, 75 ka, 10.5 – 12.7 \u002b\/- 0.5 Ma    Storage Age Field for institutions that arrange collections by geologic time or biostratigraphy. The physical location of a specimen within the collection space. Wasatchian, Paleocene, Bridgerian    Biota (Flora\/Fauna) Name given to collections of fossils of the same age from a single locality or multiple localities within a specific geographic area. Chalk Bluffs, Stewart Valley, Bridge Creek, Mazon Creek    Biostratigraphy (Biozone) The name of the lowest possible geological biostratigraphic zone of the stratigraphic horizon from which the cataloged item was collected. Wa0, Uvigerinella sparsicostata Zone, Ogygiocaris lowestBiostratigraphicZone, highestBiostratigraphicZone   Group The name of the lithostratigraphic group from which the cataloged specimen was collected. Bathurst, Lower Wealden, Monte Cristo, Contra Costa, Panoche group   Formation The name of the lithostratigraphic formation from which the cataloged specimen was collected. The National Geologic Map Database Geolex Search is a great resource for the named lithostratigraphic units accepted by the USGS. Notch Peak, House Limestone, Fillmore, Chinle formation   Member The name of the lithostratigraphic member from which the cataloged item was collected. Lava Dam, Hellnmaria, Brown Mountain Sandstone member   Bed The name of the lithostratigraphic bed from which the cataloged item was collected. Harlem coal bed   Taxon Environment The depositional environment of the rock unit from which cataloged specimen was collected. marine, lacustrine, non-marine, marine-non-marine    Lithology Field for terms to describe the types of rock\/sediment from which the cataloged specimen was collected. sandstone, mudstone, siltstone, shale lithostratigraphicTerms   Strat Remarks Field to record additional details about the geology, stratigraphy, more detailed lithology description, palynological sampling info, core data, etc.     Element Field to record type of plant organ cataloged specimen represents. stem, strobilus, sterile leaf, fertile leaf, pinnule(s), rooting organ, rootlet, megasporangium, sporangium, spore, sterile axis, fertile axis, root    Slide Properties Field to record types of prepared slides of specimens, noting the type of preparation and mounting medium, and to provide England Finder coordinates for palynomorph slides. strewn, petrographic thin-section, mounted peel    Geological Context ID An identifier for the set of information associated with a GeologicalContext (the location within a geological context, such as stratigraphy). May be a global unique identifier or an identifier specific to the data set. https:\/\/opencontext.org\/subjects\/e54377f7-4452-4315-b676-40679b10c4d9 geologicalContextID    </p><p>edit</p><p>fields</p><p>data fields</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/editor\/edit\/fields\/"},{value:"About",label:"<p>Symbiota Docs is a central repository for documentation regarding Symbiota-based data portals. This site is maintained by the Symbiota Support Hub, but all Symbiota users are encouraged to contribute. Contact Us to contribute or provide feedback on this site.\nSymbiota is an open-source software for managing and mobilizing biodiversity data that serves over 1,400 natural history collections and publishes over 70 million occurrence (specimen or observation) records.\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/about\/"},{value:"Categories",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/categories\/"},{value:"Citing This Website",label:"<p>To cite the entire website use this snippet: Symbiota Support Hub (2021). Symbiota Documentation. https:\/\/biokic.github.io\/symbiota-docs\/\nTo cite one particular page, you can use this template, or navigate to the bottom of the desired page and copy it from there: [Content authors, editors and translators full names]. [Page Title]. In: Symbiota Support Hub (2021). Symbiota Documentation. [Permalink]. Created on [DD month YYYY], last edited on [DD month YYYY].\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/citing\/"},{value:"Frequently Asked Questions",label:"<p> What is this site?    Symbiota Docs is a central repository for documentation regarding installing, using, and managing Symbiota-based portals and the data they contain.   Who is writing these docs?    Documentation is sourced from the Symbiota user community and the Symbiota Support Hub team.   Where can I find a list of Symbiota portals?    A list of existing Symbiota portals can be found here.   I would like to use Symbiota for my collection. What should I do?     Visit the list of existing Symbiota portals and determine whether your collection fits within the scope of this portal. Contact the portal administrator or help desk email of the portal you would like to join. If you can\u0026rsquo;t find this, email the Symbiota Support Hub, and we can help to put you in contact with the right people. Work with the portal administrator to create a collection and contribute your data. Don\u0026rsquo;t see a portal that you think applies to your data? Contact the Symbiota Support Hub for more information.     How can I contribute to Symbiota Docs?    We welcome and encourage new contributions or suggestions for the Symbiota Docs site. Please email us to provide content or explore other ways to contribute to the project.   Which version of Symbiota does this documentation apply to?    The majority of documentation on this site pertains to Symbiota-light; however, documentation about Symbiota2 will be integrated as it is developed.   </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/faq\/"},{value:"Still Have Questions?",label:"<p>Send us an email!\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contact\/"},{value:"Tags",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/tags\/"},];$("#search").autocomplete({source:projects}).data("ui-autocomplete")._renderItem=function(ul,item){return $("<li>").append("<a href="+item.url+" + \" &quot;\" +  >"+item.value+"</a>"+item.label).appendTo(ul);};});</script></div></div></div></div></header><section class=section><div class=container><div class="row justify-content-center"><div class="col-12 text-center"><h2 class=section-title>Browse articles by subject</h2></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/user/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-search icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">User Guide</h3><p class=mb-0>Learn to access, view, and download data and checklists.</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/editor/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-marker-alt icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Editor Guide</h3><p class=mb-0>Learn to edit, manage, import, and export data in Symbiota portals</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/coll_manager/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-panel icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Collection Manager Guide</h3><p class=mb-0>Learn to clean data, edit permissions, upload data, and perform other administrator functions.</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/dev/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-harddrive icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Developer Guide</h3><p class=mb-0>Learn to edit and develop Symbiota code</p></a></div></div></div></section><section><div class=container><div class=row><div class=col-12><div class="section px-3 bg-white shadow text-center"><h2 class=mb-4>Still need help?</h2><p class=mb-4></p><a href=/symbiota-docs/contact class="btn btn-primary">contact us</a></div></div></div></div></section><footer class="section pb-4"><div class=container><div class="row align-items-center"><div class="col-md-8 text-md-left text-center"><p class="mb-md-0 mb-4">© 2021 <a href=https://symbiota.org>Symbiota Support Hub</a></p></div><div class="col-md-4 text-md-right text-center"><ul class=list-inline><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li></ul></div></div></div></footer><script src=https://biokic.github.io/symbiota-docs/js/script.min.js></script></body></html>